{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Content of antibiotics-08-00023.pdf:\n",
      "[Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 0}, page_content='antibiotics \\nArticle\\nPrevalence of Antibiotic Resistance Genes in\\nMultidrug-Resistant Enterobacteriaceae on\\nPortuguese Livestock Manure\\nPaula Amador 1, *\\n , Ruben Fernandes 2, Cristina Prudêncio 2 and Isabel Duarte 1\\n1 Environment Department, Research Centre for Natural Resources, Environment and Society (CERNAS),\\nCollege of Agriculture, Polytechnic of Coimbra, 3045-601 Coimbra, Portugal; iduarte@esac.pt\\n2 Department Chemical Sciences and Biomolecules, School Allied Health Sciences, Polytechnic of Porto,\\n4200-072 Porto, Portugal; rfernandes@ess.ipp.pt (R.F.); cprudencio@estsp.ipp.pt (C.P .)\\n* Correspondence: paula_amador@esac.pt; Tel.: +351-129-802-940\\nReceived: 8 January 2019; Accepted: 23 February 2019; Published: 13 March 2019\\n/gid00030/gid00035/gid00032/gid00030/gid00038/gid00001/gid00033/gid00042/gid00045/gid00001\\n/gid00048/gid00043/gid00031/gid00028/gid00047/gid00032/gid00046\\nAbstract: The exposure of both crop ﬁelds and humans to antibiotic-resistant bacteria in animal\\nexcreta is an emergent concern of the One Health initiative. This study assessed the contamination\\nof livestock manure from poultry, pig, dairy farms and slaughterhouses in Portugal with resistance\\ndeterminants. The resistance proﬁles of 331 Enterobacteriaceae isolates to eight β-lactam (amoxicillin,\\ncefoxitin, cefotaxime, cefpirome, aztreonam, ceftazidime, imipenem and meropenem) and to ﬁve\\nnon-β-lactam antibiotics (tetracycline (TET), trimethoprim/sulfamethoxazole (SXT), ciproﬂoxacin\\n(CIP), chloramphenicol (CHL) and gentamicin) was investigated. Forty-nine integron and\\nnon-β-lactam resistance genes were also screened for. Rates of resistance to the 13 antibiotics ranged\\nfrom 80.8% to 0.6%. Multidrug resistance (MDR) rates were highest in pig farm samples (79%). Thirty\\ndifferent integron and resistance genes were identiﬁed. These were mainly associated with resistance\\nto CHL (catI and catII), CIP (mainly, qnrS, qnrB and oqx), TET (mainly tet(A) and tet(M)) and SXT\\n(mostly dfrIa group and sul3). In MDR isolates, integron presence and non-β-lactam resistance to TET,\\nSXT and CHL were positively correlated. Overall, a high prevalence of MDR Enterobacteriaceae was\\nfound in livestock manure. The high gene diversity for antibiotic resistance identiﬁed in this study\\nhighlights the risk of MDR spread within the environment through manure use.\\nKeywords: antibiotic-resistance genes; multidrug resistance; livestock; animal excreta;\\nenvironmental contaminants\\n1. Introduction\\nThe active ingredients of antibiotics (AB) legally prescribed in Portugal for veterinarian use in\\nlivestock prophylactic and metaphylactic treatments, amounted to 179,832 tons in 2013 [ 1]. Up to\\n90% of the ingested doses may be excreted unmodiﬁed or partially metabolised through urine\\nand faeces [ 2]. Consequently, the excreta of AB-treated livestock became important reservoirs of\\nAB residues, antibiotic-resistant bacteria (ARB) and antibiotic-resistance genes (ARGs) [ 3], where\\nhorizontal gene transference plays an essential role in the acquisition, spreading and assembly\\nof various ARGs. Although integrons are not considered genetic elements per se, their location\\non plasmids and transposons enables gene transmission in an inter- and intra-species manner\\nin a single event. For this reason, they are increasingly reported worldwide, especially among\\nEnterobacteriaceae [4,5]. Large volumes of excreta enriched with Enterobacteriaceae end up as manure,\\nslurry and wastewater, potentially vectoring these emergent contaminants. As such, they pose\\nsigniﬁcant environmental concerns.\\nAntibiotics 2019, 8, 23; doi:10.3390/antibiotics8010023 www.mdpi.com/journal/antibiotics'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 1}, page_content=\"Antibiotics 2019, 8, 23 2 of 18\\nThe slurry and the manure of intensively reared animals, composted or raw, are generally spread\\ninto soil for fertilisation; however, animal manure and wastewater are regarded as important potential\\nAB-resistance reservoirs. Therefore, this reuse practice can lead to an increase of those pollutants in\\nsoil [6,7] and in endophytic bacteria of crops grown in manured soil [8]. The excreta ARB and ARGs\\ncan be spread over longer distances through other routes, such as anemophily [9], entomophily [10] or\\nwater run-off from farms and rural settlements, posing serious health problems. Despite the awareness\\nabout these spread routes, there are no available cost-effective tools to control the contaminants in\\nmunicipal wastewater treatment plants (WWTPs) and livestock manure [11].\\nAfter recognising the importance of these emergent concerning contaminants, countries are\\ndeveloping their own national strategies to implement the Global Action Plan on antimicrobial\\nresistance [ 12,13]. Moreover, the recent awareness that the health of humans and animals are\\ninseparably interconnected with their environment led to an integrated One Health approach,\\nparticularly focusing on food safety, zoonosis surveillance and ARB control [ 14]. Dependence on\\nlivestock animals is considered one of the most critical pathogens associated risk factors to human\\ngiven the transmission of AB resistance that occurs through consumption of contaminated animal\\nproducts. Enterobacteriaceae is the major family associated with AB resistance and this is particularly\\nrelevant to this issue [15].\\nUnderstanding the diversity and the distribution of ARB and ARGs at a farm level is of great\\nimportance in order to identify the genetic background of this problem and, thus, educate and inform\\nfarm management. This study aimed to assess the level of contamination of livestock enterprises\\nand the meat industry, in the central region of Portugal, through identiﬁcation of multidrug resistant\\n(MDR) isolates in manure. The analysis focused on the ARB and ARGs in resistant Enterobacteriaceae.\\n2. Materials and Methods\\n2.1. Study Area\\nThe study was conducted in three farm types and slaughterhouses representative of the livestock\\nproduction sector in the central region of Portugal. The 12 selected small-scale enterprises included\\nintensive raising of dairy cattle (D1, D2 and D3), poultry (A1, A2 and A3), pig (P1, P2 and P3) and\\nthree slaughterhouse companies (S1, S2 and S3). These sampling sites are located in the Coimbra and\\nLeiria administrative regions, within Lis, Mondego and Vouga watersheds, at altitudes between 100 m\\nand 200 m, higher than the ﬂat alluvial river valleys (Figure 1).\\nAntibiotics 2019, 8, x 2 of 18 \\nslurry and wastewater, potentially vectoring thes e emergent contaminants. As such, they pose \\nsignificant environmental concerns. \\nThe slurry and the manure of intensively rear ed animals, composted or raw, are generally \\nspread into soil for fertilisation; however, animal manure and wastewater are regarded as important \\npotential AB-resistance reservoirs. Therefore, this reuse practice can lead to  a n  i n c r e a s e  o f  t h o s e  \\npollutants in soil [6,7] and in endophytic bacteria of crops grown in manured soil [8]. The excreta \\nARB and ARGs can be spread over longer distances through other routes, such as anemophily [9], \\nentomophily [10] or water run-off from farms and rural settlements, posing serious health problems. \\nDespite the awareness about these spread routes, there are no available cost-effective tools to control \\nthe contaminants in municipal wastewater treatment plants (WWTPs) and livestock manure [11].  \\nAfter recognising the importance of these emer gent concerning contam inants, countries are \\ndeveloping their own national strategies to impl ement the Global Action Plan on antimicrobial \\nresistance [12,13]. Moreover, the recent awareness that the health of humans and animals are \\ninseparably interconnected with their environment led to an integrated One Health approach, \\nparticularly focusing on food sa fety, zoonosis surveillance and ARB control [14]. Dependence on \\nlivestock animals is considered one of the most cr itical pathogens associated risk factors to human \\ngiven the transmission of AB resistance that occurs through consumption of contaminated animal \\nproducts. Enterobacteriaceae is the major family associated with AB resistance and this is particularly \\nrelevant to this issue [15].  \\nUnderstanding the diversity and the distribution of ARB and ARGs at a farm level is of great \\nimportance in order to identify the genetic back ground of this problem and, thus, educate and \\ninform farm management. This study aimed to assess the level of contamination of livestock \\nenterprises and the meat industry, in the central region of Portugal, thro ugh identification of \\nmultidrug resistant (MDR) isolates in manure. The analysis focused on the ARB and ARGs in \\nresistant Enterobacteriaceae. \\n2. Materials and Methods  \\n2.1. Study Area \\nThe study was conducted in three farm types and slaughterhouses re presentative of the \\nlivestock production sector in the central region of Portugal. The 12 selected small-scale enterprises \\nincluded intensive raising of dairy cattle (D1, D2 and D3), poultry (A1, A2 and A3), pig (P1, P2 and \\nP3) and three slaughterhouse companies (S1, S2 an d S3). These sampling sites are located in the \\nCoimbra and Leiria administrative regions, within Lis, Mondego and Vouga watersheds, at altitudes \\nbetween 100 m and 200 m, higher than the flat alluvial river valleys (Figure 1).  \\n \\n \\nFigure 1. Location of sampling sites depicted on (a) a map of Europe (approximate scale 1:100,000,000,\\n(b) a map of Portugal (approximate scale 1:10,000,000) and ( c) a map of Coimbra and Leiria\\nadministrative regions (dashed lines) (approximate scale 1:1,000,000). Enterprise types are represented\\nas follows:\\nAntibiotics 2019, 8, x 3 of 18 \\n                 \\nAs per information obtained from the farms' personnel, all animals had adequate veterinary \\nmonitoring and a few, when necessary, received  medical treatments. The manure and slurry \\ngenerated in these farms were either composted or di rectly used as raw soil fertilizers in the animal \\nfeed crop fields of each enterprise. According to  the slaughterhouse personnel, all animals were \\nsubjected to the required withdrawal time as per public health safety regulations and the defined \\nholding time before slaughter was followed. Each slaughterhouse had its own protocol for treating \\nwastewater.  \\n2.2. Sampling \\nBetween March 2016 and March 2017, three samp les were collected per site. Dairy cows or \\nswine manure samples were collected from open tanks with different maturation ages, whereas \\npoultry manure from was collected from waste pools. \\nIn the slaughterhouses, samples were collect ed from reservoirs holding the wastewaters \\ngenerated by the daily slaughter of animals. The number of animals slaughtered daily is presented \\nin Table 1. \\nSamples were collected in sterile plastic bottles  or bags and maintained at 4 °C until the \\nmicrobiological processing, which was always carried out within 4 h after sampling. In the \\nlaboratory, the three samples from each site were vigorously mixed and processed conjointly. \\n2.3. Microbiological Analysis and Phenotypic Characterisation \\nThe enumeration, detection and identification  of bacteria was carried out according to \\nprotocols previously described [16]. Decimal dilutions of the samples were prepared in sterile saline \\n0.9% NaCl. From each dilution, 100 mL were filtered through cellulose membranes of 0.45 µm \\n(Millipore, Bedford, MA, USA) and the filters pl aced on the surface of selective medium for \\nEnterobacteriaceae VRBG (Violet Red Bile Glucose) agar (Oxoid, Hampshire, England). The isolates \\nwere counted after overnight aerobic incubation at  37 °C. The colonies with different morphotypes \\nwere selected, picked out three times and their pu rity further confirmed microscopically. Species \\nidentification was performed according to the ma nufacturer’s recommendations by the standard \\nAPI 20E galleries (BioMerieux, Marcy l’Etoile, Lyon, France). \\nThirteen AB were selected for the phenotypic ch aracterisation the isolates. These were chosen \\nto represent the main AB classes used in human medicine and livestock production in Portugal, \\nnamely: amoxicillin/clavulanic acid combination (AMC) 30 μg/10 μg, respectively; ceftazidime \\n(CAZ) 30 μg; cefotaxime (CTX) 30 μg; cefpirome (CPO) 30 μg; aztreonam (ATM) 30 μg; cefoxitin \\n(FOX) 30 μg; imipenem (IPM) 10 μg; meropenem (MEM) 10 μg; chloramphenicol (CHL) 30 μg; \\ngentamicin (GEN) 10 μg; ciprofloxacin (CIP) 5 μg; trimethoprim/sulfamethoxazol (SXT) \\ncombination (1:19) and tetracycline (TET) 30 μg. The disk diffusion Kirby-Bauer method was \\nperformed in agreement with the guidelines for an timicrobial susceptibility tests defined by the \\nClinical Laboratory Standards Institute [17], using Escherichia coli ATCC25922 (Liofilchem S.R.L., \\nRoseto degli Abruzzi, Italy), E. coli J53-Az R (provided by Instituto de Ciências Biomédicas Abel \\nSalazar, Portugal) and E. coli HB101-StpR–(Bio-Rad Laboratories Lda, Lisbon, Portugal) as quality \\ncontrols, Mueller-Hinton agar and AB disks from Oxoid (Hampshire, England). The isolates with a \\nresistance phenotype against three or more structurally unrelated antimicrobial agents were \\ndefined as multidrug resistant (MDR) [18].  \\n2.4. Multiplex PCR (Polymerase Chain Reaction) for Genes Detection \\nThe screening for resistance genes was focused on a subset of isolates chosen according to their \\nphenotypic profile of resistance. The presence of the most frequent Enterobacteriaceae resistance \\ngenes in these isolates was determined by diff erent multiplex PCR assays. Total DNA of these \\nisolates was extracted as described by Amador et al [19] and 2 µL of each was subjected to dairy cattle farm;\\nAntibiotics 2019, 8, x 3 of 18 \\n                 \\nAs per information obtained from the farms' personnel, all animals had adequate veterinary \\nmonitoring and a few, when necessary, received  medical treatments. The manure and slurry \\ngenerated in these farms were either composted or di rectly used as raw soil fertilizers in the animal \\nfeed crop fields of each enterprise. According to  the slaughterhouse personnel, all animals were \\nsubjected to the required withdrawal time as per public health safety regulations and the defined \\nholding time before slaughter was followed. Each slaughterhouse had its own protocol for treating \\nwastewater.  \\n2.2. Sampling \\nBetween March 2016 and March 2017, three samp les were collected per site. Dairy cows or \\nswine manure samples were collected from open tanks with different maturation ages, whereas \\npoultry manure from was collected from waste pools. \\nIn the slaughterhouses, samples were collect ed from reservoirs holding the wastewaters \\ngenerated by the daily slaughter of animals. The number of animals slaughtered daily is presented \\nin Table 1. \\nSamples were collected in sterile plastic bottles  or bags and maintained at 4 °C until the \\nmicrobiological processing, which was always carried out within 4 h after sampling. In the \\nlaboratory, the three samples from each site were vigorously mixed and processed conjointly. \\n2.3. Microbiological Analysis and Phenotypic Characterisation \\nThe enumeration, detection and identification  of bacteria was carried out according to \\nprotocols previously described [16]. Decimal dilutions of the samples were prepared in sterile saline \\n0.9% NaCl. From each dilution, 100 mL were filtered through cellulose membranes of 0.45 µm \\n(Millipore, Bedford, MA, USA) and the filters pl aced on the surface of selective medium for \\nEnterobacteriaceae VRBG (Violet Red Bile Glucose) agar (Oxoid, Hampshire, England). The isolates \\nwere counted after overnight aerobic incubation at  37 °C. The colonies with different morphotypes \\nwere selected, picked out three times and their pu rity further confirmed microscopically. Species \\nidentification was performed according to the ma nufacturer’s recommendations by the standard \\nAPI 20E galleries (BioMerieux, Marcy l’Etoile, Lyon, France). \\nThirteen AB were selected for the phenotypic ch aracterisation the isolates. These were chosen \\nto represent the main AB classes used in human medicine and livestock production in Portugal, \\nnamely: amoxicillin/clavulanic acid combination (AMC) 30 μg/10 μg, respectively; ceftazidime \\n(CAZ) 30 μg; cefotaxime (CTX) 30 μg; cefpirome (CPO) 30 μg; aztreonam (ATM) 30 μg; cefoxitin \\n(FOX) 30 μg; imipenem (IPM) 10 μg; meropenem (MEM) 10 μg; chloramphenicol (CHL) 30 μg; \\ngentamicin (GEN) 10 μg; ciprofloxacin (CIP) 5 μg; trimethoprim/sulfamethoxazol (SXT) \\ncombination (1:19) and tetracycline (TET) 30 μg. The disk diffusion Kirby-Bauer method was \\nperformed in agreement with the guidelines for an timicrobial susceptibility tests defined by the \\nClinical Laboratory Standards Institute [17], using Escherichia coli ATCC25922 (Liofilchem S.R.L., \\nRoseto degli Abruzzi, Italy), E. coli J53-Az R (provided by Instituto de Ciências Biomédicas Abel \\nSalazar, Portugal) and E. coli HB101-StpR–(Bio-Rad Laboratories Lda, Lisbon, Portugal) as quality \\ncontrols, Mueller-Hinton agar and AB disks from Oxoid (Hampshire, England). The isolates with a \\nresistance phenotype against three or more structurally unrelated antimicrobial agents were \\ndefined as multidrug resistant (MDR) [18].  \\n2.4. Multiplex PCR (Polymerase Chain Reaction) for Genes Detection \\nThe screening for resistance genes was focused on a subset of isolates chosen according to their \\nphenotypic profile of resistance. The presence of the most frequent Enterobacteriaceae resistance \\ngenes in these isolates was determined by diff erent multiplex PCR assays. Total DNA of these \\nisolates was extracted as described by Amador et al [19] and 2 µL of each was subjected to poultry farm;\\nAntibiotics 2019, 8, x 3 of 18 \\n                 \\nAs per information obtained from the farms' personnel, all animals had adequate veterinary \\nmonitoring and a few, when necessary, received  medical treatments. The manure and slurry \\ngenerated in these farms were either composted or di rectly used as raw soil fertilizers in the animal \\nfeed crop fields of each enterprise. According to  the slaughterhouse personnel, all animals were \\nsubjected to the required withdrawal time as per public health safety regulations and the defined \\nholding time before slaughter was followed. Each slaughterhouse had its own protocol for treating \\nwastewater.  \\n2.2. Sampling \\nBetween March 2016 and March 2017, three samp les were collected per site. Dairy cows or \\nswine manure samples were collected from open tanks with different maturation ages, whereas \\npoultry manure from was collected from waste pools. \\nIn the slaughterhouses, samples were collect ed from reservoirs holding the wastewaters \\ngenerated by the daily slaughter of animals. The number of animals slaughtered daily is presented \\nin Table 1. \\nSamples were collected in sterile plastic bottles  or bags and maintained at 4 °C until the \\nmicrobiological processing, which was always carried out within 4 h after sampling. In the \\nlaboratory, the three samples from each site were vigorously mixed and processed conjointly. \\n2.3. Microbiological Analysis and Phenotypic Characterisation \\nThe enumeration, detection and identification  of bacteria was carried out according to \\nprotocols previously described [16]. Decimal dilutions of the samples were prepared in sterile saline \\n0.9% NaCl. From each dilution, 100 mL were filtered through cellulose membranes of 0.45 µm \\n(Millipore, Bedford, MA, USA) and the filters pl aced on the surface of selective medium for \\nEnterobacteriaceae VRBG (Violet Red Bile Glucose) agar (Oxoid, Hampshire, England). The isolates \\nwere counted after overnight aerobic incubation at  37 °C. The colonies with different morphotypes \\nwere selected, picked out three times and their pu rity further confirmed microscopically. Species \\nidentification was performed according to the ma nufacturer’s recommendations by the standard \\nAPI 20E galleries (BioMerieux, Marcy l’Etoile, Lyon, France). \\nThirteen AB were selected for the phenotypic ch aracterisation the isolates. These were chosen \\nto represent the main AB classes used in human medicine and livestock production in Portugal, \\nnamely: amoxicillin/clavulanic acid combination (AMC) 30 μg/10 μg, respectively; ceftazidime \\n(CAZ) 30 μg; cefotaxime (CTX) 30 μg; cefpirome (CPO) 30 μg; aztreonam (ATM) 30 μg; cefoxitin \\n(FOX) 30 μg; imipenem (IPM) 10 μg; meropenem (MEM) 10 μg; chloramphenicol (CHL) 30 μg; \\ngentamicin (GEN) 10 μg; ciprofloxacin (CIP) 5 μg; trimethoprim/sulfamethoxazol (SXT) \\ncombination (1:19) and tetracycline (TET) 30 μg. The disk diffusion Kirby-Bauer method was \\nperformed in agreement with the guidelines for an timicrobial susceptibility tests defined by the \\nClinical Laboratory Standards Institute [17], using Escherichia coli ATCC25922 (Liofilchem S.R.L., \\nRoseto degli Abruzzi, Italy), E. coli J53-Az R (provided by Instituto de Ciências Biomédicas Abel \\nSalazar, Portugal) and E. coli HB101-StpR–(Bio-Rad Laboratories Lda, Lisbon, Portugal) as quality \\ncontrols, Mueller-Hinton agar and AB disks from Oxoid (Hampshire, England). The isolates with a \\nresistance phenotype against three or more structurally unrelated antimicrobial agents were \\ndefined as multidrug resistant (MDR) [18].  \\n2.4. Multiplex PCR (Polymerase Chain Reaction) for Genes Detection \\nThe screening for resistance genes was focused on a subset of isolates chosen according to their \\nphenotypic profile of resistance. The presence of the most frequent Enterobacteriaceae resistance \\ngenes in these isolates was determined by diff erent multiplex PCR assays. Total DNA of these \\nisolates was extracted as described by Amador et al [19] and 2 µL of each was subjected to slaughterhouse;\\nAntibiotics 2019, 8, x 3 of 18 \\n                 \\nAs per information obtained from the farms' personnel, all animals had adequate veterinary \\nmonitoring and a few, when necessary, received  medical treatments. The manure and slurry \\ngenerated in these farms were either composted or di rectly used as raw soil fertilizers in the animal \\nfeed crop fields of each enterprise. According to  the slaughterhouse personnel, all animals were \\nsubjected to the required withdrawal time as per public health safety regulations and the defined \\nholding time before slaughter was followed. Each slaughterhouse had its own protocol for treating \\nwastewater.  \\n2.2. Sampling \\nBetween March 2016 and March 2017, three samp les were collected per site. Dairy cows or \\nswine manure samples were collected from open tanks with different maturation ages, whereas \\npoultry manure from was collected from waste pools. \\nIn the slaughterhouses, samples were collect ed from reservoirs holding the wastewaters \\ngenerated by the daily slaughter of animals. The number of animals slaughtered daily is presented \\nin Table 1. \\nSamples were collected in sterile plastic bottles  or bags and maintained at 4 °C until the \\nmicrobiological processing, which was always carried out within 4 h after sampling. In the \\nlaboratory, the three samples from each site were vigorously mixed and processed conjointly. \\n2.3. Microbiological Analysis and Phenotypic Characterisation \\nThe enumeration, detection and identification  of bacteria was carried out according to \\nprotocols previously described [16]. Decimal dilutions of the samples were prepared in sterile saline \\n0.9% NaCl. From each dilution, 100 mL were filtered through cellulose membranes of 0.45 µm \\n(Millipore, Bedford, MA, USA) and the filters pl aced on the surface of selective medium for \\nEnterobacteriaceae VRBG (Violet Red Bile Glucose) agar (Oxoid, Hampshire, England). The isolates \\nwere counted after overnight aerobic incubation at  37 °C. The colonies with different morphotypes \\nwere selected, picked out three times and their pu rity further confirmed microscopically. Species \\nidentification was performed according to the ma nufacturer’s recommendations by the standard \\nAPI 20E galleries (BioMerieux, Marcy l’Etoile, Lyon, France). \\nThirteen AB were selected for the phenotypic ch aracterisation the isolates. These were chosen \\nto represent the main AB classes used in human medicine and livestock production in Portugal, \\nnamely: amoxicillin/clavulanic acid combination (AMC) 30 μg/10 μg, respectively; ceftazidime \\n(CAZ) 30 μg; cefotaxime (CTX) 30 μg; cefpirome (CPO) 30 μg; aztreonam (ATM) 30 μg; cefoxitin \\n(FOX) 30 μg; imipenem (IPM) 10 μg; meropenem (MEM) 10 μg; chloramphenicol (CHL) 30 μg; \\ngentamicin (GEN) 10 μg; ciprofloxacin (CIP) 5 μg; trimethoprim/sulfamethoxazol (SXT) \\ncombination (1:19) and tetracycline (TET) 30 μg. The disk diffusion Kirby-Bauer method was \\nperformed in agreement with the guidelines for an timicrobial susceptibility tests defined by the \\nClinical Laboratory Standards Institute [17], using Escherichia coli ATCC25922 (Liofilchem S.R.L., \\nRoseto degli Abruzzi, Italy), E. coli J53-Az R (provided by Instituto de Ciências Biomédicas Abel \\nSalazar, Portugal) and E. coli HB101-StpR–(Bio-Rad Laboratories Lda, Lisbon, Portugal) as quality \\ncontrols, Mueller-Hinton agar and AB disks from Oxoid (Hampshire, England). The isolates with a \\nresistance phenotype against three or more structurally unrelated antimicrobial agents were \\ndefined as multidrug resistant (MDR) [18].  \\n2.4. Multiplex PCR (Polymerase Chain Reaction) for Genes Detection \\nThe screening for resistance genes was focused on a subset of isolates chosen according to their \\nphenotypic profile of resistance. The presence of the most frequent Enterobacteriaceae resistance \\ngenes in these isolates was determined by diff erent multiplex PCR assays. Total DNA of these \\nisolates was extracted as described by Amador et al [19] and 2 µL of each was subjected to pig farm.\"), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 2}, page_content='Antibiotics 2019, 8, 23 3 of 18\\nAs per information obtained from the farms’ personnel, all animals had adequate veterinary\\nmonitoring and a few, when necessary, received medical treatments. The manure and slurry generated\\nin these farms were either composted or directly used as raw soil fertilizers in the animal feed crop\\nﬁelds of each enterprise. According to the slaughterhouse personnel, all animals were subjected to the\\nrequired withdrawal time as per public health safety regulations and the deﬁned holding time before\\nslaughter was followed. Each slaughterhouse had its own protocol for treating wastewater.\\n2.2. Sampling\\nBetween March 2016 and March 2017, three samples were collected per site. Dairy cows or swine\\nmanure samples were collected from open tanks with different maturation ages, whereas poultry\\nmanure from was collected from waste pools.\\nIn the slaughterhouses, samples were collected from reservoirs holding the wastewaters generated\\nby the daily slaughter of animals. The number of animals slaughtered daily is presented in Table 1.\\nSamples were collected in sterile plastic bottles or bags and maintained at 4 ◦C until the\\nmicrobiological processing, which was always carried out within 4 h after sampling. In the laboratory,\\nthe three samples from each site were vigorously mixed and processed conjointly.\\n2.3. Microbiological Analysis and Phenotypic Characterisation\\nThe enumeration, detection and identiﬁcation of bacteria was carried out according to protocols\\npreviously described [16]. Decimal dilutions of the samples were prepared in sterile saline 0.9% NaCl.\\nFrom each dilution, 100 mL were ﬁltered through cellulose membranes of 0.45 µm (Millipore, Bedford,\\nMA, USA) and the ﬁlters placed on the surface of selective medium for Enterobacteriaceae VRBG (Violet\\nRed Bile Glucose) agar (Oxoid, Hampshire, England). The isolates were counted after overnight aerobic\\nincubation at 37 ◦C. The colonies with different morphotypes were selected, picked out three times\\nand their purity further conﬁrmed microscopically. Species identiﬁcation was performed according to\\nthe manufacturer’s recommendations by the standard API 20E galleries (BioMerieux, Marcy l’Etoile,\\nLyon, France).\\nThirteen AB were selected for the phenotypic characterisation the isolates. These were chosen to\\nrepresent the main AB classes used in human medicine and livestock production in Portugal, namely:\\namoxicillin/clavulanic acid combination (AMC) 30 µg/10 µg, respectively; ceftazidime (CAZ) 30\\nµg; cefotaxime (CTX) 30 µg; cefpirome (CPO) 30 µg; aztreonam (ATM) 30 µg; cefoxitin (FOX) 30 µg;\\nimipenem (IPM) 10 µg; meropenem (MEM) 10 µg; chloramphenicol (CHL) 30 µg; gentamicin (GEN) 10\\nµg; ciproﬂoxacin (CIP) 5 µg; trimethoprim/sulfamethoxazol (SXT) combination (1:19) and tetracycline\\n(TET) 30 µg. The disk diffusion Kirby-Bauer method was performed in agreement with the guidelines\\nfor antimicrobial susceptibility tests deﬁned by the Clinical Laboratory Standards Institute [17], using\\nEscherichia coli ATCC25922 (Lioﬁlchem S.R.L., Roseto degli Abruzzi, Italy), E. coli J53-AzR (provided by\\nInstituto de Ciências Biomédicas Abel Salazar, Portugal) and E. coli HB101-StpR–(Bio-Rad Laboratories\\nLda, Lisbon, Portugal) as quality controls, Mueller-Hinton agar and AB disks from Oxoid (Hampshire,\\nEngland). The isolates with a resistance phenotype against three or more structurally unrelated\\nantimicrobial agents were deﬁned as multidrug resistant (MDR) [18].\\n2.4. Multiplex PCR (Polymerase Chain Reaction) for Genes Detection\\nThe screening for resistance genes was focused on a subset of isolates chosen according to their\\nphenotypic proﬁle of resistance. The presence of the most frequent Enterobacteriaceae resistance genes\\nin these isolates was determined by different multiplex PCR assays. Total DNA of these isolates was\\nextracted as described by Amador et al [19] and 2 µL of each was subjected to multiplex PCR in a 25\\nµL reaction mixture containing 1× PCR buffer (200 mM Tris-HCl, pH 8.4, 500 mM KCl) and, according\\nto the target gene, a variable concentration of primers (Table S1), MgCl2, dNTPs and 0.5 U of Taq DNA\\npolymerase (Invitrogen, Thermo Fisher Scientiﬁc, Carlsbad, California, USA) (Table S2) . The PCR\\nconditions described by referenced authors in Table S1 were modiﬁed as speciﬁed in Table S2.'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 3}, page_content='Antibiotics 2019, 8, 23 4 of 18\\nThe chloramphenicol-resistance genes were detected by a single multiplex PCR with four forward\\nprimers and one reverse primer to target four genes, namely,catI, catII, catIII and catIV . The detection\\nof sul1, sul2 and sul3 genes, that confer resistance to sulphonamide, was performed by two multiplex\\nPCR assays given the different annealing temperature for sul3 gene ampliﬁcation (Table S2).\\nThe search for 14 genes involved in three tetracycline resistance mechanisms, namely efﬂux pump,\\nribosomal protection and tetracycline enzymatic alteration, was performed by two multiplex PCRs,\\none targeting genes tet(A), tet(E), tet(G), tet(K), tet(L), tet(M), tet(O) and tet(S) and the other tet(B), tet(C),\\ntet(D), tet(Q), tetA(P) and tet(X).\\nFor the detection of genes conferring resistance to trimethoprim, four distinct multiplex PCRs\\nwere performed according to the dfr gene group. The ﬁrst PCR was used to detect genes belonging to\\ngroups Ia, (dfrA1, dfrA15, dfrA15b, dfrA16, dfrA16b, dfrA28), Ib (dfrA8) and Ic (dfrA12, dfrA13, dfrA21,\\ndfrA22); the second for the group genes IIa (dfrA5, dfrA14, dfrA25, dfrA27), IIb (dfrA7, dfrA17) and IIc\\n(dfrA3b); the third for groups III and IV , namely, IIIa (dfrA3), IIIb (dfrA10), IIIc (dfrA26), IVa (dfrA6),\\nIVb (dfrA24), IVc (dfrA23); and the fourth for the groups Va (dfrB1, dfrB2, dfrB3, dfrB4, dfrB5, dfrB6), Vb\\n(dfrA9), Vc (dfrA19) and Vd (dfrA20) [20].\\nRegarding plasmid-mediated quinolone resistance, nine genes responsible for conferring three\\ndifferent resistance mechanisms were analysed, namely, DNA gyrase protection from the action of\\nthe quinolones ( qnrA, qnrB, qnrS, qnrC and qnrD), AB acetylation ( aac(6’)-Ib-cr) and efﬂux pumps\\nproduction (qepA, oqxA, oqxB). For this purpose, four different multiplex PCR assays were performed\\naccording to the annealing temperature (Table S2); one for the qnrA, qnrB and aac(6’)-Ib-cr; one for\\nqnrD, qnrC and qepA; one for qnrS and oqxA; and a last one for oqxB gene.\\nTo determine whether the ARGs were putatively disseminated via mobile genetic elements, three\\nintegrons classes, known to circulate in food-animal Enterobacteriaceae, were searched among the MDR\\nisolates under study. Classes 1 and 2 of these genes were chosen for being most frequent among\\nEnterobacteriaceae, whereas class 3 was chosen for being presently considered an emergent [21]. For this\\nreason, a PCR was performed to detect intI1, intI2 and intI3 genes.\\nMultiplex PCRs were performed in a thermocycler (iCycler, Bio-Rad, Thermal Cycler, Hercules,\\nCA, USA) and the ampliﬁcation products obtained were separated by electrophoresis on a 1% agarose\\ngel (BioRad), stained with ethidium bromide (125µg/mL) and visualized under a UV transillumination\\n(Vilber Lourmat, Marne-la-Vallé, France).\\n3. Results and Discussion\\nMDR information is crucial to support environmental and public health mitigation actions.\\nEnvironmental systems have very complex microbiome communities, which are difﬁcult to study\\ndue to the inability to artiﬁcially culture the vast majority of the species present in a given sample.\\nMetagenomic approaches have overcome this difﬁculty, revealing the vast genetic diversity of the\\ncollective molecular signature of the samples’ resistomes. However, these molecular methods alone do\\nnot allow attributing bacterial ARG to a single organism and therefore cannot be used to recognise\\nMDR. This study is based on an organismal approach, selecting thein vitro cultivable ARB, speciﬁcally\\nwith the Enterobacteriaceae family. Consequently, the isolates characterized are a slice of the sample\\nbacteria: the AB resistant Enterobacteriaceae.\\n3.1. Microbiological Parameters\\nThe higher enumeration of the Enterobacteriaceae was recorded in manure samples from poultry\\nfarms when compared to those of slaughterhouse, swine and dairy cows (four orders of magnitude\\nsmaller than poultry). The average temperature observed in the samples ranged between 9.4–24.0 ◦C.\\nThe pH values of poultry and pig manure samples were neutral, while dairy cows manure slightly\\nalkaline. Slaughterhouse samples varied from neutral to acid (Table 1).'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 4}, page_content='Antibiotics 2019, 8, 23 5 of 18\\nTable 1. Animal heads per sampling site, average microbiological and physical-chemical parameters\\nof samples.\\nCharacterisation of Sampling Sites Characterisation of Samples\\nSite Code Enterprise Type Animal Heads Enterobacteriaceae\\n(cfu/mL)\\nTemperature\\n(◦C) pH\\nA1 Poultry 10,000 chicks Legorne/pavilion 1.4×107 10.2 6.80\\nA2 Poultry 72,000 caged laying hens Legorne 6.9×109 24.1 7.24\\nA3 Poultry 28,000 soil, cage-free laying hens Legorne 7.6×109 24.0 7.06\\nD1 Dairy cattle 100 Holstein Friesian 1.7×105 9.4 8.81\\nD2 Dairy cattle 30 Holstein Friesian 5.7×105 14.6 7.46\\nD3 Dairy cattle 54 Holstein Friesian 2.4×105 16.9 8.47\\nP1 Pig 48 breeding sow Large Write 3.4×105 14.7 7.71\\nP2 Pig 700 breeding sow, 5300 piglets Large Write 5.0×105 14.0 7.16\\nP3 Pig 1640 fattening pigs Large Write 1.7×106 13.8 7.33\\nS1 Slaughterhouse 505 piglets, 62 cattle (daily abattoir) 1.0×106 14.1 6.52\\nS2 Slaughterhouse 381 hogs, 740 cattle (daily abattoir) 1.7×106 16.0 4.92\\nS3 Slaughterhouse 190 hogs, 40 cattle (daily abattoir) 1.1×109 23.8 7.31\\nTemperature (◦C)—determined by thermometry; pH—determined by potentiometry.\\nOut of the 331 isolates recovered from 12 sampling sites, grouped by four enterprise types,\\n90 were originated from poultry, 71 from dairy, 81 from pig and 89 from slaughterhouse samples.\\nThe Enterobacteriaceae species, detected, by decreasing order of prevalence, were: E. coli, unidentiﬁed\\nisolates, Citrobacter freundii, Raoultella ornithinolytica, Salmonella enterica, C. koseri, Enterobacter cloacae,\\nMorganella morganii, C. braakii, E. hermannii, E. vulneris, Klebsiella oxytoca, Kluyvera spp. A higher\\nfrequency of E. coli was registered in samples from slaughterhouses, pig and poultry farms, whereas C.\\nfreundii was more prevalent in dairy farm samples (Table 2).\\n3.2. Antibiotic Resistance Proﬁle\\nThe analysis of AB resistance proﬁles of the isolates under study revealed by decreasing frequency\\norder, among the β-lactam group, AMC, FOX, CPO, CTX, ATM and CAZ. The carbapenems, IPM\\nand MEM, had the lowest frequencies. With regards to the non-β-lactam group, TET, SXT and CHL\\nrevealed to be less effective, whereas CIP and GEN were the most effectives (Figure 2). These ﬁndings\\ncan be explained by the intensive and recurrent use of tetracyclines, sulphonamides, penicillins and\\n2nd, 3rd and 4th cephalosporins generation in livestock production in Portugal [ 1] and all over\\nthe world [22,23]. The last report of the national control plan of drug use in Portugal referred the\\ntetracyclines (38.9%), penicillins (18.8%), quinolones (4.5%) and sulphonamides (3.4%), as the AB\\nclasses for veterinary use with higher sales [1]. Curiously, as chloramphenicol (CHL) was banned in\\nlivestock according to the European legislation [24], the low sales of amphenicols (0.7%), apparently\\nnot justify the high frequency of CHL resistant isolates.\\nThe AB resistance patterns per enterprise type revealed that the isolates from poultry farms had\\nthe highest resistance rates to TET, SXT, CHL and AMC (Figure 2). These results are in agreement\\nwith reports from: (i) Ghana, with high resistance frequencies to TET (88.9%), sulphonamide (75.0%),\\nampicillin (69.4%) and trimethoprim (66.7%) [25]; (ii) Madagascar broil farms with 97.6% to TET [26];\\n(iii) Chinese carcasses with ampicillin (98.9%), CHL (92.2%), TET (78.9%) [ 27]; and (iv) Portuguese\\nhealthy chickens with TET (70%), ampicillin (63%) and CIP (49%), however, with lower frequencies\\nfor SXT (33%), CHL (12%) and AMC (18%) [ 28]. It is noteworthy a higher frequency of poultry\\nisolates resistant to CIP (30.0%) compared with those from the other enterprise types. An increasing\\nresistance to this AB in poultry farming have been reported, for example from Canada [29], China [30],\\nBrazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing administration\\nof quinolones to treat avian infections [21].'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 5}, page_content='Antibiotics 2019, 8, 23 6 of 18\\nTable 2. List of AB resistance proﬁle, AB resistance and integrons genes harboured per isolate species and sample origin.\\nSample(a) Species Resistance Phenotype(b) AB Resistance Genes intIAMC FOX CTX CAZ CPO ATM IPM MEM CHL CIP GEN SXT TET cat qnr /oqx tet sul dfr\\nPoultry\\nS. enterica x x x x x x x x - - A, M, L, K 1, 2, 3 Ia, IIb -\\nEnterobacter cloacaex x x x x x x x x I D, S A, E, B, K, O 1, 3 Ia -\\nE. hermannii x x x x x x x x x - - C 1, 2 Ic -\\nC. freundii x x x x x x x x I C, S A 1, 2 Ia, IIb, IIIc -\\nC. freundii x x x x x x x x I S A, L, K 1, 2 Ia, Ic, IIIa, IIIc -\\nR. ornithinolytica x x x x - nt - 3 Ia 1\\nE. coli x x x x x x x x x - nt A, C 1, 2, 3 Ia, Ib, IIIa -\\nE. coli x x x x x - nt A, E, C, M, L 2, 3 Ia, IIa -\\nE. coli x x x x x x x - nt A 1, 3 Ia, Ib -\\nE. coli x x x x x x x x - nt A 1, 3 Ia, Ib, Ic, IIIa -\\nE. coli x x x x x - nt A 2 Ia, Ic, IIb -\\nE. coli x x x x x x x - nt A 3 Ia, Ib -\\nnt x x x x x x x x I S A, M 1, 2, 3 Ia, IIb -\\nnt x x x x x x x x x I S A, A(P) 1, 3 Ia -\\nnt x x x x x x x - C, S A 1, 3 Ia 1\\nnt x x x x x x x x I C, S A 1, 2, 3 Ia 1\\nnt x x x x x x I B, S A 1, 3 - I\\nnt x x x x x x I B, C, D, S - 1 Ia 1\\nnt x x x x - nt A, A(P), M - IIb -\\nnt x x x x x x - nt A - - -\\nnt x x x x x - B, S A, L, K 3 - -\\nnt x x x x x - B, S A, L 1, 3 Ia, Ic, IIb -\\nPig\\nS. enterica x x x x x - nt A - Ia, Ic 2\\nK. oxytoca x x x x x - oqxB A 1, 2, 3 Ic, IIa -\\nR. ornithinolytica x x x x x x x I B M 2, 3 - -\\nKluyveraspp. x x x x x x x x x - nt A 2 Ia 2\\nE. coli x x x x - nt A, B - Ic -\\nE. coli x x x x x x x I - A 1, 2, 3 Ia, Ic 1\\nE. coli x x x x x x - B A 1, 2, 3 Ia 2\\nE. coli x x x x x x - - - - Ia, IIIb -\\nE. coli x x x x x x x x I - B, M 3 Ia, IIa 2\\nE. coli x x x x x x x - B A - Ia -\\nE. coli x x x x x x x x x - B A - Ia, IIIb 1\\nE. coli x x x x x x x x x - B A 1, 2 Ia, Ic -\\nnt x x x x - oqxB A, M 1, 2, 3 - 1\\nnt x x x x x x - nt - - - 2\\nnt x x x x x x - B, S A 2 IIIb -\\nnt x x x x x x - nt A, M, K 2, 3 - 2\\nnt x x x x x x - S A, M 3 - 1\\nnt x x x x x x x - nt A 2 IIIb -\\nnt x x x x x x x - nt A 2, 3 - 1'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 6}, page_content='Antibiotics 2019, 8, 23 7 of 18\\nTable 2. Cont.\\nSample(a) Species Resistance Phenotype(b) AB Resistance Genes intIAMC FOX CTX CAZ CPO ATM IPM MEM CHL CIP GEN SXT TET cat qnr /oqx tet sul dfr\\nDairy\\nM. morganii x x x x x II nt A, O 1, 2 - -\\nM. morganii x x x x x x x II nt A, K 1, 2 - -\\nC. freundii x x x x x x x I S A 1, 2 Ia -\\nC. freundii x x x x x x I C, S - 1, 3 Ia 1\\nC. freundii x x x x x x x I C, S A 1, 3 Ia 1\\nC. braakii x x x x x x x - nt A - Ib -\\nC. koseri x x x x x x x x x - nt A, E 1, 3 Ib, IIIb -\\nR. ornithinolytica x x x x - nt A, E - Ia, Ib -\\nC. koseri x x x x - nt - - IIIc -\\nE. coli x x x x - nt A 2 Ia, IIa -\\nE. coli x x x x x x - nt A, E 1 IIc -\\nE. coli x x x x x - nt - - Ia -\\nnt x x x x x x - nt M 3 IIIb, IIIc -\\nSlaughterhouse\\nE. coli x x x - nt A(P), L - Ia, Ic 2\\nE. coli x x x x x - nt O, M 3 Ia -\\nE. coli x x x x x x - nt A, M - Ic -\\nE. coli x x x x x x - - A, B, O, M 3 Ic -\\nE. coli x x x x x x x x x - nt K 1, 3 Ia 1\\nE. coli x x x x x x x x - nt A 2, 3 IIb 1\\nE. coli x x x x x x x x I - A, M 2, 3 Ic, IIb 1\\nS. enterica x x x x - nt - 3 Ia, IIIb 2\\nE. vulneris x x x x x - - B, M 1, 2, 3 Ic, IIb -\\nEnterobacter cloacaex x x x x - nt - - - -\\nC. freundii x x x x x x x - nt - 3 IIIb -\\nnt x x x - nt A(P) - - 1\\nnt x x x x - B, S A, M 3 - -\\nnt x x x x x - B, S B 3 - -\\nnt x x x x x - nt A, A(P), M 3 IIIb 2\\nnt x x x x x - nt A 1 - -\\n(a) Isolates origin per enterprise type: Poultry, Pig, Dairy and Slaughterhouse; (b) AB (antibiotics): AMC, amoxicillin/clavulanic acid; FOX, cefoxitin; CTX, cefotaxime; CAZ, ceftazidime;\\nCPO, cefpirome; ATM, aztreonam; IPM, imipenem; MEM, meropenem; CIP , ciproﬂoxacin; GEN, gentamicin; CHL, chloramphenicol; SXT, trimethoprim/sulfamethoxazol; TET, tetracycline;\\nnt: not tested; -: gene not present.'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 7}, page_content='Antibiotics 2019, 8, 23 8 of 18\\nAntibiotics 2019, 8, x 6 of 18 \\nisolates resistant to CIP (30.0%) compared with those from the other enterprise types. An increasing \\nresistance to this AB in poultry farming have been reported, for example from Canada [29], China \\n[30], Brazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing \\nadministration of quinolones to treat avian infections [21]. \\nNo isolate resisted to MEM and IMP (Figure 2), which agrees wi th many other studies carried \\nout in livestock environments, showing only sporadic reports of isolates resistant to carbapenemases \\n[28,33]. \\n \\nFigure 2. Antibiotic resistance of Enterobacteriaceae isolates obtained from 12 sampling sites. Relative \\nfrequency of resistance patterns exhibited by resistant and intermediate resistant isolates to 13 AB. \\nTested AB: eight β-lactams: amoxicillin/clavulanic acid (A MC), ceftazidime (CAZ), cefotaxime \\n(CTX), cefpirome (CPO), aztreonam (ATM), cefo xitin (FOX), imipenem (IPM), meropenem (MEM) \\nand non- β-lactams: chloramphenicol (CHL), tetracyclin e (TET), gentamicin (GEN), trimethoprim/ \\nsulfamethoxazol (SXT) and cipro ﬂoxacin (CIP). Enterprise type:  poultry,  pig,  dairy,  \\nslaughterhouse. \\nRegarding pig farms, the resistance rates of th e isolates were higher for TET, SXT and CHL, \\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to \\ncarbapenems, IPM and MEM were not surprising, as  previously mentioned. Similar results were \\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34]. \\nIn the same country, another study revealed iden tical results towards carbapenems but all isolates \\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence \\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance \\nphenotypes ampicillin–doxycycline–TET–SXT was reported as the predominant in poultry and \\nswine production in China [36]. \\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by \\nTET, CHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These \\nresults are in agreement with many studies carried out in dairy farms, which usually report high \\nTET and β-lactamic group resistances [37] and low or null carbapenem resistance. However, \\ndisagreeing with most published results, SXT resistances were lower than to CHL.  \\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar \\nto those of the other three enterpri se types. These are unsurprising results, once the majority of the \\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these \\nresults are consistent with many other studies carried out in agricultural environments. For instance, \\na Portuguese study on S. enterica and E. coli strains isolated from pork  and beef products revealed \\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal \\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET \\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39]. \\nFigure 2. Antibiotic resistance of Enterobacteriaceae isolates obtained from 12 sampling sites.\\nRelative frequency of resistance patterns exhibited by resistant and intermediate resistant isolates\\nto 13 AB. Tested AB: eight β-lactams: amoxicillin/clavulanic acid (AMC), ceftazidime (CAZ),\\ncefotaxime (CTX), cefpirome (CPO), aztreonam (ATM), cefoxitin (FOX), imipenem (IPM), meropenem\\n(MEM) and non- β-lactams: chloramphenicol (CHL), tetracycline (TET), gentamicin (GEN),\\ntrimethoprim/sulfamethoxazol (SXT) and ciproﬂoxacin (CIP). Enterprise type:\\nAntibiotics 2019, 8, x 6 of 18 \\n[30], Brazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing \\nadministration of quinolones to treat avian infections [21]. \\nNo isolate resisted to MEM and IMP (Figure 2), which agrees wi th many other studies carried \\nout in livestock environments, showing only sporadic reports of isolates resistant to carbapenemases \\n[28,33]. \\n \\n                            \\nRegarding pig farms, the resistance rates of th e isolates were higher for TET, SXT and CHL, \\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to \\ncarbapenems, IPM and MEM were not surprising, as  previously mentioned. Similar results were \\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34]. \\nIn the same country, another study revealed iden tical results towards carbapenems but all isolates \\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence \\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance \\nphenotypes ampicillin –doxycycline–TET–SXT was reported as the predominant in poultry and \\nswine production in China [36]. \\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by \\nTET, CHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These \\nresults are in agreement with many studies carried out in dairy farms, which usually report high \\nTET and β-lactamic group resistances [37] and low or null carbapenem resistance. However, \\ndisagreeing with most published results, SXT resistances were lower than to CHL.  \\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar \\nto those of the other three enterpri se types. These are unsurprising results, once the majority of the \\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these \\nresults are consistent with many other studies carried out in agricultural environments. For instance, \\na Portuguese study on S. enterica and E. coli strains isolated from pork and beef products revealed \\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal \\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET \\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39]. \\nThe comprehensive analysis of AB resistances rates by enterprise type revealed differences \\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other \\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry. The \\nhighest rates of resistance to all the AB were found among the E. coli  and the unidentified \\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii \\nand to MEM in Kluyvera spp. (Figure S1). \\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless \\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven \\n poultry,\\nAntibiotics 2019, 8, x 6 of 18 \\n[30], Brazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing \\nadministration of quinolones to treat avian infections [21]. \\nNo isolate resisted to MEM and IMP (Figure 2), which agrees wi th many other studies carried \\nout in livestock environments, showing only sporadic reports of isolates resistant to carbapenemases \\n[28,33]. \\n \\n                            \\nRegarding pig farms, the resistance rates of th e isolates were higher for TET, SXT and CHL, \\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to \\ncarbapenems, IPM and MEM were not surprising, as  previously mentioned. Similar results were \\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34]. \\nIn the same country, another study revealed iden tical results towards carbapenems but all isolates \\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence \\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance \\nphenotypes ampicillin –doxycycline–TET–SXT was reported as the predominant in poultry and \\nswine production in China [36]. \\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by \\nTET, CHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These \\nresults are in agreement with many studies carried out in dairy farms, which usually report high \\nTET and β-lactamic group resistances [37] and low or null carbapenem resistance. However, \\ndisagreeing with most published results, SXT resistances were lower than to CHL.  \\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar \\nto those of the other three enterpri se types. These are unsurprising results, once the majority of the \\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these \\nresults are consistent with many other studies carried out in agricultural environments. For instance, \\na Portuguese study on S. enterica and E. coli strains isolated from pork and beef products revealed \\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal \\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET \\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39]. \\nThe comprehensive analysis of AB resistances rates by enterprise type revealed differences \\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other \\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry. The \\nhighest rates of resistance to all the AB were found among the E. coli  and the unidentified \\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii \\nand to MEM in Kluyvera spp. (Figure S1). \\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless \\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven \\n pig,\\nAntibiotics 2019, 8, x 6 of 18 \\n[30], Brazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing \\nadministration of quinolones to treat avian infections [21]. \\nNo isolate resisted to MEM and IMP (Figure 2), which agrees wi th many other studies carried \\nout in livestock environments, showing only sporadic reports of isolates resistant to carbapenemases \\n[28,33]. \\n \\n                            \\nRegarding pig farms, the resistance rates of th e isolates were higher for TET, SXT and CHL, \\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to \\ncarbapenems, IPM and MEM were not surprising, as  previously mentioned. Similar results were \\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34]. \\nIn the same country, another study revealed iden tical results towards carbapenems but all isolates \\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence \\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance \\nphenotypes ampicillin –doxycycline–TET–SXT was reported as the predominant in poultry and \\nswine production in China [36]. \\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by \\nTET, CHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These \\nresults are in agreement with many studies carried out in dairy farms, which usually report high \\nTET and β-lactamic group resistances [37] and low or null carbapenem resistance. However, \\ndisagreeing with most published results, SXT resistances were lower than to CHL.  \\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar \\nto those of the other three enterpri se types. These are unsurprising results, once the majority of the \\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these \\nresults are consistent with many other studies carried out in agricultural environments. For instance, \\na Portuguese study on S. enterica and E. coli strains isolated from pork and beef products revealed \\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal \\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET \\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39]. \\nThe comprehensive analysis of AB resistances rates by enterprise type revealed differences \\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other \\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry. The \\nhighest rates of resistance to all the AB were found among the E. coli  and the unidentified \\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii \\nand to MEM in Kluyvera spp. (Figure S1). \\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless \\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven \\ndairy,\\nAntibiotics 2019, 8, x 6 of 18 \\n[30], Brazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing \\nadministration of quinolones to treat avian infections [21]. \\nNo isolate resisted to MEM and IMP (Figure 2), which agrees wi th many other studies carried \\nout in livestock environments, showing only sporadic reports of isolates resistant to carbapenemases \\n[28,33]. \\n \\n                            \\nRegarding pig farms, the resistance rates of th e isolates were higher for TET, SXT and CHL, \\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to \\ncarbapenems, IPM and MEM were not surprising, as  previously mentioned. Similar results were \\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34]. \\nIn the same country, another study revealed iden tical results towards carbapenems but all isolates \\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence \\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance \\nphenotypes ampicillin –doxycycline–TET–SXT was reported as the predominant in poultry and \\nswine production in China [36]. \\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by \\nTET, CHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These \\nresults are in agreement with many studies carried out in dairy farms, which usually report high \\nTET and β-lactamic group resistances [37] and low or null carbapenem resistance. However, \\ndisagreeing with most published results, SXT resistances were lower than to CHL.  \\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar \\nto those of the other three enterpri se types. These are unsurprising results, once the majority of the \\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these \\nresults are consistent with many other studies carried out in agricultural environments. For instance, \\na Portuguese study on S. enterica and E. coli strains isolated from pork and beef products revealed \\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal \\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET \\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39]. \\nThe comprehensive analysis of AB resistances rates by enterprise type revealed differences \\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other \\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry. The \\nhighest rates of resistance to all the AB were found among the E. coli  and the unidentified \\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii \\nand to MEM in Kluyvera spp. (Figure S1). \\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless \\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven \\n slaughterhouse.\\nNo isolate resisted to MEM and IMP (Figure 2), which agrees with many other studies carried out in\\nlivestock environments, showing only sporadic reports of isolates resistant to carbapenemases [28,33].\\nRegarding pig farms, the resistance rates of the isolates were higher for TET, SXT and CHL,\\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to\\ncarbapenems, IPM and MEM were not surprising, as previously mentioned. Similar results were\\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34].\\nIn the same country, another study revealed identical results towards carbapenems but all isolates\\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence\\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance phenotypes\\nampicillin–doxycycline–TET–SXT was reported as the predominant in poultry and swine production\\nin China [36].\\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by TET,\\nCHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These results\\nare in agreement with many studies carried out in dairy farms, which usually report high TET and\\nβ-lactamic group resistances [37] and low or null carbapenem resistance. However, disagreeing with\\nmost published results, SXT resistances were lower than to CHL.\\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar\\nto those of the other three enterprise types. These are unsurprising results, once the majority of the\\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these\\nresults are consistent with many other studies carried out in agricultural environments. For instance,\\na Portuguese study on S. enterica and E. coli strains isolated from pork and beef products revealed\\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal\\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET\\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39].\\nThe comprehensive analysis of AB resistances rates by enterprise type revealed differences\\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other\\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry.\\nThe highest rates of resistance to all the AB were found among the E. coli and the unidentiﬁed'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 8}, page_content='Antibiotics 2019, 8, 23 9 of 18\\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii and to\\nMEM in Kluyvera spp. (Figure S1).\\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless\\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven\\nAB, when compared with slaughterhouse samples, where a higher rate of isolates simultaneously\\nresistant to a lower number of AB was recorded (Figure 3).\\nAntibiotics 2019, 8, x 7 of 18 \\nThe comprehensive analysis of AB resistances ra tes by enterprise type revealed differences \\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other \\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry. The \\nhighest rates of resistance to all the AB were found among the E. coli  and the unidentified \\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii and to \\nMEM in Kluyvera spp. (Figure S1). \\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless \\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven \\nAB, when compared with slaughterhouse samples, where a higher rate of isolates simultaneously \\nresistant to a lower number of AB was recorded (Figure 3). \\n \\nFigure 3. Relative frequency of the number of antibiotic s to which each isolate resists. Enterprise \\ntype:  poultry,  pig,  dairy,  slaughterhouse. \\nThe MDR rates observed in this study, poultr y (71%), pig (79%), dairy cattle (69%) and \\nslaughterhouse (63%), were higher if compared with other three Portuguese studies. One with pork \\nsamples, (60.7%) [40], other with chicken carcasses samples (56%) [28] and the third one with faecal \\nand tissues samples of chickens and pigs (over 50%) [41]. The increasing trend of the AB co-resistant \\nphenotypes numbers in humans and animals is a consequence over time of the intensive broad \\nspectrum AB use and the introduction of newer compounds in human and veterinary medicine. This \\nincrease of MDR isolates, all over the world, can be explained by the mobile genetic elements in \\nbacteria isolated from facilities farms, including WWTP, farm surrounding air, faeces, carcasses and \\norgans of the animals, identified in numerous studies [21].  \\n3.3. Screening of AB Resistance and Integrons Genes \\nBased on phenotypically characterised isolates, 70 MDR (22, 19, 13 and 16 isolated from poultry, \\npig, dairy and slaughterhouse samples, respectively) were selected for molecular characterisation \\n(Tables 2,3). \\n3.3.1. Chloramphenicol  \\nAll the isolates harbouring CHL resistance genes revealed an according phenotypic resistance \\nprofile (Table 2). However, in 61.4% (43/70) of those isolates no cat genes were detected. These \\nresults may be explained by the CHL resistance mechanisms, which are not exclusively determined \\nby the cat genes family (I, II, III and IV). Althou gh, the plasmid-mediat ed chloramphenicol \\n0 2 04 06 08 0 1 0 0 1 2 0\\n1\\n2\\n3\\n4\\n5\\n6\\n7\\n8\\n9\\nRelative frequency (%)\\nNumber of AB resistances / isolate\\nFigure 3. Relative frequency of the number of antibiotics to which each isolate resists. Enterprise type:\\nAntibiotics 2019, 8, x 6 of 18 \\n[30], Brazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing \\nadministration of quinolones to treat avian infections [21]. \\nNo isolate resisted to MEM and IMP (Figure 2), which agrees wi th many other studies carried \\nout in livestock environments, showing only sporadic reports of isolates resistant to carbapenemases \\n[28,33]. \\n \\n                            \\nRegarding pig farms, the resistance rates of th e isolates were higher for TET, SXT and CHL, \\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to \\ncarbapenems, IPM and MEM were not surprising, as  previously mentioned. Similar results were \\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34]. \\nIn the same country, another study revealed iden tical results towards carbapenems but all isolates \\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence \\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance \\nphenotypes ampicillin –doxycycline–TET–SXT was reported as the predominant in poultry and \\nswine production in China [36]. \\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by \\nTET, CHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These \\nresults are in agreement with many studies carried out in dairy farms, which usually report high \\nTET and β-lactamic group resistances [37] and low or null carbapenem resistance. However, \\ndisagreeing with most published results, SXT resistances were lower than to CHL.  \\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar \\nto those of the other three enterpri se types. These are unsurprising results, once the majority of the \\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these \\nresults are consistent with many other studies carried out in agricultural environments. For instance, \\na Portuguese study on S. enterica and E. coli strains isolated from pork and beef products revealed \\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal \\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET \\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39]. \\nThe comprehensive analysis of AB resistances rates by enterprise type revealed differences \\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other \\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry. The \\nhighest rates of resistance to all the AB were found among the E. coli  and the unidentified \\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii \\nand to MEM in Kluyvera spp. (Figure S1). \\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless \\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven \\npoultry,\\nAntibiotics 2019, 8, x 6 of 18 \\n[30], Brazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing \\nadministration of quinolones to treat avian infections [21]. \\nNo isolate resisted to MEM and IMP (Figure 2), which agrees wi th many other studies carried \\nout in livestock environments, showing only sporadic reports of isolates resistant to carbapenemases \\n[28,33]. \\n \\n                            \\nRegarding pig farms, the resistance rates of th e isolates were higher for TET, SXT and CHL, \\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to \\ncarbapenems, IPM and MEM were not surprising, as  previously mentioned. Similar results were \\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34]. \\nIn the same country, another study revealed iden tical results towards carbapenems but all isolates \\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence \\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance \\nphenotypes ampicillin –doxycycline–TET–SXT was reported as the predominant in poultry and \\nswine production in China [36]. \\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by \\nTET, CHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These \\nresults are in agreement with many studies carried out in dairy farms, which usually report high \\nTET and β-lactamic group resistances [37] and low or null carbapenem resistance. However, \\ndisagreeing with most published results, SXT resistances were lower than to CHL.  \\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar \\nto those of the other three enterpri se types. These are unsurprising results, once the majority of the \\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these \\nresults are consistent with many other studies carried out in agricultural environments. For instance, \\na Portuguese study on S. enterica and E. coli strains isolated from pork and beef products revealed \\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal \\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET \\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39]. \\nThe comprehensive analysis of AB resistances rates by enterprise type revealed differences \\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other \\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry. The \\nhighest rates of resistance to all the AB were found among the E. coli  and the unidentified \\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii \\nand to MEM in Kluyvera spp. (Figure S1). \\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless \\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven \\n pig,\\nAntibiotics 2019, 8, x 6 of 18 \\n[30], Brazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing \\nadministration of quinolones to treat avian infections [21]. \\nNo isolate resisted to MEM and IMP (Figure 2), which agrees wi th many other studies carried \\nout in livestock environments, showing only sporadic reports of isolates resistant to carbapenemases \\n[28,33]. \\n \\n                            \\nRegarding pig farms, the resistance rates of th e isolates were higher for TET, SXT and CHL, \\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to \\ncarbapenems, IPM and MEM were not surprising, as  previously mentioned. Similar results were \\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34]. \\nIn the same country, another study revealed iden tical results towards carbapenems but all isolates \\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence \\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance \\nphenotypes ampicillin –doxycycline–TET–SXT was reported as the predominant in poultry and \\nswine production in China [36]. \\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by \\nTET, CHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These \\nresults are in agreement with many studies carried out in dairy farms, which usually report high \\nTET and β-lactamic group resistances [37] and low or null carbapenem resistance. However, \\ndisagreeing with most published results, SXT resistances were lower than to CHL.  \\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar \\nto those of the other three enterpri se types. These are unsurprising results, once the majority of the \\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these \\nresults are consistent with many other studies carried out in agricultural environments. For instance, \\na Portuguese study on S. enterica and E. coli strains isolated from pork and beef products revealed \\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal \\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET \\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39]. \\nThe comprehensive analysis of AB resistances rates by enterprise type revealed differences \\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other \\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry. The \\nhighest rates of resistance to all the AB were found among the E. coli  and the unidentified \\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii \\nand to MEM in Kluyvera spp. (Figure S1). \\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless \\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven \\n dairy,\\nAntibiotics 2019, 8, x 6 of 18 \\n[30], Brazil [31] and India [32]. This raise of resistance to CIP might be due to the increasing \\nadministration of quinolones to treat avian infections [21]. \\nNo isolate resisted to MEM and IMP (Figure 2), which agrees wi th many other studies carried \\nout in livestock environments, showing only sporadic reports of isolates resistant to carbapenemases \\n[28,33]. \\n \\n                            \\nRegarding pig farms, the resistance rates of th e isolates were higher for TET, SXT and CHL, \\nfollowed by the β-lactams AMC, ATM and CTX (Figure 2). The lower resistance frequencies to \\ncarbapenems, IPM and MEM were not surprising, as  previously mentioned. Similar results were \\npublished from China with over 60% for TET and CHL and no resistances to carbapenem (IMP) [34]. \\nIn the same country, another study revealed iden tical results towards carbapenems but all isolates \\nwere resistant to TET, ampicillin, CAZ and CTX. However, contrary to our results, a lower incidence \\nfor CHL (26.3%) and SXT (52.6%) were detected [35]. The association among the resistance \\nphenotypes ampicillin –doxycycline–TET–SXT was reported as the predominant in poultry and \\nswine production in China [36]. \\nAmong dairy farm isolates, higher resistance prevalence was observed to AMC, followed by \\nTET, CHL and SXT. A single isolate resistant to carbapenem, MEM was found (Figure 2). These \\nresults are in agreement with many studies carried out in dairy farms, which usually report high \\nTET and β-lactamic group resistances [37] and low or null carbapenem resistance. However, \\ndisagreeing with most published results, SXT resistances were lower than to CHL.  \\nThe AB resistance patterns (TET, SXT, CHL and AMC) of slaughterhouses isolates were similar \\nto those of the other three enterpri se types. These are unsurprising results, once the majority of the \\nanimals slaughtered in the abattoirs in this region, are locally raised cattle and pigs. Moreover, these \\nresults are consistent with many other studies carried out in agricultural environments. For instance, \\na Portuguese study on S. enterica and E. coli strains isolated from pork and beef products revealed \\nsimilar resistances to TET, sulfamethoxazole and ampicillin [38]. In addition, a study in swine faecal \\nsamples obtained from Spanish slaughterhouses, showed the following resistance percentages: TET \\n(90%), AMC (78%); SXT (67.5%); CHL (26%); GEN (6%) and CTX (0%) [39]. \\nThe comprehensive analysis of AB resistances rates by enterprise type revealed differences \\nbetween poultry and pig isolates compared to those of dairy cattle. This seems consistent with other \\nresults [21] also reporting a lower prevalence of resistant E. coli in cattle than in pig and poultry. The \\nhighest rates of resistance to all the AB were found among the E. coli  and the unidentified \\nEnterobacteriaceae isolates. Worth noting was the high resistance to CTX in Citrobacter freundii \\nand to MEM in Kluyvera spp. (Figure S1). \\nThe majority of the isolates under study resisted simultaneously to three or four AB, regardless \\nthe sample origin. Poultry farm samples had the highest rate of resistant isolates to more than seven \\n slaughterhouse.\\nThe MDR rates observed in this study, poultry (71%), pig (79%), dairy cattle (69%) and\\nslaughterhouse (63%), were higher if compared with other three Portuguese studies. One with\\npork samples, (60.7%) [40], other with chicken carcasses samples (56%) [28] and the third one with\\nfaecal and tissues samples of chickens and pigs (over 50%) [ 41]. The increasing trend of the AB\\nco-resistant phenotypes numbers in humans and animals is a consequence over time of the intensive\\nbroad spectrum AB use and the introduction of newer compounds in human and veterinary medicine.\\nThis increase of MDR isolates, all over the world, can be explained by the mobile genetic elements in\\nbacteria isolated from facilities farms, including WWTP , farm surrounding air, faeces, carcasses and\\norgans of the animals, identiﬁed in numerous studies [21].\\n3.3. Screening of AB Resistance and Integrons Genes\\nBased on phenotypically characterised isolates, 70 MDR (22, 19, 13 and 16 isolated from poultry,\\npig, dairy and slaughterhouse samples, respectively) were selected for molecular characterisation\\n(Tables 2 and 3).\\n3.3.1. Chloramphenicol\\nAll the isolates harbouring CHL resistance genes revealed an according phenotypic resistance\\nproﬁle (Table 2). However, in 61.4% (43/70) of those isolates no cat genes were detected. These results\\nmay be explained by the CHL resistance mechanisms, which are not exclusively determined by the\\ncat genes family (I, II, III and IV). Although, the plasmid-mediated chloramphenicol acetyltransferase\\n(cat) gene is the most common mechanism, other non-enzymatic resistance plasmidic genes, cmlA and\\nﬂoR, encoding efﬂux pumps, have also been identiﬁed in Enterobacteriaceae. The latter gene encodes a\\nﬂorfenicol/chloramphenicol transporter that confers resistance to both AB. A less common resistance\\nmechanism involves mutations in 50S ribosomal subunit [42,43].'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 9}, page_content='Antibiotics 2019, 8, 23 10 of 18\\nNo CHL resistant isolates harboured genes catIII and catIV and those which had a gene, had\\neither catI or catII (Table 3). The highest frequencies of cat genes were detected in samples from poultry\\nand dairy cattle farms. The gene catI was more prevalent (21.4%) than catII (2.9%), the latter only\\nrepresented by two M. morganii isolates from dairy samples (Table 3). Similarly, a study on MDRE. coli\\nfrom Irish cattle and the farm environment, reported 9% CHL resistant isolates, mainly mediated by\\ncatI and ﬂoR [42]. Other reports also reveal that catI gene together with catIII gene is the most widely\\ndistributed, being catII less frequent [44]. However, a Korean study on ﬁsh pathogens isolates detected\\nthe catII gene as the most frequent, although catIV was also found in some isolates [45].\\nThe high resistance to CHL found in this and other studies may have several explanatory\\nhypotheses. Assuming that this AB is no longer used in human and veterinary medicine due to its\\nproven chronic toxicity, the bacteria continue to resist because the determinant genes (often integrated\\nin gene cassettes inserted in genetic elements, including integrons) could be transferred among them,\\neven without high selective drug pressure. Another explanation is the licensed use of ﬂorfenicol for\\ntreatment of respiratory infections in cattle and pigs (Regulation (EEC) No 2377/90). This AB is a\\nderivative of CHL, which shares one gene that determines the resistance mechanism [21].\\nTable 3. Absolute and relative frequency of isolates with cat, qnr, oqx, aac(6’)-Ib,qep, tet, sul, dfr and intI\\ngenes per enterprise type.\\nTarget Gene/Group Enterprise Type\\nTotal *Poultry * Pig * Dairy * Slaughterhouse *\\ncat\\nI 8 (36.4) 3 (15.7) 3 (23.1) 1 (6.3) 15 (21.4)\\nII 2 (15.4) 2 (2.9)\\nIII, IV 0 (0.0)\\nqnr\\nA 0 (0.0)\\nB 4 (30.8) 6 (50.0) 2 (40.0) 12 (36.4)\\nC 4 (30.8) 2 (66.7) 6 (18.1)\\nD 2 (15.4) 2 (6.1)\\nS 11 (84.6) 2 (16.7) 3 (100.0) 2 (40.0) 18 (54.6)\\noqx A 0 (0.0)\\nB 2 (16.7) 2 (6.1)\\naac(6’)-Ib 0 (0.0)\\nqep A 0 (0.0)\\ntet\\nA 19 (86.4) 15 (78.9) 9 (69.2) 7 (43.8) 50 (71.4)\\nB 1 (4.6) 2 (10.5) 3 (18.8) 6 (8.6)\\nC 3 (13.6) 3 (4.3)\\nE 2 (9.1) 3 (23.1) 5 (7.1)\\nK 4 (18.2) 1 (5.3) 1 (7.7) 1 (6.3) 7 (10.0)\\nL 5 (22.7) 1 (6.3) 6 (8.6)\\nM 4 (18.2) 5 (26.3) 1 (7.7) 7 (43.8) 17 (24.3)\\nO 1 (4.6) 1 (7.7) 2 (12.5) 4 (5.7)\\nA(P) 2 (9.1) 3 (18.8) 5 (7.1)\\nD, G, S, Q, X 0 (0.0)\\nsul\\n1 15 (68.2) 5 (26.3) 7 (53.8) 3 (18.8) 30 (42.9)\\n2 9 (40.9) 11 (57.9) 4 (30.8) 3 (18.8) 27 (38.6)\\n3 15 (68.2) 9 (47.4) 4 (30.8) 11 (68.8) 40 (57.1)\\ndfr\\nIa 17 (77.2) 9 (47.4) 6 (46.2) 4 (25.0) 36 (51.4)\\nIb 4 (18.2) 3 (23.1) 7 (10.0)\\nIc 5 (22.7) 5 (26.3) 5 (31.3) 15 (21.4)\\nIIa 1 (4.6) 2 (10.5) 1 (7.7) 4 (5.7)\\nIIb 6 (27.3) 3 (18.8) 9 (12.9)\\nIIc 1 (7.7) 1 (1.4)\\nIIIa 3 (13.6) 3 (4.3)\\nIIIb 4 (21.1) 2 (15.4) 3 (18.8) 9 (12.9)\\nIIIc 2 (9.1) 2 (15.4) 4 (5.7)\\nIVa, IVb, IVc 0 (0.0)\\nVa, Vb, Vc, Vd 0 (0.0)\\nintI 1 5 (22.7) 5 (26.3) 2 (15.4) 4 (25.0) 16 (22.9)\\n2 6 (31.6) 3 (18.8) 9 (12.9)\\n3 0 (0.0)\\n* n (%), absolute and (relative) frequency of isolates carrying the gene. The blank space means a negative result, i.e.,\\n0 (0%).'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 10}, page_content='Antibiotics 2019, 8, 23 11 of 18\\n3.3.2. Quinolones\\nThe detection of the genes conferring resistance to the quinolones class, showed a higher incidence\\namong isolates of poultry farms (Table 2). Quinolones, such as diﬂoxacin, enroﬂoxacin, danoﬂoxacin,\\nmarboﬂoxacin, pradoﬂoxacin represent 4.5% of the AB sales, mainly for the avian sector in Portugal [1].\\nLikewise, ciproﬂoxacin (CIP), levoﬂoxacin, norﬂoxacin, oﬂoxacin are widely used clinically to treat\\navian colibacillosis in many other countries [ 30]. The intensiﬁcation of this AB class usage might\\nexplain the higher relative frequencies of CIP resistance in poultry isolates than others, sampled in\\nother enterprise types (Figure 2). These quinolones class resistances might difﬁcult the future therapy\\nin this livestock sector.\\nNo qnrA, oqxA, aac(6’)–Ib-cr andqepA genes were detected in any isolate. The qnrS gene was\\nthe most prevalent, mainly in the assayed poultry isolates (Table 3; Table 4), which is consistent with\\nprevious works carried out worldwide. An Italian work showed a higher frequency of the qnrS1 and\\nqnrB19 genes for poultry isolates, respectively but no genes of qnrC, qnrD, qepA and aac(6’)-Ib-cr [46].\\nOther study involving E. coli isolates from chicken farms and turkeys at slaughterhouses in the Czech\\nRepublic, showed through screening of several genes, namely, qnrB1, qnrB4, qnrB8, qnrB10, qnrB19,\\nqnrD, qnrS1, qnrS2, aac(6’)-Ib-cr andoqxAB, the same trend of results, a higher prevalence of qnrS1\\nfollowed by qnrB19 [47]. Another Chinese study, investigating the quinolones resistance in E. coli\\nisolated from septicaemic broilers, namely, qnrA, qnrB, qnrS and aac(6’)-Ib-cr, recorded the highest\\nprevalence of the gene aac(6’)-Ib-cr (36.0%), followed byqnrS (8.1%), qnrB (0.9%) and qnrA (0%) [30].\\nConversely, in Nigerian poultry faecal samples no isolates containing theqnrS gene were registered,\\nbeing oqxB the most prevalent [48].\\nTable 4. Resistance phenotype of isolates from four livestock enterprises to non-β-lactamic AB and\\ncorresponding presence of integrons class.\\nAB Resistant Isolates (N) Resistant Isolates with Integrons, N (%) Resistant Isolates without\\nIntegrons, N (%)intI1 intI2\\nCHL 58 14 (24.1%) 8 (13.8%) 36 (62.1%)\\nCIP 29 10 (34.5%) 1 (3.4%) 18 (62.1%)\\nGEN 7 1 (14.3%) 0 (0.0%) 6 (85.7%)\\nSXT 56 15 (26.8%) 8 (14.3%) 33 (58.9%)\\nTET 63 13 (20.6%) 8 (12.7%) 42 (66.7%)\\nN (%), absolute and (relative) frequency of isolates carrying the gene.\\nConsidering the pig farm isolates, our data revealed a higher prevalence for the qnrB gene,\\nfollowed by the same prevalence for qnrS and oqxB, with 2 isolates each (Table 3). A study on sewage\\nand soil adjacent to swine feedlots, showed a higher frequency for the qnrD, qepA and oqxB genes,\\nwhereas qnrS and oqxA were only present in the samples of residual waters [49].\\n3.3.3. Tetracycline\\nThe screening of the 14 most frequent tet genes conferring resistance to TET revealed that\\ngenotypes are consistent with the phenotypes observed, excepting for three isolates, from pig and\\ndairy farms (Tables 2 and 3). The TET resistance in those isolates without tet genes, might be due\\nto other resistance genes, not targeted in this study. There are at least 47 distinct genes identiﬁed,\\nresponsible for four main mechanisms by which the bacteria acquire resistance to tetracyclines [50].\\nAdditionally, no isolates harboured the following genes: tet(D) and tet(G), responsible for the\\nefﬂux pump resistance mechanism; tet(S) and tet(Q), responsible for the mechanism of ribosomal\\nprotection and tet(X), for the degradation of AB (Table 3). The most incident genes were tet(A) and\\ntet(M) (Table 3), encoding an efﬂux pump and providing ribosomal protection, respectively. All the\\nother surveyed genes had frequencies below or equal 10%. The gene tet(A) prevailed in isolates of\\nall the four enterprise types; however, the second most frequent gene varied according to the sample'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 11}, page_content='Antibiotics 2019, 8, 23 12 of 18\\nsource: tet(L) in poultry farms, tet(E) in dairy farms and tet(M) in pig farms and slaughterhouses. The\\nprevalence of tet(A) gene in this study is in agreement with the dominant tet gene in different animal\\nand environmental samples reported worldwide in a myriad of studies [42,51–54].\\nThe isolates from poultry farms harboured the highest diversity of different tet genes (9/14),\\nfollowed by slaughterhouses isolates with (7/14), conversely to those from dairy and pig farms, which\\nshowed lower genes diversity. The tet(C) gene was unique from poultry isolates group. Regarding\\nthe number of different tet genes per isolate, 14.3%, 45.7%, 27.1% and 7.1% of them contained, zero,\\none, two and three, respectively (Table 2). It should also be noted that one isolate of E. cloacae and one\\nof E. coli from the poultry farms presented ﬁve different genes, while two other isolates, a S. enterica\\nfrom laying hens samples and an E. coli, from the slaughterhouse, presented four genes. The analysis\\nby sample source showed that the poultry farm isolates had the higher number of resistant isolates\\ncontaining more tet gene types simultaneously. Identical results were reported in a study carried out\\nin swine sewage ponds, where the isolates most often harbour multiple tet genes [55].\\n3.3.4. Sulphonamide\\nThe most prevalent resistance genes to the sulphonamide class were sul3, followed by sul1 and at\\nlast sul2 (Table 3); however, this ranking varies according to the sample source. For instance, among\\nthe poultry farm isolates genes sul1 and sul3 were more frequent than sul2. Curiously, the isolates\\nfrom dairy farms presented a higher incidence of sul1, whereas sul3 prevailed in slaughterhouses\\nisolates and sul2 in those from pig farms. These genes prevalence can vary greatly with the matrixes,\\ngeographic region, as well as the species.\\nOur results are partially in accordance with two studies carried out in Denmark on isolates of\\nE. coli resistant to sulphonamide obtained from swine faeces. The ﬁrst work presented the highest\\nfrequency for the sul1 gene (55%), followed by sul2 (50%) and ﬁnally sul3 (11%) [56], whereas the\\nsecond work reported sul2 gene, as the most prevalent (65%) [57]. On the other hand, a Portuguese\\nstudy on resistances of E. coli isolated from carcasses and internal organs of healthy chickens from\\nintensive farms detected 37 different resistance genes and the most common were tet(A) (72%) and\\nsul1 (47%) [28]. A study in Cambodia on sulfamethoxazole-resistant isolates obtained from faecal\\nsamples of healthy chickens found a higher prevalence of thesul2 gene in E. coli (56%) and in Salmonella\\nCorvallis (53%). However, among the Salmonella Albany the most prevalent gene was sul1 (54%) [58].\\nThe analysis by isolates shows that 22.9% did not have a sul gene, 30.0% harboured one type of\\nsul gene, 34.3% contained simultaneously two and 12.9% three sul genes (Table 2). It should be noted\\nthat the latter isolates were more prevalent in poultry and pig farm samples, which corroborates the\\nidea that there is a higher AB selective pressure in these two livestock intensive raising systems.\\n3.3.5. Trimethoprim\\nRegarding the dfr genes detected in this study, a higher prevalence of thedfrIa group was found as\\na general trend in all enterprise type samples; however, being most prevalent in those of poultry farms\\n(Table 3). Genes of dfrIc, dfrIIIb, dfrIIb groups were recorded but dfrIV and dfrV were not identiﬁed.\\nMoreover, the dfrIIc group genes had the lowest detection rate, in only an isolate from a dairy farm\\nsample. The isolates of poultry and dairy farms showed the highest diversity in this gene family. These\\nresults are in accordance with those of a Lithuanian study about the prevalence of dfr genes on E. coli\\nisolates from human and animal clinical samples, concluding that 92% contained at least one of the six\\ngenes, dfrA1, dfrA5, dfrA8, dfrA12, dfrA14 and dfrA17 [20]. This gene nomenclature corresponds in our\\nstudy to the genes of the group dfrIa, dfrIIa, dfrIb, dfrIc, dfrIIa, dfrIIb, respectively). The genes dfrA1\\n(dfrIa group) and dfrA17 (dfrIIb group) were more frequently found in clinical isolates, while the dfrA1\\n(dfrIa group) and dfrA14 (dfrIIa group) genes dominated in isolates of animal origin. The genes dfrA5\\n(dfrIIa group), dfrA12 (dfrIc group) and dfrA8 (dfrIb group) were detected at lower frequencies.\\nFor the analysis of the genes involved in SXT resistance it is required to consider simultaneously\\nthe genes sul and dfr, since they act synergistically to confer resistance to an association of two'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 12}, page_content='Antibiotics 2019, 8, 23 13 of 18\\nantimicrobials, the sulfamethoxazole and the trimethoprim, both inhibitors of the biosynthetic pathway\\nof folic acid. Resistance to both AB occurs by the acquisition of genes encoding enzymes that have no\\nafﬁnity for binding to these antimicrobial compounds, inactivating them [43].\\nAll isolates listed in Table 2 with at least one type of gene from each family,sul or dfr exhibited\\nSXT resistance, as for example some SXT resistant isolates without dfr genes but with one or more sul\\ngenes. This occurrence may be explained, as previously stated, by the presence of other genes that\\nwere not searched in this study, since there are more than 30 different known genes of trimethoprim\\nresistance [20]. On the contrary, the phenotype of SXT resistant isolates lacking the sul genes might be\\nexplained by chromosomal mutations, resulting in molecular variants providing this resistance [43],\\neven in absence of the resistance genes sul1, sul2 and sul3.\\n3.3.6. Integrons\\nThe analysis of the AB resistance proﬁles reveals their clear MDR phenotype (Table 2).\\nThose isolates with co-resistant phenotypes showed that the underlying resistance determinants\\nof the non- β-lactams are based on three or more gene groups: tetracycline ( tet(A) and tet(M)),\\nsulfamethoxazole ( sul3 and sul1), trimethoprim ( dfrA1-like) and CHL ( catI) (Table 4). Although\\nit was not the scope of this work, the information on the Extended Spectrum Beta-Lactamases (ESBL)\\ngenotyping of these isolates would have been interesting, as these enzymes are often associated with\\nthe MDR [ 59]. Nevertheless, our results show the prevalence of a common resistance association\\npattern of several gene classes, which are known to be specially spread and persistent in livestock\\nenvironments [21]. The widespread dissemination of the same MDR phenotypes mediating AB\\nresistance was also reported by the European surveys on healthy food animals [22,60]. Other studies\\nalso show a predominant association among ampicillin-tetracycline-sulfamethoxazole/trimethoprim\\nin Chinese poultry and pig farming [36].\\nThe search for three integrons classes among the 65 MDR isolates under study, revealing 25\\nintegrons (38.5%) of classes 1 and 2 and none of class 3. This latter, result is expected, as it is not\\nprevalent in Enterobacteriaceae. The integrase gene IntI2 (class 2), initially present in Tn7 transposon\\nand derivatives, is more frequent than the class 3 integrons [ 61]. Moreover, no isolate harboured\\nsimultaneously two integrons classes.\\nClass 1 and class 2, with 16 and 9 integrons, respectively were the most prevalent in isolates of\\nall enterprise type samples (Table 3). These results are consistent with those of a Portuguese study\\nconducted in poultry and pigs farms, having identiﬁed ﬁve isolates with class 1 of integrons and\\none isolate with class 2 [ 41]. Likewise, other US study on Enterobacteriaceae isolates obtained from\\nﬁsh, poultry, swine and cattle reported that 52% of them contained either class 1 or class 2 integrases,\\nwith the frequencies of 46% intI1, 6% intI2 and 5% intI1 plus intI2 [62]. Additionally, another study\\nperformed on MDR K. pneumoniae isolates from Iranian hospitals showed similar trends regarding the\\nprevalence of the two integrons classes and a positive association between the class 1 integrons and\\nMDR [63]. A Polish investigation on clinical samples from patients not undergoing AB therapy and\\nfrom a WWTP detected 12.1% of the E. coli isolates with integrase genes, 10.9% of which with class 1,\\n1.4% with class 2 and 0% with class 3 [64].\\nContrary to what was expected, the integrons were not prevalent among the isolates of poultry\\nfarm samples, which contradict the idea that the prevalence of integrons could reﬂect a strong AB\\npressure, selecting for those isolates with integrons [64]. In fact, the highest prevalence was recorded\\nin pig farm isolates, with 11 integrons of both classes (Table 3). These frequency trends of the integrons\\nclasses are somewhat consistent with those found in E. coli isolated from samples of different livestock\\nenvironments in Georgia reported for: (i) poultry isolates, intI1 (66%), intI2 (14%) and intI1 plus intI2\\n(11%); (ii) swine isolates, intI1 (86%), intI2 (0%) and intI1 plus intI2 (0%); and iii) beef cattle isolates,\\nintI1 (75%), intI2 (23%) and intI1 plus intI2 (20%) [62].\\nComparing the resistances to the non- β-lactamic AB with the presence of integrons in all the\\nisolates under study, a positive relation between the SXT, CHL and TET resistant phenotype with'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 13}, page_content='Antibiotics 2019, 8, 23 14 of 18\\nthe presence of integrons was observed. This relation is less evident with the quinolone CIP and\\naminoglycoside GEN (Table 4).\\nA Portuguese study on isolates from several polluted rivers, due to heavy domestic, industrial\\nand agricultural inﬂuence, registered among the cefotaxime-resistant isolates, the prevalence of class 1\\nintegrase (56.41% in those ESBL+) and the gene cassettes identiﬁed conferred resistance to β-lactams,\\ntrimethoprim and chloramphenicol [59]. The different sampling matrices may account to dissimilar\\nresults to ours, regarding CIP and GEN. However, several studies revealing the diversity of class 1\\nintegrons and the gene cassettes in Enterobacteriaceae, indicate that the integrons generate new bindings\\nof AB resistances, mainly in areas highly exposed to antimicrobial agents [65].\\n4. Conclusions\\nThis study demonstrated a high prevalence of MDR Enterobacteriaceae and high resistance-gene\\ndiversity in livestock manure, possibly a consequence of intensive animal farming. Therefore, ﬁeld\\namended with manure containing these biological contaminants can potentially contribute to an\\nincreased frequency of MDR bacteria in surrounding soil and aquatic ecosystems. As untreated\\nmanure is often used as a fertiliser of maize, rice and horticultural crops in agricultural ﬁelds of the\\nVouga, Lis and Mondego valleys, the common practices of nutrient and organic matter reuse for\\ncrop production has become a matter of concern, since the resistance of zoonotic bacteria should\\nalso be considered, as those contaminants may be transferred to livestock and humans through the\\nconsumption of raw crops fertilised with contaminated manure.\\nIt should be important not only to reinforce the prudent, ethical and professional use of AB but\\nalso to improve appropriate treatments in the WWTP of the farms and slaughterhouses. Our ﬁndings\\nprovide an insight into the real impact of livestock farming manure to the environmental and public\\nhealth problem of these contaminants in the Central Region of mainland Portugal.\\nSupplementary Materials: The following are available online at http://www.mdpi.com/2079-6382/8/1/23/s1,\\nFigure S1: Relative frequency of antibiotics resistance by species. Table S1: Primers used for the identiﬁcation\\nof chloramphenicol ( cat), trimethoprim ( dfr), quinolones ( qnr, aac(6’)-Ib, oqx, qep), sulphonamides ( sul) and\\ntetracyclines (tet) resistance genes and for integron class, Table S2: Multiplex PCR conditions for target genes.\\nAuthor Contributions: P .A. and I.D. conceived and design the study, carried out the sampling and the biological\\nanalysis, worked on the microbiological and molecular characterisation of antibiotic resistance bacteria. P .A.\\nperformed the data analysis and worked on the manuscript. C.P . and R.F. contributed with reagents and with I.D.\\nworked on the manuscript.\\nFunding: This research received no external funding and the APC was funded by the authors.\\nAcknowledgments: This research was conducted within project Grupo Operacional para a Gestão da Água no\\nVale do Lis, PDR2020-1.0.1-FEADER-030911, funded by Program PDR2020, co-funded by FEDER, Innovation\\nMeasure, Portugal and by Research Centre for Natural Resources, Environment and Society—CERNAS—under\\nproject UID/AMB/00681/2019, funded by the Portuguese Science and Technology Foundation (FCT). The authors\\nwould like to thank Engineer Diogo Vilão for technical support and advice selecting the sampling sites.\\nConﬂicts of Interest: The authors declare that there are no conﬂicts of interest.\\nReferences\\n1. DGAV .Relatório Nacional de Monitorização do Consumo de Antimicrobianos Ano 2013—Portugal ; Direcção-Geral\\nde Alimentaç ão e Veterin ária/Direcção de Serviços de Medicamentos e Produtos de Uso Veterin ário.\\nMinistério da Agricultura, Mar, Ambiente e Ordenamento do Territ ório; Governo de Portugal: Lisbon,\\nPortugal, 2014; pp. 1–13.\\n2. Jury, K.L.; Vancov, T.; Stuetz, R.M.; Khan, S.J. Antibiotic resistance dissemination and sewage treatment\\nplants. In Current Research, Technology and Education Topics in Applied Microbiology and Microbial Biotechnology;\\nMéndez-Vilas, A., Ed.; Formatex Research Center: Badajoz, Spain, 2010; pp. 509–519. ISBN 978-84-614-6194-3.\\n3. Faldynova, M.; Videnska, P .; Havlickova, H.; Sisak, F.; Juricova, H.; Babak, V .; Steinhauser, L.; Rychlik, I.\\nPrevalence of antibiotic resistance genes in faecal samples from cattle, pigs and poultry. Vet. Med. 2013, 58,\\n298–304. [CrossRef]'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 14}, page_content='Antibiotics 2019, 8, 23 15 of 18\\n4. Fluit, A.C.; Schmitz, F.J. Class 1 integrons, gene cassettes, mobility and epidemiology.Eur. J. Clin. Microbiol.\\nInfect. Dis. 1999, 18, 761–770. [CrossRef] [PubMed]\\n5. Kaushik, M.; Kumar, S.; Kapoor, R.K.; Virdi, J.S.; Gulati, P . Integrons in Enterobacteriaceae: Diversity,\\ndistribution and epidemiology. Int. J. Antimicrob. Agents 2017, 51, 167–176. [CrossRef]\\n6. Martinez-Carballo, E.; Gonzalez-Barreiro, C.; Scharf, S.; Gans, O. Environmental monitoring study of selected\\nveterinary antibiotics in animal manure and soils in Austria. Environ. Pollut. 2007, 148, 570–579. [CrossRef]\\n7. Ji, X.L.; Shen, Q.H.; Liu, F.; Ma, J.; Xu, G.; Wang, Y.L.; Wu, M.H. Antibiotic resistance gene abundances\\nassociated with antibiotics and heavy metals in animal manures and agricultural soils adjacent to feedlots in\\nShanghai; China. J. Hazard. Mater. 2012, 235, 178–185. [CrossRef] [PubMed]\\n8. Yang, Q.; Zhang, H.; Guo, Y.; Tian, T. Inﬂuence of Chicken Manure Fertilization on Antibiotic-Resistant\\nBacteria in Soil and the Endophytic Bacteria of Pakchoi. Int. J. Environ. Res. Public Health 2016, 13, 662.\\n[CrossRef] [PubMed]\\n9. Gao, L.; Tan, Y.; Zhang, X.; Hu, J.; Miao, Z.; Wei, L.; Chai, T. Emissions of Escherichia coli carrying\\nextended-spectrum β-lactamase resistance from pig farms to the surrounding environment. Int. J. Environ.\\nRes. Public Health 2015, 12, 4203–4213. [CrossRef]\\n10. Songe, M.M.; Hang’ombe, B.M.; Knight-Jones, T.J.; Grace, D. Antimicrobial resistant enteropathogenic\\nEscherichia coli and Salmonella spp. in houseﬂies infesting ﬁsh in food markets in Zambia. Int. J. Environ. Res.\\nPublic Health 2016, 14, 21. [CrossRef]\\n11. Massé, D.I.; Saady, N.M.; Gilbert, Y. Potential of biological processes to eliminate antibiotics in livestock\\nmanure: An overview. Animals 2014, 4, 146–163. [CrossRef]\\n12. WHO. Global Action Plan on Antimicrobial Resistance; World Health Organization: Geneva, Switzerland, 2015;\\np. 28. ISBN 978-92-4-150976-3.\\n13. WHO. Antimicrobial Resistance: A Manual for Developing National Action Plans; Published by the WHO and the\\nFood and Agriculture Organization of the United Nations and the World Organization for Animal Health;\\nWorld Health Organization: Geneva, Switzerland, 2016; p. 26. ISBN 978-92-4-154953-0.\\n14. King, L.J.; Anderson, L.R.; Blackmore, C.G.; Blackwell, M.J.; Lautner, E.A.; Marcus, L.C.; Meyer, T.E.;\\nMonath, T.P .; Nave, J.E.; Ohle, J.; et al. Executive summary of the AVMA one health initiative task force\\nreport. J. Am. Vet. Med. Assoc. 2008, 233, 259–261. [CrossRef]\\n15. Rousham, E.K.; Unicomb, L.; Islam, M.A. Human, animal and environmental contributors to antibiotic\\nresistance in low-resource settings: Integrating behavioural, epidemiological and One Health approaches.\\nProc. R. Soc. Lond. B 2018, 285, 20180332. [CrossRef]\\n16. Amador, P .; Fernandes, R.; Prud êncio, C.; Barreto, M.; Duarte, I. Antibiotic resistance in wastewater:\\nOccurrence and fate of Enterobacteriaceae producers of Class A and Class C β-lactamases. J. Environ. Sci.\\nHealth A 2015, 50, 26–39. [CrossRef]\\n17. CLSI. Performance Standards for Antimicrobial Susceptibility Testing, 26th ed.; CLSI Supplement M100S; Clinical\\nand Laboratory Standards Institute: Wayne, PA, USA, 2016.\\n18. Magiorakos, A.P .; Carey, R.B.; Carmeli, Y.; Falagas, M.E.; Giske, C.G.; Harbarth, S.; Hindler, J.F.;\\nKahlmeter, G.; Olsson-Liljequist, B.; Paterson, D.L.; et al. Multidrug-resistant, extensively drug-resistant and\\npandrug-resistant bacteria: An international expert proposal for interim standard deﬁnitions for acquired\\nresistance. Clin. Microbiol. Infect. 2012, 18, 268–281. [CrossRef]\\n19. Amador, P .; Fernandes, R.; Brito, L.; Prudêncio, C. Antibiotic resistance in Enterobacteriaceae isolated from\\nPortuguese deli meats. J. Food Saf. 2011, 31, 1–20. [CrossRef]\\n20. Šeputien˙e, V .; Povilonis, J.; Ružauskas, M.; Pavilonis, A.; Sužied ˙elien˙e, E. Prevalence of trimethoprim\\nresistance genes in Escherichia coli isolates of human and animal origin in Lithuania. J. Med. Microbiol. 2010,\\n59, 315–322. [CrossRef]\\n21. Szmolka, A.; Nagy, B. Multidrug resistant commensalEscherichia coli in animals and its impact for public\\nhealth. Front. Microbiol. 2013, 4, 258. [CrossRef]\\n22. De Jong, A.; Smet, A.; Ludwig, C.; Stephan, B.; De Graef, E.; Vanrobaeys, M.; Haesebrouck, F. Antimicrobial\\nsusceptibility of Salmonella isolates from healthy pigs and chickens (2008–2011). Vet. Microbiol. 2014, 171,\\n298–306. [CrossRef]\\n23. Moawad, A.A.; Hotzel, H.; Awad, O.; Tomaso, H.; Neubauer, H.; Hafez, H.M.; El Adawy, H. Occurrence of\\nSalmonella enterica and Escherichia coli in raw chicken and beef meat in northern Egypt and dissemination of\\ntheir antibiotic resistance markers. Gut Pathog. 2017, 9, 57. [CrossRef]'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 15}, page_content='Antibiotics 2019, 8, 23 16 of 18\\n24. EC. Community procedure on measures to monitor certain substances and residues thereof in live animals\\nand animal products and repealing. Directives 85/358/EEC and 86/469/EEC and Decisions 89/187/EEC\\nand 91/664/EEC. Council Directive 96/23/EC of 29 April 1996. Ofﬁc. J. Eur. Communities 1996, 125, 10.\\n25. Rasmussen, M.M.; Opintan, J.A.; Frimodt-Møller, N.; Styrishave, B. Beta-lactamase producing Escherichia\\ncoli isolates in imported and locally produced chicken meat from Ghana. PLoS ONE 2015, 10, e0139706.\\n[CrossRef]\\n26. Gay, N.; Leclaire, A.; Laval, M.; Miltgen, G.; Jégo, M.; Stéphane, R.; Jaubert, J.; Belmonte, O.; Cardinale, E.\\nRisk factors of extended-spectrum β-lactamase producing Enterobacteriaceae occurrence in farms in Reunion,\\nMadagascar and Mayotte Islands, 2016–2017. Vet. Sci. 2018, 5, 22. [CrossRef]\\n27. Wu, H.; Wang, M.; Liu, Y.; Wang, X.; Wang, Y.; Lu, J.; Xu, H. Characterization of antimicrobial resistance in\\nKlebsiella species isolated from chicken broilers. Int. J. Food Microbiol. 2016, 232, 95–102. [CrossRef]\\n28. Mendonça, N.; Figueiredo, R.; Mendes, C.; Card, R.M.; Anjum, M.F.; Silva, G.J. Microarray evaluation of\\nantimicrobial resistance and virulence of Escherichia coli isolates from Portuguese poultry. Antibiotics 2016,\\n5, 4. [CrossRef]\\n29. Agunos, A.; Léger, D.; Avery, B.P .; Parmley, E.J.; Deckert, A.; Carson, C.A.; Dutil, L. Ciproﬂoxacin-resistant\\nCampylobacter spp. in retail chicken, western Canada. Emerg. Infect. Dis. 2013, 19, 1121–1124. [CrossRef]\\n30. Xie, R.; Huo, S.; Li, Y.; Chen, L.; Zhang, F.; Wu, X. Molecular epidemiological survey on quinolone resistance\\ngenotype and phenotype of Escherichia coli in septicemic broilers in Hebei, China. Poult. Sci. 2014, 93, 335–339.\\n[CrossRef]\\n31. Gouvêa, R.; dos Santos, F.F.; de Aquino, M.H.C. Fluoroquinolones in industrial poultry production, bacterial\\nresistance and food residues: A review. Rev. Bras. Cienc. Avic. 2015, 17, 1–10. [CrossRef]\\n32. Bhushan, C.; Khurana, A.; Sinha, R.; Nagaraju, M. Antibiotic Resistance in Poultry Environment: Spread of\\nResistance from Poultry Farm to Agricultural Field ; Centre for Science and Environment: New Delhi, India,\\n2017; 36p, Available online: www.cseindia.org (accessed on 14 July 2018).\\n33. Franco, A.; Leekitcharoenphon, P .; Feltrin, F.; Alba, P .; Cordaro, G.; Iurescia, M.; Tolli, R.; D’Incau, M.;\\nStaffolani, M.; Di Giannatale, E.; et al. Emergence of a clonal lineage of multidrug-resistant ESBL-producing\\nSalmonella Infantis transmitted from broilers and broiler meat to humans in Italy between 2011 and 2014.\\nPLoS ONE 2015, 10, e0144802. [CrossRef]\\n34. Gao, L.; Hu, J.; Zhang, X.; Wei, L.; Li, S.; Miao, Z.; Chai, T. Application of swine manure on agricultural\\nﬁelds contributes to extended-spectrum β-lactamase-producing Escherichia coli spread in Tai’an, China.Front.\\nMicrobiol. 2015, 6, 313. [CrossRef]\\n35. Li, L.; Ye, L.; Yu, L.; Zhou, C.; Meng, H. Characterization of extended spectrum β-lactamase producing\\nEnterobacteria and methicillin-resistantStaphylococcus aureusisolated from raw pork and cooked pork products\\nin South China. J. Food Sci. 2016, 81, M1773–M1777. [CrossRef]\\n36. Jiang, H.X.; Lü, D.H.; Chen, Z.L.; Wang, X.M.; Chen, J.R.; Liu, Y.H.; Liao, X.P .; Liu, J.H.; Zeng, Z.L. High\\nprevalence and widespread distribution of multi-resistant Escherichia coli isolates in pigs and poultry in\\nChina. Vet. J. 2011, 187, 99–103. [CrossRef]\\n37. Koovapra, S.; Bandyopadhyay, S.; Das, G.; Bhattacharyya, D.; Banerjee, J.; Mahanti, A.; Samanta, I.;\\nNanda, P .K.; Kumar, A.; Mukherjee, R.; et al. Molecular signature of extended spectrum β-lactamase\\nproducing Klebsiella pneumoniae isolated from bovine milk in eastern and north-eastern India. Infect. Genet.\\nEvol. 2016, 44, 395–402. [CrossRef]\\n38. Clemente, L.; Manageiro, V .; Jones-Dias, J.; Correia, C.; Patricia Themudo, P .; Albuquerque, T.; Geraldes, M.;\\nMatos, F.; Almendra, C.; Ferreira, E.; et al. Antimicrobial susceptibility and oxymino-β-lactam resistance\\nmechanisms in Salmonella enterica and Escherichia coli isolates from different animal sources. Res. Microbiol.\\n2015, 166, 574–583. [CrossRef]\\n39. Marchant, M.; Vinué, L.; Torres, C.; Moreno, M.A. Change of integrons over time inEscherichia coli isolates\\nrecovered from healthy pigs and chickens. Vet. Microbiol. 2013, 163, 124–132. [CrossRef]\\n40. Clemente, L.; Manageiro, V .; Ferreira, E.; Jones-Dias, D.; Correia, I.; Themudo, P .; Albuquerque, T.; Caniça, M.\\nOccurrence of extended-spectrum β-lactamases among isolates of Salmonella enterica subsp. enterica from\\nfood-producing animals and food products, in Portugal. Int. J. Food Microbiol. 2013, 167, 221–228. [CrossRef]\\n41. Jones-Dias, D.; Manageiro, V .; Martins, A.P .; Ferreira, E.; Caniça, M. New class 2 integron in 2-4 among\\nIncI1-positive Escherichia coli isolates carrying ESBL and PMA β genes from food animals in Portugal.\\nFoodborne Pathog. Dis. 2015, 13, 36–39. [CrossRef]'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 16}, page_content='Antibiotics 2019, 8, 23 17 of 18\\n42. Karczmarczyk, M.; Walsh, C.; Slowey, R.; Leonard, N.; Fanning, S. Molecular characterization of\\nmultidrug-resistant Escherichia coli isolates from Irish cattle farms. Appl. Environ. Microbiol. 2011, 77,\\n7121–7127. [CrossRef]\\n43. Frye, J.G.; Jackson, C.R. Genetic mechanisms of antimicrobial resistance identiﬁed in Salmonella enterica,\\nEscherichia coli and Enteroccocus spp. isolated from U.S. food animals. Front. Microbiol. 2013, 4, 135. [CrossRef]\\n44. Shaw, W.V . Chloramphenicol acetyltransferase: Enzymology and molecular biology.Crit. Rev. Biochem. 1983,\\n14, 1–46. [CrossRef]\\n45. Yoo, M.H.; Huh, M.D.; Kim, E.H.; Lee, H.H.; Do Jeong, H. Characterization of chloramphenicol\\nacetyltransferase gene by multiplex polymerase chain reaction in multidrug-resistant strains isolated from\\naquatic environments. Aquaculture 2003, 217, 11–21. [CrossRef]\\n46. Niero, G.; Bortolaia, V .; Vanni, M.; Intorre, L.; Guardabassi, L.; Piccirillo, A. High diversity of genes and\\nplasmids encoding resistance to third-generation cephalosporins and quinolones in clinical Escherichia coli\\nfrom commercial poultry ﬂocks in Italy. Vet. Microbiol. 2018, 216, 93–98. [CrossRef]\\n47. Röderova, M.; Halova, D.; Papousek, I.; Dolejska, M.; Masarikova, M.; Hanulik, V .; Pudova, V .; Broz, P .;\\nHtoutou-Sedlakova, M.; Sauer, P .; et al. Characteristics of quinolone resistance inEscherichia coli isolates from\\nhumans, animals and the environment in the Czech Republic. Front. Microbiol. 2017, 7, 2147. [CrossRef]\\n48. Akinbami, O.R.; Oloﬁnsae, S.; Ayeni, F.A. Prevalence of extended spectrum beta lactamase and plasmid\\nmediated quinolone resistant genes in strains of Klebsiella pneumonia , Morganella morganii , Leclercia\\nadecarboxylata and Citrobacter freundii isolated from poultry in South Western Nigeria. PeerJ 2018, 6, e5053.\\n[CrossRef]\\n49. Li, J.; Wang, T.; Shao, B.; Shen, J.; Wang, S.; Wu, Y. Plasmid-mediated quinolone resistance genes and\\nantibiotic residues in wastewater and soil adjacent to swine feedlots: Potential transfer to agricultural lands.\\nEnviron. Health Perspect. 2012, 120, 1144–1149. [CrossRef]\\n50. Nguyen, F.; Starosta, A.L.; Arenz, S.; Sohmen, D.; Dönhöfer, A.; Wilson, D.N. Tetracycline antibiotics and\\nresistance mechanisms. Biol. Chem. 2014, 395, 559–575. [CrossRef]\\n51. Zhang, S.; Gu, J.; Wang, C.; Wang, P .; Jiao, S.; He, Z.; Han, B. Characterization of antibiotics and antibiotic\\nresistance genes on an ecological farm system. J. Chem. 2015, 2015, 526143. [CrossRef]\\n52. Ghosh, S.; LaPara, T.M. The effects of subtherapeutic antibiotic use in farm animals on the proliferation and\\npersistence of antibiotic resistance among soil bacteria. ISME J. 2007, 1, 191–203. [CrossRef]\\n53. Koo, H.J.; Woo, G.J. Distribution and transferability of tetracycline resistance determinants in Escherichia coli\\nisolated from meat and meat products. Int. J. Food Microbiol. 2011, 145, 407–413. [CrossRef]\\n54. McNeece, G.; Naughton, V .; Woodward, M.J.; Dooley, J.S.G.; Naughton, P .J. Array based detection of\\nantibiotic resistance genes in Gram negative bacteria isolated from retail poultry meat in the UK and Ireland.\\nInt. J. Food Microbiol. 2014, 179, 24–32. [CrossRef]\\n55. Chee-Sanford, J.; Maxwell, S.; Tsau, K.; Merrick, K.; Aminov, R. Antibiotic resistance in\\nswine-manure-impacted environments. In Antimicrobial Resistance in the Environment ; Keen, P .L.,\\nMark, H.M.M., Eds.; John Wiley & Sons, Inc.: Hoboken, NJ, USA, 2012; pp. 203–223. ISBN 9780470905425.\\n56. Hammerum, A.M.; Sandvang, D.; Andersen, S.R.; Seyfarth, A.M.; Porsbo, L.J.; Frimodt-Møller, N.; Heuer, O.E.\\nDetection of sul1, sul2 and sul3 in sulphonamide resistant Escherichia coli isolates obtained from healthy\\nhumans, pork and pigs in Denmark. Int. J. Food Microbiol. 2006, 106, 235–237. [CrossRef]\\n57. Wu, S.; Dalsgaard, A.; Hammerum, A.M.; Porsbo, L.J.; Jensen, L.B. Prevalence and characterization of\\nplasmids carrying sulfonamide resistance genes among Escherichia coli from pigs, pig carcasses and human.\\nActa Vet. Scand. 2010, 52, 47. [CrossRef]\\n58. Vuthy, Y.; Lay, K.S.; Seiha, H.; Kerleguer, A.; Aidara-Kane, A. Antibiotic susceptibility and molecular\\ncharacterization of resistance genes among Escherichia coli and among Salmonella subsp. in chicken food\\nchains. Asian Pac. J. Trop. Biomed. 2017, 7, 670–674. [CrossRef]\\n59. Tacão, M.; Moura, A.; Correia, A.; Henriques, I. Co-resistance to different classes of antibiotics among\\nESBL-producers from aquatic systems. Water Res. 2014, 48, 100–107. [CrossRef]\\n60. EFSA—European Food Safety Authority and European Centre for Disease Prevention and Control (EFSA\\nand ECDC). The European Union Summary Report on antimicrobial resistance in zoonotic and indicator\\nbacteria from humans, animals and food in 2010. EFSA J. 2012, 10, 2598. [CrossRef]\\n61. Dillon, B.; Thomas, L.; Mohmand, G.; Zelynski, A.; Iredell, J. Multiplex PCR for screening of integrons in\\nbacterial lysates. J. Microbiol. Methods 2005, 62, 221–232. [CrossRef]'), Document(metadata={'source': 'texts\\\\antibiotics-08-00023.pdf', 'page': 17}, page_content='Antibiotics 2019, 8, 23 18 of 18\\n62. Goldstein, C.; Lee, M.D.; Sanchez, S.; Hudson, C.; Phillips, B.; Register, B.; Grady, M.; Liebert, C.;\\nSummers, A.O.; White, D.G.; et al. Incidence of class 1 and 2 integrases in clinical and commensal bacteria\\nfrom livestock, companion animals and exotics. Antimicrob. Agents Chemother. 2001, 45, 723–726. [CrossRef]\\n63. Akya, A.; Azam Elahi, A.; Chegenelorestani, R.; Rezaee, M. Dissemination of multidrug-resistant, class I and\\nII integrons and molecular typing of CTX-M-producing Klebsiella pneumoniae. Int. J. Appl. Basic Med. Res.\\n2018, 8, 100–105. [CrossRef]\\n64. Koczura, R.; Mokracka, J.; Jabło ´ nska, L.; Gozdecka, E.; Kubek, M.; Kaznowski, A. Antimicrobial resistance of\\nintegron-harboring Escherichia coli isolates from clinical samples, wastewater treatment plant and river water.\\nSci. Total Environ. 2012, 414, 680–685. [CrossRef]\\n65. Ravi, A.; Avershina, E.; Ludvigsen, J.; L’Abée-Lund, T.M.; Rudi, K. Integrons in the intestinal microbiota as\\nreservoirs for transmission of antibiotic resistance genes. Pathogens 2014, 3, 238–248. [CrossRef]\\n© 2019 by the authors. Licensee MDPI, Basel, Switzerland. This article is an open access\\narticle distributed under the terms and conditions of the Creative Commons Attribution\\n(CC BY) license (http://creativecommons.org/licenses/by/4.0/).')]\n",
      "\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Content of IMPART, plate validation, 2022.pdf:\n",
      "[Document(metadata={'source': 'texts\\\\IMPART, plate validation, 2022.pdf', 'page': 0}, page_content='Journal of Microbiological Methods 193 (2022) 106418\\nAvailable online 15 January 2022\\n0167-7012/© 2022 The Authors. Published by Elsevier B.V. This is an open access article under the CC BY license (http://creativecommons.org/licenses/by/4.0/).\\nA European multicenter evaluation study to investigate the performance on \\ncommercially available selective agar plates for the detection of \\ncarbapenemase producing Enterobacteriaceae \\nCindy Dierikx\\na\\n, Stefan B ¨orjesson\\nb , c\\n, Agn `es Perrin-Guyomard\\nd\\n, Marisa Haenni\\ne\\n, \\nMadelaine Norstr ¨om\\nf\\n, Hege H. Divon\\nf\\n, Hanna Karin Ilag\\nf\\n, Sophie A. Granier\\nd\\n, \\nAnnette Hammerum\\ng\\n, Jette Sejer Kjeldgaard\\nh\\n, Natalie Pauly\\ni\\n, Luke Randall\\nj\\n, Muna F. Anjum\\nj\\n, \\nAleksandra Smialowska\\nk\\n, Alessia Franco\\nl\\n, Kees Veldman\\nm\\n, Jannice Schau Slettemeås\\nf , *\\n, on \\nbehalf of the One Health EJP IMPART project\\n1 \\na\\nNational Institute for Public Health and the Environment, Antonie van Leeuwenhoeklaan 9, 3721, MA, Bilthoven, the Netherlands \\nb\\nNational Veterinary Institute (SVA), SE-751 89 Uppsala, Sweden \\nc\\nPublic Health Agency of Sweden, SE-171 82 Solna, Sweden \\nd\\nFrench Agency for Food, Environmental and Occupational Health & Safety, Foug `eres Laboratory, 10B rue Claude Bourgelat, Javen ´e CS 40608 35306 Foug `eres Cedex, \\nFrance \\ne\\nFrench Agency for Food, Environmental and Occupational Health & Safety – Lyon University, Lyon Laboratory, 31, avenue Tony Garnier 69394 Lyon Cedex 07, France \\nf\\nNorwegian Veterinary Institute, P.O. Box 64, 1431 Ås, Norway \\ng\\nStatens Serums Institut, Artillerivej 5, DK-2300 Copenhagen S, Denmark \\nh\\nTechnical University of Denmark, DTU F ø devareinstituttet, Kemitorvet, Bygning 202, DK-2800 Kgs Lyngby, Denmark \\ni\\nGerman Federal Institute for Risk Assessment, Max-Dohrn-Str. 8-10, 10589 Berlin, Germany \\nj\\nAnimal and Plant Health Agency, Woodham Ln, Addlestone KT15 3NB, United Kingdom \\nk\\nNational Veterinary Research Institute, 57 Partyzant ´ow Avenue, 24-100 PU Ł AWY, Poland \\nl\\nIstituto Zooprofilattico Sperimentale del Lazio e della Toscana M. Aleandri, via Appia Nuova, 1411 - 00178 Roma, Italy \\nm\\nWageningen Bioveterinary Research, P.O. Box 65, 8200 AB Lelystad, the Netherlands   \\nARTICLE INFO  \\nKeywords: \\nSelective agar \\nCarbapenem resistance \\nCarbapenemases \\nEnterobacteriaceae \\nSalmonella \\nAnimal \\nABSTRACT  \\nThe European Food Safety Authority (EFSA) advised to prioritize monitoring carbapenemase producing \\nEnterobacteriaceae (CPE) in food producing animals. Therefore, this study evaluated the performance of different \\ncommercially available selective agars for the detection of CPE using spiked pig caecal and turkey meat samples \\nand the proposed EFSA cultivation protocol. Eleven laboratories from nine countries received eight samples (four \\ncaecal and four meat samples). For each matrix, three samples contained approximately 100 CFU/g CPE, and one \\nsample lacked CPE. After overnight enrichment in buffered peptone water, broths were spread upon Brilliance ™ \\nCRE Agar (1), CHROMID ® CARBA (2), CHROMagar ™ mSuperCARBA ™ (3), Chromatic ™ CRE (4), CHROMID ® \\nOXA-48 (5) and Chromatic ™ OXA-48 (6). From plates with suspected growth, one to three colonies were selected \\nfor species identification, confirmation of carbapenem resistance and detection of carbapenemase encoding \\ngenes, by methods available at participating laboratories. Of the eleven participating laboratories, seven reported \\nspecies identification, susceptibility tests and genotyping on isolates from all selective agar plates. Agars 2, 4 and \\n5 performed best, with 100% sensitivity. For agar 3, a sensitivity of 96% was recorded, while agar 1 and 6 \\nperformed with 75% and 43% sensitivity, respectively. More background flora was noticed for turkey meat \\nsamples than pig caecal samples. Based on this limited set of samples, most commercially available agars per -\\nformed adequately. The results indicate, however, that OXA-48-like and non-OXA-48-like producers perform \\nvery differently, and one should consider which CPE strains are of interest to culture when choosing agar type.  \\n* Corresponding author. \\nE-mail address: jannice.schau-slettemeas@vetinst.no (J.S. Slettemeås).   \\n1 \\nThe additional partners, contributing to the OH EJP IMPART project and this publication are given in the “ Acknowledgements ” at the end of this article. \\nContents lists available at ScienceDirect \\nJournal of Microbiological Methods \\njournal homep age: www.elsevi er.com/locate /jmicmeth \\nhttps://doi.org/10.1016/j.mimet.2022.106418 \\nReceived 26 November 2021; Received in revised form 12 January 2022; Accepted 12 January 2022   '), Document(metadata={'source': 'texts\\\\IMPART, plate validation, 2022.pdf', 'page': 1}, page_content='Journal of Microbiological Methods 193 (2022) 106418\\n2\\n1. Introduction \\nCarbapenemase producing Enterobacteriaceae (CPE) are emerging in \\nthe health care sector all over the world, including Europe, and are \\nalready endemic in some areas such as in Mediterranean areas. In the \\nEuropean Union (EU), although regulations can differ between member \\ncountries, carbapenem use is prohibited in livestock and restricted to \\nexceptional cascade use in companion animals ( European Commission, \\n2015 ). Nevertheless, occurrence of CPE has been reported in pig and \\nbroiler farms in Germany and meat products in Belgium ( Borowiak \\net al., 2017 ; Fischer et al., 2013 ; Garcia-Graells et al., 2020 ; Guerra et al., \\n2014 ), and was recently reported in Italian fattening pigs ( Diaconu et al., \\n2020 ). Furthermore, CPE has also been reported in livestock from non- \\nEuropean countries, where they sometimes appear to be more common \\n( Hamza et al., 2016 ; Li et al., 2019 ; Mollenkopf et al., 2018 ; Tang et al., \\n2019 ; Wang et al., 2017 ). \\nThe occurrence of CPE in livestock indicates that livestock could \\nconstitute a potential reservoir for CPE, and in China transfer of CPE has \\nbeen described between humans and livestock ( Li et al., 2019 ). In \\naddition, the EFSA recognized in 2013 that presence of CPE in food \\nproducing animals could constitute a reservoir for humans, and they \\ntherefore advised to prioritize monitoring CPE in food producing ani -\\nmals ( European Food Safety Authority, 2013 ). However, the monitoring \\nof carbapenemase producing (CP) Escherichia coli using selective isola -\\ntion remained optional until 2021 when it became mandatory for the \\nnational monitoring programs in Decision 2020/1729/EU ( European \\nCommission, 2020 ). \\nIn an early warning system, a selective and sensitive culture method \\nto detect CPE is needed and such a method should be capable of \\ndetecting low concentrations ( < 100 CFU/g) of CPE in faecal samples of \\nanimals and in food samples. The method should also at least be able to \\ndetect CPE that produce OXA-48 like enzymes (OXA-48 CPE) as well as \\nCPE that produce other carbapenemases like NDM, IMP, VIM, and KPC \\n(non-OXA-48 CPE). As both these CPE-groups are the most common in \\nhuman clinical settings and are the ones that primarily have been \\ndetected in food or animals ( Bonardi and Pitino, 2019 ). \\nSeveral studies have already been conducted on evaluating the per -\\nformance of different CPE selective agar media ( Cohen Stuart et al., \\n2013 ; Garcia-Fernandez et al., 2017 ; Girlich et al., 2013a ; Girlich et al., \\n2013b ; Girlich et al., 2019 ; Gottig et al., 2020 ). However, to our \\nknowledge, no cross-country multicenter studies have been performed, \\nnor a study including all commercially available selective agar plates on \\nthe market. To this end, a consortium called IMPART (IMproving \\nPhenotypic Antimicrobial Resistance Testing) within the One Health \\nEuropean Joint Programme ( https://onehealthejp.eu/ ) of both veteri -\\nnary and public health laboratories was established in 2018 focusing on \\nthe development of sensitive screening assays for emerging resistances \\nsuch as CPE. The aim was to evaluate the performance, sensitivity, and \\nspecificity, of different commercial selective agars available for the \\ndetection of CPE from spiked samples of animal origin, i.e. meat and \\ncaecal samples. This was carried out by performing a multicenter study, \\nusing the enrichment protocol used in the EU Decision 2013/652/EU as \\ndescribed by the EURL-AR. \\n2. Material and methods \\n2.1. Participants \\nIn total, twelve institutes (eleven participants and one organizer) \\nfrom nine countries participated in the multicenter evaluation study \\nconducted in 2019, see Table 1 . \\nBefore starting the multicenter evaluation study, all participants \\nwere asked to fill in a questionnaire on the methods used in their \\ninstitute to detect CPE in samples (see Supplementary file 1). \\nPrior to the execution of the multicenter evaluation study, a pre-test \\nwas conducted in 2018 that involved four of the laboratories (Anses \\nFoug `eres Laboratory, NVI, RIVM and WBVR). The purpose of this pre- \\ntest was to evaluate growth conditions and available selective agar \\nplates in order to narrow down test conditions for the eleven partici -\\npants in the main multicenter evaluation study (see Supplementary file 2 \\nand for the report see https://zenodo.org/record/3676437#.YW \\n1r-xpBxPY ). In summary the pre-test showed that: i) incubating selec -\\ntive agar plates at 44 \\n◦\\nC compared to 35 – 37 \\n◦\\nC resulted in false negative \\nsamples, ii) ready-to-order plates gave the same results as in-house made \\nselective agar plates from the same manufacturer, and iii) there was no \\ninterference by the matrix i.e. caecal and meat, to detect the spiked \\nisolate. \\n2.2. Matrices and strains \\nMeat from specific pathogen free (SPF) turkey and caecal samples \\nfrom pigs collected at a French slaughterhouse were used. Matrices were \\nchecked to be negative for CPE using the protocol from the EURL-AR \\n(accessed on June 2019) ( Hasman et al., 2018a ; Hasman et al., \\n2018b ). Verified CPE free samples were frozen at \\x00 20 \\n◦\\nC until spiked. In \\ntotal eight samples, six spiked samples and two blank samples were \\nprepared ( Table 2 ). \\nStrains used for spiking are listed in Table 2 , and inclusion criteria for \\nthe strains were: i) strains should be well defined (reference strains), ii) \\nstrains should carry a CPE gene, and iii) included strains should express \\ndifferent minimum inhibitory concentration (MIC) values to the carba -\\npenems meropenem, ertapenem and imipenem. Antimicrobial suscep -\\ntibility towards beta-lactam antibiotics including different carbapenems \\nwas determined by the organizer using the broth microdilution method \\nwith the Sensititre ™ TREK EUVSEC2 panel (Thermo Fisher Scientific, \\nWaltham, Massachusetts, USA). Strains were grown on blood agar and \\nincubated overnight at 37 \\n◦\\nC. From the overnight cultures, a 0.5 \\nMcFarland ( ≈ 10\\n8 \\nCFU/mL) bacterial suspension for each strain was \\nprepared in saline (0.9% NaCl). Suspensions were then diluted to obtain \\nthe target final concentration (100 CFU/g sample) in the pooled minced \\nmeat or caecal content samples. Spiked samples were homogenized in a \\nmixer for meat and with a stirrer for caecal content for approximately 5 \\nmin and aliquoted on a Friday. All aliquots were stored at 4 \\n◦\\nC until \\nshipping the following Monday at 4 \\n◦\\nC in accordance with UN3373 \\nregulation. For each sample, homogeneity was tested on ten aliquots \\nrandomly selected from the positive test samples and on three aliquots \\nrandomly selected from the negative test samples. The stability of each \\nsample was tested on three aliquots randomly chosen among positive \\ntest samples. Stability was assessed by comparing the data from the day \\nof shipment (data obtained in the homogeneity study) and from one day \\nafter reception and analysis of the samples by the participating labora -\\ntories. All laboratories were instructed to start analyses immediately \\nTable 1 \\nOverview of participating institutes.  \\nNo. Participant lab Country \\n1 Anses, Foug `eres Laboratory France \\n(organizer) \\n2 Norwegian Veterinary Institute (NVI) Norway \\n3 Anses, Lyon Laboratory France \\n4 Statens Serums Institut (SSI) Denmark \\n5 National Veterinary Institute (SVA) Sweden \\n6 Technical University of Denmark (DTU) Denmark \\n7 German Federal Institute for Risk Assessment (BfR) Germany \\n8 National Institute for Public Health and the Environment \\n(RIVM) \\nThe \\nNetherlands \\n9 Animal and Plant Health Agency (APHA) United \\nKingdom \\n10 Pa ´nstwowy Instytut Weterynaryjny (PIWET) Poland \\n11 Istituto Zooprofilattico Sperimentale del Lazio e della \\nToscana M. Aleandri (IZSLT) \\nItaly \\n12 Wageningen Bioveterinary Research (WBVR) The \\nNetherlands  \\nC. Dierikx et al.                                                                                                                                                                                                                                 '), Document(metadata={'source': 'texts\\\\IMPART, plate validation, 2022.pdf', 'page': 2}, page_content='Journal of Microbiological Methods 193 (2022) 106418\\n3\\nupon arrival of the samples. \\n2.3. Selective CPE agar plates \\nTo be included in the multicenter evaluation study, the commercial \\nselective agar plates had to fulfill the following criteria: i) plates should \\nbe available in all countries, ii) they should be available as ready-to-use \\nplates, iii) no bi/dual plates or similar should be included. One exception \\nto the criteria was done for the CHROMagar ™ mSuperCARBA ™ which \\nis only available as an in-house medium. To avoid any preparation \\nbiases of the CHROMagar ™ mSuperCARBA ™ plates these were pre -\\npared for the participating laboratories by the MAST group (MAST DI -\\nAGNOSTICS, Amiens, France). Table 3 lists commercial agar plates \\nincluded and whether the plate detects OXA-48 CPE, non-OXA-48 CPE \\nor both. \\n2.4. Protocol \\nFig. 1 outlines the protocol used in this multicenter evaluation study. \\nIn brief, all samples were diluted 1:10 in buffered peptone water (BPW) \\nand incubated overnight at 37 \\n◦\\nC. Per lab, three control strains, one \\nstrain without a carbapenemase gene, one with an OXA-48-like carba -\\npenemase gene and one with a non-OXA-48-like carbapenemase gene \\n( Table 4 ) were included to validate growth on the selective agar plates \\naccording to the validation protocol described by the EURL-AR ( Hasman \\net al., 2015 ). The next day, 10 μ L of each enrichment was plated on each \\nof the six agar plates. After overnight incubation at 35 \\n◦\\nC or 37 \\n◦\\nC \\n(depending on the recommendation for the selective agar plate) a \\nmaximum of three suspected CPE colonies were picked from each plate \\naccording to the colony morphology described by the manufacturer. \\nSuspected colonies were sub-streaked on the respective selective agar \\nplate and blood agar. On single colonies species identification was \\nperformed according to the methodology available at the participating \\nlaboratory (see Supplementary file 3), with Matrix-assisted laser \\ndesorption ionization – time-of-flight mass spectrometry (MALDI-TOF \\nMS) being the most common method used (nine out of eleven labora -\\ntories). On isolates with the correct species, i.e. Klebsiella pneumoniae, \\nSalmonella spp. or E. coli, susceptibility testing was performed by either \\nbroth microdilution or disk diffusion, with the minimum criteria of \\nincluding meropenem. On all presumptive pure cultured CPE isolates, \\ngenotypic identification was performed by either PCR or real-time PCR, \\nusing established protocols at the participating laboratories, which at a \\nminimum included the following genotypes: bla\\nIMP\\n, bla\\nNDM\\n, bla\\nKPC\\n, \\nbla\\nVIM \\nand bla\\nOXA-48 \\n(for details on PCRs see Supplementary file 3). For \\nreporting the results of each step, all participants were provided with a \\npre-prepared report scheme. \\n3. Results \\n3.1. Questionnaire \\nIn total eleven institutes returned the questionnaire (see Supple -\\nmentary file 1). Eight laboratories normally used pre-enrichment of the \\nsamples overnight in BPW at 37 \\n◦\\nC and one laboratory used Tryptic Soy \\nBroth (TSB), while one laboratory normally did not use pre-enrichment \\nand one laboratory never cultured samples for CPE. Two of the eight \\nlaboratories performing the BPW pre-enrichment, also reported the use \\nof a second selective enrichment step. One of these two laboratories \\nfollowed the BPW pre-enrichment using a lysogenic broth with 1 mg/L \\ncefotaxime incubated at 44 \\n◦\\nC prior to plating on selective agars, while \\nthe other laboratory followed the BPW pre-enrichment using BPW with \\n0.25 mg/L ertapenem and 50 mg/L vancomycin incubated at 37 \\n◦\\nC prior \\nto their PCR detection procedure. Regarding selective agar plates used at \\nthe different laboratories, all laboratories reported that they used agar \\nplates from bioM ´erieux, either both the CHROMID ® CARBA and the \\nCHROMID ® OXA-48 as separate plates, or the combined bi/dual plate \\nCHROMID ® CARBA SMART ® . In addition, one laboratory also used the \\nCHROMagar ™ KPC. \\n3.2. Validation of CPE selective agar plates using reference strains \\nTo validate the selective agar plates, all laboratories used the E. coli \\nATCC 25922 as a carbapenem-susceptible control strain, and none of the \\neleven participants reported growth of this strain on any of the six se -\\nlective agar plates ( Table 4 ). All data from the multicenter study is \\npresented in Supplementary file 4. \\nTo validate growth of OXA-48 strains on the selective agar plates, ten \\nof eleven laboratories used the strain E. coli 16874 bla\\nOXA-48 \\ndistributed \\nby the EURL-AR, which is used for validation of the selective agars in the \\nTable 2 \\nOverview of samples, matrices and bacteria/gene combinations.  \\nSample Matrix Matrix \\norigin \\nBacterial species Carbapenemase \\ngene \\nMEM\\n# \\nMIC (mg/ \\nL) \\nERT\\n# \\nMIC (mg/ \\nL) \\nIMI\\n# \\nMIC (mg/ \\nL) \\nReference \\nC-1 Caecal Pig BLANK – – – – – \\nC-2 Caecal Pig E. coli NCTC 13476 bla\\nIMP \\n2 > 2 2 NCTC \\nC-3 Caecal Pig K. pneumoniae NCTC 13442 bla\\nOXA-48 \\n4 > 2 16 NCTC \\nC-4 Caecal Pig E. coli R1180 bla\\nVIM-1 \\n0.25 0.06 4 ( Irrgang et al., 2017 ) \\nM-1 Meat SPF * Turkey BLANK – – – – – \\nM-2 Meat SPF * Turkey K. pneumoniae ATCC BAA- \\n1705 \\nbla\\nKPC-2 \\n> 16 > 2 16 ATCC \\nM-3 Meat SPF * Turkey E. coli 16874 bla\\nOXA-48 \\n1 2 2 ( Hasman et al., \\n2015 ) \\nM-4 Meat SPF * Turkey S. Kentucky 2014LSAL00827 bla\\nNDM-1 \\n2 1 4 ( Hawkey et al., \\n2019 )  \\n*\\nSPF - specific pathogen free. \\n#\\nAll MICs were performed at Anses, Foug `eres Laboratory, France. (MEM) indicates meropenem, (ERT) indicates ertapenem, and (IMI) indicates imipenem. \\nTable 3 \\nSelective agar plates included in the multicenter evaluation study.  \\nName Producer Ready-to-use or In- \\nhouse media \\nCPE detected \\nBrilliance ™ CRE Agar Oxoid Ready-to-use Non-OXA-48 \\nand OXA-48 \\nCHROMID ® CARBA \\nAgar \\nbioM ´erieux Ready-to-use Only non-OXA- \\n48 \\nCHROMID ® OXA-48 \\nAgar \\nbioM ´erieux Ready-to-use Only OXA-48 \\nChromatic ™ CRE Liofilchem Ready-to-use Non-OXA-48 \\nand OXA-48 \\nChromatic ™ OXA-48 Liofilchem Ready-to-use Only OXA-48 \\nCHROMagar ™ \\nmSuperCARBA ™ \\nCHROMagar In-house * Non-OXA-48 \\nand OXA-48  \\n*\\nManufactured by MAST DIAGNOSTICS, Amiens, France, for the multicenter \\nevaluation study. \\nC. Dierikx et al.                                                                                                                                                                                                                                 '), Document(metadata={'source': 'texts\\\\IMPART, plate validation, 2022.pdf', 'page': 3}, page_content='Journal of Microbiological Methods 193 (2022) 106418\\n4\\nEU monitoring program ( European Commission, 2020 ; Hasman et al., \\n2015 ). One laboratory used a K. pneumoniae 32516 bla\\nOXA-48 \\nstrain. \\nE. coli 16874 strain was detected on the plates expected to detect bla\\nOXA- \\n48 \\nisolates (i.e. CHROMID ® OXA-48, Chromatic ™ OXA-48, Chromatic ™ \\nCRE, and CHROMagar ™ mSuperCARBA ™ ), but not on Brilliance ™ CRE \\nAgar in any of the ten laboratories using this strain. However, this strain \\nshould not be able to grow on the included non-OXA-48 selective agar \\nCHROMID ® CARBA Agar growth was recorded in four laboratories, \\nalthough three of them reported growth of only a few colonies. \\nK. pneumoniae 32516 has a higher meropenem MIC value compared to \\nE. coli 16874, 4 mg/L and 1 mg/L, respectively, and grew on all plates \\ntested, including CHROMID ® CARBA Agar where it was not expected to \\ngrow ( Table 4 ). \\nTo validate growth of non-OXA-48 CPE strains, nine of eleven lab -\\noratories used the strain E. coli TZ 3638 bla\\nGES-5 \\ndistributed by the EURL- \\nAR used for validation of the selective agars in the EU monitoring pro -\\ngram. Further, one laboratory used E. coli 40336 bla\\nKPC-3 \\nstrain ( Mani \\net al., 2017 ) and one laboratory E. coli TZ 116 bla\\nVIM-1 \\nstrain which was \\nalso distributed by the EURL-AR as a second choice for validation of \\nselective agar plates. On all plates the detection of the non-OXA-48 CPE \\nwere seen, except for Brilliance ™ CRE Agar which could not detect the \\nE. coli TZ 3638 bla\\nGES-5\\n, ( Table 4 ). One laboratory reported growth of \\nE. coli TZ 3638 bla\\nGES-5 \\non CHROMID ® OXA-48 Agar, and two \\nlaboratories reported growth of this strain on Chromatic ™ OXA-48. \\nE. coli TZ 116 was also reported to grow on Chromatic ™ OXA-48 \\n( Table 4 ). None of these strains were expected to grow on the two \\nOXA-48 selective agars. \\n3.3. Performance of selective agar plates using spiked and blank samples \\nIn this part of the study, four laboratories were excluded and the \\nresults from only seven out of the eleven laboratories were included. The \\nreason for this exclusion was that only the results of laboratories that \\nperformed species identification, susceptibility testing and genotyping \\non isolates from all selective agar plates were included. \\nNone of the seven laboratories was able to culture the bla\\nVIM-1 \\npos -\\nitive E. coli strain spiked in caecum sample C-4. From meat sample M-4 \\nspiked with Salmonella Kentucky carrying bla\\nNDM-1 \\ngrowth of E. coli and/ \\nor Enterobacter spp. carrying a bla\\nNDM \\ngene (gene variant not deter -\\nmined) was detected by five of the participating laboratories, and one \\nlaboratory reported detecting an E. coli carrying both bla\\nNDM \\nand bla\\nVIM\\n. \\nThe homogeneity and stability studies at Agency for Food, Environ -\\nmental and Occupational Health & Safety (Anses) Foug `eres laboratory \\nbefore and after shipping the samples verified the findings of \\nmeropenem-resistant E. coli and Enterobacter spp. together with S. \\nKentucky. Further, no E. coli bla\\nVIM-1 \\nwas detected from sample C-4 after \\nFig. 1. Schematic outline of the multicenter evaluation study. \\nCreated with BioRender.com \\nC. Dierikx et al.                                                                                                                                                                                                                                 '), Document(metadata={'source': 'texts\\\\IMPART, plate validation, 2022.pdf', 'page': 4}, page_content='Journal of Microbiological Methods 193 (2022) 106418\\n5\\nthe homogeneity and stability studies. \\nWhen including samples C-4 and M-4 in the analyses, sensitivity \\nranged from 43% to 100% ( Table 5 ). The only agar-plate that performed \\nwith 100% sensitivity was the CHROMID ® OXA-48 when examining the \\nbla\\nOXA-48 \\npositive strains. However, if C-4 and M-4 were excluded from \\nthe analyses, CHROMID ® OXA-48, CHROMID ® CARBA Agar and \\nChromatic ™ CRE performed with 100% sensitivity, and CHROMagar ™ \\nmSuperCARBA ™ performed with 96% sensitivity. Chromatic ™ OXA-48 \\nand Brilliance ™ CRE Agar performed with a sensitivity of 43% and 75%, \\nrespectively. \\nIn at least one sample, all seven laboratories reported growth of other \\nbacteria than the expected one, i.e. background flora. Overall, unspecific \\ngrowth was reported more frequently from meat samples than from \\ncaecal samples ( Fig. 2 A). Pseudomonas spp. were repeatedly identified, \\nbut other Gram-negative bacteria like Acinetobacter spp. , Steno -\\ntrophomonas spp. , Enterobacter spp., and Ochrobactrum spp. , were also \\nreported. In sample M-4, spiked with bla\\nNDM-1 \\npositive S. Kentucky, \\nother bla\\nNDM-1 \\npositive species ( E. coli and/or Enterobacter spp.) were \\nalso detected. The bla\\nNDM \\ngene was found in one Enterobacter and in nine \\nE. coli isolated from sample M-4. \\nAll selective agar plates showed growth of background flora. How -\\never, difference between plates exists, e.g. CHROMagar ™ mSu -\\nperCARBA ™ displayed the most background flora, while Brilliance ™ \\nCRE Agar displayed the least background flora of all media ( Fig. 2 B). \\nIn sample C-3, four laboratories observed either different colony \\nsizes on the selective agar plates or mucoid and non-mucoid colonies of \\nK. pneumoniae colonies after sub culturing on blood agar plates from all \\nselective agar plates with growth. MICs of these different colonies \\nranged from 1 to 16 mg/L, but OXA-48 was confirmed in all isolates. \\nSupplementary file 4. \\nFig. 2 A and B show a qualitative overview of the background flora \\nsorted by sample (A) and by agar plate (B). \\n4. Discussion \\nCompared to earlier studies performed on evaluating selective agar \\nplates for the detection of CPE, the current study had the large advan -\\ntage that it was a multicenter cross-country study focusing on spiked \\nsamples instead of already purified bacterial strains. Hence, the study \\nmore closely mimicked the experimental conditions in a routine sur -\\nveillance. However, this setup also resulted in that only a relatively \\nsmall number of samples were included, due to the number of partici -\\npants and the combination of matrices, genes, and bacterial strains \\nneeded to be tested. In our study, Brilliance ™ CRE Agar was shown to \\nsupport growth of both bla\\nIMP \\nand bla\\nKPC\\n, but could not detect the bla\\nOXA- \\n48 \\nstrain with low carbapenem MIC (meropenem 1 mg/L). In addition, \\nall nine laboratories using the E. coli TZ 3638 bla\\nGES-5 \\nstrain for valida -\\ntion of the selective agars, provided as a positive control by the EURL- \\nAR, reported that this strain did not grow on the Brilliance ™ CRE \\nAgar. Cohen Stuart et al. (2013) arrived at a similar conclusion that \\nBrilliance ™ CRE Agar possessed a high sensitivity but a low specificity, \\nwith the difference that they tested a high number of isolates: 95 CPE \\nand 160 non-CPE strains, including AmpC beta-lactamase and ESBL- \\nproducing bacteria. The low specificity for this agar was mainly due to \\ngrowth of AmpC beta-lactamase or ESBL-producing bacterial strains and \\nthat it was less sensitive for bla\\nOXA-48 \\npositive strains. The Brilliance ™ \\nCRE Agar was also tested in a study by Girlich et al. (2013b) using 131 \\nCPE strains, where growth was described for both OXA-48 producers as \\nwell as other carbapenemase producers (KPC, VIM and IMP). However, \\nin that study, the SuperCARBA medium, from which the CHROMagar ™ \\nmSuperCARBA ™ had been derived ( Amar et al., 2017 ), performed \\nbetter than the Brilliance ™ CRE Agar in detecting strains tested. This was \\nalso comparable to our study in which the CHROMagar ™ mSu -\\nperCARBA ™ performed better than the Brilliance ™ CRE Agar. The \\nbetter performance of the CHROMagar ™ mSuperCARBA ™ plate was \\nmainly due to the detection of the E. coli 16874 bla\\nOXA-48 \\nstrain by all \\nTable 4 \\nNumber of laboratories showing growth of control strains on different selective agars.      \\nMIC (mg/L) Detection of OXA-48 CPE isolates Detection of OXA-48 and non-OXA-48 CPE isolates Detection of non-OXA-48 \\nCPE isolates  \\nStrain CP- \\ngene \\nNo. of labs \\n(n) * \\nMEM\\n# \\nIMI\\n# \\nERT\\n# \\nCHROMID ® \\nOXA-48 \\nChromatic ™ \\nOXA-48 \\nBrilliance ™ CRE \\nAgar \\nCHROMagar ™ \\nmSuperCARBA ™ \\nChromatic ™ \\nCRE \\nCHROMID ® CARBA \\nNon-CPE E. coli ATCC 25922 – 11 ≤ 0.03 ≤ 0.12 ≤ 0.015 0 0 0 0 0 0 \\nOXA-48 CPE E. coli 16874 bla\\nOXA- \\n48 \\n10 1 2 2 10 10 0 10 10 4 \\nK. pneumoniae \\n32516 \\nbla\\nOXA- \\n48 \\n1 4 > 2 > 32 1 1 1 1 1 1 \\nNon-OXA-48 \\nCPE \\nE. coli TZ 3638 bla\\nGES-5 \\n9 0.5 1 0.5 1 2 0 9 9 9 \\nE. coli 40336 bla\\nKPC-3 \\n1 3 2 1.5 0 0 1 1 1 1 \\nE. coli TZ 116 bla\\nVIM-1 \\n1 2 4 1 0 1 1 1 1 1  \\n*\\nNumber of laboratories that used the strain. (MEM) indicates meropenem, (IMI) indicates imipenem, and (ERT) indicates ertapenem. \\n#\\nMICs were derived from the EURL-AR, except for E. coli 40336 ( Mani et al., 2017 ) and K. pneumoniae 32516 (Anses, Foug `eres Laboratory, France). \\nC. Dierikx et al.                                                                                                                                                                                                                                 '), Document(metadata={'source': 'texts\\\\IMPART, plate validation, 2022.pdf', 'page': 5}, page_content='Journal of Microbiological Methods 193 (2022) 106418\\n6\\nlaboratories when validating the agar plates (Table 4), while none of \\nthese laboratories detected this strain with the Brilliance™ CRE Agar. It \\nshould, however, be mentioned that the Brilliance™ CRE Agar did \\nperform better in a recent study by Gottig et al. (2020) in which it was \\ncompared to Chromatic™ CRE, CHROMID® CARBA, CHROMID® OXA- \\n48 and two in-house made media called McCARB and McCARB-T. In that \\nstudy, the better sensitivity of Brilliance™ CRE Agar (98.6%) compared \\nto Chromatic™ CRE (94.2%) was based primarily on the failure to detect \\ntwo K. pneumoniae strains with bla\\nKPC-2 \\nby Chromatic™ CRE, and that \\nthe CHROMID® CARBA agar failed to detect one of the K. pneumoniae \\nstrains together with a Providencia stuartii strain with bla\\nNDM-1\\n. The au -\\nthors also pointed out that the CHROMID® CARBA agar again missed to \\ndetect eight out of twenty investigated bla\\nOXA-48 \\nstrains resulting in a \\nsensitivity of 85.5%. However, sensitivity raises to 96% when bla\\nOXA-48 \\nstrains are excluded, as the CHROMID® CARBA agar is not designed to \\ndetect bla\\nOXA-48 \\nstrains. In summary, performance of the studied plates \\nhave been described to vary depending on the choice of strains tested, \\nbut all included plates in the current study have previously been \\ndescribed to perform overall well in detecting CPE (Girlich et al., 2013a; \\nGirlich et al., 2013b). Although in our current study the Chromatic™ \\nCRE, CHROMID® CARBA agar and CHROMID® OXA-48 agar performed \\nslightly better than the other included agars (Table 5). However, this \\nbetter performance could be biased by the strains that were included. In \\naddition, the CHROMID® CARBA agar and CHROMID® OXA-48 were \\nalso the agars that most participating laboratories reported to use \\nroutinely. Thus, improved performance may be because these labora -\\ntories have more experience in handling and interpreting suspected \\ngrowth on these plates. Despite the better performance by CHROMID® \\nCARBA agar in our study a recent German study did highlight that this \\nagar have difficulties detecting CPE with low MICs (Pauly et al., 2020). \\nBased on the German experience on detecting CPE in food producing \\nanimals where mostly CPE with low MICs has been detected it was also \\nconcluded that if CHROMID® CARBA agar are used in screening it might \\nunderestimate the occurrence of CPE. \\nUnfortunately, the strains chosen for the current study might have \\ninfluenced some inconsistencies in the outcome of the evaluation of the \\nagar plates. One large issue was that caecal sample C-4, which was re -\\nported as negative by all laboratories. This sample was spiked with an \\nE. coli bla\\nVIM-1 \\nisolate originating from German fattening pigs that \\nalready had proved challenging to detect using the non-selective \\nenrichment in BPW (Irrgang et al., 2019). Another issue that arose \\nduring the evaluation was that in meat sample M-4 spiked with a bla\\nNDM- \\n1 \\ncarrying S. Kentucky, a potential plasmid transfer might have occurred \\nas eight out of the eleven laboratories reported findings of Enterobacter \\nspp. and/or E. coli isolates carrying bla\\nNDM\\n. An alternative, but a more \\nunlikely, explanation could be that a contamination of the sample could \\nhave occurred. The detection of the S. Kentucky strain was also likely \\nfurther hampered due to the difficulty of separating it from the back -\\nground flora as it, like most of the environmental bacteria, is non- \\nchromogenic. There was a higher presence of background flora on the \\nselective agar plates cultured with enriched turkey meat samples \\ncompared to the pig caeca samples, which was not unexpected as bac -\\nterial contamination of poultry meat at slaughter has been well \\ndescribed (Rouger et al., 2017). Because of the difficulty to detect bla\\nVIM- \\n1 \\ncarrying E. coli and bla\\nNDM-1 \\ncarrying S. Kentucky in samples C-4 and \\nM-4, respectively, these samples were excluded from the comparison in \\nthe outcome of the evaluation. \\nAnother difficulty arose in sample C-3 with the K. pneumoniae bla\\nOXA- \\n48 \\nstrain. This sample resulted in growth of mucoid and non-mucoid \\ncolonies of Klebsiella with different meropenem MICs. This phenome -\\nnon is well described by Chiarelli et al. (2020). As this strain was \\ndetected in all seven laboratories on all except one agar plate (Table 5), \\nit is therefore very unlikely that this has influenced the interpretation of \\nthe growth on these plates. \\nThe inability of the current EURL-AR protocol used in the EFSA AMR \\nsurveillance program, and the current study, in combination with all \\ninvestigated agar plates to detect the bla\\nVIM-1 \\npositive strain in sample C- \\n4 is worrisome, since this gene has first been identified in an E. coli strain \\nisolated from food producing animals in Germany in 2013 (Fischer et al., \\n2013), and afterwards in a Salmonella Infantis and in an E. coli isolated \\nfrom a minced pork meat sample in Germany and Belgium, respectively \\n(Borowiak et al., 2017; Garcia-Graells et al., 2020). All three isolates did \\nhave a meropenem MIC equal to or below 1 mg/L. In addition, two E. coli \\nisolates with bla\\nVIM-1 \\nwere also detected from fattening pigs in the \\n2017–2018 German monitoring program (European Food Safety Au -\\nthority and European Centre for Disease Prevention and Control, 2019.). \\nTable 5 \\nNumber of laboratories detecting growth of the expected strain.    \\nOXA-48 detection OXA-48 and non-OXA-48 CPE detection non-OXA-48 \\ndetection \\nMaterial Strain CHROMID® \\nOXA-48 \\nChromatic™ \\nOXA-48 \\nBrilliance™ CRE \\nAgar \\nCHROMagar™ \\nmSuperCARBA™ \\nChromatic™ \\nCRE \\nCHROMID® \\nCARBA \\nCaecal C-1 \\nunspiked \\n0* 0* 0* 0* 0* 0* \\nC-2 EC IMP 0* 0* 7 7 7 7 \\nC-3 KP OXA- \\n48 \\n7 1 7 7 7 0* \\nC-4 EC VIM- \\n1 \\n0* 0* 0 0 0 0 \\nMeat M-1 \\nunspiked \\n0* 0* 0* 0* 0* 0* \\nM-2 KP KPC- \\n2 \\n0* 0* 7 7 7 7 \\nM-3 EC \\nOXA-48 \\n7 5 0 6 7 0* \\nM-4 SK \\nNDM-1 \\n0* 0* 6 2 4 1 \\nTotal score n 14 6 27 29 32 15 \\nSpecificity % 100 100 100 100 100 100 \\nSensitivity % 100 43 64 69 76 54 \\nSensitivity (excluding \\nsample C-4 and M-4) \\n% 100 43 75 96 100 100 \\n(EC) indicates E. coli, (KP) indicates K. pneumoniae, (SK) indicates Salmonella Kentucky. (*) indicates that the strain should not grow on this plate. Sample C-4 contained \\na strain with very low meropenem MIC, which made it difficult to detect. In sample M-4, plasmid conjugation or contamination had occurred, as different bacterial \\nspecies were detected positive for bla\\nNDM\\n-gene, which resulted that the correct gene group, bla\\nNDM\\n, but an incorrect species determination were reported from eight of \\neleven participants. \\nC. Dierikx et al.                                                                                                                                                                                                                                 '), Document(metadata={'source': 'texts\\\\IMPART, plate validation, 2022.pdf', 'page': 6}, page_content='Journal of Microbiological Methods 193 (2022) 106418\\n7\\nOne of the isolates had a meropenem MIC of 2 mg/L, ertapenem MIC of \\n0.5 mg/L and imipenem MIC of 4 mg/L. In Pauly et al. (2021) , it was \\ndescribed that this isolate was not detected using the EURL-AR protocol \\nfor CP E. coli and the CHROMID ® CARBA agar plate, but instead it was \\ndetected using the EURL-AR protocol for the detection of ESBL/AmpC \\nproducing E. coli on a MacConkey agar plate with 1 mg/L cefotaxime. \\nThe reason it was identified in the ESBL/AmpC screening was likely due \\nto the fact that the bla\\nVIM-1 \\nwas co-located on a plasmid with a bla\\nSHV-12 \\ngene. Thus, it might be a risk that the current method is underestimating \\nthe incidence of CPE in European livestock, and meats thereof, based on \\nthe current methodology used in the harmonized monitoring protocol \\nsuggested by Decision 2013/652/EU ( European Commission, 2013 ). \\nThe study by Irrgang et al. (2019) was also unable to detect the E. coli TZ \\n116 bla\\nVIM-1 \\nstrain using the EURL-AR protocol, as we were in the \\ncurrent study, and came to a similar conclusion: “ For monitoring programs \\naccording to the guideline 2013/652/EU it might be suitable to use two serial \\nenrichment steps: first an unselective pre-enrichment step in buffered peptone \\nwater, using this culture for plating on selective OXA-48 media. In a second \\nstep, cultivating an inoculum of the unselective pre-enrichment culture in LB \\nsupplemented with 1 mg/L Cefotaxime at 44\\n◦\\nC (for isolation of E. coli) or \\n37\\n◦\\nC (for Enterobacteriaceae in general). ” . The combined results of the \\ncurrent study and the Irrgang et al. (2019) study, highlights the need for \\nfurther evaluation of the suitability of using BPW without antibiotics \\nregarding the sensitivity and specificity of identifying CPEs. The urgency \\nof performing this evaluation is high as the selective screening for CP \\nE. coli in the harmonized monitoring in European livestock became \\nmandatory as of 2021 according to the Decision 2020/1729/EU ( Euro -\\npean Commission, 2020 ). However, one should be aware that the \\n0\\n5\\n10\\n15\\n20\\n25\\nC-1 C-2 C-3 C-4 M-1 M-2 M-3 M-4\\nTotal reported \\nSamples\\n2A Background ﬂora sorted by sample\\nOther\\nPseudomonas\\nAcinetobacter\\nStenotrophomonas\\nE. coli\\nEnterobacter\\nOchrobactrum\\n0\\n5\\n10\\n15\\n20\\n25\\nCHROMID®\\nOXA-48\\nChroma/g415c™ \\nOXA-48 \\nBrilliance™ CRE \\nAgar\\nCHROMagar™ \\nmSuperCARBA™ \\nChroma/g415c™ \\nCRE \\nCHROMID®\\nCARBA\\nTotal reported\\nCPE selec/g415ve agar plates\\n2B Background ﬂora sorted by agar plate\\nOther\\nPseudomonas\\nAcinetobacter\\nStenotrophomonas\\nE. coli\\nEnterobacter\\nOchrobactrum\\nFig. 2. Overview of background flora (organism/genus) total reported by sample (2A) and total reported by agar plate (2B). Presence and absence of accompanying \\nflora were scored as “ 1 ” or “ 0 ” and sorted according to samples (2A) or selective media (2B). The scoring is qualitative only and presence of a genus in a sample was \\nscored as “ 1 ” for the specific genus and sample. The y-axis presents total reported of the identified species per participant. “ Other ” was registered when: a) mixed \\nculture with no name, b) pure culture with no name, or c) correct identification but mixed culture. From the eighteen E. coli isolates reported in sample M-4, ten were \\nconfirmed to carry a bla\\nNDM \\ngene, and from the three Enterobacter isolates reported in sample M-4, one was confirmed for a bla\\nNDM \\ngene. E. coli with bla\\nNDM \\nwas \\nreported from Brilliance ™ CRE Agar ( n = 1), CHROMID ® CARBA agar ( n = 3), Chromatic ™ CRE (n = 3) and the CHROMagar ™ mSuperCARBA ™ (n = 3). The \\nEnterobacter carrying bla\\nNDM \\nwas reported from the CHROMagar ™ mSuperCARBA ™ plate. \\nC. Dierikx et al.                                                                                                                                                                                                                                 '), Document(metadata={'source': 'texts\\\\IMPART, plate validation, 2022.pdf', 'page': 7}, page_content='Journal of Microbiological Methods 193 (2022) 106418\\n8\\nreasoning behind choosing the current EURL-AR protocol included other \\ncriteria than sensitivity and specificity. For example, the protocol \\nneeded to be cheap and easy to implement in all member states, so it was \\npreferable that the protocol could be combined with already utilized \\nprotocols, such as the isolation of Salmonella, and not requiring addi -\\ntional sample collection for the national reference laboratories. \\n5. Conclusions \\nThe evaluation showed that the investigated agars were not suitable \\nfor detection of non-chromogenic CPE strains nor for the detection of \\nCPE with low meropenem MICs. All plates showed 100% specificity, and \\nwhen including all eight samples the Chromatic ™ CRE and the CHRO -\\nMagar ™ mSuperCARBA ™ showed the greatest sensitivity, 76% and \\n69% respectively, while the CHROMID ® CARBA performed less well \\nwith 54% sensitivity. However, when excluding the meat sample with \\nthe Salmonella Kentucky carrying bla\\nNDM-1 \\nand the caecal sample with \\nthe E. coli carrying bla\\nVIM-1\\n, the CHROMID ® CARBA, Chromatic ™ CRE \\nand CHROMagar ™ mSuperCARBA ™ showed 96 – 100% sensitivity. For \\nthe OXA-48 agars, the CHROMID ® OXA-48 agar performed with 100% \\nsensitivity compared to the Chromatic ™ OXA-48 at 47%. The results \\nemphasize that when choosing a selective agar one should always \\nconsider which carbapenemase and which bacterial species to detect. \\nThe results of the current study also show the need for further \\ndevelopment of the pre-enrichment step in the current EURL-AR pro -\\ntocol for detection of CPE ’ s. However, the consequence of such a change \\nwould likely be that the protocol would only be applicable for the spe -\\ncific monitoring of CP E. coli , instead of as now when used for several \\nscreenings. \\nSupplementary data to this article can be found online at https://doi. \\norg/10.1016/j.mimet.2022.106418 . \\nDeclaration of Competing Interest \\nNone. \\nAcknowledgement \\nWe thank the European Commission for funding the European Joint \\nProgramme One Health. This programme falls under the European \\nUnion ’ s Horizon 2020 research and innovation programme under Grant \\nAgreement No. 773830. The project received co-funding from all \\ninvolved institutions. Furthermore we would like to thank all IMPART \\nparticipants listed below for their technical assistance: M ¨arit Pringle, \\nAnnica Land ´en and Boel Harbom from the National Veterinary Institute, \\nSweden; Tifaine H ´echard, Pamela Hou ´ee, Charlotte Valentin and Pierre \\nCh ˆatre from the French Agency for Food, Environmental and Occupa -\\ntional Health & Safety, Foug `eres Laboratory and Lyon Laboratory, \\nFrance; Alexandra Irrgang and Jens Andr ´e Hammerl from the German \\nFederal Institute for Risk Assessment, Germany; Paul Hengeveld from \\nthe National Institute for Public Health and the Environment, the \\nNetherlands; Yvon Geurts from Wageningen Bioveterinary Research, the \\nNetherlands; Fabiola Feltrin, Angela Ianzano, Antonio Battisti and Vir -\\nginia Carfora from the Istituto Zooprofilattico Sperimentale del Lazio e \\ndella Toscana M. Aleandri , Italy. \\nReferences \\nAmar, M., Shalom, O., Adler, A., 2017. Comparative evaluation of a new commercial \\nmedia, the CHROMAgar mSuperCARBA, for the detection of carbapenemase- \\nproducing Enterobacteriaceae. Diagn. Microbiol. Infect. Dis. 88, 20 – 22 . \\nBonardi, S., Pitino, R., 2019. Carbapenemase-producing bacteria in food producing \\nanimals, wildlife and environment: a challenge for human health. Ital J. Food Saf. 8, \\n7956 . \\nBorowiak, M., Szabo, I., Baumann, B., Junker, E., Hammerl, J.A., Kaesbohrer, A., \\nMalorny, B., Fischer, J., 2017. VIM-1-producing Salmonella Infantis isolated from \\nswine and minced pork meat in Germany. J. Antimicrob. Chemother. 72, \\n2131 – 2133 . \\nChiarelli, A., Cabanel, N., Rosinski-Chupin, I., Dieudonn ´e Zongo, P., Naas, T., Bonnin, R. \\nA., Glaser, P., 2020. Diversity of mucoid to non-mucoid switch among \\ncarbapenemase-producing Klebsiella pneumoniae . BMC Microbiol. 20, 325 . \\nCohen Stuart, J., Voets, G., Rottier, W., Voskuil, S., Scharringa, J., Van Dijk, K., Fluit, A. \\nC., Leverstein-Van Hall, M., 2013. Evaluation of the Oxoid Brilliance CRE Agar for \\nthe detection of carbapenemase-producing Enterobacteriaceae. Eur. J. Clin. \\nMicrobiol. Infect. Dis. 32, 1445 – 1449 . \\nDiaconu, E.L., Carfora, V., Alba, P., Di Matteo, P., Stravino, F., Buccella, C., Dell ’ Aira, E., \\nOnorati, R., Sorbara, L., Battisti, A., Franco, A., 2020. Novel IncFII plasmid \\nharbouring bla\\nNDM-4 \\nin a carbapenem-resistant Escherichia coli of pig origin, Italy. \\nJ. Antimicrob. Chemother. 75, 3475 – 3479 . \\nEuropean Commission, 2013. Commission Implementing Decision on the Monitoring and \\nReporting of Antimicrobial Resistance in Zoonotic and Commensal Bacteria. \\nDecision 2013/652/EU 2013 (updated 14 November 2013. 40-7). Available from. \\nhttps://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri = CELEX:32013D0652 \\n& from = ES . Official Journal of the European Union, 40-47.  \\nEuropean Commission, 2015. Commission Notice Guidelines for the Prudent Use of \\nAntimicrobials in Veterinary Medicine (2015/C 299/04). Available from. \\nhttps://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri = CELEX:52017XC0701 \\n(01) . Official Journal of the European Union, C299/7.  \\nEuropean Commission, 2020. Commission Implementing Decision (EU) 2020/1729 of 17 \\nNovember 2020 on the Monitoring and Reporting of Antimicrobial Resistance in \\nZoonotic and Commensal Bacteria and Repealing Implementing Decision 2013/652/ \\nEU (Notified under Document C(2020) 7894). Available from. https://eur-lex. \\neuropa.eu/eli/dec_impl/2020/1729/oj . Official Journal of the European Union, \\nL387/8.  \\nEuropean Food Safety Authority, 2013. Scientific opinion on carbapenem resistance in \\nfood animal ecosystems. EFSA J. 11 (12), 3501 . \\nEuropean Food Safety Authority and European Centre for Disease Prevention and \\nControl, 2019. The European Union summary report on antimicrobial resistance in \\nzoonotic and indicator bacteria from humans, animals and food in 2017. EFSA J. 17, \\ne05598 . \\nFischer, J., Rodriguez, I., Schmoger, S., Friese, A., Roesler, U., Helmuth, R., Guerra, B., \\n2013. Salmonella enterica subsp. enterica producing VIM-1 carbapenemase isolated \\nfrom livestock farms. J. Antimicrob. Chemother. 68, 478 – 480 . \\nGarcia-Fernandez, S., Hernandez-Garcia, M., Valverde, A., Ruiz-Garbajosa, P., \\nMorosini, M.I., Canton, R., 2017. CHROMagar mSuperCARBA performance in \\ncarbapenem-resistant Enterobacteriaceae isolates characterized at molecular level \\nand routine surveillance rectal swab specimens. Diagn. Microbiol. Infect. Dis. 87, \\n207 – 209 . \\nGarcia-Graells, C., Berbers, B., Verhaegen, B., Vanneste, K., Marchal, K., Roosens, N.H.C., \\nBotteldoorn, N., De Keersmaecker, S.C.J., 2020. First detection of a plasmid located \\ncarbapenem resistant bla\\nVIM-1 \\ngene in E. coli isolated from meat products at retail in \\nBelgium in 2015. Int. J. Food Microbiol. 324, 108624 . \\nGirlich, D., Anglade, C., Zambardi, G., Nordmann, P., 2013a. Comparative evaluation of \\na novel chromogenic medium (chromID OXA-48) for detection of OXA-48 producing \\nEnterobacteriaceae. Diagn. Microbiol. Infect. Dis. 77, 296 – 300 . \\nGirlich, D., Poirel, L., Nordmann, P., 2013b. Comparison of the SUPERCARBA, \\nCHROMagar KPC, and Brilliance CRE screening media for detection of \\nEnterobacteriaceae with reduced susceptibility to carbapenems. Diagn. Microbiol. \\nInfect. Dis. 75, 214 – 217 . \\nGirlich, D., Grosperrin, V., Naas, T., Dortet, L., 2019. CHROMagar ESBL/mSuperCARBA \\nbi-plate medium for detection of ESBL- and carbapenemase-producing \\nEnterobacteriaceae from spiked stools. Diagn. Microbiol. Infect. Dis. 95, 107 – 112 . \\nGottig, S., Walker, S.V., Saleh, A., Koroska, F., Sommer, J., Stelzer, Y., Steinmann, J., \\nHamprecht, A., 2020. Comparison of nine different selective agars for the detection \\nof carbapenemase-producing Enterobacterales (CPE). Eur. J. Clin. Microbiol. Infect. \\nDis. 39, 923 – 927 . \\nGuerra, B., Fischer, J., Helmuth, R., 2014. An emerging public health problem: acquired \\ncarbapenemase-producing microorganisms are present in food producing animals, \\ntheir environment, companion animals and wild birds. Vet. Microbiol. 171, \\n290 – 297 . \\nHamza, E., Dorgham, S.M., Hamza, D.A., 2016. Carbapenemase-producing Klebsiella \\npneumoniae in broiler poultry farming in Egypt. J. Glob. Antimicrob. Resist. 7, 8 – 10 . \\nHasman, H., Agers ø , Y., Hendriksen, R., Cavaco, L.M., Guerra-Roman, B., 2015. \\nValidation of Selective and Indicative Agar Plates for Monitoring of Carbapenemase- \\nProducing E. coli . Available from. https://www.eurl-ar.eu/protocols.aspx . \\nHasman, H., Agers ø , Y., Hendriksen, R., Cavaco, L.M., Guerra-Roman, B., 2018a. AmpC- \\nand Carbapenemase-Producing E. coli from Caecal Samples. Version 6. Available \\nfrom. https://www.eurl-ar.eu/protocols.aspx . \\nHasman, H., Agers ø , Y., Hendriksen, R., Cavaco, L.M., Guerra-Roman, B., 2018b. \\nIsolation of ESBL-, AmpC- and Carbapenemase-Producing E. coli from Fresh Meat. \\nVersion 6. Available from. https://www.eurl-ar.eu/protocols.aspx . \\nHawkey, J., Le Hello, S., Doublet, B., Granier, S.A., Hendriksen, R.S., Fricke, W.F., \\nCeyssens, P.J., Gomart, C., Billman-Jacobe, H., Holt, K.E., Weill, F.X., 2019. Global \\nphylogenomics of multidrug-resistant Salmonella enterica serotype Kentucky ST198. \\nMicrob. Genom. 5 . \\nIrrgang, A., Fischer, J., Grobbel, M., Schmoger, S., Skladnikiewicz-Ziemer, T., \\nThomas, K., Hensel, A., Tenhagen, B.A., Kasbohrer, A., 2017. Recurrent detection of \\nVIM-1-producing Escherichia coli clone in German pig production. J. Antimicrob. \\nChemother. 72, 944 – 946 . \\nIrrgang, A., Tenhagen, B.A., Pauly, N., Schmoger, S., Kaesbohrer, A., Hammerl, J.A., \\n2019. Characterization of VIM-1-producing E. coli isolated from a German fattening \\npig farm by an improved isolation procedure. Front. Microbiol. 10, 2256 . \\nLi, J., Bi, Z., Ma, S., Chen, B., Cai, C., He, J., Schwarz, S., Sun, C., Zhou, Y., Yin, J., \\nHulth, A., Wang, Y., Shen, Z., Wang, S., Wu, C., Nilsson, L.E., Walsh, T.R., \\nC. Dierikx et al.                                                                                                                                                                                                                                 '), Document(metadata={'source': 'texts\\\\IMPART, plate validation, 2022.pdf', 'page': 8}, page_content='Journal of Microbiological Methods 193 (2022) 106418\\n9\\nBorjesson, S., Shen, J., Sun, Q., Wang, Y., 2019. Inter-host transmission of \\ncarbapenemase-producing Escherichia coli among humans and backyard animals. \\nEnviron. Health Perspect. 127, 107009. \\nMani, Y., Mansour, W., Mammeri, H., Denamur, E., Saras, E., Boujaafar, N., \\nBouallegue, O., Madec, J.Y., Haenni, M., 2017. KPC-3-producing ST167 Escherichia \\ncoli from mussels bought at a retail market in Tunisia. J. Antimicrob. Chemother. 72, \\n2403–2404. \\nMollenkopf, D.F., Mathys, D.A., Feicht, S.M., Stull, J.W., Bowman, A.S., Daniels, J.B., \\nWittum, T.E., 2018. Maintenance of carbapenemase-producing Enterobacteriaceae \\nin a farrow-to-finish swine production system. Foodborne Pathog. Dis. 15, 372–376. \\nPauly, N., Hammerl, J.A., Grobbel, M., Tenhagen, B.A., Kasbohrer, A., Bisenius, S., \\nFuchs, J., Horlacher, S., Lingstadt, H., Mauermann, U., Mitro, S., Muller, M., \\nRohrmann, S., Schiffmann, A.P., Stuhrenberg, B., Zimmermann, P., Schwarz, S., \\nMeemken, D., Irrgang, A., 2020. ChromID((R)) CARBA agar fails to detect \\ncarbapenem-resistant Enterobacteriaceae with slightly reduced susceptibility to \\ncarbapenems. Front. Microbiol. 11, 1678. \\nPauly, N., Hammerl, J.A., Schwarz, S., Grobbel, M., Meemken, D., Malorny, B., \\nTenhagen, B.A., Kasbohrer, A., Irrgang, A., 2021. Co-occurrence of the bla\\nVIM-1 \\nand \\nbla\\nSHV-12 \\ngenes on an IncHI2 plasmid of an Escherichia coli isolate recovered from \\nGerman livestock. J. Antimicrob. Chemother. 76, 531–533. \\nRouger, A., Tresse, O., Zagorec, M., 2017. Bacterial contaminants of poultry meat: \\nsources, species, and dynamics. Microorganisms. 5, 50. \\nTang, B., Chang, J., Cao, L., Luo, Q., Xu, H., Lyu, W., Qian, M., Ji, X., Zhang, Q., Xia, X., \\nYang, H., 2019. Characterization of an NDM-5 carbapenemase-producing Escherichia \\ncoli ST156 isolate from a poultry farm in Zhejiang, China. BMC Microbiol. 19, 82. \\nWang, Y., Zhang, R., Li, J., Wu, Z., Yin, W., Schwarz, S., Tyrrell, J.M., Zheng, Y., \\nWang, S., Shen, Z., Liu, Z., Liu, J., Lei, L., Li, M., Zhang, Q., Wu, C., Zhang, Q., \\nWu, Y., Walsh, T.R., Shen, J., 2017. Comprehensive resistome analysis reveals the \\nprevalence of NDM and MCR-1 in Chinese poultry production. Nat. Microbiol. 2, \\n16260. \\nC. Dierikx et al.                                                                                                                                                                                                                                 ')]\n",
      "\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Content of Irrgang, GES-5, 2020.pdf:\n",
      "[Document(metadata={'source': 'texts\\\\Irrgang, GES-5, 2020.pdf', 'page': 0}, page_content='microorganisms\\nCommunication\\nFirst Detection of GES-5-Producing Escherichia coli\\nfrom Livestock—An Increasing Diversity of\\nCarbapenemases Recognized from German\\nPig Production\\nAlexandra Irrgang 1, *\\n , Simon H. Tausch 1\\n , Natalie Pauly 1, Mirjam Grobbel 1\\n ,\\nAnnemarie Kaesbohrer 1,2 and Jens A. Hammerl 1\\n1 Department Biological Safety, German Federal Institute for Risk Assessment (Bundesinstitut für\\nRisikobewertung, BfR), D-10589 Berlin, Germany; simon.tausch@bfr.bund.de (S.H.T.);\\nnatalie.pauly@bfr.bund.de (N.P .); mirjam.grobbel@bfr.bund.de (M.G.);\\nannemarie.kaesbohrer@bfr.bund.de (A.K.); jens-andre.hammerl@bfr.bund.de (J.A.H.)\\n2 Institute for Veterinary Public Health, University of Veterinary Medicine, 1210 Vienna, Austria\\n* Correspondence: alexandra.irrgang@bfr.bund.de; Tel.: +49-30-18412-24310\\nReceived: 29 September 2020; Accepted: 14 October 2020; Published: 16 October 2020\\n/gid00030/gid00035/gid00032/gid00030/gid00038/gid00001/gid00033/gid00042/gid00045/gid00001\\n/gid00048/gid00043/gid00031/gid00028/gid00047/gid00032/gid00046\\nAbstract: Resistance to carbapenems due to carbapenemase-producing Enterobacteriaceae (CPE)\\nis an increasing threat to human health worldwide. In recent years, CPE could be found only\\nsporadically from livestock, but concern rose that livestock might become a reservoir for CPE. In 2019,\\nthe ﬁrst GES carbapenemase-producingEscherichia colifrom livestock was detected within the German\\nnational monitoring on antimicrobial resistance. The isolate was obtained from pig feces and was\\nphenotypically resistant to meropenem and ertapenem. The isolate harbored three successive blaGES\\ngenes encoding for GES-1, GES-5 and GES-5B in an incomplete class-I integron on a 12 kb plasmid\\n(pEC19-AB02908; Acc. No. MT955355). The strain further encoded for virulence-associated genes\\ntypical for uropathogenic E. coli, which might hint at an increased pathogenic potential. The isolate\\nproduced the third carbapenemase detected from German livestock. The ﬁnding underlines the\\nimportance CPE monitoring and detailed characterization of new isolates.\\nKeywords: carbapenemase; CPE; GES-5; GES-5B; whole-genome sequencing; livestock\\n1. Introduction\\nCarbapenemase-producing Enterobacteriaceae (CPE) are a global threat to human health.\\nCarbapenemases are often associated with nosocomial infections (esp. KPC) but are also disseminated\\nin the community [1]. While OXA-48, VIM, NDM and KPC carbapenemases are detected frequently\\nfrom human infections in Germany, GES carbapenemase-producing bacteria were only isolated\\nsporadically [2]. In 2018, blaGES carbapenemase genes were recovered from 5 Enterobacteria and 14\\nPseudomonas aeruginosaisolates from human infections, representing ~1% of detected CPE from human\\ninfections in Germany [2]. GES enzymes are serine proteases of the Ambler class A [ 3]. GES-1 was\\nﬁrst described in Klebsiella pneumoniae. It exhibits strong activity against most β-lactams and results\\nin an extended-spectrum β-lactamase (ESBL) phenotype [4]. Point mutations can lead to increased\\nhydrolyzing activity and carbapenem substrate utilization (i.e., GES-5) [5].\\nWhile the carbapenemases VIM-1 and OXA-48 had been reported sporadically from German pig\\nproduction in recent years [6,7], other carbapenemases have not been observed in German livestock\\nso far. Here, we report the phenotypic and genotypic properties of the ﬁrst GES-5-producing E. coli\\nfrom livestock.\\nMicroorganisms 2020, 8, 1593; doi:10.3390/microorganisms8101593 www.mdpi.com /journal/microorganisms'), Document(metadata={'source': 'texts\\\\Irrgang, GES-5, 2020.pdf', 'page': 1}, page_content='Microorganisms 2020, 8, 1593 2 of 5\\n2. Materials and Methods\\nThe E. coli isolate 19-AB02908 was obtained within the German national monitoring on\\nantimicrobial resistance from a fecal sample of a fattening pig following the protocol of the European\\nReference Laboratory for Antimicrobial Resistance (EURL-AR) for selective CPE isolation (Available\\nonline: https://www.eurl-ar.eu/protocols.aspx). Phenotypic resistance was conﬁrmed by determining\\nminimal inhibitory concentration (MIC) values using the broth microdilution method according to\\nCLSI guidelines (CLSI M07-A9). MIC values were interpreted according to EUCAST epidemiological\\ncut-oﬀ values deﬁned in 2013. Initial determination of the genotype was carried out by routine\\nreal-time PCR adapted from van der Zee et al. [8] and Swayne et al. [9], followed by diagnostic PCR for\\nblaGES (F: 5′-ATGCGCTTCATTCACGCAC-3′/R: 5′-TCCGTGCTCAGGATGAGTTG-3′) and subsequent\\nSanger sequencing of the PCR products. For genetic in-depth dissection, the isolate was subjected\\nto long-read (Nanopore) and short-read (Illumina, PRJNA660949) sequencing. Hybrid assembly\\nof the plasmid was carried out using Unicycler v.0.44 (provided by PATRIC 3.6.6). The complete\\nplasmid sequence of pEC19-AB02908 is available under the GenBank Acc.No MT955355. Genome\\ncharacterization was conducted with our in-house developed pipeline Bakcharak v1.0.0 (Available\\nonline: https://gitlab.com/bfr_bioinformatics/bakcharak) which implements ABRicate v1.0.1 (Available\\nonline: https://github.com/tseemann/abricate) for screening of antimicrobial resistance genes (using\\nthe NCBI amrﬁnder database [ 10]), plasmid markers (using the PlasmidFinder database [ 11]) and\\nvirulence factors (using the VFDB [12]). Plasmid contig identiﬁcation was performed using platon\\n(Available online: https: //github.com/oschwengers/platon). Furthermore the MLST sequence type\\nwas inferred using mlst (Available online: https://github.com/tseemann/mlst) based on the pubmlst\\ndatabase [13].\\n3. Results\\nThe isolate 19-AB02908 was obtained from a fecal sample of a fattening pig, which was taken in\\nthe course of the German resistance monitoring on CPE. Phenotypic antimicrobial resistances were\\ndetected for all tested penicillins and cephalosporins, as well as for tetracycline (MIC ≥64 mg/L),\\ntrimethoprim (MIC > 32 mg/L), and sulfamethoxazole (MIC > 1024 mg/L) (Table 1). The isolate further\\nshowed resistance to meropenem (MIC ≥0.25 mg/L) and ertapenem (MIC ≥0.25 mg/L), but only\\nslightly reduced susceptibility to imipenem (MIC ≥0.5 mg/L).\\nTable 1. Main characteristics of the GES-producing E. coli isolate 19-AB02908. Abbreviations\\nof antimicrobials: AMP—ampicillin; ETP—ertapenem; FEP—cefepime, FOT—cefotaxime;\\nFOX—cefoxitin; MERO—meropenem; SMX—sulfamethoxazole; TAZ—ceftazidime;\\nTET—tetracycline; TMP—trimethoprim.\\nPhylogeny Phenotypic\\nResistance Resistance Genes Plasmids Virulence\\nAssociated Genes\\n-ST1084\\n-phylogenetic\\ngroup B1\\nAMP , ETP , FEP ,\\nFOT, FOX, MERO,\\nSMX, TAZ, TET,\\nTMP\\naac(6’)-Ib3, ant(3′’)-Ia,\\naph(3”)-Ib, aph(6)-Id, blaGES-1,\\nblaGES-5, blaGES-5B, blaTEM-1B,\\ndfrA1, mph(B), sul-1, sul-2,\\ntet(A)\\n12 kb\\n(pEC19-AB02908)\\n227 kb\\nafaA, afaD, cma,\\ncvaC, hlyF, hra, iroN,\\niss, lpfA, ompT, sitA,\\nterC, traT\\nThe results of PCR sequencing indicated the presence of a GES-5 carbapenemase. Based on\\ncompeting nucleotides at deﬁned positions in the sequence chromatograms, the presence of up to two\\nmore GES variants seemed to be likely. The hybrid assembly of whole-genome sequencing revealed\\nthat there are three copies of theblaGES genes separated by 157 bp intergenic regions on a 12 kb plasmid\\n(GenBank Acc.No MT955355). The three-fold repetition of the blaGES genes was veriﬁed by PCR using\\nreverse complementary primers. As initially predicted, the prevailing blaGES genes diﬀered in one to\\ntwo nucleotide positions from each other. One is the ESBL-encoding geneblaGES-1, followed by blaGES-5\\nand a second blaGES-5 with a silent mutation on nucleotide position 54 (G54A) (Figure 1). This gene'), Document(metadata={'source': 'texts\\\\Irrgang, GES-5, 2020.pdf', 'page': 2}, page_content='Microorganisms 2020, 8, 1593 3 of 5\\nvariant was formerly named blaGES-3 (AY494717) [14], but was later renamed to blaGES-5 on the basis of\\nits amino acid sequence relationship [15]. The current blaGES-3 gene (AB113580.1) was described by\\nWachino et al. [16]. We suggest naming the GES-5 variant with a silent mutation GES-5B according\\nto the classiﬁcations of blaTEM variants. Similar structures of class-I integrons with duplicated blaGES\\nvariants have been found in Pseudomonas aeruginosa(GQ337064) and Enterobacter cloacae(KX230795)\\n(Figure 1) [17,18].\\nMicroorganisms 2020, 8, x FOR PEER REVIEW 3 of 5 \\n \\nwas described by Wachino et al. [16]. We suggest naming the GES-5 variant with a silent mutation \\nGES-5B according to the classifications of blaTEM variants. Similar structures of class-I integrons with \\nduplicated blaGES variants have been found in Pseudomonas aeruginosa (GQ337064) and Enterobacter \\ncloacae (KX230795) (Figure 1) [17,18]. \\n \\nFigure 1. Comparison of the sequence organization of the blaGES region with similar integrons \\navailable at GenBank [17,18]. The reading frames are presented with arrows; arrowheads indicate the \\ndirection of transcription. \\nHere, the unusual formation of three successive blaGES variants was part of an incomplete \\nintegron, as the conserved CS3 ′ region was missing. In 2018, an incomplete class-1 integron with a \\nquadruple of blaGES-5 gene cassettes was reported [19] . The authors suggested that the blaGES gene \\ncassettes tend to duplicate by si te-specific recombination. Apart from antimicrobial resistances, the \\nplasmid backbone is represented by a DNA region of 4.7 kb that can be found in several \\nEnterobacteriaceae, like Klebsiella pneumoniae (e.g., LN824137) or Salmonella Typhimurium (e.g., \\nCP050743). It is likely that a class-I integron with two GES variants has been integrated into this small \\nplasmid. The presence of the third blaGES gene might be a result of gene duplication by a \\nrecombination event in which the CS3′ site of the integron was deleted. Viedma et al. [18] suggested \\na direct repeat sequence (5 ′-ACAAA-3′) that might be involved in the gene duplication. In silico  \\nanalysis revealed that this sequence is present 17 times on the pEC19-AB02908 plasmid, indicating \\nthat this recognition sequence might not be specific enough. \\nBesides the blaGES carrying plasmid, the isolate harbored an additional 227 kb IncF plasmid \\nconferring resistance to a variety of antimicrobials (aminoglycosides, beta-lactams, macrolides, and \\nsulfonamides) as well as heavy metal resistance (Table 1). Therefore, a co-selection for the \\ncarbapenemase resistance can occur even if carbapenems or cephalosporins are not applied to the \\nanimals. \\nTo determine a potential persistence of the plasmid in livestock, the farm was investigated \\ncomprehensively three months after the isolate was detected within the German national monitoring \\nof antimicrobial resistance. Livestock and farm surrounding samples were taken and processed as \\ndescribed [20]. Additionally, real-time PCR from the enrichment cultures targeting the blaGES gene \\nwas performed. However, no CPE could be detected in the samples indicating that a further spread \\nof GES-producing enterobacteria did not occur. \\nBesides others, the isolate harbored a variet y of virulence-associated genes typical for \\nuropathogenic E. coli (i.e., afaA, afaD, hylF, sfaX, iroN, iss) suggesting an increased pathogenic potential \\n[21]. This is untypical for phylogenetic group B1 as  this phylogenetic group is often associated with \\nhigh resistance while exhibiting only low virulence [22]. \\nAfter VIM-1 and OXA-48, GES-5 is the third carbapenemase, which could be detected in German \\npig production. The entry source of these isolat es remains speculative bu t the set of different \\ncarbapenemases found in livestock  reflects the increasing diversity reported for CPE from human \\nsources in Germany. The repeated detection of carbapenemases with presumed human origin in pig \\nFigure 1. Comparison of the sequence organization of theblaGES region with similar integrons available\\nat GenBank [17,18]. The reading frames are presented with arrows; arrowheads indicate the direction\\nof transcription.\\nHere, the unusual formation of three successiveblaGES variants was part of an incomplete integron,\\nas the conserved CS3′region was missing. In 2018, an incomplete class-1 integron with a quadruple of\\nblaGES-5 gene cassettes was reported [19]. The authors suggested that the blaGES gene cassettes tend to\\nduplicate by site-speciﬁc recombination. Apart from antimicrobial resistances, the plasmid backbone\\nis represented by a DNA region of 4.7 kb that can be found in several Enterobacteriaceae, like Klebsiella\\npneumoniae (e.g., LN824137) or Salmonella Typhimurium (e.g., CP050743). It is likely that a class-I\\nintegron with two GES variants has been integrated into this small plasmid. The presence of the third\\nblaGES gene might be a result of gene duplication by a recombination event in which the CS3′site of the\\nintegron was deleted. Viedma et al. [18] suggested a direct repeat sequence (5′-ACAAA-3′) that might\\nbe involved in the gene duplication. In silico analysis revealed that this sequence is present 17 times on\\nthe pEC19-AB02908 plasmid, indicating that this recognition sequence might not be speciﬁc enough.\\nBesides the blaGES carrying plasmid, the isolate harbored an additional 227 kb IncF plasmid\\nconferring resistance to a variety of antimicrobials (aminoglycosides, beta-lactams, macrolides,\\nand sulfonamides) as well as heavy metal resistance (Table 1). Therefore, a co-selection for the\\ncarbapenemase resistance can occur even if carbapenems or cephalosporins are not applied to\\nthe animals.\\nTo determine a potential persistence of the plasmid in livestock, the farm was investigated\\ncomprehensively three months after the isolate was detected within the German national monitoring\\nof antimicrobial resistance. Livestock and farm surrounding samples were taken and processed as\\ndescribed [20]. Additionally, real-time PCR from the enrichment cultures targeting the blaGES gene\\nwas performed. However, no CPE could be detected in the samples indicating that a further spread of\\nGES-producing enterobacteria did not occur.\\nBesides others, the isolate harbored a variety of virulence-associated genes typical for\\nuropathogenic E. coli (i.e., afaA, afaD, hylF, sfaX, iroN, iss) suggesting an increased pathogenic\\npotential [ 21]. This is untypical for phylogenetic group B1 as this phylogenetic group is often\\nassociated with high resistance while exhibiting only low virulence [22].'), Document(metadata={'source': 'texts\\\\Irrgang, GES-5, 2020.pdf', 'page': 3}, page_content='Microorganisms 2020, 8, 1593 4 of 5\\nAfter VIM-1 and OXA-48, GES-5 is the third carbapenemase, which could be detected in German\\npig production. The entry source of these isolates remains speculative but the set of di ﬀerent\\ncarbapenemases found in livestock reﬂects the increasing diversity reported for CPE from human\\nsources in Germany. The repeated detection of carbapenemases with presumed human origin in pig\\nproduction shows that hygiene concepts should be followed. These are necessary not only to facilitate\\nanimal welfare, but also to prevent the transfer of zoonotic bacteria from humans to animals and vice\\nversa. The newly reported isolate combines the zoonotic potential of a probable pathogenic E. coli\\nwith the limited therapeutic options due to its broad resistance features. This ﬁnding conﬁrms the\\nneed for continuous monitoring in order to detect any spread of new resistance mechanisms in animal\\npopulations immediately.\\nAuthor Contributions: Conceptualization, A.I.; Methodology, A.I.; Validation, A.I., S.H.T.; Investigation, A.I.,\\nN.P .; Resources, S.H.T., J.A.H.; Data Curation, M.G.; Writing–Original Draft Preparation, A.I.; Writing–Review &\\nEditing, S.H.T., N.P .; M.G., A.K. Visualization, J.A.H.; Supervision, A.K.; Project Administration, A.I.; Funding\\nAcquisition, A.K.; J.A.H. All authors have read and agreed to the published version of the manuscript.\\nFunding: The work of Natalie Pauly was supported by funding from the European Union’s Horizon 2020 Research\\nand Innovation programme under grant agreement No 773830: One Health European Joint Programme. The work\\nwas also supported by grants of the German Federal Institute for Risk Assessment (43-001 and 43-002).\\nAcknowledgments: We thank Romy Fuhrmeister and Anja Seeger for her their assistance with the farm\\ninvestigations. Thanks to the farmer for taking part in this study and Dr. Torsten Jäger from the LK Harburg for\\nhis cooperation and support. We further thank Silvia Schmoger and Maria Borowiak for sequencing the isolate.\\nConﬂicts of Interest: The authors declare no conﬂict of interest.\\nReferences\\n1. Potter, R.F.; D ′Souza, A.W.; Dantas, G. The rapid spread of carbapenem-resistant Enterobacteriaceae.\\nDrug Resist. Updat.2016, 29, 30–46. [CrossRef] [PubMed]\\n2. Pfennigwerth, N. Bericht des Nationalen Referenzzentrums für gramnegative Krankenhauserreger, 2018.\\nEpid. Bull. 2019, 31, 289–294. [CrossRef]\\n3. Bush, K.; Jacoby, G.A. Updated functional classiﬁcation of beta-lactamases. Antimicrob Agents Chemother\\n2010, 54, 969–976. [CrossRef] [PubMed]\\n4. Poirel, L.; Le Thomas, I.; Naas, T.; Karim, A.; Nordmann, P . Biochemical sequence analyses of GES-1, a\\nnovel class A extended-spectrum beta-lactamase, and the class 1 integron In52 from Klebsiella pneumoniae.\\nAntimicrob. Agents Chemother.2000, 44, 622–632. [CrossRef] [PubMed]\\n5. Stewart, N.K.; Smith, C.A.; Frase, H.; Black, D.J.; Vakulenko, S.B. Kinetic and structural requirements for\\ncarbapenemase activity in GES-type β-lactamases. Biochemistry 2015, 54, 588–597. [CrossRef]\\n6. Irrgang, A.; Pauly, N.; Tenhagen, B.-A.; Grobbel, M.; Kaesbohrer, A.; Hammerl, A.J.A. Spill-Over from Public\\nHealth? First Detection of an OXA-48-Producing Escherichia coli in a German Pig Farm. Microorganisms\\n2020, 8, 855. [CrossRef] [PubMed]\\n7. Borowiak, M.; Szabo, I.; Baumann, B.; Junker, E.; Hammerl, J.A.; Kaesbohrer, A.; Malorny, B.; Fischer, J.\\nVIM-1-producing Salmonella Infantisisolated from swine and minced pork meat in Germany. J. Antimicrob.\\nChemother. 2017, 72, 2131–2133. [CrossRef] [PubMed]\\n8. van der Zee, A.; Roorda, L.; Bosman, G.; Fluit, A.C.; Hermans, M.; Smits, P .H.M.; van der Zanden, A.G.M.;\\nTe Witt, R.; van Bruijnesteijn Coppenraet, L.E.S.; Cohen Stuart, J.; et al. Multi-centre evaluation of real-time\\nmultiplex PCR for detection of carbapenemase genes OXA-48, VIM, IMP , NDM and KPC.BMC Infect Dis.\\n2014, 14, 27. [CrossRef]\\n9. Swayne, R.L.; Ludlam, H.A.; Shet, V .G.; Woodford, N.; Curran, M.D. Real-time TaqMan PCR for rapid\\ndetection of genes encoding ﬁve types of non-metallo- (class A and D) carbapenemases in Enterobacteriaceae.\\nInt. J. Antimicrob. Agents2011, 38, 35–38. [CrossRef]\\n10. Feldgarden, M.; Brover, V .; Haft, D.H.; Prasad, A.B.; Slotta, D.J.; Tolstoy, I.; Tyson, G.H.; Zhao, S.; Hsu, C.-H.;\\nMcDermott, P .F.; et al. Validating the AMRFinder Tool and Resistance Gene Database by Using Antimicrobial\\nResistance Genotype-Phenotype Correlations in a Collection of Isolates. Antimicrob. Agents Chemother.2019,\\n63. [CrossRef]'), Document(metadata={'source': 'texts\\\\Irrgang, GES-5, 2020.pdf', 'page': 4}, page_content='Microorganisms 2020, 8, 1593 5 of 5\\n11. Carattoli, A.; Zankari, E.; Garcia-Fernandez, A.; Voldby Larsen, M.; Lund, O.; Villa, L.; Moller Aarestrup, F.;\\nHasman, H. In silico detection and typing of plasmids using PlasmidFinder and plasmid multilocus sequence\\ntyping. Antimicrob. Agents Chemother.2014, 58, 3895–3903. [CrossRef] [PubMed]\\n12. Chen, L.; Yang, J.; Yu, J.; Yao, Z.; Sun, L.; Shen, Y.; Jin, Q. VFDB: A reference database for bacterial virulence\\nfactors. Nucleic Acids Res.2005, 33, D325–D328. [CrossRef] [PubMed]\\n13. Jolley, K.A.; Maiden, M.C.J. BIGSdb: Scalable analysis of bacterial genome variation at the population level.\\nBMC Bioinform.2010, 11, 595. [CrossRef] [PubMed]\\n14. Vourli, S.; Giakkoupi, P .; Miriagou, V .; Tzelepi, E.; Vatopoulos, A.C.; Tzouvelekis, L.S. Novel GES /IBC\\nextended-spectrum beta-lactamase variants with carbapenemase activity in clinical enterobacteria.\\nFEMS Microbiol. Lett.2004, 234, 209–213. [CrossRef]\\n15. Jeong, S.H.; Bae, I.K.; Kim, D.; Hong, S.G.; Song, J.S.; Lee, J.H.; Lee, S.H. First outbreak ofKlebsiella pneumoniae\\nclinical isolates producing GES-5 and SHV-12 extended-spectrum beta-lactamases in Korea. Antimicrob.\\nAgents Chemother.2005, 49, 4809–4810. [CrossRef]\\n16. Wachino, J.-I.; Doi, Y.; Yamane, K.; Shibata, N.; Yagi, T.; Kubota, T.; Ito, H.; Arakawa, Y. Nosocomial spread of\\nceftazidime-resistant Klebsiella pneumoniae strains producing a novel class a beta-lactamase, GES-3, in a\\nneonatal intensive care unit in Japan. Antimicrob. Agents Chemother.2004, 48, 1960–1967. [CrossRef]\\n17. Cuzon, G.; Bogaerts, P .; Bauraing, C.; Huang, T.-D.; Bonnin, R.A.; Glupczynski, Y.; Naas, T. Spread of\\nPlasmids Carrying Multiple GES Variants. Antimicrob. Agents Chemother.2016, 60, 5040–5043. [CrossRef]\\n18. Viedma, E.; Juan, C.; Acosta, J.; Zamorano, L.; Otero, J.R.; Sanz, F.; Chaves, F.; Oliver, A. Nosocomial spread of\\ncolistin-only-sensitive sequence type 235 Pseudomonas aeruginosaisolates producing the extended-spectrum\\nbeta-lactamases GES-1 and GES-5 in Spain. Antimicrob. Agents Chemother.2009, 53, 4930–4933. [CrossRef]\\n19. Xu, T.; Wang, J.; Ying, J.; Zhu, T.; Liu, Y.; Xu, L.; Li, P .; Li, P .; Ying, J.; Li, K.; et al. Characterisation of a class 1\\nintegron associated with the formation of quadruple blaGES-5 cassettes from an IncP-1β group plasmid in\\nPseudomonas aeruginosa. Int. J. Antimicrob. Agents2018, 52, 485–491. [CrossRef]\\n20. Irrgang, A.; Tenhagen, B.-A.; Pauly, N.; Schmoger, S.; Kaesbohrer, A.; Hammerl, J.A. Characterization of\\nVIM-1-Producing E. coliIsolated From a German Fattening Pig Farm by an Improved Isolation Procedure.\\nFront. Microbiol. 2019, 10, 2256. [CrossRef]\\n21. Sarowska, J.; Futoma-Koloch, B.; Jama-Kmiecik, A.; Frej-Madrzak, M.; Ksiazczyk, M.; Bugla-Ploskonska, G.;\\nChoroszy-Krol, I. Virulence factors, prevalence and potential transmission of extraintestinal pathogenic\\nEscherichia coli isolated from diﬀerent sources: Recent reports. Gut Pathog.2019, 11, 10. [CrossRef]\\n22. Johnson, J.R.; Murray, A.C.; Gajewski, A.; Sullivan, M.; Snippes, P .; Kuskowski, M.A.; Smith, K.E. Isolation\\nand molecular characterization of nalidixic acid-resistant extraintestinal pathogenic Escherichia colifrom\\nretail chicken products. Antimicrob. Agents Chemother.2003, 47, 2161–2168. [CrossRef]\\nPublisher’s Note:MDPI stays neutral with regard to jurisdictional claims in published maps and institutional\\naﬃliations.\\n© 2020 by the authors. Licensee MDPI, Basel, Switzerland. This article is an open access\\narticle distributed under the terms and conditions of the Creative Commons Attribution\\n(CC BY) license (http://creativecommons.org/licenses/by/4.0/).')]\n",
      "\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Content of Irrgang, VIM_2019.pdf:\n",
      "[Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 0}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 1\\nORIGINAL RESEARCH\\npublished: 01 October 2019\\ndoi: 10.3389/fmicb.2019.02256\\nEdited by:\\nCostas C. Papagiannitsis,\\nUniversity of Thessaly, Greece\\nReviewed by:\\nJian-Hua Liu,\\nSouth China Agricultural University,\\nChina\\nBalázs Libisch,\\nAgricultural Biotechnology Institute,\\nHungary\\n*Correspondence:\\nA. Irrgang\\nalexandra.irrgang@bfr.bund.de\\nSpecialty section:\\nThis article was submitted to\\nAntimicrobials, Resistance\\nand Chemotherapy,\\na section of the journal\\nFrontiers in Microbiology\\nReceived: 13 May 2019\\nAccepted: 17 September 2019\\nPublished: 01 October 2019\\nCitation:\\nIrrgang A, Tenhagen B-A, Pauly N,\\nSchmoger S, Kaesbohrer A and\\nHammerl JA (2019) Characterization\\nof VIM-1-Producing E. coli Isolated\\nFrom a German Fattening Pig Farm\\nby an Improved Isolation Procedure.\\nFront. Microbiol. 10:2256.\\ndoi: 10.3389/fmicb.2019.02256\\nCharacterization of VIM-1-Producing\\nE. coliIsolated From a German\\nFattening Pig Farm by an Improved\\nIsolation Procedure\\nA. Irrgang1*, B.-A. Tenhagen1, N. Pauly1, S. Schmoger1, Annemarie Kaesbohrer1,2 and\\nJ. A. Hammerl1\\n1 Unit of Epidemiology, Zoonoses and Antimicrobial Resistance, Department of Biological Safety, German Federal Institute\\nfor Risk Assessment (Bundesinstitut für Risikobewertung, BfR), Berlin, Germany, 2 Institute for Veterinary Public Health,\\nUniversity of Veterinary Medicine Vienna, Vienna, Austria\\nA few reports indicate that livestock might represent a new reservoir for carbapenemase-\\nproducing Enterobacteriaceae (CPE). In 2015, VIM-1-producing Escherichia coli were\\ndetected at slaughter in colon contents of animals from a German fattening pig farm\\nwithin the national monitoring on ESBL-producing E. coli. In this study, pooled faces\\nsamples from pigs, as well as samples from the barn surrounding environment of this\\nfattening farm were taken, to evaluate the dissemination of CPEs. Several modiﬁcations\\nof the culture-dependent detection procedure were investigated for their potential to\\nimprove the sensitivity of the CPE isolation method. The current reference procedure\\nwas adapted by adding a real-time PCR pre-screening and additional enrichment steps.\\nIt was possible to isolate 32 VIM-1-producing E. coli from four fecal samples of three\\ndifferent barns using two serial enrichment steps in combination with real-time PCR\\nand selective agar plates. By genetic typing, we conﬁrmed the presence of two E. coli\\nclonal lineages circulating on this particular farm: one was harboring the blaVIM−1 on\\nan IncHI2 plasmid while the second lineage carried the gene on the chromosome.\\nDespite its different locations, the blaVIM−1 gene was harbored on a class 1 integron\\nin both clonal lineages. Whole-genome sequencing revealed that the VIM-1-carrying\\nplasmids exhibited only slight variability in its compositions and sizes. We assume that\\nthe prevalence of CPEs in animal production in Germany and other European countries\\nmight be underestimated and there is a concern of further spread of VIM-1-producing\\nbacteria in German livestock and food.\\nKeywords: carbapenemase, metallo-ß-lactamase, plasmid, sequencing, chromosomally encoded, VIM-1\\nINTRODUCTION\\nCarbapenems are broad-spectrum beta (ﬂ)-lactams that are considered as “critically important\\nantimicrobials” by the World Health Organization (WHO). They often represent the last treatment\\noption in human medicine for infections with multidrug-resistant bacteria (WHO Advisory\\nGroup on Integrated Surveillance of Antimicrobial Resistance, 2017). These antibiotics inhibit the\\nbacterial cell wall synthesis by interacting with the penicillin-binding protein (PBP). Carbapenems\\nFrontiers in Microbiology | www.frontiersin.org 1 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 1}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 2\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nare not degraded by most ﬂ-lactamases, including pAmpC and\\nextended-spectrum ﬂ-lactamases (ESBL). Acquired resistance\\nto carbapenems can be mediated by carbapenem degrading\\nenzymes (carbapenemases) or by changes in membrane\\npermeability due to the loss of speciﬁc outer membrane porins\\n(Miriagou et al., 2010). Carbapenemases confer resistance\\nto almost all ﬂ-lactams and typically are associated with\\nfurther resistance genes for other antimicrobial classes,\\nwhich strongly limit therapeutic options (Woodford et al.,\\n2014). Since the genetic information for the synthesis of\\ncarbapenemases is mostly encoded on plasmids or transposons,\\nthese resistance genes are horizontally transmissible to other\\nbacteria, which might further promote their spread (Patel and\\nBonomo, 2013). Carbapenemases have been mainly reported in\\nEnterobacteriaceae, Pseudomonas aeruginosa and Acinetobacter\\nbaumannii (Cantón et al., 2012).\\nCarbapenem-resistant Enterobacteriaceae (CRE) are an\\nemerging concern in the medical setting, especially in intensive\\nhealth care units. Enterobacteriaceae include common (Klebsiella\\npneumoniae, Escherichia coli, Salmonella enterica ) and rare\\nhuman pathogens (i.e., Proteus mirabilis, Raoultella planticola,\\nCitrobacter freundii) with increasing antibiotic resistance (Potter\\net al., 2016). Main risk factors for acquiring CRE among\\nhumans are intensive care therapies or hospital admission in\\nthe preceding 6 months, as well as previous travel outside the\\ncountry of residence (Grundmann et al., 2017).\\nCarbapenems are not approved for use in veterinary\\nmedicine (OIE, 2015). However, carbapenemase-producing\\nEnterobacteriaceae (CPE) were also reported from livestock,\\ncompanion animals, wildlife, food and frequently also from\\nrivers and sewage (Woodford et al., 2014; Köck et al., 2018).\\nNevertheless, carbapemases from non-human sources are up to\\nnow only sporadically isolated.\\nIn Germany, most carbapenemases detected in human\\nmedicine are OXA-48 and VIM-1 (Robert Koch-Institut, 2018).\\nWhereas OXA-48 and variants of it have not been detected from\\nGerman livestock yet, VIM-1-producing Enterobacteriaceae (i.e.,\\nSalmonella Infantis and E. coli) were detected in chicken and\\npig farms in Germany occasionally since 2011 (Fischer et al.,\\n2012, 2013). The respective Salmonella and E. coliisolates carried\\nthe blaVIM−1 gene on IncHI2 plasmids, ranging between 300 kb\\nand 220 kb in size, respectively (Falgenhauer et al., 2015). Due\\nto the lack of the Tra1-transfer regions the E. coli plasmid\\nwas determined to be non-self-transmissible. However, apart\\nfrom three deletion events, both plasmids were closely related\\n(Borowiak et al., 2017). Almost 5 years later, VIM-1-producing\\nE. coliwere detected from two batches of pigs at slaughter within\\nthe German national ESBL E. coli monitoring (Irrgang et al.,\\n2017b). Strikingly, isolates showed a high degree of similarity to\\nthe isolates from 2011. It is remarkable, that one of the isolates\\nobtained from a pig at slaughter harbored the blaVIM−1 gene and\\nits integron within the chromosome. In Figure 1 all recovered\\nVIM-1-producing E. coliisolates from Germany are summarized\\non a schematic time scale.\\nIn this study, the origin of CPE found in pigs at slaughter\\n(Irrgang et al., 2017b) was traced back to a pig fattening\\nfarm in Germany. Further investigations on the farm and the\\nsubsequent isolation of carbapenemase-producing E. coli from\\npooled fecal samples were carried out. Furthermore, the genetic\\nrelationship of the isolates in comparison to previous found\\nVIM-1-producing E. coliwas elucidated.\\nMATERIALS AND METHODS\\nSampling\\nThe pig farm consisted of six barns for fattening pigs.\\nThree barns were in historical buildings at the main site\\nof the farm. The other three barns were modern identical\\nbuildings located approximately 100 m away from the old\\nbuildings. The barns were subdivided in eight to fourteen\\npens with thirty pigs per pen. All in all, around 2,000\\nanimals were housed at the farm. The six barns were\\nmanaged in an all in/all out system per barn. Composite\\nfecal samples were collected from the ﬂoor at a minimum\\nof ten diﬀerent points of two opposing pens per sample\\nresulting in 33 fecal samples. Additionally boot swab samples\\nwere taken from the alleys of each barn. Furthermore,\\nenvironmental wipes were collected from trough, toys,\\nwalls, and dust.\\nIsolation Procedure\\nAll samples were tested using the method recommended by the\\nEURL-AR for the isolation of carbapenemase-producing E. coli\\nfrom caeca samples (Version 3; actual version can be found at:\\nhttps://www.eurl-ar.eu/protocols.aspx, accessed on September 3,\\n2018). According to this procedure, the fecal samples were diluted\\n1:10 in buﬀered peptone water (w/v) without antimicrobial\\nadditives and pre-incubated 16ñ20 h at 37 ◦C. Thereafter, this\\npre-enrichment culture was applied onto chromID\\nR⃝Carba plates\\n(Biomerieux) for selective cultivation. An aliquot of the pre-\\nenrichment culture was stored at ñ80◦C in glycerin.\\nAlternatively, a selective pre-enrichment was conducted in\\nparallel by applying 10 g of the pooled fecal samples to 90 ml\\nlysogeny broth (LB) supplemented with 1 mg/L cefotaxime\\n(CTX) and plated on chromID\\nR⃝Carba. To exclude any inhibitory\\neﬀects of the chromID\\nR⃝Carba agar plates, 100 µl of the\\nunselective and selective pre-enrichments were additionally\\nplated on MacConkey agar supplemented with 1 mg/L CTX.\\nPrevious experiences had revealed that an additional selective\\nenrichment step in LB +CTX following the unselective pre-\\nenrichment might yield a higher recovery rate of CPEs. Thus, a\\nloop from the glycerin stock was inoculated in LB +CTX at 37\\nand 44◦C (Supplementary Figure S1A).\\nThe detection of VIM-1-producing E. coli was supported\\nby real-time PCR (described below), carried out from the\\nselective enrichment broth (37 ◦C). In case of a positive signal,\\nindividual presumptive E. coli colonies were picked from the\\ncorresponding MacConkey plates into LB supplemented with\\n0.125 mg/L meropenem for 24 h at 37 ◦C in a 96 well microtiter\\nplate. CPE veriﬁcation was conducted by real-time PCR. Species\\ndetermination of bacterial colonies was performed by MALDI-\\nTOF MS (Biotyper, Bruker, Bremen, Germany).\\nFrontiers in Microbiology | www.frontiersin.org 2 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 2}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 3\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nFIGURE 1 | Time scale of detected VIM-1-producing E. coli from German pig production. Colors cyan and red indicate the two different farms, from which the\\nsamples origin. ∗Isolates were described previously (Falgenhauer et al., 2015; Fischer et al., 2017). ∗∗Study described by Irrgang et al. (2017b).\\nAntimicrobial Susceptibility Testing\\nSusceptibility to antimicrobials was determined by broth\\nmicrodilution according to CLSI standard and minimal\\ninhibitory concentration (MIC) values were interpreted\\naccording to EUCAST deﬁnitions. In case of azithromycin and\\ntemocillin tentative ECOFFS provided by EFSA were used for\\ninterpretation.\\nReal-Time PCR\\nPCR-based detection was used to conﬁrm the presence/absence\\nof VIM-1 CPE in enrichment cultures. Real-time PCR was\\nperformed on a Bio-Rad CFX system. The blaVIM primer and\\nprobe sequences were adapted from van der Zee et al. (2014) with\\nCAL Flour\\nR⃝Gold 540 as ﬂuorophore and BHQ-1 as quencher\\n(van der Zee et al., 2014). Ampliﬁcation was conducted using\\nABsolute QPCR Mix, no ROX (Thermo Fisher Scientiﬁc) and the\\nfollowing conditions: initially 15 min at 95◦C, 30 cycles with 15 s\\nat 95◦C and 60 s at 60◦C for product ampliﬁcation.\\nMolecular Characterization\\nPulsed-ﬁeld gel electrophoresis (PFGE) was performed according\\nto the PulseNet protocol 1. Restriction proﬁles of the genomes\\nwere obtained by using XbaI, whereas plasmid content was\\nanalyzed using S1-nuclease. PFGE analysis and cluster analysis\\nwere conducted with BioNumerics 7.5 (Applied Maths, Sint-\\nMartens-Latem, Belgium). To determine whether the blaVIM−1\\ngene was located on plasmids, S1-PFGE gels and subsequent\\nSouthern blot hybridization following DNAñDNA hybridization\\nwere performed. For blotting, DNA from PFGE gels was\\ntransferred to a nitrocellulose membrane by vacuum blotting\\n(1.5 h at 5 bar) with 10 ×SSC. After crosslinking DNA for\\n2 min under UV light (312 nm), membrane was hybridized\\nwith a digoxigenin (DIG) labeled blaVIM probe and DIG\\nlabeled Lambda DNA as positive control using DIG Easy\\nHyb and DIG Wash and Block Buﬀer Set (Roche Diagnostics,\\nMannheim, Germany) as recommended by the manufacturer.\\n1https://www.cdc.gov/pulsenet/pdf/ecoli-shigella-salmonella-pfge-protocol-\\n508c.pdf\\nTransformation of blaVIM−1-carrying plasmids was carried\\nout by electroporation into competent E. coli DH10B cells\\n(ElectroMAXTM DH10BTM Cells; InvitrogenTM, Thermo Fisher\\nScientiﬁc, Schwerte, Germany).\\nSequencing and Bioinformatic Analyses\\nWhole-genome sequencing (WGS) of selected isolates ( n = 10)\\nwas conducted on a Miseq benchtop platform (Illumina, CA,\\nUnited States) with 2 × 251 bp paired-end reads. Isolates\\nrepresenting diﬀerent clonal lineages and/or plasmid proﬁles\\nwere chosen for WGS. De novo genome assembling was\\ncarried out using CLC Genome workbench 9.5.2 (Qiagen\\nBioinformatics) or PATRIC database 2. Sequences were\\ndeposited at NCBI with the accession numbers: RYCV00000000\\n(R1176), RYCW00000000 (R1180), RYCX00000000 (R1182),\\nRYCY00000000 (R1183), RYDA00000000 (R1184), RYDA\\n00000000 (R1191), RYDB00000000 (R1203), RYDC00000000\\n(R1207), RYDD00000000 (R1208), and RYDE00000000 (R1209).\\nInformation on MLST-type, plasmid content, pMLST and\\nacquired resistance genes was obtained from sequencing data\\nusing the web-based tools provided by the Center for Genomic\\nEpidemiology3. For single nucleotide polymorphism (SNP)\\nwhole-genome data were processed and interpreted using\\nBioNumerics 7.5 (Applied Maths). Raw reads ﬁrst underwent a\\nlength trimming (settings: 300 bases; minimum sequence length:\\n20, maximum homopolymer length: 20 bases), followed by a\\nquality trimming (minimum quality: 5; average quality: 20). For\\nSNP analysis SNP ﬁlters for absolute coverage, relative coverage,\\nunreliable bases, ambiguous bases, non-informative SNPs and\\ninter-SNP distance were applied with default parameters (Irrgang\\net al., 2018). Plasmid related sequences were mapped using BRIG\\nsoftware (Alikhan et al., 2011).\\nEthics Statement\\nAccording to the national law, permission from an ethical\\ncommittee for this study is not needed. The samples were taken\\n2https://www.patricbrc.org/\\n3http://www.genomicepidemiology.org\\nFrontiers in Microbiology | www.frontiersin.org 3 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 3}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 4\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nfrom the ﬂoor of the boxes. The animals of the farm were not\\ntested directly.\\nRESULTS AND DISCUSSION\\nIsolation of VIM-1-Producing E. coli\\nThe pig farm was investigated in May 2016, 6 month after the\\nﬁrst detection of a VIM-1-producing E. coli from the colon\\ncontent of a fattening pig at slaughter (Irrgang et al., 2017b).\\nUsing the EURL-AR reference method for isolation of CPEs, no\\ncarbapenemase-producing E. coliwere visually detected in fecal\\nand environmental samples. However, most of the agar plates\\nexhibited complex lawns of Pseudomonas bacteria that impede\\nreliable detection of CPEs.\\nIn contrast to the EURL-AR reference method, CPE detection\\nusing the selective enrichment with LB +CTX and subsequent\\nplating on ChromID\\nR⃝Carba we recovered VIM-1-producing\\nE. coli isolates R1182 to R1184 from one of the samples (B6S3,\\nsample No. 3 of barn No. 6; see Table 1 ). This indicated\\nthat we might have missed further positive samples due to\\nthe extensive growth of accompanying bacterial ﬂora (i.e.,\\nPseudomonas) on most of the ChromID\\nR⃝Carba plates. Thus, a\\nloop of the stored glycerin stocks of the unselective enrichment\\nwas transferred into LB +CTX for selective enrichment. By\\nreal-time PCR using bacteria of the selective enrichment as\\ntemplate blaVIM was identiﬁed in three of the samples (B1S1,\\nB5S5 and B6S6). In contrast to the samples B1S1, B5S5 and\\nB6S6, the sampleB6S3 could not be detected by real-time\\nPCR but was positive by cultivation. From the blaVIM positive\\nsamples individual colonies were randomly taken from the\\ncorresponding MacConkey +CTX plates and sub-cultured in LB\\nsupplemented with 0.125 mg/L meropenem in 96-well microtiter\\nplates. Bacterial growth was observed in most of the wells of\\nsamples B5S5 and B6S6. Encoded blaVIM could be conﬁrmed\\nby real-time PCR.\\nNo bacterial growth was observed in wells with colonies from\\nthe third positive sample B1S1. Here, we reverted to the glycerin\\nstock of the unselective pre-enrichment. This stock was sub-\\ncultivated in LB +CTX at 44◦C for 24 h. This inhibited growth\\nof P. aeruginosa and allowed the detection of isolates from the\\nforth sample by subsequent plating on chromID\\nR⃝ Carba plates.\\nIn total, 32 VIM-1-producing E. coli isolates from four fecal\\nsamples of three diﬀerent barns (see Table 1) were recovered\\nby the combination of the diﬀerent procedures. This indicates a\\nspread of VIM-1-producing E. coliwithin the farm but at a very\\nlow level. Boot swab samples as well as wipe samples from the\\nenvironment were analyzed in a similar way to the fecal samples.\\nFor wipe samples the parallel enrichment step in LB +CTX\\nat 37 ◦C was dropped because the sample was not divisible.\\nAll of them were negative for CPE. There was no detection of\\nSalmonella sp. from the samples, although there was a VIM-\\npositive Salmonella strain isolated from a diseased pig 2 months\\nbefore from the same farm (Borowiak et al., 2017).\\nFailure to identify positive samples using the reference method\\nshowed, that the sensitivity of this procedure is not suﬃcient\\nto detect CPEs with a low prevalence in such samples. This\\nwas mostly due to the overgrowth by Pseudomonas sp. that\\nare frequently intrinsic resistant to carbapenems (Livermore,\\n2001). In this study, a second selective enrichment step at\\n44◦C was useful to suppress the growth of Pseudomonas, but\\nEnterobacteriaceae other than E. colimight be also inhibited.\\nThe current used EURL reference method is based on\\nisolation with an appropriate selective medium. The sensitivity\\nand speciﬁcity of commercially available media diﬀers a lot\\nbetween diﬀerent studies (Aguirre-Quiñonero and Martínez-\\nMartínez, 2017). For example sensitivity ranged from 33.3 to\\n100% and speciﬁcity from 54.1 to 98.9% for chromID\\nR⃝Carba,\\nwhich was used in this study. So, a combination of microbiologic\\nand molecular methods might improve the sensitivity for the\\ndetection of CPEs.\\nFor research purpose, analyzing the enrichment cultures by\\n(real-time) PCR for the presence of respective genes is suitable.\\nHowever, such a molecular screening of the samples is limited\\nto the detection of some carbapenemase groups, which are\\ndepending on the established (multiplex) PCR. In our case we\\nfocus on the most prevalent carbapenemase groups (NDM, VIM,\\nOXA-48, KPC and GES). Other or novel carbapenem genes\\nwould not be detected by routine. For monitoring programs\\naccording to the guideline 2013/652/EU it might be suitable to use\\ntwo serial enrichment steps ( Supplementary Figure S1B ): ﬁrst\\nan unselective pre-enrichment step in buﬀered peptone water,\\nusing this culture for plating on selective OXA-48 media. In\\na second step, cultivating an inoculum of the unselective pre-\\nenrichment culture in LB supplemented with 1 mg/L CTX at\\n44◦C (for isolation of E. coli) or 37◦C (for Enterobacteriaceae in\\ngeneral). As the suggested protocol is derived only from isolation\\nof blaVIM−1-producing E. coli, further investigations are needed,\\nto clarify its suitability for other CPEs and other carbapenemase\\ngenes. Nevertheless, the protocol recommended by EURL-AR is\\nused from the laboratories of the federal states at the moment,\\nso an underestimation of the occurrence of carbapenemases in\\nGerman livestock is assumed.\\nMolecular Characteristics of the\\nVIM-1-Producing E. coliFrom German\\nFattening Pigs\\nThese isolates were further characterized and compared with\\nthe isolates obtained from colon samples and isolates obtained\\nfrom a German pig farm in 2011 as given in Figure 1\\n(Fischer et al., 2017).\\nXbaI-PFGE proﬁles of the obtained isolates showed great\\nsimilarities ( Figure 2 ). Minor diﬀerences between the isolates\\nare based on variations in the plasmid contents ( Figure 3 ).\\nThe XbaI-PFGE pattern revealed also a close phylogenetic\\nrelationship to the isolates from colon content and to the ones,\\nwhich were detected in 2011 ( Figure 2 ) (Fischer et al., 2017;\\nIrrgang et al., 2017b). In concordance to the PFGE analyses,\\nall isolates belonged to the multi-locus sequence type ST-88\\nthat was also identiﬁed for the previously characterized VIM-1-\\nproducing E. coli. DNAñDNA hybridization of blotted S1-PFGE\\ngels indicated that the blaVIM−1 gene was located on plasmids\\nrepresenting sizes of more than 200 kb. Respective plasmids\\nFrontiers in Microbiology | www.frontiersin.org 4 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 4}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 5\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nTABLE 1 | Summary of the VIM-1-producing E. coli isolates detected from fecal samples of pigs from a German fattening farm and their basic characteristics.\\nIsolate Barn No. Sample\\nNo.\\nBeta-lactamases Localization\\nblaVIM−1\\nSize and\\nincompatibility group\\nof VIM-1 plasmid\\nResistance proﬁle phenotype/genotype∗\\nR1182¤ 6 3 VIM-1, ACC-1,\\nTEM-206\\nPlasmid 375 kb; IncHI2 ß-lactams ∗∗, AZI, CHL, SMX, TMP/ aadA2, aph(6)-Id,\\naph(3\")-Ib, blaACC−1, blaVIM−1, blaTEM, cmlA1, dfrA8,\\ndfrA12, mef(C)∗∗∗, mph(G), sul1, sul2\\nR1183¤ 6 3 VIM-1, ACC-1,\\nTEM-206\\nPlasmid 208 kb; IncHI2 ß-lactams ∗∗, AZI, CHL, SMX, TMP/, aadA1, aadA2,\\naph(6)-Id, blaACC−1, blaVIM−1, blaTEM, cmlA1∗∗∗, dfrA8,\\ndfrA12, mef(C)∗∗∗, mph(G), strA, sul1, sul2,\\nR1184¤ 6 3 VIM-1, ACC-1,\\nTEM-206\\nPlasmid 207 kb; IncHI2 ß-lactams ∗∗, AZI, CHL, SMX, TMP/aadA1, aadA2,\\naph(6)-Id, aph(3\")-Ib, blaACC−1, blaVIM−1, blaTEM,\\ncmlA1∗∗∗, dfrA8, dfrA12, mef(C)∗∗∗, mph(G), sul1, sul2,\\nR1185 6 6 VIM-1 Chromosomal ß-lactams ∗∗, SMX\\nR1186 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1187 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1188 6 6 VIM-1 Chromosomal ß-lactams ∗∗, SMX\\nR1189 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1190 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1191¤ 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP/aadA1, aadA2, blaTEM−1B,\\nblaVIM−1, cmlA1∗∗∗, dfrA8, dfrA12, mef(C)∗∗∗, mph(G),\\nstrA, sul1∗∗∗, sul2\\nR1192 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1193 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1194 6 6 VIM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1195 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1196 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1197 6 6 VIM-1 Chromosomal ß-lactams ∗∗, SMX\\nR1198 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1199 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, SMX\\nR1200 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1201 6 6 VIM-1 Chromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1202 6 6 VIM-1, TEM-1 Chromosomal ß-lactams ∗∗, SMX\\nR1203¤ 5 5 VIM-1, ACC-1,\\nTEM-1\\nPlasmid 212 kb, IncHI2 ß-lactams ∗∗, AZI, CHL, SMX, TMP/aadA1, aadA2,\\naph(6)-Id blaACC−1, blaTEM−1B, blaVIM−1, cmlA1∗∗∗, dfrA8,\\ndfrA12, strA, mef(C)∗∗∗, mph(G), strA, sul1∗∗∗ sul2\\nR1204 5 5 VIM-1, ACC-1,\\nTEM-1\\nPlasmid 253 kb, IncHI2 ß-lactams ∗∗, CHL, SMX, TMP\\nR1205 5 5 VIM-1, ACC-1 Plasmid 208 kb, IncHI2 ß-lactams ∗∗, SMX\\nR1206 5 5 VIM-1, ACC-1 Plasmid 212 kb, IncHI2 ß-lactams ∗∗, SMX\\nR1207¤ 5 5 VIM-1, ACC-1 Plasmid 238 kb, IncHI2 ß-lactams ∗∗, SMX, aadA1, aph(6)-Id, blaACC−1, blaVIM−1,\\nstrA, sul1∗∗∗, sul2\\nR1208¤ 5 5 VIM-1, ACC-1,\\nTEM-1\\nPlasmid 210 kb, IncHI2 ß-lactams ∗∗, AZI, CHL, SMX, TMP/aadA1, aadA2,\\naph(6)-Id, blaACC−1, blaTEM−1B, blaVIM−1, cmlA1∗∗∗,\\ndfrA12, strA, sul1∗∗∗\\nR1209¤ 1 1 VIM-1, TEM-1,\\nTEM-206\\nChromosomal ß-lactams ∗∗, AZI, CHL, SMX, TMP/aadA1, aadA2,\\naph(6)-Id, blaACC−1, blaTEM−206, blaVIM−1, cmlA1∗∗∗,\\ndfrA12, mef(C)∗∗∗, mph(G), strA, sul1, sul2\\nR1210 1 1 VIM-1, TEM-1,\\nTEM-206\\nChromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1211 1 1 VIM-1, TEM-1,\\nTEM-206\\nChromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1212 1 1 VIM-1, TEM-1,\\nTEM-206\\nChromosomal ß-lactams ∗∗, CHL, SMX, TMP\\nR1213 1 1 VIM-1, TEM-1,\\nTEM-206\\nChromosomal ß-lactams ∗∗, CHL, SMX, TMP\\n¤Strains sequenced. ∗Only genes with 100% coverage and identity of 100% included; all detected genes can be found in Supplementary Table S1. ∗∗Resistance\\nconﬁrmed for the ß-lactams: ampicillin, cefepime, cefotaxime, ceftazidime, cefoxitin, imipenem; for temocillin, meropenem and ertapenem, please see theSupplementary\\nTable S1, if the strains showed a decreased susceptibility. ∗∗∗Identity less than 100% (>99%), included as functionality is assumed by MIC values.\\nFrontiers in Microbiology | www.frontiersin.org 5 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 5}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 6\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nwere shown in isolates from two of the samples (R1182-1184\\nfrom sample B6S3, R1203-R1208 from sample B5S5), whereas the\\nisolates from the other two samples (B1S1, B6S6) harbored the\\ngene on the chromosome. This indicates the spread of two very\\nsimilar clonal lineages within the farm. Transformation of the\\nblaVIM−1 plasmid by electroporation with subsequent replicon\\ntyping conﬁrmed that these plasmids belong to IncHI2.blaVIM−1\\ncarrying plasmids detected in the past from Salmonella and\\nE. coliisolates of samples from pig production were typed equally\\n(Fischer et al., 2013; Irrgang et al., 2017b). Interestingly, the\\ncontent of plasmids among the analyzed isolates showed a high\\ndegree of variation, indicating a great plasticity in the mobilome\\nof the VIM-1-producing E. coli. All isolates harbored a 135 kb\\nIncFII/FIB plasmid (pMLST [2:A-:1]). In some of the isolates\\nthis plasmid was fused with a ∼35 kb IncX1 plasmid, which\\nwas harbored frequently. Isolate R1882 harbored a huge plasmid,\\ncontaining loci of IncHI2, IncF and IncX1 plasmids, suggesting\\na fusion of these plasmids. It is questionable, whether plasmid\\nfusion was original or an artifact from isolation procedure. An\\nadditional ∼70 kb plasmid could be found in some of the isolates.\\nIt is possible that due to the formation of plasmid co-integrates\\nthe properties and host range of these mobile elements might\\nbe changed (Papagiannitsis et al., 2013). The IncH12 plasmids\\ndescribed here are non-conjugative variants of wide-host range\\nplasmids. Co-integration with a plasmid of incompatibility group\\nP , L/M, N, R, W or U could make the plasmid conjugative\\nagain with an extend of the host range to a broad spectrum of\\nbacterial species (Rozwandowicz et al., 2018). It was even shown,\\nthat co-integration sometimes only takes place for the time of\\ntransmission and plasmids separate again in the new host (Xie\\net al., 2018). With this strategy, localization of the blaVIM−1 gene\\non a non-conjugative plasmid does not limit its spread.\\nAnalysis of the stability of the blaVIM−1 plasmid was carried\\nout by subcultivation of selected strains without antibiotic\\npressure for at least 500 bacterial generations ( >50 passages).\\nQuantiﬁcation of bacteria under selective and non-selective\\nconditions revealed that the plasmid was highly stable in E. coli\\n(data not shown). This is in concordance to the repeated isolation\\nof the clones over a period of 6 months at one farm and also with\\nthe detection of similar E. coli from diﬀerent farms 5 years ago\\n(Fischer et al., 2017; Irrgang et al., 2017b). This stability concerns\\nin regard of a further spread of VIM-1-producing E. coliwithin\\nthe German livestock.\\nAs described before, based on the blaVIM−1 localization two\\nclonal lineages were detected. One clonal lineage harbored the\\nblaVIM−1 gene on plasmids and in the other clonal lineage it\\nwas chromosomally located. In one barn we detected both clonal\\nlineages in diﬀerent samples. Regardless its localization within the\\ngenome, the blaVIM−1 gene was located on a class 1 integron as\\ndescribed (Falgenhauer et al., 2015). As the blaVIM−1-carrying\\nplasmids are not self-transmissible, transmission of the gene\\nis only possible through mobilization of the integron or the\\nplasmid. The chromosomal localization of the blaVIM−1 gene is\\nunique for isolates obtained from this farm. None of the VIM-\\n1-producing Enterobacteriaceae isolated in Germany before,\\nharbored the gene on the chromosome (Fischer et al., 2012, 2013;\\nRoschanski et al., 2017). Integration in the chromosome provides\\nFIGURE 2 | PFGE XbaI restriction pattern of VIM-1 producing E. coli from pig\\nsamples. PFGE conditions: 2.2–54.2 s; 20 h; 14◦C; 6 V/cm. Lanes: 1: R1180\\n(Irrgang et al., 2017b); 2: R1209; 3: R1210; 4: R1211; 5: R1203; 6: R1207; 7:\\nR1208; 8: R1182; 9: R1283; 10: R1284; 11: R1186; 12: R1191; 13: 1197;\\n14: R178 (Fischer et al., 2012); M: S. Braenderup H9812.\\nthe advantage, that the resistance determinants will be vertically\\ntransmitted stable without antimicrobial pressure whereas it\\nremains transmissible through its integron localization. Such\\na transmission was shown for chromosomal located blaVIM\\nin Aeromonas caviae from Israel (Adler et al., 2014). The\\ndetection of the plasmid-mediated AmpC β-lactamase AAC-\\n1 was a suitable marker for the localization of the VIM-1\\nintegron as it was only present on strains harboring the blaVIM−1\\ngene on a plasmid.\\nTo get a deeper insight into the genome and to determine the\\nintegrated sequence length, a subset of eight isolates from the\\nfarm and two isolates from slaughter samples (R1176 and R1180;\\nIrrgang et al., 2017b) was sequenced on an Illumina Miseq. The\\nSNP analysis of the sequence data of the isolates (mobilome\\nexcluded) conﬁrmed the close phylogenetic relationship of the\\nisolates ( Figure 4 ). The isolates exhibited slight variations\\nranging between 0 and 17 SNPs to each other. This result\\nsuggested that all isolates originating from a common ancestor.\\nSimilar results are observed using wgMLST for phylogenetic\\ncomparison (data not shown). By SNP analysis, isolates carrying\\nthe blaVIM−1 on the chromosome are phylogenetically slightly\\ndistinct from the ones harboring the gene on the plasmid. In\\naddition, isolates from the same barn form separate clusters. The\\nmethod was found to be robust as there were no diﬀerences\\nobserved using diﬀerent reference genome for cluster analysis.\\nIt is also in concordance to cluster analysis of the XbaI-PFGE\\n(Supplementary Figure S2).\\nThe mapping of the sequence data to the E. coli blaVIM−1\\nplasmid pRH-R178 (Acc-No.: HG530658.1) which was detected\\nin 2011 is shown in Figure 5A . All isolates lacked a fragment\\nof ∼10 kb length. This fragment harbored some hypothetical\\nproteins as well as genes coding for e.g., a copper receptor, a\\nFrontiers in Microbiology | www.frontiersin.org 6 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 6}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 7\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nFIGURE 3 | S1-nuclease PFGE. Running conditions: 1–25 s; 6 V/cm, 120◦, 14◦C. blaVIM−1-harboring plasmids are marked with an asterisk. 1: R1182; 2: R1183; 3:\\nR1184; 4: R1185; 5: R1186; 6: R1187; 7: R1188; 8: R1189; 9: R1190; 10: R1191; 11: R1192; 12: R1193; 13: R1194; 14: R1195; 15: R1196; 16: R1197; 17:\\nR1198; 18: R1199; 19: R1200; 20: R1201; 21: R1202; 22: R1203; 23: R1204; 24: R1205; 25: R1206; 26: R1207; 27: R1208; 28: R1209; 29: R1210; 30: R1211;\\n31: R1212; 32: R1213; 33: R178 (Fischer et al., 2012); 34: R29 (Fischer et al., 2012); M: S. Braenderup H9812.\\nFIGURE 4 | SNP analysis of the isolates obtained from the pig farm (R1182-1209) in comparison with the isolates R1176 and R1180 detected from pigs from two\\nslaughter batches (Irrgang et al., 2017b). As reference R1184 sequenced by PacBio RS2 (to be published elsewhere) was used. Clustering was calculated using\\nNeighbor Joining method.\\nsilver-binding protein or a cation eﬄux pump ( Figure 5B). The\\ndeleted region is ﬂanked by a transposase upstream and might\\nbe a target of integration or excision events, which then might\\nchange the organization of this gene cluster. The rings 1ñ3 display\\nsequences from the isolates harboring the blaVIM−1 gene on the\\nchromosome. Results indicated that a large fragment of ∼100 kb\\nlength was transferred to the chromosome. Mapping of short read\\nWGS data to reference sequence RYCV00000000 (to be published\\nelsewhere) of the previously characterized isolate R1176 (Irrgang\\net al., 2017b) indicated, that isolates detected within this study\\nexhibited a similar chromosomal insertion of the resistance gene\\n(Figure 6 ). This fragment sequence is ﬂanked by transposases\\nindicating that recombination events might occur frequently\\nand location of the VIM-1 integron within the genome might\\nchange repeatedly.\\nPhenotypes and Their Correlation to\\nGenes and Plasmids\\nThe phenotypic resistance (MIC) was determined for all isolates.\\nRelevant phenotypes are summarized in Table 1 . Further\\nMIC values are provided in the Supplementary Material . All\\nisolates were susceptible to tetracycline, colistin, tigecycline,\\n(ﬂuoro-)quinolones and gentamicin. However, resistance genes\\nfor aminoglycosides detected via WGS (aadA, strA, strB) indicate\\nresistance to amikacin and/or kanamycin. In the EUVSEC\\nFrontiers in Microbiology | www.frontiersin.org 7 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 7}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 8\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nFIGURE 5 | (A) Mapping of whole genome sequences to plasmid pRH-R178 (Acc.-No: HG530658.1) described by Falgenhauer et al. (2015). Rings 1–10 named in\\nthe legend are numbered from in to out. Rings 1–3 blaVIM−1 located on the chromosome; rings 4–10 gene located on IncHI2 plasmid. The position of repetitive\\nsequences associated with the insertion elements IS6100 and ISEcp1 are indicated by blue and red circles, respectively.(B) Schematic illustration of the pRH-R178\\nregion that is deleted in the isolates of this study. In panel (A), the organization of the respective pRH-R178 region (nucleotide position 145,601–166,000) is shown.\\nThe organization of the respective pRH-R178 region (nucleotide position 145,601–166,000) is shown on the top. Genes of the speciﬁed functions are colored as\\ndesignated. Below, the conserved DNA regions (gray) found in the individual isolates of this study is shown by boxes.\\nFrontiers in Microbiology | www.frontiersin.org 8 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 8}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 9\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nFIGURE 6 | Schematic illustration of the chromosomal location of the blaVIM−1 gene in E. coli isolate R1176. In panel (A), a dot plot alignment (minimum 85%\\nnucleotide identity; Accelrys DS Gene, version 2.5, Accelrys Inc.) of the R178 plasmid (Accession number HG530658.1) with the respective region of the E. coli\\nR1176 (RYCV00000000) chromosome is shown. DNA regions of close relationship between R178 and the R1176 chromosome are indicated in gray. Furthermore,\\nthe position of the blaVIM−1 gene is given. In panel (B), the organization of the chromosomal region ﬂanking the inserted element of R178 (gray) as well as potential\\ntarget sites for the chromosomal insertion (repeat regions RR1 and RR2) are shown. In comparison to the chromosomal fragment ofE. coli R1176 the corresponding\\nsequence positions of the complete E. coli 14EC029 genome are given. While in E. coli R1176 approximately 100 kb of the R178 plasmid were inserted, the\\ncorresponding region of the E. coli isolate 14EC029 exhibited no disruption by insertion sequences.\\nlayout, which is used since 2014, only gentamycin is tested. In\\nolder publications, where the former test layout was used, the\\ncorrelation of the aminoglycoside resistance genes and phenotype\\nwas shown (Irrgang et al., 2017a). As seen in Supplementary\\nTable S1 some of the strains are susceptible to meropenem and\\nertapenem but all strains showed MIC values above the cut-\\noﬀ value for imipenem. This indicates that meropenem was not\\nthe appropriate indicator antimicrobial to detect carbapenemase\\nactivity. Low MIC values for carbapenems are reported for\\nother VIM-producing species like A. caviae or K. pneumoniae\\n(Psichogiou et al., 2008; Adler et al., 2014). Minor variations\\nin MIC values of cephalosporins are mostly in the range of\\nthe microdilution method. Diﬀerences in MIC could be seen\\nfor chloramphenicol, trimethoprim and azithromycin. Resistance\\nto chloramphenicol and trimethoprim was associated with the\\npresence of a ∼35 kb IncX1 plasmid harboring the genes cmlA1\\n(chloramphenicol resistance) and dfrA12/dfrA8 (trimethoprim\\nresistance), respectively. The isolates R1199 and R1202 harbored\\na 5 kb smaller plasmid and were therefore sensitive for these two\\nantimicrobials as well as the isolates where the plasmid was not\\npresent. In the isolates R1182, R1183, R1184, R1190 and R1204\\nthe IncX1 plasmid seemed to be fused to the 135 kb plasmid,\\nwhich is present in all other isolates, or to the blaVIM−1-carrying\\nplasmid. In R1207 the respective plasmid seemed to be fused to\\nFrontiers in Microbiology | www.frontiersin.org 9 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 9}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 10\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nthe smaller version of the IncX1 plasmid. This strain is sensitive\\nfor chloramphenicol and trimethoprim. IncX1 are narrow-host\\nplasmids typically present in E. coli and Salmonella sp. often\\nharboring AMR genes, especially ﬂ-lactamases (Rozwandowicz\\net al., 2018). In our isolates the presence of blaTEM genes\\ncorrelates with the presence of IncX1 sequences, except for R1194\\nand R1201 where the plasmid was smaller indicating deletion of\\nthe gene region.\\nThe resistance to azithromycin is related to another 70 kb\\nplasmid, harboring mph(G) with 100% identity and mef (C)\\nwith 99.92% identity ( Supplementary Table S2 ). This plasmid\\nwas ∼5 kb smaller in R1205, R1206 and R1207 resulting in\\nsensitivity to azithromycin which indicated the deletion of this\\ngene region. Based on the sequence information typing of the\\nplasmid was not possible.\\nCONCLUSION\\nIn conclusion, isolation of VIM-1-producing E. coli from fecal\\nsamples with a low CPE load was only possible by adapting\\nthe given EURL-AR protocol with diﬀerent enrichment steps in\\ncombination with a molecular screening via real-time PCR. Two\\nclosely related clonal lineages of the strain were detected, which\\ncould be pursued at the farm for at least 6 months. Persistence due\\nto plasmid stability and vertical gene transfer of the chromosomal\\nencoded VIM-1 carbapenemase raise the concern of further\\nspread within the German pig production.\\nDATA AVAILABILITY STATEMENT\\nThe datasets generated for this study can be found\\nat NCBI with the accession numbers: RYCV00000000\\n(R1176), RYCW00000000 (R1180), RYCX00000000 (R1182),\\nRYCY00000000 (R1183), RYDA00000000 (R1184), RYDA\\n00000000 (R1191), RYDB00000000 (R1203), RYDC00000000\\n(R1207), RYDD00000000 (R1208), and RYDE00000000\\n(R1209). Sequences are publically available as soon as the\\nmanuscript is published.\\nAUTHOR CONTRIBUTIONS\\nAI and B-AT took the samples on the farm. AI, AK,\\nand B-AT planned and organized the study. AI, SS, and\\nNP carried out laboratory work. AI and JH performed the\\nbioinformatics analyses and conceptualized the study. AI, NP ,\\nand JH wrote the manuscript. All authors contributed in\\nproofreading the manuscript.\\nFUNDING\\nAI was funded by grants of the BMBF , German Federal\\nMinistry for Education and Research in the course of RESET\\n(FKZ01Kl1313B). NP received grants from the One Health\\nEuropean Joint Programme (OHEJP) JRP1 IMPART (Grant\\nagreement n◦773830). This work was also supported by grants\\n(43-001 and 43-002) of the German Federal Institute for\\nRisk Assessment.\\nACKNOWLEDGMENTS\\nWe thank Britta Lesniewsky, Tanja Skladnikiewicz-Ziemer,\\nKatharina Thomas and Nicole Dinse for excellent technical\\nassistance. We also thank the farmer for taking part of the project\\nas such investigations are voluntary in Germany.\\nSUPPLEMENTARY MATERIAL\\nThe Supplementary Material for this article can be found\\nonline at: https://www.frontiersin.org/articles/10.3389/fmicb.\\n2019.02256/full#supplementary-material\\nREFERENCES\\nAdler, A., Assous, M. V., Paikin, S., Shulman, A., Miller-Roll, T., Hillel, S., et al.\\n(2014). Emergence of VIM-producing aeromonas caviae in israeli hospitals.\\nJ. Antimicrob. Chemother.69, 1211ñ1214. doi: 10.1093/jac/dkt505\\nAguirre-Quiñonero, A., and Martínez-Martínez, L. (2017). Non-molecular\\ndetection of carbapenemases in Enterobacteriaceae clinical isolates. J. Infect.\\nChemother. 23, 1ñ11. doi: 10.1016/j.jiac.2016.09.008\\nAlikhan, N.-F., Petty, N. K., Ben Zakour, N. L., and Beatson, S. A. (2011). BLAST\\nRing Image Generator (BRIG): simple prokaryote genome comparisons. BMC\\nGenom. 12:402. doi: 10.1186/1471-2164-12-402\\nBorowiak, M., Szabo, I., Baumann, B., Junker, E., Hammerl, J. A., Kaesbohrer,\\nA., et al. (2017). VIM-1-producing Salmonella Infantis isolated from swine\\nand minced pork meat in Germany. J. Antimicrob. Chemother.72, 2131ñ2133.\\ndoi: 10.1093/jac/dkx101\\nCantón, R., Akóva, M., Carmeli, Y., Giske, C. G., Glupczynski, Y., Gniadkowski,\\nM., et al. (2012). Rapid evolution and spread of carbapenemases among\\nEnterobacteriaceae in Europe. Clin. Infect. Dis.18, 413ñ431. doi: 10.1111/j.1469-\\n0691.2012.03821.x\\nFalgenhauer, L., Ghosh, H., Guerra, B., Y ao, Y., Fritzenwanker, M., Fischer, J.,\\net al. (2015). Comparative genome analysis of IncHI2 VIM-1 carbapenemase-\\nencoding plasmids of Escherichia coli and Salmonella enterica isolated\\nfrom a livestock farm in Germany. Vet. Microbiol. 200, 114ñ117. doi:\\n10.1016/j.vetmic.2015.09.001\\nFischer, J., Rodríguez, I., Schmoger, S., Friese, A., Roesler, U., Helmuth, R.,\\net al. (2012). Escherichia coli producing VIM-1 carbapenemase isolated on\\na pig farm. J. Antimicrob. Chemother. 67, 1793ñ1795. doi: 10.1093/jac/dk\\ns108\\nFischer, J., Rodríguez, I., Schmoger, S., Friese, A., Roesler, U.,\\nHelmuth, R., et al. (2013). Salmonella enterica subsp. enterica\\nproducing VIM-1 carbapenemase isolated from livestock farms.\\nJ. Antimicrob. Chemother. 68, 478ñ480. doi: 10.1093/jac/dk\\ns393\\nFischer, J., San José, M., Roschanski, N., Schmoger, S., Baumann, B., Irrgang,\\nA., et al. (2017). Spread and persistence of VIM-1 carbapenemase-\\nproducing Enterobacteriaceae in three German swine farms in 2011\\nand 2012. Vet. Microbiol. 200, 118ñ123. doi: 10.1016/j.vetmic.2016.\\n04.026\\nFrontiers in Microbiology | www.frontiersin.org 10 October 2019 | Volume 10 | Article 2256'), Document(metadata={'source': 'texts\\\\Irrgang, VIM_2019.pdf', 'page': 10}, page_content='fmicb-10-02256 September 30, 2019 Time: 16:42 # 11\\nIrrgang et al. VIM-1-Producing E. coli on a Pig Farm\\nGrundmann, H., Glasner, C., Albiger, B., Aanensen, D. M., Tomlinson,\\nC. T., Andrasevi ´c, A. T., et al. (2017). Occurrence of carbapenemase-\\nproducing Klebsiella pneumoniaeand Escherichia coliin the European survey\\nof carbapenemase-producing Enterobacteriaceae (EuSCAPE): a prospective,\\nmultinational study. Lancet Infect. Dis. 17, 153ñ163. doi: 10.1016/S1473-\\n3099(16)30257-2\\nIrrgang, A., Falgenhauer, L., Fischer, J., Ghosh, H., Guiral, E., Guerra, B.,\\net al. (2017a). CTX-M-15-producing E. coli isolates from food products in\\nGermany are mainly associated with an IncF-type plasmid and belong to two\\npredominant clonal E. coli lineages. Front. Microbiol.8:318. doi: 10.3389/fmicb.\\n2017.02318\\nIrrgang, A., Fischer, J., Grobbel, M., Schmoger, S., Skladnikiewicz-Ziemer, T.,\\nThomas, K., et al. (2017b). Recurrent detection of VIM-1-producingEscherichia\\ncoli clone in German pig production. J. Antimicrob. Chemother.72, 944ñ946.\\ndoi: 10.1093/jac/dkw479\\nIrrgang, A., Hammerl, J. A., Falgenhauer, L., Guiral, E., Schmoger, S., Imirzalioglu,\\nC., et al. (2018). Diversity of CTX-M-1-producing E. coli from German food\\nsamples and genetic diversity of the blaCTX-M-1 region on IncI1 ST3 plasmids.\\nVet. Microbiol.221, 98ñ104. doi: 10.1016/j.vetmic.2018.06.003\\nKöck, R., Daniels-Haardt, I., Becker, K., Mellmann, A., Friedrich, A. W., Mevius,\\nD., et al. (2018). Carbapenem-resistant Enterobacteriaceae in wildlife, food-\\nproducing, and companion animals: a systematic review.Clin. Microbiol. Infect.\\n24, 1241ñ1250. doi: 10.1016/j.cmi.2018.04.004\\nLivermore, D. M. (2001). Of Pseudomonas, porins, pumps and carbapenems.\\nJ. Antimicrob. Chemother.47, 247ñ250. doi: 10.1093/jac/47.3.247\\nMiriagou, V., Cornaglia, G., Edelstein, M., Galani, I., Giske, C. G., Gniadkowski,\\nM., et al. (2010). Acquired carbapenemases in gram-negative bacterial\\npathogens: detection and surveillance issues. Clin. Microbiol. Infect.16, 112ñ\\n122. doi: 10.1111/j.1469-0691.2009.03116.x\\nOIE (2015). OIE List of Antimicrobials of Veterinary Importance . Available\\nat: http://www.oie.int/ﬁleadmin/Home/eng/Our_scientiﬁc_expertise/docs/pdf/\\nEng_OIE_List_antimicrobials_May2015.pdf (accessed Septermber 20, 2019).\\nPapagiannitsis, C. C., Miriagou, V., Giakkoupi, P., Tzouvelekis, L. S., and\\nVatopoulos, A. C. (2013). Characterization of pKP1780, a novel IncR plasmid\\nfrom the emerging Klebsiella pneumoniaeST147, encoding the VIM-1 metallo-\\nβ-lactamase. J. Antimicrob. Chemother.68, 2259ñ2262. doi: 10.1093/jac/dkt196\\nPatel, G., and Bonomo, R. A. (2013). “Stormy waters ahead”: global emergence of\\ncarbapenemases. Front. Microbiol.4:48. doi: 10.3389/fmicb.2013.00048\\nPotter, R. F., D’Souza, A. W., and Dantas, G. (2016). The rapid spread of\\ncarbapenem-resistant Enterobacteriaceae. Drug Resist. Updat 29, 30ñ46. doi:\\n10.1016/j.drup.2016.09.002\\nPsichogiou, M., Tassios, P. T., Avlamis, A., Stefanou, I., Kosmidis, C., Platsouka,\\nE., et al. (2008). Ongoing epidemic of blaVIM-1-positive Klebsiella pneumoniae\\nin athens, greece: a prospective survey. J. Antimicrob. Chemother.61, 59ñ63.\\ndoi: 10.1093/jac/dkm443\\nRobert Koch-Institut (2018). Epidemiologisches Bulletin Des Robert Koch-Instituts\\nAusgabe 28/2018. Berlin: Robert Koch-Institut.\\nRoschanski, N., Guenther, S., Vu, T. T. T., Fischer, J., Semmler, T., Huehn, S., et al.\\n(2017). VIM-1 carbapenemase-producing Escherichia coli isolated from retail\\nseafood, Germany 2016. Euro. Surveill. 22, 17ñ32. doi: 10.2807/1560-7917.ES.\\n2017.22.43.17-00032\\nRozwandowicz, M., Brouwer, M. S. M., Fischer, J., Wagenaar, J. A., Gonzalez-Zorn,\\nB., Guerra, B., et al. (2018). Plasmids carrying antimicrobial resistance genes in\\nEnterobacteriaceae. J. Antimicrob. Chemother.73, 1121ñ1137. doi: 10.1093/jac/\\ndkx488\\nvan der Zee, A., Roorda, L., Bosman, G., Fluit, A. C., Hermans, M., Smits, P. H. M.,\\net al. (2014). Multi-centre evaluation of real-time multiplex PCR for detection\\nof carbapenemase genes OXA-48, VIM, IMP , NDM and KPC.BMC Infect. Dis.\\n14:27. doi: 10.1186/1471-2334-14-27\\nWHO Advisory Group on Integrated Surveillance of Antimicrobial Resistance\\n(2017). Critically Important Antimicrobials for Human Medicine: Ranking of\\nAntimicrobial Agents for Risk Management of Antimicrobial Resistance Due to\\nNon-Human Use. Geneva: World Health Organization.\\nWoodford, N., Wareham, D. W., Guerra, B., and Teale, C. (2014). Carbapenemase-\\nproducing Enterobacteriaceae and non- Enterobacteriaceae from animals and\\nthe environment: an emerging public health risk of our own making?\\nJ. Antimicrob. Chemother.69, 287ñ291. doi: 10.1093/jac/dkt392\\nXie, M., Li, R., Liu, Z., Chan, E. W. C., and Chen, S. (2018). Recombination of\\nplasmids in a carbapenem-resistant NDM-5-producing clinical Escherichia coli\\nisolate. J. Antimicrob. Chemother.73, 1230ñ1234. doi: 10.1093/jac/dkx540\\nConﬂict of Interest: The authors declare that the research was conducted in the\\nabsence of any commercial or ﬁnancial relationships that could be construed as a\\npotential conﬂict of interest.\\nCopyright © 2019 Irrgang, Tenhagen, Pauly, Schmoger, Kaesbohrer and Hammerl.\\nThis is an open-access article distributed under the terms of the Creative Commons\\nAttribution License (CC BY). The use, distribution or reproduction in other forums\\nis permitted, provided the original author(s) and the copyright owner(s) are credited\\nand that the original publication in this journal is cited, in accordance with accepted\\nacademic practice. No use, distribution or reproduction is permitted which does not\\ncomply with these terms.\\nFrontiers in Microbiology | www.frontiersin.org 11 October 2019 | Volume 10 | Article 2256')]\n",
      "\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Content of Pauly, 19-AB01133.pdf:\n",
      "[Document(metadata={'source': 'texts\\\\Pauly, 19-AB01133.pdf', 'page': 0}, page_content='microorganisms\\nCommunication\\nIdentiﬁcation of a blaVIM-1-Carrying IncA/C2 Multiresistance\\nPlasmid in an Escherichia coliIsolate Recovered from the\\nGerman Food Chain\\nNatalie Pauly 1, *, Jens Andre Hammerl 1\\n , Mirjam Grobbel 1\\n , Annemarie Käsbohrer 1,2 ,\\nBernd-Alois Tenhagen 1\\n , Burkhard Malorny 1, Stefan Schwarz 3\\n , Diana Meemken 4\\n and\\nAlexandra Irrgang 1, *\\n/gid00030/gid00035/gid00032/gid00030/gid00038/gid00001/gid00033/gid00042/gid00045/gid00001\\n/gid00048/gid00043/gid00031/gid00028/gid00047/gid00032/gid00046\\nCitation: Pauly, N.; Hammerl, J.A.;\\nGrobbel, M.; Käsbohrer, A.; Tenhagen,\\nB.-A.; Malorny , B.; Schwarz, S.;\\nMeemken, D.; Irrgang, A. Identification\\nof a blaVIM-1-Carrying IncA/C 2\\nMultiresistance Plasmid in anEscherichia\\ncoliIsolate Recovered from the German\\nFood Chain. Microorganisms 2021, 9,\\n29. https://dx.doi.org/10.3390/\\nmicroorganisms9010029\\nReceived: 19 November 2020\\nAccepted: 21 December 2020\\nPublished: 24 December 2020\\nPublisher’s Note: MDPI stays neu-\\ntral with regard to jurisdictional claims\\nin published maps and institutional\\nafﬁliations.\\nCopyright: © 2020 by the authors. Li-\\ncensee MDPI, Basel, Switzerland. This\\narticle is an open access article distributed\\nunder the terms and conditions of the\\nCreative Commons Attribution (CC BY)\\nlicense (https://creativecommons.org/\\nlicenses/by/4.0/).\\n1 Department Biological Safety, German Federal Institute for Risk Assessment, 10589 Berlin, Germany;\\nJens-Andre.Hammerl@bfr.bund.de (J.A.H.); mirjam.grobbel@bfr.bund.de (M.G.);\\nannemarie.kaesbohrer@bfr.bund.de (A.K.); Bernd-Alois.Tenhagen@bfr.bund.de (B.-A.T.);\\nBurkhard.Malorny@bfr.bund.de (B.M.)\\n2 Institute for Veterinary Public Health, University of Veterinary Medicine, 1210 Vienna, Austria\\n3 Institute of Microbiology and Epizootics, Freie Universität Berlin, 14163 Berlin, Germany;\\nstefan.schwarz@fu-berlin.de\\n4 Institute of Food Safety and Food Hygiene, Freie Universität Berlin, 14163 Berlin, Germany;\\nDiana.Meemken@fu-berlin.de\\n* Correspondence: natalie_pauly@hotmail.com (N.P .); alexandra.irrgang@bfr.bund.de (A.I.)\\nAbstract: Within the German national monitoring of zoonotic agents, antimicrobial resistance deter-\\nmination also targets carbapenemase-producing (CP) Escherichia coliby selective isolation from food\\nand livestock. In this monitoring in 2019, the CPE. coli19-AB01133 was recovered from pork shoulder.\\nThe isolate was assigned to the phylogenetic group B1 and exhibited the multi-locus sequence-type\\nST5869. Molecular investigations, including whole genome sequencing, of 19-AB01133 revealed\\nthat the isolate carried the resistance genes blaVIM-1, blaSHV-5 and blaCMY-13 on a self-transmissible\\nIncA/C2 plasmid. The plasmid was closely related to the previously described VIM-1-encoding plas-\\nmid S15FP06257_p from E. coliof pork origin in Belgium. Our results indicate an occasional spread\\nof the blaVIM-1 gene in Enterobacteriaceae of the European pig population. Moreover, the blaVIM-1\\nlocated on an IncA/C 2 plasmid supports the presumption of a new, probably human source of\\ncarbapenemase-producing Enterobacteriaceae (CPE) entering the livestock and food chain sector.\\nKeywords: carbapenem; metallo-β-lactamase; IncA/C2; WGS; blaVIM-1\\n1. Introduction\\nCarbapenems are last resort antimicrobial agents against several infections caused by\\nmultidrug-resistant bacteria in human medicine. The World Health Organization deﬁnes\\nthem as “Highest Priority Critically Important Antimicrobials”. Nevertheless, the reports\\nof carbapenem resistant bacteria are not limited to human medicine [ 1]. The repeated\\ndetection of carbapenemase-producing (CP) Enterobacteriaceae (CPE) in the German food-\\nproduction chain is of considerable concern [ 1,2]. In general, CPE have been isolated\\nsporadically from non-human sources (i.e., livestock, pets, wildlife) although carbapenems\\nare not approved for veterinary application [2]. One of the main mechanisms of carbapenem\\nresistance is the production of degrading enzymes, so called carbapenemases, which can\\nhydrolyze almost all available β-lactams, including carbapenems [3]. The corresponding\\ngenes are often located on mobile genetic elements, in particular plasmids, which often carry\\nadditional antimicrobial, biocide, or heavy metal resistance genes [4]. These plasmids can be\\nhorizontally spread to bacteria under the selective pressure also imposed by non-β-lactam\\nagents [5,6]. Since 2011, CPE isolates, which harbored VIM-1 encoding IncHI2 plasmids,\\nwere detected in livestock and food [7–12]. The ﬁrst report described Salmonella Infantis and\\nMicroorganisms 2021, 9, 29. https://dx.doi.org/10.3390/microorganisms9010029 https://www.mdpi.com/journal/microorganisms'), Document(metadata={'source': 'texts\\\\Pauly, 19-AB01133.pdf', 'page': 1}, page_content='Microorganisms 2021, 9, 29 2 of 7\\nE. coliisolates detected in 2011 from fattening pigs in Germany [10]. They could persist in\\nthe animal host and closely related isolates could be detected in fattening pigs and minced\\nmeat in the following years [7,8]. Another VIM-1 report described the detection of this gene\\non an IncY plasmids inE. colioriginated from seafood samples in Germany [13]. Besides the\\nincreasing reports of VIM-1 producing Enterobacteriaceae in the German food chain,\\nthe variants of carbapenemase genes increased as well. In 2019 within the annual resistance\\nmonitoring in the food chain in Germany, three CPE. coliwere isolated. These include an\\nOXA-48 producing E. coli[14] and a GES-5 producing E. coli[15], both isolated from pig\\nfeces, and another VIM-1 producing E. colifrom a pork sample. This blaVIM-1 harboring\\nE. coliisolate 19-AB01133 and its comprehensive molecular characteristics are described in\\nthis study. It shows great similarities to an E. coliisolate described by Garcia-Graells et al.\\n(2020) [16] that was isolated from a pork sample in Belgium. Both isolates harbored almost\\nidentical IncA/C2 plasmids, which encode the blaVIM-1 gene. Furthermore, the genome\\ndata hint to a speciﬁc human entry source.\\n2. Materials and Methods\\n2.1. Isolate Origin and Antimicrobial Susceptibility Testing\\nThe E. coliisolate 19-AB01133 was obtained in 2019 in the framework of the annual\\nresistance monitoring in the food chain in Germany. The selective isolation of the CP\\nE. coliwas performed at a Federal State Laboratory, following Commission Implementing\\nDecision (CID) 2013/652/EC and protocols provided by the European Union Reference Lab-\\noratory for Antimicrobial Resistance (EURL-AR) (https://www.eurl-ar.eu/protocols.aspx).\\nThe E. coliisolate 19-AB01133 was recovered from a shoulder meat sample of a pig raised\\nin Germany. The pig was fattened and slaughtered in Brandenburg. The isolate was\\nsent to the German Federal Institute for Risk Assessment for further conﬁrmation and\\nphenotypic and genotypic characterization. Antimicrobial susceptibility testing according\\nto the EN ISO20776-1:2006 was conducted by broth microdilution using commercial plates\\n(Thermo Fisher Scientiﬁc, Schwerte, Germany) with antimicrobial substances and con-\\ncentrations deﬁned by CID 2013/652/EC. The minimum inhibitory concentration (MIC)\\nvalues were interpreted based on epidemiological cut-off values deﬁned by the European\\nCommittee on Antimicrobial Susceptibility Testing (EUCAST) (www.eucast.org) and ﬁxed\\nin CID 2013/652/EU.\\n2.2. Phenotypic and Genotypic Characterization\\nSpecies conﬁrmation of the isolate was conducted by MALDI-TOF Microﬂex LT/SH\\n(Bruker Daltonics, Bremen, Germany) according to the manufacturer’s recommenda-\\ntions. Therefore, α-Cyano-4-hydroxycinnamic acid (HCCA, Bruker, MA, USA) was used\\nas the matrix. Molecular determination of the carbapenem-resistance genes and ini-\\ntial phylogenetic typing were performed as previously described [ 14]. The E. coli 19-\\nAB01133 was subjected to S1-nuclease pulsed-ﬁeld gel electrophoresis (S1-PFGE) [ 17]\\n(https://www.cdc.gov/pulsenet/pathogens/protocols.html) followed by Southern blot\\nhybridization [18] against a digoxigenin-labelled blaVIM-1 probe while using DIG EasyHyb\\nand DIG Wash and Block Buffer Set (Roche Diagnostics, Mannheim, Germany) according to\\nthe manufacturer’s recommendations. The transmissibility of the plasmid was investigated\\nby in vitro ﬁlter-mating studies using E. coliK12 J53 as the recipient. Moreover, the plasmid\\npEC19-AB01133 was extracted by using the CosMCPrep Plasmid Puriﬁcation Kit (Beckman\\nCoulter, Krefeld, Germany) according to the manufacturer’s recommendations. The plas-\\nmids were transformed into highly competent E. coliDH10B (ElectroMAXTM DH10BTM\\nCells; InvitrogenTM, Thermo Fisher Scientiﬁc, Schwerte, Germany) by electroporation\\nusing disposable cuvettes with 1 mm gap and, 1.8 kV (E = 18 kV/cm) in a Bio-Rad Mi-\\ncroPulser (Bio-Rad Laboratories, Feldkirchen, Germany). Potential transconjugants and\\ntransformants were analyzed for their plasmid content and resistance phenotype. TheE. coli\\nisolate 19-AB01133 was subjected to short-read (MiSeq, Illumina, CA, USA) and long-read\\nwhole genome sequencing (MinIon; Oxford Nanopore, Oxford, UK) followed by a hybrid'), Document(metadata={'source': 'texts\\\\Pauly, 19-AB01133.pdf', 'page': 2}, page_content='Microorganisms 2021, 9, 29 3 of 7\\nassembly of the obtained sequences using Unicycler v0.4.4 under default parameters [19].\\nBased on this assembly, the multi-locus sequence-type (MLST) as well as resistance and\\nvirulence genes were determined using online tools that were provided by the Danish\\nTechnical University (http://www.genomicepidemiology.org). The annotation was carried\\nout by RAST2 provided by PATRIC (www.patricbrc.org). The assembly of the plasmid\\npEC19-AB01133 was deposited in GenBank (NCBI) under the accession number MT682138.\\n2.3. Farm Investigation\\nThree months after detecting the isolate 19-AB01133, the farm and the corresponding\\nslaughterhouse were re-investigated. Therefore, composite fecal samples (n = 8) were taken\\nfrom all pens. Moreover, ten environmental samples were taken in the farm. These samples\\ninclude cobwebs, dust, water trough, wet residues and sock swabs. In the slaughterhouse,\\nadditional 17 samples were taken, including minced and sausage meat, fat, a smear of work\\nsurfaces and drains, curing water, hand washing water and more. These samples were analyzed\\naccording to the EURL method combined with a second enrichment step in lysogeny broth\\n(LB) supplemented with 1 mg/L cefotaxime (CTX) and in LB supplemented with 0.125 mg/L\\nmeropenem (MEM). Afterwards, 10µL of all enrichments (BPW, LB + CTX and LB + MEM)\\nwere streaked out on self-made selective agar(McConkey agar (McC) supplemented with\\n0.125 mg/L MEM and McC supplemented with 0.125 mg/L MEM and 1 mg/L CTX) and\\non chromID® CARBA SMART (bioMérieux, Nürtingen, Germany). Plates were incubated\\nfor 16–18 h at 37 ±2 ◦C. Up to ten colonies with different morphologies were picked and\\nfurther analyzed by real-time PCR to conﬁrm the presence or absence of blaGES, blaKPC,\\nblaNDM, blaOXA-48 and blaVIM [14] and by MALDI-ToF MS for species conﬁrmation of\\npresumptive CPE.\\n3. Results and Discussion\\nThe phenotypic analysis of E. coli19-AB01133 indicated a non-wildtype phenotype for\\nthe tested β-lactam antimicrobial agents, including penicillins (ampicillin MIC > 64 mg/L),\\ncephalosporins (cefotaxime MIC 64 mg/L, ceftazidime MIC > 128 mg/L, cefepime\\nMIC > 32 mg/L, cefoxitin MIC > 64 mg/L, ceftazidime MIC > 8 mg/L) and carbapenems\\n(imipenem MIC 8 mg/L, ertapenem MIC > 2 mg/L and meropenem MIC 8 mg/L), and also\\nfor (ﬂuoro)quinolones (ciproﬂoxacin MIC > 8 mg/L, nalidixic acid MIC > 128 mg/L)\\nand aminoglycosides (gentamicin MIC > 32 mg/L) (Table 1). Further typing assigned\\nthe isolate to the phylogenetic group B1. This group is often associated with enhanced\\nantimicrobial resistance but low virulence potential [ 20,21]. The virulence potential of\\nthe E. coli 19-AB01133 is composed of the genes iss, ifpA and gad, which code for an\\nincreased serum survival, for long polar ﬁmbriae and for a glutamate decarboxylase,\\nrespectively. The S1-PFGE with subsequent Southern blot hybridization and in vitro ﬁlter-\\nmating and transformation studies identiﬁed a single, conjugative ~190 kb plasmid carrying\\nthe blaVIM-1 gene.\\nThe bioinformatic analysis revealed assigned E. coli19-AB01133 to the sequence type\\nST5869, which has not been described before from livestock and food in Germany. The plas-\\nmid pEC19-AB01133 (190,205 bp) exhibited an IncA/C2 replicon type. The characteristics\\nof the presented isolate 19-AB01133 and its plasmid pEC19-AB01133 are summarized in the\\nSupplemental Material Table S1. The encoded blaVIM-1 gene is highly likely to be responsi-\\nble for the resistance to the tested carbapenems (imipenem, ertapenem and meropenem)\\nin the E. coliisolate 19-AB01133. To the best of our knowledge, blaVIM-1-carrying IncA/C2\\nplasmids were detected sporadically in human samples [22,23], but so far only once in live-\\nstock and food in Europe [16]. Recently, Garcia-Grealls and her colleagues (2020) described\\nan almost identical VIM-1 encoding plasmid (99.99% identical over the entire plasmid\\nnucleotide sequence, Acc. No. PRJNA564835) of E. coli(Acc. No. MN477204.1) isolated\\nfrom minced pork in Belgium in 2015 [16]. The corresponding isolate was assigned to the\\nsame sequence type, which might indicate a clonal relationship. This observation leads to\\nthe question of repeated contamination of pigs and pork with this strain, or to a persistence'), Document(metadata={'source': 'texts\\\\Pauly, 19-AB01133.pdf', 'page': 3}, page_content='Microorganisms 2021, 9, 29 4 of 7\\nof this strain in Central European pig production. The E. coliisolate 19-AB01133 further\\nshowed a close relationship to the E. coliisolate TZ 116, which is used by the EURL-AR as\\nsecond (backup) strain for the validation of selective plates for the detection of CP E. coli\\n(https://www.eurl-ar.eu/protocols.aspx). This backup strain was isolated from a human\\nsample. According to personal communication, the strain was not used in the isolating\\nlaboratory at that time or in our lab and thus, a cross-contamination can be excluded.\\nApart from this control strain, some moreblaVIM-1 harboring IncA/C2 plasmids of isolates\\nfrom human origin showed a close relationship to the described plasmid pEC19-AB01133.\\nFor example, Drieux et al. (2012) [ 23] described a plasmid (Acc. No. JQ824049) from\\na Greek Providencia stuartii multiresistant strain. By comparing this plasmid with the\\nplasmid pEC19-AB01133, a query coverage of 87% was observed. Further, plasmids from\\nEnterobacteriaceae of clinical and environmental samples from Canada showed also high\\nsimilarities [24] to the described plasmid pEC17-AB01133. The plasmid pIncAC-KP4898\\n(Acc. No. KY882285.1) and the plasmid pRIVM0001_VIM-1 (Acc. No. MH220284.1) had a\\nquery coverage of 70% and the plasmid pKPC_CAV1344 (Acc. No. CP011622.1) a query\\ncoverage of 81%. Taken together, the close relationship and the repeated detection of the\\nstrain in food chain samples hints to a spill-over of the strain from humans to food.\\nThe plasmid pEC19-AB01133 harbored a variety of antimicrobial resistance genes, which\\nare located within a ~60 kb multiresistance region (Figure 1). TheblaVIM-1 gene is located\\non a class 1 integron. In addition, the displayed region exhibited three further interesting\\nsegments. The first segment comprised two mercury resistance operons (Figure 1), which con-\\nfer narrow-spectrum resistance to inorganic mercury [ 25]. Two tnpA genes encompass\\nthe 5768 bp segment 2. This DNA-segment, originally identiﬁed in Citrobacter freundii\\n(Acc. No. AY339625.2), encodes the small multidrug resistance efﬂux transporter SugE\\nand the AmpC β-lactamase CMY-13 [26]. To date, the occurrence of blaCMY-13 has only\\nsporadically been reported [26]. Another neighboring class 1 integron conferred resistance\\nto aminoglycosides [aac(6′)-Il, aac(3)-l, ant(3”)-Ia] and to sulfonamides (sul1). Downstream\\nof this class 1 integron, the third segment was integrated. This segment carried the ESBL\\nβ-lactamase gene blaSHV-5. The blaSHV-5 gene has been previously reported from isolates of\\nE. coli, Klebsiella pneumoniaeand Providencia stuartiifrom humans, livestock and wildlife [27],\\nbut not yet in animals or food in Germany.\\nThe farm and the corresponding slaughter facility were re-investigated three months\\nlater. Thereby, 35 samples from feces, farm and slaughter facility environment and different\\nmeat products were taken. No CPE were detected in these samples. This suggests that\\nthe strain was unable to persist on the farm and spread of the VIM-1 plasmid did not\\noccur. Nevertheless, the detection of an isolate with such high similarities to the Belgian\\nisolates might hint to the ability for it to persist within the pork production chain [ 15].\\nA re-introduction by human, i.e., by farmers or workers, is also possible as there was no\\nobvious link between the two fattening pig farms.\\nThe detection of multiresistance IncA/C2 plasmids identiﬁed in E. coliisolates from\\nfood chain is a threat to public health. Previous studies from Germany described the\\nlocation of blaVIM-1 only on IncHI2 or IncY plasmids. However, for IncA/C 2 plasmids,\\na high impact on the dissemination of antimicrobial resistance genes has been reported [20].\\nThe detection of new plasmid types and the recurrence of carbapenemase-producing\\nisolates in the German pork production chain emphasizes the importance of the CPE\\nmonitoring [4,8,9]. The occurrence of the blaVIM-1 located on an IncA/C2 plasmid in an\\nE. coli isolate from pork suggests a new, probably human source of CPE entering the\\nfood chain.'), Document(metadata={'source': 'texts\\\\Pauly, 19-AB01133.pdf', 'page': 4}, page_content='Microorganisms 2021, 9, 29 5 of 7\\nTable 1. Minimal inhibitory concentrations (MIC) of following antimicrobial agents (mg/L) to 19-AB01133and its transkonjugant TK_19AB01133. MIC values classiﬁed as non-wildtype are\\ncolored in red. Ampicillin (AMP); Azithromycin (AZI); Cefepime (FEP); Chloramphenicol (CHL); Ciproﬂoxacin (CIP); Colistin (COL); Ertapenem (ERP); Cefotaxime (FOT); Cefoxitin (FOX);\\nGentamicin (GEN); Imipenem (IMI); Kanamycin (KAN); Meropenem (MERO); Nalidixic acid (NAL); Sulfamethoxazole (SMX); Ceftazidime (TAZ); Tetracycline (TET); Tigecycline (TGC);\\nTrimethophrim (TMP).\\nIsolate AMP AZI CHL CIP COL ERP FEP FOT FOX GEN IMI MERO NAL SMX TAZ TET TGC TMP\\n19-AB01133 >64 16 16 >8 ≤1 >2 >32 >64 >64 32 8 8 >128 >1024 >8 4 ≤0.25 >32\\nTK_19-AB01133 >64 8 8 ≤0.015 ≤1 >2 >32 >64 >64 4 8 4 ≤4 >1024 >8 4 ≤0.25 >32\\nK12 J53 4 8 ≤8 ≤0.015 ≤1 ≤0.015 ≤0.06 ≤0.25 4 2 0.25 ≤0.03 ≤4 ≤8 ≤0.5 4 0.5 ≤0.25'), Document(metadata={'source': 'texts\\\\Pauly, 19-AB01133.pdf', 'page': 5}, page_content='Microorganisms 2021, 9, 29 6 of 7\\nMicroorganisms 2020, 8, x FOR PEER REVIEW 6 of 9 \\n \\nApart from this control strain, some more blaVIM-1 harboring \\nIncA/C2 plasmids of isolates from human origin showed a close rela-\\ntionship to the described plasmid pEC19 -AB01133. For example, \\nDrieux et al. (2012) [23] described a plasmid (Acc. No. JQ824049) from \\na Greek Providencia stuartii  multiresistant st rain. By comparing this \\nplasmid with the plasmid pEC19 -AB01133, a query coverage of 87 % \\nwas observed. Further, plasmids from Enterobacteriaceae  of clinical \\nand environmental samples from Canada showed also high similarities \\n[24] to the described plasmid pEC17 -AB01133. The plasmid pIncAC -\\nKP4898 (Acc. No. KY882285.1) and the plasmid pRIVM0001_VIM -1 \\n(Acc. No. MH220284.1) had a query coverage of 70 % and the plasmid \\npKPC_CAV1344 (Acc. No. CP011622.1) a query coverage of 81%. Taken \\ntogether, the close relationship and the repeated detection of the strain \\nin food chain samples hints to a spill-over of the strain from humans to \\nfood.  \\nThe plasmid pEC19-AB01133 harbored a variety of antimicrobial \\nresistance genes, which are located within a ~60 kb multiresistance re-\\ngion (Figure 1). The blaVIM-1 gene is located on a class 1 integron. In ad-\\ndition, the displayed region exhibited three furth er interesting seg-\\nments. The first segment comprised two mercury resistance operons \\n(Figure 1), which confer narrow-spectrum resistance to inorganic mer-\\ncury [25]. Two tnpA genes encompass the 5768 bp segment 2. This \\nDNA-segment, originally identified in Citrobacter freundii  (Acc. No. \\nAY339625.2), encodes the small multidrug resistance efflux transporter \\nSugE and the AmpC β-lactamase CMY-13 [26]. To date, the occurrence \\nof blaCMY-13 has only sporadically been reported [26]. Another neighbor-\\ning class 1 integron conferred resistance to aminoglycosides [aac(6′)-Il, \\naac(3)-l, ant(3”)-Ia] and to sulfonamides (sul1). Downstream of this class \\n1 integron, the third segment was integrated. This segment carried the \\nESBL β-lactamase gene blaSHV-5. The blaSHV-5 gene has been previously \\nreported from isolates of E. coli, Klebsiella pneumoniae and Providencia \\nstuartii from humans, livestock and wildlife [27], but not yet in animals \\nor food in Germany.  \\n \\nFigure 1. Schematic illustration of the multidrug -resistance region of plasmid pEC19 -AB01133. The \\nreading frames are displayed as arrows with the arrowhead showing the direction of transcription. \\nThe numbers refer to the whole plasmid sequence of pEC19-AB01133, which is deposited in the Gen-\\nBank database under accession no. MT682138. \\nThe farm and the corresponding slaughter facility were re-investi-\\ngated three months later. Thereby, 35 samples from feces, farm and \\nslaughter facility environment and different meat products were taken. \\nNo CPE were detected in these samples. This suggests that the strain \\nwas unable to persist on the farm and spread of the VIM-1 plasmid did \\nFigure 1. Schematic illustration of the multidrug-resistance region of plasmid pEC19-AB01133. The reading frames are\\ndisplayed as arrows with the arrowhead showing the direction of transcription. The numbers refer to the whole plasmid\\nsequence of pEC19-AB01133, which is deposited in the GenBank database under accession no. MT682138.\\nSupplementary Materials: The following are available online at https://www.mdpi.com/2076-2607/9/\\n1/29/s1. Table S1: General information about the isolate 19-AB01133 and its plasmid pEC19-AB01133.\\nAuthor Contributions: Data curation, N.P .; formal analysis, S.S.; investigation, N.P ., J.A.H., M.G.,\\nB.-A.T. and A.I.; project administration, A.I.; visualization, N.P .; writing—original draft, N.P ., J.A.H.\\nand A.I.; writing—review and editing, J.A.H., M.G., A.K., B.-A.T., B.M., S.S., D.M. and A.I. All authors\\nhave read and agreed to the published version of the manuscript.\\nFunding: This work was supported by the German Federal Institute for Risk Assessment (BfR, 43-001\\nand 1322-648) and the European Joint Project (EJP) IMPART funded by the European Union’s Horizon\\n2020 research and innovation programme under Grant Agreement No 773830. The work of JAH\\nwas supported by the BMG project GÜCCI. The work of SS was supported by the German Federal\\nMinistry of Education and Research (BMBF) under project number 01KI1727D as part of the Research\\nNetwork Zoonotic Infectious Diseases.\\nData Availability Statement: Data is contained within the article and supplementary material.\\nMoreover, the plasmid sequence was deposited in GenBank (NCBI) under the accession number\\nMT682138.\\nAcknowledgments: We gratefully acknowledge the support of the regional laboratories and authori-\\nties in collecting the samples and providing the isolates in the framework of the monitoring. We also\\nthank the laboratory team, especially Romy Fuhrmeister and Silvia Schmoger, for excellent technical\\nassistance.\\nConﬂicts of Interest: The authors declare no conﬂict of interest.\\nReferences\\n1. WHO; Advisory Group on Integrated Surveillance of Antimicrobial Resistance (AGISAR). Critically Important Antimicrobials for\\nHuman Medicine: Ranking of Antimicrobial Agents for Risk Management of Antimicrobial Resistance due to Non-Human Use, 5th ed.;\\nWorld Health Organization: Geneva, Switzerland, 2017.\\n2. Köck, R.; Daniels-Haardt, I.; Becker, K.; Mellmann, A.; Friedrich, A.W.; Mevius, D.; Schwarz, S.; Jurke, A. carbapenem-resistant\\nenterobacteriaceae in wildlife, food-producing, and companion animals: A systematic review. Clin. Microbiol. Infect.2018, 24,\\n1241–1250. [CrossRef] [PubMed]\\n3. Salahuddin, P .; Kumar, A.; Kha, A.U. Structure, function of serine and Metallo-β-lactamases and their Inhibitors. Curr. Protein.\\nPept. Sci.2018, 19, 130–144. [CrossRef] [PubMed]\\n4. Bush, K.; Fisher, J.F. Epidemiological expansion, structural studies, and clinical challenges of new β-lactamases from gram-\\nnegative bacteria. Annu. Rev. Microbiol.2011, 65, 455–478. [CrossRef]\\n5. Partridge, S.R.; Kwong, S.M.; Firth, N.; Jensen, S.O. Mobile genetic elements associated with antimicrobial resistance.Clin. Microbial. Rev.\\n2018, 31, e00088-17. [CrossRef] [PubMed]\\n6. Potter, R.F.; D′Souza, A.W.; Dantas, G. The rapid spread of carbapenem-resistant enterobacteriaceae. Drug Resist. Updat.2016, 29,\\n30–46. [CrossRef]'), Document(metadata={'source': 'texts\\\\Pauly, 19-AB01133.pdf', 'page': 6}, page_content='Microorganisms 2021, 9, 29 7 of 7\\n7. Irrgang, A.; Fischer, J.; Grobbel, M.; Schmoger, S.; Skladnikiewicz-Ziemer, T.; Thomas, K.; Hensel, A.; Tenhagen, B.-A.; Kasbohrer,\\nA.; Käsbohrer, A. Recurrent detection of VIM-1-producingEscherichia coliclone in German pig production. J. Antimicrob. Chemother.\\n2017, 72, 944–946.\\n8. Borowiak, M.; Szabo, I.; Baumann, B.; Junker, E.; Hammerl, J.A.; Kaesbohrer, A.; Malorny, B.; Fischer, J. VIM-1-producing\\nSalmonella infantis isolated from swine and minced pork meat in Germany. J. Antimicrob. Chemother. 2017, 72, 2131–2133.\\n[CrossRef]\\n9. Fischer, J.; Rodriguez, I.; Schmoger, S.; Friese, A.; Roesler, U.; Helmuth, R.; Guerra, B.Salmonella enterica subsp. enterica producing\\nVIM-1 carbapenemase isolated from livestock farms. J. Antimicrob. Chemother.2013, 68, 478–480. [CrossRef]\\n10. Fischer, J.; San Jose, M.; Roschanski, N.; Schmoger, S.; Baumann, B.; Irrgang, A.; Friese, A.; Roesler, U.; Helmuth, R.; Guerra, B.\\nSpread and persistence of VIM-1 carbapenemase-producing enterobacteriaceae in three German swine farms in 2011 and 2012.\\nVet. Microbiol.2017, 200, 118–123. [CrossRef]\\n11. Irrgang, A.; Tenhagen, B.-A.; Pauly, N.; Schmoger, S.; Kaesbohrer, A.; Hammerl, J.A. Characterization of VIM-1-producing,\\nE. coliisolated from a German fattening pig farm by an improved isolation procedure. Front. Microbiol.2019, 10, 2256. [CrossRef]\\n12. Pauly, N.; Hammerl, J.A.; Schwarz, S.; Grobbel, M.; Meemken, D.; Malorny, B.; Tenhagen, B.-A.; Käsbohrer, A.; Irrgang, A.\\nCo-occurrence of the blaVIM-1 and blaSHV-12 genes on an IncHI2 plasmid of an Escherichia coliisolate recovered from German\\nlivestock. J. Antimicrob. Chemother.2020. [CrossRef] [PubMed]\\n13. Roschanski, N.; Guenther, S.; Vu, T.T.T.; Fischer, J.; Semmler, T.; Huehn, S.; Alter, T.; Roesler, U. VIM-1 carbapenemase-producing\\nEscherichia coliisolated from retail seafood, Germany 2016. Eurosurveillance 2017, 22, 17-00032. [CrossRef] [PubMed]\\n14. Irrgang, A.; Pauly, N.; Tenhagen, B.-A.; Grobbel, M.; Kaesbohrer, A.; Hammerl, A.J. A spill-over from public health? First detection\\nof an OXA-48-producing Escherichia coliin a German pig farm. Microorganisms 2020, 8, 855. [CrossRef] [PubMed]\\n15. Irrgang, A.; Tausch, S.H.; Pauly , N.; Grobbel, M.; Kaesbohrer, A.; Hammerl, J.A. First detection of GES-5-producing escherichia coli from\\nlivestock-an increasing diversity of carbapenemases recognized from german pig production.Microorganisms2020, 8, 1593. [CrossRef]\\n16. Garcia-Graells, C.; Berbers, B.; Verhaegen, B.; Vanneste, K.; Marchal, K.; Roosens, N.H.C.; Botteldoorn, N.; De Keersmaecker,\\nS.C.J. First detection of a plasmid located carbapenem resistant bla(VIM-1) gene in E. coliisolated from meat products at retail in\\nBelgium in 2015. Int. J. Food Microbiol.2020, 324, 108624. [CrossRef]\\n17. Guerra, B.; Junker, E.; Miko, A.; Helmuth, R.; Mendoza, M.C. Characterization and localization of drug resistance determinants in\\nmultidrug-resistant, integron-carrying Salmonella enterica serotype typhimurium strains. Microb. Drug Resist.2004, 10, 83–91.\\n[CrossRef]\\n18. Rodriguez, I.; Barownick, W.; Helmuth, R.; Mendoza, M.C.; Rodicio, M.R.; Schroeter, A.; Guerra, B. Extended-spectrum {beta}-\\nlactamases and AmpC {beta}-lactamases in ceftiofur-resistant Salmonella enterica isolates from food and livestock obtained in\\nGermany during 2003-07. J. Antimicrob. Chemother.2009, 64, 301–309. [CrossRef]\\n19. Hadziabdic, S.; Fischer, J.; Borowiak, M.; Malorny, B.; Juraschek, K.; Kaesbohrer, A.; Guerra, B.; Deneke, C.; Gonzalez-\\nZorn, B.; Szabo, I. The blaNDM-1-carrying IncA/C2 plasmid underlies structural alterations and cointegrate formation in vivo.\\nAntimicrob. Agents Chemother.2019, 63. [CrossRef]\\n20. Harmer, C.J.; Hall, R.M. The A to Z of A/C plasmids. Plasmid 2015, 80, 63–82. [CrossRef]\\n21. Johnson, J.R.; Kuskowski, M.A.; Owens, K.; Gajewski, A.; Winokur, P .L. Phylogenetic origin and virulence genotype in relation to\\nresistance to ﬂuoroquinolones and/or extended-spectrum cephalosporins and cephamycins among Escherichia coliisolates from\\nanimals and humans. J. Infect. Dis.2003, 188, 759–768. [CrossRef]\\n22. Antonelli, A.; D’Andrea, M.M.; Montagnani, C.; Bartolesi, A.M.; Di Pilato, V .; Fiorini, P .; Torricelli, F.; Galli, L.; Rossolini, G.M.\\nNewborn bacteraemia caused by an aeromonas caviae producing the VIM-1 and SHV-12 beta-lactamases, encoded by a transfer-\\nable plasmid. J. Antimicrob. Chemother.2016, 71, 272–274. [CrossRef] [PubMed]\\n23. Drieux, L.; Decré, D.; Frangeul, L.; Arlet, G.; Jarlier, V .; Sougakoff, W. Complete nucleotide sequence of the large conjugative pTC2\\nmultireplicon plasmid encoding the VIM-1 metallo-β-lactamase. J. Antimicrob. Chemother.2013, 68, 97–100. [CrossRef] [PubMed]\\n24. Kohler, P .; Tijet, N.; Kim, H.C.; Johnstone, J.; Edge, T.; Patel, S.N.; Seah, C.; Willey, B.; Coleman, B.; Green, K.; et al. The Toronto\\ninvasive bacterial diseases network. Dissemination of Verona integron-encoded Metallo- β-lactamase among clinical and\\nenvironmental enterobacteriaceae isolates in Ontario, Canada. Sci. Rep.2020, 10, 18580. [CrossRef] [PubMed]\\n25. Osborn, A.M.; Bruce, K.D.; Strike, P .; Ritchie, D.A. Distribution, diversity and evolution of the bacterial mercury resistance (mer)\\noperon. FEMS Microbiol. Rev.1997, 19, 239–262. [CrossRef]\\n26. Miriagou, V .; Tzouvelekis, L.S.; Villa, L.; Lebessi, E.; Vatopoulos, A.C.; Carattoli, A.; Tzelepi, E. CMY-13, a novel inducible\\ncephalosporinase encoded by an Escherichia coliplasmid. Antimicrob. Agents Chemother.2004, 48, 3172–3174. [CrossRef]\\n27. Liakopoulos, A.; Mevius, D.; Ceccarelli, D. A review of SHV extended-spectrum β-lactamases: Neglected yet ubiquitous.\\nFront. Microbial.2016, 7, 1374. [CrossRef]')]\n",
      "\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Content of Pauly, Carba plate validation, 2020.pdf:\n",
      "[Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 0}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 1\\nMETHODS\\npublished: 11 August 2020\\ndoi: 10.3389/fmicb.2020.01678\\nEdited by:\\nShaolin Wang,\\nChina Agricultural University, China\\nReviewed by:\\nRong Zhang,\\nZhejiang University, China\\nRuichao Li,\\nYangzhou University, China\\n*Correspondence:\\nNatalie Pauly\\nnatalie.pauly@bfr.bund.de\\nAlexandra Irrgang\\nalexandra.irrgang@bfr.bund.de;\\nalex.irrgang@gmail.com\\nSpecialty section:\\nThis article was submitted to\\nAntimicrobials, Resistance\\nand Chemotherapy,\\na section of the journal\\nFrontiers in Microbiology\\nReceived: 30 March 2020\\nAccepted: 26 June 2020\\nPublished: 11 August 2020\\nCitation:\\nPauly N, Hammerl JA, Grobbel M,\\nTenhagen B-A, Käsbohrer A,\\nBisenius S, Fuchs J, Horlacher S,\\nLingstädt H, Mauermann U, Mitro S,\\nMüller M, Rohrmann S,\\nSchiffmann AP , Stührenberg B,\\nZimmermann P , Schwarz S,\\nMeemken D and Irrgang A (2020)\\nChromID R⃝CARBA Agar Fails\\nto Detect Carbapenem-Resistant\\nEnterobacteriaceae With Slightly\\nReduced Susceptibility\\nto Carbapenems.\\nFront. Microbiol. 11:1678.\\ndoi: 10.3389/fmicb.2020.01678\\nChromID\\nR⃝ CARBA Agar Fails to\\nDetect Carbapenem-Resistant\\nEnterobacteriaceae With Slightly\\nReduced Susceptibility to\\nCarbapenems\\nNatalie Pauly1*, Jens A. Hammerl1, Mirjam Grobbel1, Bernd-Alois Tenhagen1,\\nAnnemarie Käsbohrer1,2, Sandra Bisenius3, Jannika Fuchs4, Sabine Horlacher5,\\nHolger Lingstädt6, Ute Mauermann7, Silke Mitro8, Margit Müller9, Stefan Rohrmann10,\\nArthur P. Schiffmann11, Birgit Stührenberg12, Pia Zimmermann13, Stefan Schwarz14,\\nDiana Meemken15 and Alexandra Irrgang1*\\n1 German Federal Institute for Risk Assessment, Berlin, Germany, 2 Unit for Veterinary Public Health and Epidemiology,\\nUniversity of Veterinary Medicine, Vienna, Austria, 3 Institute for Fish and Fishery Products (LAVES), Cuxhaven, Germany,\\n4 Chemical and Veterinary Investigation Ofﬁce, Karlsruhe, Germany, 5 Chemical and Veterinary Investigation Ofﬁce, Fellbach,\\nGermany, 6 State Ofﬁce for Consumer Protection Saxony-Anhalt, Stendal, Germany, 7 Food and Veterinary Institute,\\nOldenburg, Germany, 8 State Investigation Institute for Health and Veterinary Services, Chemnitz, Germany, 9 Chemical\\nand Veterinary Investigation Ofﬁce Rhein-Ruhr-Wupper, Krefeld, Germany,10 Chemical and Veterinary Investigation Ofﬁce,\\nArnsberg, Germany, 11 Hessian State Laboratory, Gießen, Germany, 12 Chemical and Veterinary Investigation Ofﬁce,\\nDetmold, Germany, 13 Bavarian Health and Food Safety Authority, Oberschleißheim, Germany, 14 Institute of Microbiology\\nand Epizootics, Freie Universität Berlin, Berlin, Germany, 15 Institute of Food Safety and Food Hygiene, Freie Universität\\nBerlin, Berlin, Germany\\nAfter ﬁrst detections of carbapenemase-producing Enterobacteriaceae (CPE) in animals,\\nthe European Union Reference Laboratory for Antimicrobial Resistance has provided\\na protocol for the isolation of carbapenemase-producing Escherichia (E.) coli from\\ncecum content and meat. Up to now, only few isolates were recovered using this\\nprocedure. In our experience, the choice of the selective agar is important for the\\nefﬁcacy of the method. Currently, the use of the prevailing method fails to detect CPE\\nthat exhibit a low resistance against carbapenems. Thus, this study aims to evaluate\\nthe suitability of selective media with antibiotic supplements and commercial ChromID\\nR⃝\\nCARBA agar for a reliable CPE detection. For comparative investigations, detection\\nof freeze-dried carbapenemase-resistant bacteria was studied on different batches\\nof the ChromID\\nR⃝\\nCARBA agar as well as on MacConkey agar supplemented with\\n1 mg/L cefotaxime and 0.125 mg/L meropenem (McC +CTX+MEM). The suitability\\nof the different media was assessed within a time of 25 weeks, starting at least\\nsix weeks before expiration of the media. Carbapenem-resistant isolates exhibiting a\\nserine-based hydrolytic resistance mechanism (e.g., blaKPC genes) were consistently\\ndetected over 25 weeks on the different media. In contrast, carbapenemase producers\\nwith only slightly reduced susceptibility and exhibiting a zinc-catalyzed activity (e.g.,\\nblaVIM, blaNDM, and blaIMP) could only be cultivated on long-time expired ChromID\\nR⃝\\nCARBA, but within the whole test period on McC+CTX+MEM. Thus, ChromID\\nR⃝\\nCARBA\\nagar appears to be not suitable for the detection of CPE with slightly increased minimum\\ninhibitory concentrations (MIC) against carbapenems, which have been detected in\\nGerman livestock and thus, are of main interest in the national monitoring programs.\\nFrontiers in Microbiology | www.frontiersin.org 1 August 2020 | Volume 11 | Article 1678'), Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 1}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 2\\nPauly et al. Validation of Selective CPE Agar\\nOur data are in concordance with the results of eleven state laboratories that had\\nparticipated in this study with their ChromID\\nR⃝\\nCARBA batches routinely used for the\\nGerman CPE monitoring. Based on the determined CPE detection rate, we recommend\\nthe use of McC +CTX+MEM for monitoring purposes. This study indicates that the\\nuse of ChromID\\nR⃝\\nCARBA agar might lead to an underestimation of the current CPE\\noccurrence in food and livestock samples.\\nKeywords: carbapenemase, isolation, media, speciﬁcity, sensitivity\\nINTRODUCTION\\nCarbapenems are important antimicrobial substances with\\nbroad activity against almost all β-lactams. They are essential\\nto treat severe human infections with multidrug-resistant\\nGram-negative bacteria. Due to their high impact in human\\nmedicine, dissemination of carbapenem-resistant bacteria\\nshould be avoided (Sharland et al., 2018; European Medicines\\nAgency [EMA], 2019). Actually, more than 2,000 variants\\nof carbapenemase resistance genes have been described that\\nwere allocated to the β-lactamases of Ambler classes A, B,\\nand D (Codjoe and Donkor, 2017; Grundmann et al., 2017).\\nKlebsiella pneumoniae carbapenemases (KPC) and deﬁned OXA\\nenzymes are common Ambler class A and D carbapenemases,\\nrespectively, with a serine residue in their active sites (Codjoe\\nand Donkor, 2017). New Delhi metallo- β-lactamases (NDM)\\nand Verona integron-encoded metallo- β-lactamases (VIM)\\nare predominant Ambler class B enzymes. Their resistance\\nmechanism depends on the interaction of the carbapenems with\\nzinc ions in the active site of the enzymes (Codjoe and Donkor,\\n2017). Carbapenemase genes are often located on mobile genetic\\nelements (i.e., transposons/integrons, plasmids, phages), which\\nsupport horizontal transmission of the genes between diﬀerent\\nbacteria (Carattoli, 2009; Miriagou et al., 2010; Tato et al., 2010).\\nThe genes are often co-located with genes mediating resistance to\\nother antimicrobial classes. This leads to an important reduction\\nof therapeutic options in human medicine (Woodford et al.,\\n2014). In addition, various chromosomal mechanisms are also\\ninvolved in resistance development against carbapenems, i.e.,\\nchanges in membrane permeability due to the loss of speciﬁc\\nouter membrane porins (Miriagou et al., 2010).\\nFor a long time, carbapenemase-producing\\nEnterobacteriaceae (CPE) were almost exclusively associated\\nwith human medicine. However, the number of reported isolates\\nfrom wildlife, companion animals, livestock, and food increases\\nworldwide (Köck et al., 2018). To get deeper insights into the\\ndynamics of occurring CPE, the European Union (EU) set up a\\ncarbapenem-resistance monitoring for Escherichia coli isolates\\nfrom food products and livestock (Schrijver et al., 2018). Aside\\nfrom clinical settings, the carbapenemase enzymes OXA, VIM,\\nNDM, and KPC possess the highest impact in wildlife, pets, and\\nthe food chain (Grundmann et al., 2017; Köck et al., 2018). In\\nGermany, France, Italy, Spain, and the Netherlands, reports have\\nbeen published about carbapenem-resistant Enterobacteriaceae\\n(CRE) in companion animals (i.e., dogs, cats, horses) (Stolle\\net al., 2013; Schmiedel et al., 2014; Gonzalez-Torralba et al.,\\n2016; Melo et al., 2017; Vittecoq et al., 2017; Pulss et al., 2018),\\nseafood (i.e., shrimps, blue mussels, cockles) (Roschanski et al.,\\n2016; Ceccarelli et al., 2017), wild animals (i.e., yellow-legged\\ngulls, black kite) (Fischer et al., 2013b; Vergara et al., 2017;\\nVittecoq et al., 2017), and food of livestock animals (i.e., beef,\\nchicken meat, pork) (Schwaiger et al., 2008; Leverstein-van Hall\\net al., 2011; Poirel et al., 2012; Zurﬂuh et al., 2016; Ceccarelli\\net al., 2017; Fischer et al., 2017; Pulss et al., 2017; Randall et al.,\\n2017; Roschanski et al., 2017, 2018). In 2011, VIM-1-producing\\nEnterobacteriaceae (Salmonella Infantis and E. coli) were found\\nin several chicken and pig farms in Germany (Borowiak et al.,\\n2017; Falgenhauer et al., 2017; Fischer et al., 2017; Irrgang\\net al., 2017, 2019; Roschanski et al., 2018). The European Union\\nReference Laboratory for Antimicrobial Resistance (EURL-AR)\\ntogether with national experts on antimicrobial resistances\\ndeveloped a culture-dependent procedure for the isolation of\\ncarbapenemase-producing E. coli from meat and cecum content\\nsamples1. As no validated method for isolates from food exists,\\nthis method is currently used in the annual monitoring on\\ncarbapenemase-producing E. coli from livestock and food in\\nGermany and other European member states. In Germany,\\nwe recognized on several occasions within the monitoring and\\nduring additional studies that the method failed to detect CPE\\nthat were identiﬁed in the same sample using a diﬀerent method.\\nOne example was an isolate in 2017, which was detected in\\nthe monitoring on ESBL/AmpC β-lactamases but could not be\\ndetected within the German CPE monitoring. It was a VIM-\\n1-producing E. coli from the cecum content of a fattening pig\\nat slaughter that exhibited minimum inhibitory concentrations\\n(MIC) ranging around the current cutoﬀ values for the tested\\ncarbapenems. These cutoﬀ values are considerably lower than\\nthe MIC of CPE of human clinical origin (European Food Safety\\nAuthority [EFSA], and European Centre for Disease Prevention\\nand Control [ECDC],, 2017, 2019). Thus, there was the necessity\\nto improve the isolation method for CPE with low carbapenem\\nMIC values from food and livestock samples.\\nDue to the prevailing deﬁcit in reliable detection of CPE\\nexhibiting low MIC values, this study aims to determine the\\nsensitivity of the selective cultivation after enrichment of the\\nsamples for the laboratory routine in detecting CPE. Thus, the\\nsuitability of diﬀerent selective media for the recovery of CPE\\nwas assessed. Beside the frequently used ChromID\\nR⃝ CARBA\\nagar (bioMérieux, Nürtingen, Germany) (Nordmann et al.,\\n2012; Girlich et al., 2013; Papadimitriou-Olivgeris et al., 2014),\\n1https://www.eurl-ar.eu/protocols.aspx\\nFrontiers in Microbiology | www.frontiersin.org 2 August 2020 | Volume 11 | Article 1678'), Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 2}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 3\\nPauly et al. Validation of Selective CPE Agar\\nwhich is also predominantly used by the German federal state\\nlaboratories, we also determined the sensitivity of MacConkey\\nagar supplemented with cefotaxime and meropenem over a time\\ninterval of 25 weeks.\\nMATERIALS AND EQUIPMENT\\nBacterial Strains\\nFive carbapenem-resistant isolates were used for the evaluation\\nof the diﬀerent selective media ( Table 1 ). We chose three\\nCPE ( E. coli : CP-3 and CP-9; Salmonella Corvallis: CP-5),\\nwhich had previously been recovered from German food and\\nlivestock samples (Fischer et al., 2013b; Irrgang et al., 2017), one\\ncarbapenemase-producing Vibrio parahaemolyticus (CP-8) from\\nseafood and one carbapenem-resistant K. pneumoniae isolate\\nfrom a pig slaughterhouse that lacks carbapenemase production\\n(CR-1). As references for quality assurance concerning the\\nselectivity of the media, the ESBL-producing E. coli isolate AR-1\\n(Wilkinson et al., 2012) and E. coli ATCC 25922 were used.\\nFurther information on the bacterial isolates used in this\\nstudy is summarized in Table 1. Detailed information on the\\nMIC values of the isolates for 14 antimicrobials (ampicillin,\\ncefotaxime, ceftazidime, chloramphenicol, ciproﬂoxacin, colistin,\\nﬂorfenicol, gentamicin, kanamycin, nalidixic acid, streptomycin,\\nsulfamethoxazole, tetracycline, and trimethoprim) is provided in\\nthe Supplementary Table S1.\\nMedia and Supplements\\nIf not stated otherwise, all bacteria were cultivated in lysogeny\\nbroth (LB) at 37 ◦C for 16ñ20 h under shaking conditions (120ñ\\n160 rpm). Solid media agars containing 0.8% (w/v) Bacto agar\\n(Oxoid, Dassel, Germany) were used. If necessary, antimicrobial\\nsubstances were supplemented to the media as indicated. All\\nmedia (listed in Table 2) of this study were stored at 4ñ6 ◦C in\\na refrigerator and only exposed to light during processing of\\nthe experiments.\\nMETHODS\\nAntimicrobial Susceptibility Testing\\nThe susceptibility to antimicrobials of the isolates was\\ndetermined by broth microdilution using deﬁned antimicrobial\\nsubstances and concentrations from the harmonized EU\\npanel [plates EUVSEC and EUVSEC2; TREK Diagnostic\\nSystems (Thermo Fisher Scientiﬁc, Schwerte, Germany)].\\nResistance testing was conducted according to the EN ISO20776-\\n1:2006 (European Food Safety Authority [EFSA], and European\\nCentre for Disease Prevention and Control [ECDC],, 2018), and\\nMIC values were interpreted based on EUCAST deﬁnitions 2\\nusing epidemiological cutoﬀ values. The testing was repeated\\nthree times (Supplementary Table S1). As quality control strain,\\nE. coli ATCC 25922, was used.\\n2www.eucast.org\\nPreparation of Freeze-Dried Bacteria\\nTo ensure that equal amounts of bacterial cells were used\\nthroughout the whole study, a single batch of freeze-dried\\nisolates was prepared using an Epsilon 2-10D LSC lyophilization\\ninstrument (Christ, Osterode am Harz, Germany) with the\\nfollowing procedure. Bacterial enrichments were prepared by\\ninoculating 2 mL LB broth, supplemented with 1 mg/L\\ncefotaxime (CTX), with a single colony of the carbapenem-\\nresistant or ESBL-producing isolates, while cultivation of E. coli\\nstrain ATCC 25922 was conducted in LB without antimicrobials.\\nThe inoculated broths were cultured for 16ñ20 h at 37 ±2◦C\\non a rotational shaker (180 rpm). Following incubation, 150 µL\\nof the respective cultures was transferred into 2 mL of fresh\\nLB for further incubation at 37 ±2◦C until an optical density\\n(OD600 nm ) between 0.5 and 0.6 was reached. Thereafter, the\\nculture was diluted to 1:10,000 (10 3 to 10 4 CFU/mL) and\\ncentrifuged at 4,000 g for 10 min. The bacterial cell pellet was\\nresuspended in lyophilization medium [glutamic acid 10.0 g/L;\\nNaOH (8%) 30.0 mL/L; Bacto tryptone 25.0 g/L; sucrose 50.0 g/L;\\nthiourea (CH 4N2S) 2.0 g/L] and was applied in aliquots of\\n1 mL to the borosilicate glasses. Lyophilization was conducted\\naccording to the standardized and validated procedure of the\\nGerman Federal Institute for Risk Assessment (BfR) (freezing;\\ncondenser cooling to −50◦C; vacuuming the drying chamber;\\nmain drying and post-drying). After lyophilization, the bacterial\\nconcentrations (CFU/mL) of the freeze-dried bacteria were\\ndetermined under selective and non-selective conditions. The\\nbacterial concentration after resuspension, plating on LB agar\\nwithout any supplement and incubation overnight at 37 ◦C\\naccounted between 7.5 and 45 ×103 CFU/mL.\\nEnumeration of Bacteria on Different\\nMedia\\nTo evaluate the performance of diﬀerent agar types and the\\nimpact of increasing storage time of ChromID\\nR⃝ CARBA plates,\\nbacterial growth on diﬀerent media was determined weekly over\\na period of 25 weeks as described below. Therefore, the respective\\nlyophilisates of the bacteria were resuspended in 1 mL double-\\ndistilled water and incubated for 30ñ60 min at room temperature\\nfor adaption of the bacteria. To exclude potential eﬀects of the\\ncommercial agar of the growth performance of the bacteria,\\nﬁve diﬀerent ChromID\\nR⃝ CARBA batches were tested. Aliquots\\n(50 µL) of the lyophilisate suspensions were plated on ChromID\\nR⃝\\nCARBA plates. For comparative analysis of the commercial\\nagar to other media, the same lyophilisate suspensions were\\nalso applied to LB agar and to MacConkey agar supplemented\\nwith 1 mg/L CTX and 0.125 mg/L meropenem (MEM)\\n(McC+CTX+MEM). Both LB agar and McC+CTX+MEM were\\nproduced in-house at the BfR. After an incubation period of 16ñ\\n18 h at 37 ±2◦C, the bacterial concentration (CFU/mL) was\\ndetermined. For comparability, the storage time the bacterial\\ngrowth on the ﬁve ChromID\\nR⃝ CARBA batches was calculated\\naccording to the expiry date.\\nFreeze-dried bacteria were also provided to 11 participating\\nfederal state laboratories for comparative investigation. The\\nlaboratories of Baden-Württemberg (Fellbach, Karlsruhe),\\nFrontiers in Microbiology | www.frontiersin.org 3 August 2020 | Volume 11 | Article 1678'), Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 3}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 4\\nPauly et al. Validation of Selective CPE Agar\\nBavaria (Oberschleiﬂheim), Hesse (Gieﬂen), Lower Saxony\\n(Cuxhaven, Oldenburg), North Rhine-Westphalia (Arnsberg,\\nDetmold, Krefeld), Saxony (Chemnitz), and Saxony-Anhalt\\n(Stendal) conducted growth investigation according to the\\nspeciﬁcations of the BfR. Overall, the selectivity of the ChromID\\nR⃝\\nCARBA agar was tested on seven diﬀerent batches. In contrast to\\nthe long-term procedure of the BfR, the federal state laboratories\\ninvestigated the growth performance of the diﬀerent isolates on\\ndiﬀerent selective media at three deﬁned time points: (i) at the\\nexpiry date, (ii) 2 weeks before, and (iii) 5 weeks after the expiry\\ndate. The systematic procedure of all steps is shown in Figure 1.\\nRESULTS\\nCommercial selective medium is not suitable for\\nroutine monitoring purposes on isolates from\\nlivestock and food that exhibit low carbapenem\\nMIC values.\\nTherefore, the commercial ChromID\\nR⃝ CARBA agar was\\ntested out of date because of previous observations and\\nexperiences. Freeze-dried bacteria, which were produced\\nwithin this study, showed growth between 5 ×102 CFU/mL\\nand 4.5 × 104 CFU/mL ( Figure 2 ). Over the 25-week\\nperiod, the bacterial titer of the isolates did not change\\nsigniﬁcantly under non-selective conditions (LB). However,\\nwe observed a slight decrease in the bacterial titer for\\nthe isolates AR-1, CR-1, and CP-8 over the ﬁrst time\\npoints of enumeration ( Figure 2 ). After 25 weeks, all\\nisolates were still reliably detectable under non-selective\\nconditions (Figure 2).\\nThe results of the comparative analysis are summarized in\\nSupplementary Table S2 . Results of comparative testing of\\nthe diﬀerent federal state laboratories were very similar. They\\nTABLE 1 | Strains and species used in this study with their resistance determinants (threefold determined MIC values, variations are indicated with a “/” between\\nthe values).\\nIsolate Bacterium Origin Carbapenemase\\ngenes\\nCarbapenemase\\nproduction\\nERP\\n(mg/L)\\nIMI (mg/L) MEM\\n(mg/L)\\nReferences\\nATCC\\n25922\\nE. coli DSMZ None − ≤ 0.015 0.25 ≤0.03 DSMZ\\nAR-1 E. coli Bovine cecum None − 0.25 0.5 0.6 This work\\nCP-3 E. coli Colon content\\nof a slaughter\\npig\\nblaVIM−1 + 0.12 4 1 Irrgang et al.,\\n2017\\nCP-5 Salmonella\\nCorvallis\\nWild bird blaNDM−1 + >2 4 8 Fischer et al.,\\n2013b\\nCP-8 V. parahaemolyticus Seafood blaNDM−1 + 0.25 1/2/4 0.25 This work\\nCP-9 E. coli Human blaKPC−2 + >2 4 8 This work\\nCR-1 K. pneumoniae Porcine cecum None − 2 0.5/1 0.5 This work\\nERP , ertapenem; IMI, imipenem; MEM, meropenem. DSMZ, German collection of microorganisms and cell cultures. Moreover CP , carbapenemase-producing; CR,\\ncarbapenem-resistant; and AR, antibiotic resistant.\\nTABLE 2 | List of media, reagents, materials, and equipment.\\nMaterial Produced by Further information Country\\nLysogeny broth In-house Yeast extract 5 g/L; peptone from casein 10 g/L; NaCl\\n10 g/L\\nLysogeny agar In-house Yeast extract 5 g/L; peptone from casein 10 g/L; NaCl\\n10 g/L; agar 12 g/L\\nBacto agar Oxoid Dassel, Germany\\nLyophilization medium In-house Glutamic acid 10.0 g/L; NaOH (8%) 30.0 mL/L; Bacto\\ntryptone 25.0 g/L; sucrose 50.0 g/L; thiourea (CH4N2S)\\n2.0 g/L\\nMacConkey agar In-house Peptone from casein 17 g/L; peptone from meat 3 g/L;\\nNaCl 5 g/L; lactose 10 g/L; bile salt mixture 1.5 g/L, neutral\\nred 0.03 g/L; crystal violet 0.001 g/L, agar 13.5 g/L\\nChromID R⃝CARBA bioMérieux Nürtingen, Germany\\nCefotaxime Sigma-Aldrich Taufkirchen, Germany\\nMeropenem Sigma-Aldrich Taufkirchen, Germany\\nPlates EUVSEC and EUVSEC2 Thermo Fisher Scientiﬁc Schwerte, Germany\\nEpsilon 2-10D LSC lyophilization\\ninstrument\\nChrist Osterode am Harz, Germany\\nSensititre AIMTM Automated Inoculation\\nDelivery System\\nTREK Diagnostic East Grinstead, United Kingdom\\nSensititreTM SWINTM Software System\\n3.3\\nThermo Fisher Scientiﬁc Schwerte, Germany\\nFrontiers in Microbiology | www.frontiersin.org 4 August 2020 | Volume 11 | Article 1678'), Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 4}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 5\\nPauly et al. Validation of Selective CPE Agar\\nFIGURE 1 | Flow diagram of the systematic procedure.\\nFIGURE 2 | The growth of all lyophilisates on non-selective LB agar over the entire test period of 25 weeks. The x-axis (abscissa) shows the time in weeks, and the\\ny-axis (ordinate) shows CFU/mL.\\nindicated that only one isolate (CP-9) could be detected on the\\nused agar batches at all three time points. This respective isolate,\\na human E. coli which carried the gene blaKPC−2, exhibited\\na high MIC values against the tested carbapenems ( Table 1).\\nNone of the other isolates showed bacterial growth at the\\nselected time points.\\nTo investigate the suitability of the ChromID\\nR⃝ CARBA agar\\nfor the detection of CPE exhibiting low carbapenem MIC\\nvalues, the growth ability of the aforementioned isolates was\\ninvestigated at the NRL-AR over a time period of 25 weeks\\nusing ﬁve diﬀerent batches of the agar. The diagrams 2Añ2E\\nshow that at the beginning of the test, bacterial growth could\\nonly be detected for isolate CP-9, exhibiting high MIC values\\nfor the carbapenems ertapenem, imipenem, and meropenem\\n(Table 1 and Supplementary Table S1). Slight bacterial growth\\ncould be detected for the isolates CP-3, CP-5, and CP-8\\n2 weeks before expiry date (week 7) ( Figures 3A–C ). This\\ndetection increased over time and also after the expiry date. The\\ncarbapenemase-producing isolate CP-9 showed a highly reliable\\naverage growth rate between 5.0 ×103 and 2.5 ×104 CFU/mL\\nat all points of investigations over the 25 weeks ( Figure 3D ).\\nAt the end of the trial, four carbapenem-resistant isolates\\n(CP-3; CP-5; CP-8; and CR-1) showed increased growth rates\\n(average: CP-3 with blaVIM−1: 4 to 32 CFU/mL; CP-5 with\\nblaNDM−1: 4 to 368 CFU/mL; and CP-8 with blaVIM−1: 4 to\\n24 CFU/mL). The CR-1 isolate that exhibits no carbapenemase-\\nproduction showed growth rates between 20 and 3920 CFU/mL\\nand was ﬁrst detected 1 week before the expiry date of the\\nmedia ( Figure 3E ). The quality control isolate AR-1 ( Table 1\\nand Supplementary Table S1 ) and the control strain ATCC\\n25922 did not grow on ChromID\\nR⃝ CARBA plates during the\\ninvestigated period.\\nFrontiers in Microbiology | www.frontiersin.org 5 August 2020 | Volume 11 | Article 1678'), Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 5}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 6\\nPauly et al. Validation of Selective CPE Agar\\nFIGURE 3 | The x-axis shows the time in weeks where “9” the expiry date is. They-axis shows CFU/mL. (A) Growth of CP-3 on ﬁve tested batches of ChromIDR⃝\\nCARBA agar. Moreover, the average of all batches is shown.(B) Growth of CP-5 on ﬁve tested batches of ChromIDR⃝CARBA agar. Moreover, the average of all\\nbatches is shown. (C) Growth of CP-8 on ﬁve tested batches of ChromIDR⃝CARBA agar. Moreover, the average of all batches is shown.(D) Growth of CP-9 on ﬁve\\ntested batches of ChromID R⃝CARBA agar. Moreover, the average of all batches is shown.(E) Growth of CR-1 on ﬁve tested batches of ChromIDR⃝CARBA agar.\\nMoreover, the average of all batches is shown. (F) Growth of the both control strains NK and AR-1 on ﬁve tested batches of ChromIDR⃝CARBA agar.\\nSupplementing 1 mg/L CTX and 0.125 mg/L MEM to the agar\\nbasis of the MacConkey medium can overcome growth inhibition\\nduring cultivation of low-carbapenem MIC-exhibiting isolates.\\nThe growth performance of the abovementioned isolates\\nwas also determined on McC +CTX+MEM agar. In contrast\\nto ChromID\\nR⃝ CARBA agar, all carbapenem-resistant isolates\\nwere reliably detected at all time points within the 25-\\nweek period of investigation ( Table 3 ). The growth of the\\nCP-9 isolate was similar on McC +CTX+MEM, ChromID\\nR⃝\\nCARBA agar as well as under non-selective conditions (LB\\nagar) ranging between 2.4 ×103 and 3.3 ×104 CFU/mL.\\nWith the focus on the isolates (AR-1 and NK) used as\\nquality controls, McC +CTX+MEM has a reliable speciﬁcity\\nof up to 22 weeks, as within this period no bacterial growth\\nwas observed.\\nDISCUSSION\\nPrevious experiments in our laboratory showed that the\\nChromID\\nR⃝ CARBA agar becomes more sensitive with increasing\\nstorage time. Based on the observations and experiences\\nwithin the laboratory routine (International Organization\\nfor Standardization, 2006; Fischer et al., 2013a,b, 2017;\\nIrrgang et al., 2017, 2019), detection of carbapenemase-\\nproducing bacteria is challenging using the EURL-AR method\\nin combination with ChromID\\nR⃝ CARBA agar. We assume\\nthat the concentration of diﬀerent ingredients and thus the\\nselectivity will decrease with time or after the expiry date.\\nThis justiﬁes the choice of times to determine a possible\\ndecrease in antimicrobial concentration after the shelf life of\\nChromID\\nR⃝ CARBA agar.\\nFrontiers in Microbiology | www.frontiersin.org 6 August 2020 | Volume 11 | Article 1678'), Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 6}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 7\\nPauly et al. Validation of Selective CPE Agar\\nTABLE 3 | Growth of the seven lyophilisates tested once a week over 25 weeks on MacConkey agar supplemented with 0.125 mg/L meropenem and 1 mg/L\\ncefotaxime.\\nWeek 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25\\nAR-1 − − − − − − − − − − − − − − − − − − − − − − + + +\\nCP-3 + + + + + + + + + + + + + ++ + ++ ++ ++ ++ + + ++ ++ ++ ++\\nCP-5 + + + + + + + + + + + + + + + + + + + + + + + + +\\nCP-8 + − − + + + + + + + + − + + + + + + + + − + + + +\\nCP-9 +++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++\\nCR-1 ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++ ++\\nNK − − − − − − − − − − − − − − − − − − − − − − − − −\\n“−” means no growth; “+” means 1–100 cfu/mL; “++” means 100–1000 cfu/mL; “+++” means≥1000 cfu/mL.\\nWithin this study, we observed strong diﬀerences in the\\ngrowth performance of the used carbapenem-resistant isolates\\non the selective media. These diﬀerent growth performances\\nexceed the observed slight decrease of AR-1, CR-1, and CP-\\n8 during the ﬁrst time points of enumeration, which seems\\nto represent a natural adaption of the speciﬁc isolates to the\\nprevailing cultivation conditions (Figure 2).\\nAs previously reported (Fischer et al., 2012; Köck et al.,\\n2018), carbapenem-resistant isolates from food products or\\nlivestock exhibit signiﬁcantly lower MIC values for the respective\\ncarbapenems than clinical isolates. There may be diﬀerent\\nreasons for this observation. It could be due to molecular\\nconditions, such as a variable number of gene copies, a diﬀerent\\nexpression of the genes, or a diﬀerent genetic localization\\n(plasmid versus chromosomal DNA). It could also be due to\\nthe diﬀerent antimicrobial selection pressures to which the\\nstrains are exposed. As carbapenems are not licensed for use in\\nfood animals, previous exposure of the strains with a livestock\\norigin is unlikely. The chosen isolates for the evaluation of the\\nselective cultivation step of the detection procedure represent\\ndiﬀerent bacterial genera and carbapenem resistance genes as\\nwell as resistance mechanisms (Viau et al., 2012; Köck et al.,\\n2018). The E. coli isolate CP-9 carries a blaKPC−2 gene and\\nexhibited high MIC values against all tested carbapenems.\\nCP-9 represents typical MIC values for carbapenems of human\\nisolates (Supplementary Table S1). Other target isolates, except\\nCP-5 (the NDM-producing Salmonella) exhibited carbapenem\\nresistance but showed only MICs ranging around the current\\ncutoﬀ values ( Supplementary Table S1 ). As the used isolates\\nexhibited very low MIC values, we suppose that the remaining\\nconcentration of the supplemented antimicrobials is still high\\nenough to prevent a growth of these isolates.\\nThe KPC enzymes belong to Amber class A carbapenemases\\nand use a serine-based hydrolytic mechanism to inactivate\\ncarbapenems. In contrast, the VIM-1 and NDM-1 enzymes\\nbelong to Ambler class B carbapenemases and convey the\\nresistances by a zinc-catalyzing active site (Codjoe and Donkor,\\n2017). Viau et al. (2016) reported that most of their ten\\ntested media performed reasonably well with diﬀerent screening\\nmethods for the detection of class A enzymes. Performance with\\nclass B and D enzymes was more variable. However, Viau and co-\\nworkers used up to 180 diﬀerent CRE originating from human\\nclinical sources. They determined a good performance of the\\nChromID\\nR⃝CARBA medium if tested with rectal/perirectal swabs\\n(Jeon et al., 2015). The diﬀerences between CPE from human\\nsamples and CPE from non-clinical animal samples have not\\nbeen investigated systematically so far as the number of available\\nnon-clinical animal isolates is still limited. As the ingredients\\nof the ChromID\\nR⃝ CARBA agar are unknown and have not\\nbeen provided by the manufacturers on request, it is diﬃcult to\\nassess the selective additives of the medium and to expect their\\ninﬂuence on the cultivation of CPE with low MIC. It might be\\nthat an additive supports the serine-based hydrolytic resistance\\nmechanism in a better way than the zinc-based mechanism.\\nWithin our study, CP-9 was the only isolate that was\\nconsistently detected in all participating laboratories and with all\\nselective media. The determined amount of the adjusted, freeze-\\ndried bacteria ranged between 5 ×102 and 4.5 ×104 CFU/mL\\nbetween the laboratories. The other carbapenem-resistant isolates\\n(CP-3; CP-5; CP-8; and CR-1) showed no detectable growth\\non the ChromID\\nR⃝ CARBA plates within the diﬀerent federal\\nstate laboratories up to 5 weeks after expiration date of the\\nplates ( Supplementary Table S2 ). At this time point, the\\nagar will not be used by accredited laboratories as the agar\\nwill be handled as expired according to the recommendations\\nof the manufacturers. However, each of these isolates was\\nreliably detected on McC +CTX+MEM ( Table 3). The quality\\ncontrol isolate AR-1, a CTX-M-1-producing E. coli with a\\nlow phenotypical resistance against ertapenem ( Table 1 and\\nSupplementary Table S1), and the quality control strain ATCC\\n25922 did not grow on ChromID\\nR⃝ CARBA plates at any time\\n(Figure 3F ). As they grew at the last three time points on\\nMcC+CTX+MEM, it can be assumed that the speciﬁcity of the\\nplates did not decrease considerably within a period of 20 weeks\\nafter production ( Table 3). In contrast, the number of CFU of\\nthe isolates CP-3; CP-5; CP-8; and CR-1 increased at the end of\\nthe investigated time interval. This indicates that the sensitivity\\nof the commercial agar increased over time.\\nIn contrast to our study, other groups found that ChromID\\nR⃝\\nCARBA agar was one of the most sensitive and speciﬁc\\nchromogenic medium for the detection of CPE (Vrioni et al.,\\n2012; Papadimitriou-Olivgeris et al., 2014; Simner et al., 2015;\\nViau et al., 2016). However, in these studies, the MIC values of\\nthe CPE are in general much higher than the MIC values of\\nthe isolates tested here. In comparison, Vrioni et al. (2012) used\\nclinical isolates with very high MIC values of up to over 32 mg/L\\nto MEM, but they also used some strains with MIC values below\\nthe resistance breakpoint. The detection of the isolates with these\\nFrontiers in Microbiology | www.frontiersin.org 7 August 2020 | Volume 11 | Article 1678'), Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 7}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 8\\nPauly et al. Validation of Selective CPE Agar\\nhigh MIC values might correlate much more with the detection\\nof CP-9 and is not comparable with the detection of isolates\\nexhibiting low MICs (CP-3, CP-8, and CR-1).\\nLimited detection of CPE might be also attributable to a\\nloss of the resistance-harboring plasmids during lyophilization\\n(Wagman and Weneck, 1963; Berman et al., 1968; Wein et al.,\\n2019). We can exclude this as an important factor in our study\\nbecause we observed similar bacterial titers on selective and\\nnon-selective in-house media (data not shown). We had also\\nconﬁrmed the presence of the plasmid in some representative\\ncolonies from selective and non-selective media.\\nOverall, we have detected more CFU/mL by using the\\nconventional MacConkey agar supplemented with antibiotics\\nover the entire period for almost all isolates of this study. Due\\nto the lacking resistance against carbapenems, the quality control\\nstrain ATCC 25922 could not be detected on McC+CTX+MEM,\\nindicating the appropriate speciﬁcity of the medium. Some\\nspontaneous growth of the E. coli isolate AR_1 was observed\\n22, 23, and 24 weeks after agar production, indicating that the\\nprepared agar worked with a high speciﬁcity for up to 22 weeks.\\nIn contrast to this observation, the AR_1 isolate could not be\\ndetected on the ChromID\\nR⃝ CARBA agar. The concentration of\\nmeropenem (0.125 mg/L) in the supplemented agar is lower than\\nthe current ECOFFs (0.25 mg/L) (Giske et al., 2013). Based on this\\nconcentration, CPE with low MIC values are also able to grow\\nsuccessfully and will further be detected within the monitoring\\nprograms. Viau et al. (2012) reported about “silent” dissemination\\nof K. pneumoniae , because some isolates were not identiﬁed\\nas carbapenem-resistant based on their MIC, even though they\\ncarried a carbapenemase gene and showed slightly increased MIC\\nvalues against ertapenem and meropenem (Fischer et al., 2012).\\nBerthoin et al. (2010) reported that the stability of meropenem\\nin solutions depends on time, temperature, and concentration\\n(Berthoin et al., 2010). These observations were supported by\\nKeel et al. (2011). A potential eﬀect of light on the stability of\\nmeropenem can be excluded, as the agar plates were only exposed\\nto light during plating and visual inspection. Therefore, it was\\nassumed that the stability of meropenem in the media might\\nbe limited. In our study, the McC +CTX+MEM agar was stably\\nusable for up to 22 weeks without a loss of speciﬁcity when stored\\nat 4 ±2◦C. The supplementation with cefotaxime supported the\\nselective eﬀect, but for OXA-48 producers this antibiotic should\\nbe omitted. Changes in sensitivity and speciﬁcity of MacConkey\\nagar supplemented with meropenem only over time were not\\ntested in our study. Adler et al. (2011) used MacConkey agar\\nsupplemented with imipenem at 1 mg/mL (McC +IMI) and\\ncompared this medium with a commercial selective medium\\n(CHROMagar KPC, CHROMagar Company, Paris, France).\\nThey described a similar performance of both media during\\nclinical evaluation. Moreover, they argued with the lower costs\\nof McC +IMI to prefer it for detection of CRE in their study\\n(Adler et al., 2011). We decided to use meropenem as supplement\\nbecause it is more stable than imipenem (Zhanel et al., 2007).\\nBased on our data, we recommend using McC +CTX+MEM for\\nselective cultivation of carbapenem-resistant isolates from food\\nproducts and livestock samples.\\nAnother important diﬀerence between the diﬀerent agars\\nis that the ingredients of the in-house prepared agar are\\nknown. In McC +CTX+MEM agar, no further substances\\nwere supplemented for inhibition of accompanying bacteria.\\nIt is possible that the commercial selective medium comprises\\nunspeciﬁed substances (e.g., zinc that is used to inhibit non-\\ncarbapenemase-producing bacteria), which may also suppress the\\ngrowth of some CPE, e.g., with a low MIC. Although less selective\\nsubstances are used, no increased diﬃculties were found with the\\naccompanying ﬂora in other experiments so far.\\nA limitation of this study is that only one batch of\\nMcC+CTX+MEM plates was used throughout the experiment.\\nThis was due to its function as a control medium, and it was not\\nthe primary aim to compare both media.\\nCONCLUSION\\nThe results of our study indicate that the ChromID\\nR⃝CARBA agar\\nis not suitable for the cultural detection of CPE with low MIC to\\ncarbapenems, while clinical isolates with higher MIC values can\\nbe detected reliably. Overall, the use of the ChromID\\nR⃝ CARBA\\nagar in monitoring programs may lead to an underestimation\\nof the CPE occurrence in livestock and food as the method\\nlacks sensitivity.\\nWe further provide data, that MacConkey agar with 1 mg/L\\ncefotaxime and 0.125 mg/L meropenem provides a better\\nsensitivity for the detection of such isolates and may be more\\nsuitable for the carbapenemase monitoring in livestock and food.\\nDATA AVAILABILITY STATEMENT\\nAll datasets presented in this study are included in the\\narticle/Supplementary Material.\\nETHICS STATEMENT\\nThis study uses strains obtained from humans ( E. coli CP-9),\\nlivestock ( E. coli AR-1 and CP-3, Klebsiella pneumoniae CR-\\n1), wildlife ( Salmonella Corvallis CP-5) and seafood ( Vibrio\\nparahaemolyticus CP-8). All isolates were part of the strain\\ncollection of the German NRL AR and had been submitted\\nto the NRL in the framework of its routine tasks in\\nmonitoring and surveillance programs or during previous\\nscientiﬁc collaborations. None of the isolates was collected\\nspeciﬁcally for the current study. Therefore we did not consider\\nit necessary to seek for ethical approval for using the isolates.\\nAUTHOR CONTRIBUTIONS\\nJH, NP , and AI designed the study and developed the draft\\nof the manuscript. NP performed the preparation of adjusted,\\nfreeze-dried bacteria, and the determination of the growth\\nperformance of the isolates at the NRL-AR. MG, AI, and JH\\ninitially characterized the carbapenem-resistant isolates. SB, JF ,\\nSH, HL, UM, MM, SR, AS, SS, BS, and PZ performed the\\nindividual growth investigations on the commercial agar at the\\ndiﬀerent federal state laboratories. NP , AI, JH, MG, AK, B-AT,\\nSS, and DM performed the data analysis. All authors supported\\nFrontiers in Microbiology | www.frontiersin.org 8 August 2020 | Volume 11 | Article 1678'), Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 8}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 9\\nPauly et al. Validation of Selective CPE Agar\\nthe ﬁnalization of the manuscript and supported editing of\\nthe ﬁnal version.\\nFUNDING\\nThis work was ﬁnancially supported by a grant of the German\\nFederal Institute for Risk Assessment (43-002). The work of NP\\nwas supported by the European Joint Project (EJP) IMPART\\nfunded by the European Union’s Horizon 2020 Research and\\nInnovation Program under Grant Agreement No. 773830.\\nACKNOWLEDGMENTS\\nWe gratefully acknowledge the support of the federal\\nstate laboratories and authorities in collecting the samples\\nand providing the recovered isolates in the framework\\nof the diﬀerent German monitoring programs. We also\\nthank the laboratory team (Britta Lesniewsky, Anja\\nSeeger, Tanja Skladnikiewicz-Ziemer, Silke Jahn, Romy\\nFuhrmeister, Yvonne Klaar, and Silvia Schmoger) of the\\nNRL-AR for excellent technical assistance. We further\\nacknowledge the support of Peter Bahn (German Federal\\nInstitute for Risk Assessment) during preparation of the\\nfreeze-dried bacteria.\\nSUPPLEMENTARY MATERIAL\\nThe Supplementary Material for this article can be found\\nonline at: https://www.frontiersin.org/articles/10.3389/fmicb.\\n2020.01678/full#supplementary-material\\nREFERENCES\\nAdler, A., Navon-Venezia, S., Moran-Gilad, J., Marcos, E., Schwartz, D., and\\nCarmeli, Y. (2011). Laboratory and clinical evaluation of screening agar plates\\nfor detection of carbapenem-resistant Enterobacteriaceae from surveillance\\nrectal swabs. J. Clin. Microbiol. 49, 2239ñ2242. doi: 10.1128/jcm.025\\n66-10\\nBerman, S., Altieri, P. L., Groﬃnger, A., Lowenthal, J. P., and Formal, S. B. (1968).\\nFreeze-drying various strains of Shigella. Appl. Microbiol. 16, 1779ñ1781. doi:\\n10.1128/aem.16.11.1779-1781.1968\\nBerthoin, K., Le Duﬀ, C. S., Marchand-Brynaert, J., Carryn, S., and Tulkens, P. M.\\n(2010). Stability of meropenem and doripenem solutions for administration by\\ncontinuous infusion. J. Antimicrob. Chemother. 65, 1073ñ1075. doi: 10.1093/\\njac/dkq044\\nBorowiak, M., Szabo, I., Baumann, B., Junker, E., Hammerl, J. A., Kaesbohrer,\\nA., et al. (2017). VIM-1-producing Salmonella Infantis isolated from swine\\nand minced pork meat in Germany. J. Antimicrob. Chemother. 72, 2131ñ2133.\\ndoi: 10.1093/jac/dkx101\\nCarattoli, A. (2009). Resistance plasmid families inEnterobacteriaceae. Antimicrob.\\nAgents Chemother. 53, 2227ñ2238. doi: 10.1128/aac.01707-08\\nCeccarelli, D., van Essen-Zandbergen, A., Veldman, K. T., Tafro, N., Haenen, O.,\\nand Mevius, D. J. (2017). Chromosome-Based blaOXA-48-Like Variants in\\nShewanella species isolates from food-producing animals, ﬁsh, and the aquatic\\nenvironment. Antimicrob. Agents Chemother. 61:e01013-16.\\nCodjoe, F. S., and Donkor, E. S. (2017). Carbapenem resistance: a review. Med. Sci.\\n6:1. doi: 10.3390/medsci6010001\\nEuropean Food Safety Authority [EFSA], and European Centre for Disease\\nPrevention and Control [ECDC], (2017). The European Union Summary\\nReport on Antimicrobial Resistance in Zoonotic and Indicator Bacteria from\\nHumans, Animals and Food in 2015. Parma: EFSA. 176ñ192.\\nEuropean Food Safety Authority [EFSA], and European Centre for Disease\\nPrevention and Control [ECDC], (2018). The European Union summary report\\non antimicrobial resistance in zoonotic and indicator bacteria from humans,\\nanimals and food in 2016. EFSA J. 16, 223ñ250.\\nEuropean Food Safety Authority [EFSA], and European Centre for Disease\\nPrevention and Control [ECDC], (2019).The European Union Summary Report\\non Antimicrobial Resistance in Zoonotic and Indicator Bacteria from Humans,\\nAnimals and Food in 2017. Parma: EFSA. 223ñ250.\\nEuropean Medicines Agency [EMA] (2019). List of Medicinal Products Under\\nAdditional Monitoring. Amsterdam: EMA.\\nFalgenhauer, L., Ghosh, H., Guerra, B., Y ao, Y., Fritzenwanker, M., Fischer, J.,\\net al. (2017). Comparative genome analysis of IncHI2 VIM-1 carbapenemase-\\nencoding plasmids of Escherichia coli and Salmonella enterica isolated from a\\nlivestock farm in Germany. Vet. Microbiol. 200, 114ñ117. doi: 10.1016/j.vetmic.\\n2015.09.001\\nFischer, J., Rodriguez, I., Schmoger, S., Friese, A., Roesler, U., Helmuth, R., et al.\\n(2012). Escherichia coliproducing VIM-1 carbapenemase isolated on a pig farm.\\nJ. Antimicrob. Chemother. 67, 1793ñ1795. doi: 10.1093/jac/dks108\\nFischer, J., Rodriguez, I., Schmoger, S., Friese, A., Roesler, U., Helmuth, R., et al.\\n(2013a). Salmonella enterica subsp. enterica producing VIM-1 carbapenemase\\nisolated from livestock farms. J. Antimicrob. Chemother. 68, 478ñ480. doi:\\n10.1093/jac/dks393\\nFischer, J., San Jose, M., Roschanski, N., Schmoger, S., Baumann, B., Irrgang,\\nA., et al. (2017). Spread and persistence of VIM-1 Carbapenemase-producing\\nEnterobacteriaceae in three German swine farms in 2011 and 2012. Vet.\\nMicrobiol. 200, 118ñ123. doi: 10.1016/j.vetmic.2016.04.026\\nFischer, J., Schmoger, S., Jahn, S., Helmuth, R., and Guerra, B. (2013b). NDM-1\\ncarbapenemase-producing Salmonella enterica subsp. enterica serovar Corvallis\\nisolated from a wild bird in Germany.J. Antimicrob. Chemother.68, 2954ñ2956.\\ndoi: 10.1093/jac/dkt260\\nGirlich, D., Poirel, L., and Nordmann, P. (2013). Comparison of the\\nSUPERCARBA, CHROMagar KPC, and Brilliance CRE screening media for\\ndetection of Enterobacteriaceae with reduced susceptibility to carbapenems.\\nDiagn. Microbiol. Infect. Dis. 75, 214ñ217. doi: 10.1016/j.diagmicrobio.2012.\\n10.006\\nGiske, C. G., Canton, R., Stefani, S., Skov, R., Glupczynski, Y., Nordmann, P. N.,\\net al. (2013). EUCAST Guidelines for Detection of Resistance Mechanisms and\\nSpeciﬁc Resistances of Clinical and/or Epidemiological Importance . Sweden:\\nEUCAST.\\nGonzalez-Torralba, A., Oteo, J., Asenjo, A., Bautista, V., Fuentes, E., and Alos, J. I.\\n(2016). Survey of carbapenemase-producing Enterobacteriaceae in companion\\ndogs in Madrid, Spain. Antimicrob. Agents Chemother. 60, 2499ñ2501. doi:\\n10.1128/aac.02383-15\\nGrundmann, H., Glasner, C., Albiger, B., Aanensen, D. M., Tomlinson,\\nC. T., Andrasevic, A. T., et al. (2017). Occurrence of carbapenemase-\\nproducing Klebsiella pneumoniae and Escherichia coli in the European survey\\nof carbapenemase-producing Enterobacteriaceae (EuSCAPE): a prospective,\\nmultinational study. Lancet Infect. Dis. 17, 153ñ163.\\nInternational Organization for Standardization (2006). Clinical Laboratory Testing\\nand In Vitro Diagnostic Test System - Susceptibility Testing of Infectious Agents\\nand Evaluation of Performance of Antimicrobial Susceptibility Test Devices .\\nGeneva: International Organization for Standardization.\\nIrrgang, A., Fischer, J., Grobbel, M., Schmoger, S., Skladnikiewicz-Ziemer, T.,\\nThomas, K., et al. (2017). Recurrent detection of VIM-1-producing Escherichia\\ncoli clone in German pig production. J. Antimicrob. Chemother. 72, 944ñ946.\\nIrrgang, A., Tenhagen, B.-A., Pauly, N., Schmoger, S., Kaesbohrer, A., and\\nHammerl, J. A. (2019). Characterization of VIM-1-producing E. coli isolated\\nfrom a german fattening pig farm by an improved isolation procedure. Front.\\nMicrobiol. 10:2256. doi: 10.3389/fmicb.2019.02256\\nJeon, J. H., Lee, J. H., Lee, J. J., Park, K. S., Karim, A. M., Lee, C.-R., et al. (2015).\\nStructural basis for carbapenem-hydrolyzing mechanisms of carbapenemases\\nconferring antibiotic resistance. Int. J. Mol. Sci. 16, 9654ñ9692. doi: 10.3390/\\nijms16059654\\nKeel, R. A., Sutherland, C. A., Crandon, J. L., and Nicolau, D. P. (2011). Stability\\nof doripenem, imipenem and meropenem at elevated room temperatures. Int.\\nJ. Antimicrob. Agents 37, 184ñ185. doi: 10.1016/j.ijantimicag.2010.06.043\\nFrontiers in Microbiology | www.frontiersin.org 9 August 2020 | Volume 11 | Article 1678'), Document(metadata={'source': 'texts\\\\Pauly, Carba plate validation, 2020.pdf', 'page': 9}, page_content='fmicb-11-01678 August 9, 2020 Time: 12:4 # 10\\nPauly et al. Validation of Selective CPE Agar\\nKöck, R., Daniels-Haardt, I., Becker, K., Mellmann, A., Friedrich, A. W., Mevius,\\nD., et al. (2018). Carbapenem-resistant Enterobacteriaceae in wildlife, food-\\nproducing, and companion animals: a systematic review.Clin. Microbiol. Infect.\\n24, 1241ñ1250. doi: 10.1016/j.cmi.2018.04.004\\nLeverstein-van Hall, M. A., Dierikx, C. M., Stuart, J. C., Voets, G. M., van den\\nMunckhof, M. P., van Essen-Zandbergen, A., et al. (2011). Dutch patients, retail\\nchicken meat and poultry share the same ESBL genes, plasmids and strains.Clin.\\nMicrobiol. Infect. 17, 873ñ880. doi: 10.1111/j.1469-0691.2011.03497.x\\nMelo, L. C., Boisson, M. N., Saras, E., Medaille, C., Boulouis, H. J., Madec,\\nJ. Y., et al. (2017). OXA-48-producing ST372 Escherichia coli in a French dog.\\nJ. Antimicrob. Chemother. 72, 1256ñ1258.\\nMiriagou, V., Cornaglia, G., Edelstein, M., Galani, I., Giske, C. G., Gniadkowski,\\nM., et al. (2010). Acquired carbapenemases in Gram-negative bacterial\\npathogens: detection and surveillance issues. Clin. Microbiol. Infect. 16, 112ñ\\n122. doi: 10.1111/j.1469-0691.2009.03116.x\\nNordmann, P., Girlich, D., and Poirel, L. (2012). Detection of carbapenemase\\nproducers in Enterobacteriaceae by use of a novel screening medium. J. Clin.\\nMicrobiol. 50, 2761ñ2766. doi: 10.1128/jcm.06477-11\\nPapadimitriou-Olivgeris, M., Bartzavali, C., Christoﬁdou, M., Bereksi, N., Hey,\\nJ., Zambardi, G., et al. (2014). Performance of chromID\\nR⃝ CARBA medium\\nfor carbapenemases-producing Enterobacteriaceae detection during rectal\\nscreening. Eur. J. Clin. Microbiol. Infect. Dis. 33, 35ñ40. doi: 10.1007/s10096-\\n013-1925-6\\nPoirel, L., Berçot, B., Millemann, Y., Bonnin, R. A., Pannaux, G., and Nordmann, P.\\n(2012). Carbapenemase-producing Acinetobacter spp. in Cattle, France. Emerg.\\nInfect. Dis. 18, 523ñ525.\\nPulss, S., Semmler, T., Prenger-Berninghoﬀ, E., Bauerfeind, R., and Ewers, C.\\n(2017). First report of an Escherichia coli strain from swine carrying an OXA-\\n181 carbapenemase and the colistin resistance determinant MCR-1. Int. J.\\nAntimicrob. Agents 50, 232ñ236. doi: 10.1016/j.ijantimicag.2017.03.014\\nPulss, S., Stolle, I., Stamm, I., Leidner, U., Heydel, C., Semmler, T., et al.\\n(2018). Multispecies and clonal dissemination of OXA-48 carbapenemase in\\nEnterobacteriaceae From Companion Animals in Germany, 2009-2016. Front.\\nMicrobiol. 9:1265. doi: 10.3389/fmicb.2018.01265\\nRandall, L. P., Lodge, M. P., Elviss, N. C., Lemma, F. L., Hopkins, K. L.,\\nTeale, C. J., et al. (2017). Evaluation of meat, fruit and vegetables from retail\\nstores in ﬁve United Kingdom regions as sources of extended-spectrum beta-\\nlactamase (ESBL)-producing and carbapenem-resistant Escherichia coli. Int. J.\\nFood Microbiol. 241, 283ñ290. doi: 10.1016/j.ijfoodmicro.2016.10.036\\nRoschanski, N., Friese, A., von Salviati-Claudius, C., Hering, J., Kaesbohrer,\\nA., Kreienbrock, L., et al. (2017). Prevalence of carbapenemase producing\\nEnterobacteriaceae isolated from German pig-fattening farms during the years\\n2011-2013. Vet. Microbiol. 200, 124ñ129. doi: 10.1016/j.vetmic.2015.11.030\\nRoschanski, N., Friese, A., von Salviati-Claudius, C., Hering, J., Kaesbohrer,\\nA., Kreienbrock, L., et al. (2018). Retrospective analysis of bacterial cultures\\nsampled in german chicken-fattening farms during the years 2011-2012\\nrevealed additional VIM-1 carbapenemase-producing Escherichia coli and a\\nserologically rough Salmonella enterica Serovar Infantis. Front. Microbiol.9:538.\\ndoi: 10.3389/fmicb.2018.00538\\nRoschanski, N., Guenther, S., Vu, T. T. T., Fischer, J., Semmler, T., Huehn, S., et al.\\n(2016). VIM-1 carbapenemase-producing Escherichia coli isolated from retail\\nseafood, Germany 2016. Euro Surveill. 22:17-00032.\\nSchmiedel, J., Falgenhauer, L., Domann, E., Bauerfeind, R., Prenger-Berninghoﬀ,\\nE., Imirzalioglu, C., et al. (2014). Multiresistant extended-spectrum beta-\\nlactamase-producing Enterobacteriaceae from humans, companion animals\\nand horses in central Hesse, Germany. BMC Microbiol. 14:187. doi: 10.1186/\\n1471-2180-14-187\\nSchrijver, R., Stijntjes, M., Rodriguez-Bano, J., Tacconelli, E., Babu Rajendran,\\nN., and Voss, A. (2018). Review of antimicrobial resistance surveillance\\nprogrammes in livestock and meat in EU with focus on humans.Clin. Microbiol.\\nInfect. 24, 577ñ590. doi: 10.1016/j.cmi.2017.09.013\\nSchwaiger, K., Schmied, E. M., and Bauer, J. (2008). Comparative analysis of\\nantibiotic resistance characteristics of Gram-negative bacteria isolated from\\nlaying hens and eggs in conventional and organic keeping systems in Bavaria,\\nGermany. Zoonoses Public Health 55, 331ñ341. doi: 10.1111/j.1863-2378.2008.\\n01151.x\\nSharland, M., Pulcini, C., Harbarth, S., Zeng, M., Gandra, S., Mathur, S., et al.\\n(2018). Classifying antibiotics in the WHO Essential Medicines List for optimal\\nuse. Lancet Infect. Dis. 18, 18ñ20. doi: 10.1016/s1473-3099(17)30724-7\\nSimner, P. J., Gilmour, M. W., DeGagne, P., Nichol, K., and Karlowsky, J. A. (2015).\\nEvaluation of ﬁve chromogenic agar media and the Rosco Rapid Carb screen kit\\nfor detection and conﬁrmation of carbapenemase production in Gram-negative\\nbacilli. J. Clin. Microbiol. 53, 105ñ112. doi: 10.1128/jcm.02068-14\\nStolle, I., Prenger-Berninghoﬀ, E., Stamm, I., Scheufen, S., Hassdenteufel, E.,\\nGuenther, S., et al. (2013). Emergence of OXA-48 carbapenemase-producing\\nEscherichia coli and Klebsiella pneumoniae in dogs. J. Antimicrob. Chemother.\\n68, 2802ñ2808. doi: 10.1093/jac/dkt259\\nTato, M., Coque, T. M., Baquero, F., and Canton, R. (2010). Dispersal of\\ncarbapenemase blaVIM-1 gene associated with diﬀerent Tn402 variants,\\nmercury transposons, and conjugative plasmids in Enterobacteriaceae and\\nPseudomonas aeruginosa . Antimicrob. Agents Chemother. 54, 320ñ327. doi:\\n10.1128/aac.00783-09\\nVergara, A., Pitart, C., Montalvo, T., Roca, I., Sabate, S., Hurtado, J. C., et al. (2017).\\nPrevalence of extended-spectrum-beta-lactamase- and/or carbapenemase-\\nproducing Escherichia coli isolated from yellow-legged gulls from barcelona,\\nSpain. Antimicrob. Agents Chemother. 61:e02071-16.\\nViau, R., Frank, K. M., Jacobs, M. R., Wilson, B., Kaye, K., Donskey, C. J., et al.\\n(2016). Intestinal carriage of carbapenemase-producing organisms: current\\nstatus of surveillance methods. Clin. Microbiol. Rev. 29, 1ñ27. doi: 10.1128/\\ncmr.00108-14\\nViau, R. A., Hujer, A. M., Marshall, S. H., Perez, F., Hujer, K. M., Briceño, D. F.,\\net al. (2012). \"Silent\" dissemination of Klebsiella pneumoniae isolates bearing\\nK. pneumoniae carbapenemase in a long-term care facility for children and\\nyoung adults in Northeast Ohio. Clin. Infect. Dis. 54, 1314ñ1321. doi: 10.1093/\\ncid/cis036\\nVittecoq, M., Laurens, C., Brazier, L., Durand, P., Elguero, E., Arnal, A.,\\net al. (2017). VIM-1 carbapenemase-producing Escherichia coli in gulls from\\nsouthern France. Ecol. Evol. 7, 1224ñ1232. doi: 10.1002/ece3.2707\\nVrioni, G., Daniil, I., Voulgari, E., Ranellou, K., Koumaki, V., Ghirardi,\\nS., et al. (2012). Comparative evaluation of a prototype chromogenic\\nmedium (ChromID CARBA) for detecting carbapenemase-producing\\nEnterobacteriaceae in surveillance rectal swabs. J. Clin. Microbiol. 50,\\n1841ñ1846. doi: 10.1128/jcm.06848-11\\nWagman, J., and Weneck, E. J. (1963). Preservation of bacteria by circulating-gas\\nfreeze drying. Appl. Microbiol. 11, 244ñ248. doi: 10.1128/aem.11.3.244-248.\\n1963\\nWein, T., Hülter, N. F., Mizrahi, I., and Dagan, T. (2019). Emergence of plasmid\\nstability under non-selective conditions maintains antibiotic resistance. Nat.\\nCommun. 10, 2595ñ2595.\\nWilkinson, K. M., Winstanley, T. G., Lanyon, C., Cummings, S. P., Raza, M. W.,\\nand Perry, J. D. (2012). Comparison of four chromogenic culture media for\\ncarbapenemase-producing Enterobacteriaceae. J. Clin. Microbiol. 50, 3102ñ\\n3104. doi: 10.1128/jcm.01613-12\\nWoodford, N., Wareham, D. W., Guerra, B., and Teale, C. (2014). Carbapenemase-\\nproducing Enterobacteriaceae and non- Enterobacteriaceae from animals and\\nthe environment: an emerging public health risk of our own making?\\nJ. Antimicrob. Chemother. 69, 287ñ291. doi: 10.1093/jac/dkt392\\nZhanel, G., Wiebe, R., Dilay, L., Thomson, K., Rubinstein, E., Hoban, D. J., et al.\\n(2007). Comparative review of the carbapenems. Drugs 67, 1027ñ1052. doi:\\n10.2165/00003495-200767070-00006\\nZurﬂuh, K., Hindermann, D., Nuesch-Inderbinen, M., Poirel, L., Nordmann,\\nP., and Stephan, R. (2016). Occurrence and features of chromosomally\\nencoded carbapenemases in Gram-negative bacteria in farm animals sampled\\nat slaughterhouse level. Schweiz. Arch. Tierheilkd. 158, 457ñ460. doi: 10.17236/\\nsat00072\\nConﬂict of Interest: The authors declare that the research was conducted in the\\nabsence of any commercial or ﬁnancial relationships that could be construed as a\\npotential conﬂict of interest.\\nCopyright © 2020 Pauly, Hammerl, Grobbel, Tenhagen, Käsbohrer, Bisenius,\\nFuchs, Horlacher, Lingstädt, Mauermann, Mitro, Müller, Rohrmann, Schiﬀmann,\\nStührenberg, Zimmermann, Schwarz, Meemken and Irrgang. This is an open-access\\narticle distributed under the terms of the Creative Commons Attribution License\\n(CC BY). The use, distribution or reproduction in other forums is permitted, provided\\nthe original author(s) and the copyright owner(s) are credited and that the original\\npublication in this journal is cited, in accordance with accepted academic practice. No\\nuse, distribution or reproduction is permitted which does not comply with these terms.\\nFrontiers in Microbiology | www.frontiersin.org 10 August 2020 | Volume 11 | Article 1678')]\n",
      "\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Content of Pauly,method comparison, 2021.pdf:\n",
      "[Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 0}, page_content='microorganisms\\nArticle\\nIsolation Procedure for CP E. colifrom Caeca Samples under\\nReview towards an Increased Sensitivity\\nNatalie Pauly 1, Yvonne Klaar 1, Tanja Skladnikiewicz-Ziemer 1, Katharina Juraschek 1, Mirjam Grobbel 1\\n ,\\nJens André Hammerl 1\\n , Lukas Hemmers 2, Annemarie Käsbohrer 1,3\\n , Stefan Schwarz 4,5\\n , Diana Meemken 6\\n ,\\nBernd-Alois Tenhagen 1 and Alexandra Irrgang 1, *\\n/gid00030/gid00035/gid00032/gid00030/gid00038/gid00001/gid00033/gid00042/gid00045/gid00001\\n/gid00048/gid00043/gid00031/gid00028/gid00047/gid00032/gid00046\\nCitation: Pauly, N.; Klaar, Y.;\\nSkladnikiewicz-Ziemer, T.; Juraschek,\\nK.; Grobbel, M.; Hammerl, J.A.;\\nHemmers, L.; Käsbohrer, A.; Schwarz,\\nS.; Meemken, D.; et al. Isolation\\nProcedure for CP E. coli from Caeca\\nSamples under Review towards an\\nIncreased Sensitivity. Microorganisms\\n2021, 9, 1105. https://doi.org/\\n10.3390/microorganisms9051105\\nAcademic Editor: Labrini\\nV . Athanasiou\\nReceived: 30 March 2021\\nAccepted: 14 May 2021\\nPublished: 20 May 2021\\nPublisher’s Note:MDPI stays neutral\\nwith regard to jurisdictional claims in\\npublished maps and institutional afﬁl-\\niations.\\nCopyright: © 2021 by the authors.\\nLicensee MDPI, Basel, Switzerland.\\nThis article is an open access article\\ndistributed under the terms and\\nconditions of the Creative Commons\\nAttribution (CC BY) license (https://\\ncreativecommons.org/licenses/by/\\n4.0/).\\n1 German Federal Institute for Risk Assessment (Bundesinstitut für Risikobewertung),\\nD-10589 Berlin, Germany; natalie_pauly@hotmail.com (N.P .); Yvonne.Klaar@bfr.bund.de (Y.K.);\\nTanja.Skladnikiewicz-Ziemer@bfr.bund.de (T.S.-Z.); Katharina.juraschek@bfr.bund.de (K.J.);\\nmirjam.grobbel@bfr.bund.de (M.G.); Jens-Andre.Hammerl@bfr.bund.de (J.A.H.);\\nannemarie.kaesbohrer@bfr.bund.de (A.K.); bernd-alois.tenhagen@bfr.bund.de (B.-A.T.)\\n2 State Ofﬁce for Health and Social Affairs Berlin, D-10639 Berlin, Germany; lukas.hemmers@lageso.berline.de\\n3 Unit for Veterinary Public Health and Epidemiology, University of Veterinary Medicine,\\nAT-1210 Vienna, Austria\\n4 Institute of Microbiology and Epizootics, Freie Universität Berlin, D-14163 Berlin, Germany;\\nStefan.Schwarz@fu-berlin.de\\n5 FAO Reference Centre for Antimicrobial Resistance, Freie Universität Berlin, D-14163 Berlin, Germany\\n6 Institute of Food Safety and Food Hygiene, Freie Universität Berlin, D-14163 Berlin, Germany;\\ndiana.meemken@fu-berlin.de\\n* Correspondence: Alexandra.irrgang@bfr.bund.de\\nAbstract: Due to the increasing reports of carbapenemase-producing Enterobacteriaceae (CPE) from\\nlivestock in recent years, the European Reference Laboratory for Antimicrobial Resistances (EURL-\\nAR) provided a protocol for their recovery from caecum and meat samples. This procedure exhibited\\nlimitations for the detection of CPE with low carbapenem MIC values. Therefore, it was modiﬁed by a\\nsecond, selective enrichment in lysogeny broth with cefotaxime (CTX 1 mg/L) and with meropenem\\n(MEM 0.125 mg/L) at 37 ◦C under microaerophilic conditions. By Real-time PCR, these enrichments\\nare pre-screened for the most common carbapenemase genes. Another adaptation was the use of\\nin-house prepared MacConkey agar with MEM and MEM+CTX instead of commercial selective agar.\\nAccording to the EURL-method, we achieved 100% sensitivity and speciﬁcity using the in-house\\nmedia instead of commercial agar, which decreased the sensitivity to ~75%. Comparing the method\\nwith and without the second enrichment, no substantial inﬂuence on sensitivity and speciﬁcity was\\ndetected. Nevertheless, this enrichment has simpliﬁed the CPE-isolation regarding the accompanying\\nmicrobiota and the separation of putative colonies. In conclusion, the sensitivity of the method can\\nbe increased with slight modiﬁcations.\\nKeywords: Isolation; carbapenemase; CPE detection; selective media; speciﬁcity; sensitivity\\n1. Introduction\\nThe World Health Organization deﬁnes carbapenems as ‘High Priority Critically Im-\\nportant Antimicrobials’ for human medicine [1]. This is due to their role in the treatment of\\nsevere human infections with multidrug-resistant bacteria. Up to now, in European coun-\\ntries carbapenems are not licensed for use in veterinary medicine. Nevertheless, carbapen-\\nems may be used in speciﬁc circumstances in companion animals [ 2,3]. In recent years,\\nan increasing number of carbapenem resistant enterobacteria, including carbapenemase-\\nproducers, have been reported [4]. Interestingly, the reports are not only limited to human\\nmedicine [4–6] as carbapenem-resistant bacteria (i.e., E. coli, Salmonella enterica subsp.\\nenterica) have also been detected repeatedly in environmental or animal samples [ 4,5].\\nMicroorganisms 2021, 9, 1105. https://doi.org/10.3390/microorganisms9051105 https://www.mdpi.com/journal/microorganisms'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 1}, page_content='Microorganisms 2021, 9, 1105 2 of 14\\nBesides various chromosomally encoded mechanisms (i.e., changes in membrane perme-\\nability, efﬂux pumps), resistance to carbapenems is often mediated by degrading enzymes\\ncalled carbapenemases that can hydrolyze carbapenems and most other β-lactams [7].\\nTypically, the corresponding carbapenemase genes are located on mobile (integrative and\\nconjugative) genetic elements (i.e., plasmids) [8,9]. They exhibit different mechanisms of\\naction (i.e., serine-based (i.e., KPC, OXA) and zinc-catalysed carbapenemases (i.e., NDM,\\nVIM)) [10,11]. Carbapenemases usually show a broad activity against most β-lactams and\\nare typically associated with resistance determinants against other antimicrobial classes\\noften leading to a drastic reduction of therapeutic options in case of infections [12,13]. In the\\nEuropean Union (EU), OXA, VIM, NDM, and KPC carbapenemases are most frequently\\ndetected in human clinical settings [5,14,15]. In 2012, the ﬁrst carbapenemase-producing\\n(CP) E. coli was detected in livestock [16], and voluntary monitoring of CP E. coli in the EU\\nmember states along with the obligatory monitoring on extended spectrum β-lactamase\\n(ESBL)/AmpC β-lactamases producing E. coli (CID 2013/652/EU) [17] was introduced.\\nIn 2021, through new legislation, the speciﬁc monitoring for carbapenem resistant E. coli\\nbecame mandatory (CID 2020/1729/EU). For selective CP E. coli detection, the isolation\\nmethod provided by the European Reference Laboratory for Antimicrobial Resistances\\n(EURL-AR) (https://www.eurl-ar.eu/protocols.aspx, accessed on December 2019) is rec-\\nommended. The method is based on a 1:10 non-selective enrichment of the samples in\\nbuffered peptone water (BPW), which needs to be inoculated onto suitable selective agars\\n(i.e., the commercial chromID® CARBA agar). Several studies described the chromID ®\\nCARBA agar as one of the most sensitive and speciﬁc chromogenic media for the detection\\nof CP Enterobacteriaceae (CPE) in clinical samples in human medicine [18–21]. In Germany,\\nthis agar represents one of the most commonly used selective media for CPE monitoring\\npurposes [22]. By using a highly sensitive molecular pre-screening of enrichment cultures\\n(i.e., by real-time PCR), laborious microbiological investigations might be avoided, and\\nfurther efforts can focus on the recovery of CPE from presumptive-positive samples.\\nUnfortunately, for some CPE detected in German livestock the EURL-AR detection\\nmethod failed in recent years [23,24]. For example, the isolate 17-AB02384 was not detected\\nwithin the monitoring of CP E. coli but was recovered in the monitoring on ESBL/AmpC\\nβ-lactamase producing E. coli from the same sample [24]. Thus, we identiﬁed the necessity\\nto improve the isolation method for CP E. coli from livestock samples, especially from\\ncaecum content. Difﬁculties in the microbiological processing of caeca samples are the\\ngrowth of an accompanying microbiota, the low CPE concentrations, and the considerably\\nlower MIC values of CPE from animal origin in comparison to human isolates [22]. Here,\\nwe present comparative analyses on the performance of the EURL-method with a number\\nof alterations potentially leading to an improvement of the CPE detection and recovery rate.\\nThe focus of this study was on the recovery of CP E. coli from pig caecum, as the majority\\nof the recently described CP E. coli from German livestock originate from this matrix.\\nTherefore, samples were spiked and blinded to calculate the sensitivity and speciﬁcity of\\nthe approaches, which included different selective agars.\\n2. Materials and Methods\\n2.1. Pre-Studies\\nInitial experiments were carried out to determine the growth and the survival of CPE\\nin feces. Therefore, the survival rate (colony forming units (cfu)/g pig feces) of seven\\ndifferent CPE (four VIM-1 producing E. coli, two VIM-1 producing Salmonella (S.) Infantis\\nand one NDM-1 producing S. Corvallis) was investigated over a period of 10 days. For the\\nalternative method to be tested, spiked samples were prepared. For this, the seven CPE\\nwere inoculated each in lysogeny broth (LB) supplemented with 1 mg/L cefotaxime (CTX)\\n(LB + CTX) and were incubated at 37 ◦C for 16–18 h. After incubation, 10 µL of the\\nsuspension was transferred into LB and was additionally incubated at 37 ◦C with 180 rpm\\nuntil an optical density (OD600 nm) of 0.5 was reached. The suspensions were centrifuged for\\n10 min at 4000 rpm and resuspended in 0.9 % saline solution (w/v). Thereafter, 5 mL of the'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 2}, page_content='Microorganisms 2021, 9, 1105 3 of 14\\nsolution (~108 cfu/mL) was applied to 45 g feces and homogenized for 1 min in a BagMixer\\n400SW (Interscience, Wiesbaden, Germany). The prepared samples were stored at 4–6 ◦C.\\nEvery 24 h, 1 g of the spiked samples was diluted in 0.9% saline solution to 10−6. Aliquots\\nof 100 µL of the 10−5 and the 10−6 dilutions were plated out onto selective MacConkey\\nagar (McC) supplemented with 0.125 mg/L meropenem (MEM) (McC+MEM) and 1 mg/L\\nCTX and MEM (McC + CTX + MEM), and the cfu were counted after incubation for 20–24 h\\nat 37 ◦C.\\nTo extend the time until sample processing in the laboratory, a potential improve-\\nment of the CPE survival in fecal samples was tested by adding stabilizing substances.\\nTrehalose [25,26], glycerol [27,28], and sodium chloride [ 29,30] were tested as potential\\nadditives for the improvement of the recovery rate of the seven CPE in pig feces. Previous\\nstudies reported on the protective characteristics or the enhancive inﬂuence of these sub-\\nstances for the recovery and growth of Enterobacteriaceae [25–30]. Therefore, the described\\nprevious experiments were repeated by adding 1 mL of the supplement (trehalose, glycerol,\\nand sodium acid) to 9 g of the caeca sample. The samples were thoroughly mixed for 1 min\\nand a decimal dilution series up to 10−6 in 0.9% saline solution was prepared. Afterwards,\\n100 µL of the 10−5 and the 10−6 dilution were applied onto selective McC + CTX + MEM,\\nincubated overnight at 37 ◦C, and cfu/g were determined.\\nThe influence of different liquid media (Mossel medium [31] and LB) on CPE growth\\nwas tested to determine the most appropriate culture medium for the second enrichment step.\\nFurthermore, possible influences of a sublethal concentration of MEM (0.008 µg/mL) [32] and\\nthe addition of ZnSO4 (70 µg/mL) [33] to the two liquid media were tested. Gullberg et al.\\n(2014) [34] reported that sublethal antimicrobial concentrations are able to induce bacteria\\nto retain plasmids and express corresponding genes.\\nMoreover, the growth of a CP E. coli and a CP Salmonella in LB + CTX and LB + MEM\\nunder aerobic and microaerobic conditions were compared. For spiking the samples, each\\nCPE was inoculated in 4 mL BPW and incubated at 37 ◦C overnight. Thereafter, 50 µL of\\nthe culture was transferred to BPW and incubated at 37 ◦C and 180 rpm until an optical\\ndensity (OD600 nm) of 1.0 was reached. From this suspension, 1 mL was added to 9 g fresh\\ncaeca sample. Thereafter, each 1 mL of this mix was transferred to 9 mL LB + CTX and\\nincubated at 37 ◦C either under aerobic or under microaerobic conditions for 18 h. After\\nincubation, 100 µL of the sample were plated four times onto McC + CTX + MEM and cfu\\nwere counted after 20–24 h of incubation at 37 ◦C. Potential differences induced by the two\\nincubation conditions were assessed through a two-sided Wilcoxon signed rank test for\\nindependent samples.\\n2.2. Method Comparison\\n2.2.1. Bacterial Strains\\nIn total, twelve CP E. coli were used for the evaluation of both isolation methods\\nand the comparison of the different selective media. These isolates cover a wide range of\\ncarbapenemase genes (blaVIM-1; blaGES-5; blaKPC-2; blaNDM-1 and blaOXA-48) (Supplemental\\nMaterial Table S1). As reference for quality assurance concerning the selectivity for E. coli\\nof the media, the E. coli ATCC 25922 was used. In all three approaches, six caeca samples\\nwere CPE-negative and inoculated with the E. coli ATCC 25922. In each of the other\\ntwelve caeca samples, one of the twelve CP E. coli was inoculated with a concentration of\\n100 cfu/mL each.\\n2.2.2. Sample Preparation for the Method Comparison\\nAll twelve CP E. coli and the E. coli control strain ATCC 25922 were inoculated in\\n4 mL BPW and incubated at 37 ◦C overnight. After incubation, 50 µL of the overnight\\nculture was transferred to fresh BPW and incubated at 37 ◦C at 180 rpm until an optical\\ndensity (OD600 nm) of 1 was reached. For each CP E. coli, 9 g fresh caeca sample was\\nartiﬁcially contaminated with 1 mL of the 10−5 diluted bacterial suspension. For the control\\nstrain, six caeca samples were spiked. All spiked caeca samples were homogenized by'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 3}, page_content='Microorganisms 2021, 9, 1105 4 of 14\\nthorough mixing for 1 min. After sample preparation, the samples were blinded by a\\nnon-participating person. All 18 samples were analyzed by two methods (Figure 1) as\\ndescribed below. First, 1 g of each spiked sample was added to 9 mL BPW and incubated at\\n37 ± 2 ◦C for 16–18 h. Thereafter, the samples were separately subjected to investigations\\nusing the two different isolation procedures. For reliable interpretation, the 18 blinded\\nsamples were investigated with both methods in three independent approaches.\\nMicroorganisms 2021, 9, x FOR PEER REVIEW 4 of 15 \\n \\n \\n2.2.2. Sample Preparation for the Method Comparison \\nAll twelve CP E. coli and the E. coli control strain ATCC 25922 were inoculated in 4 \\nmL BPW and incubated at 37 °C overnight. After incubation, 50 µL of the overnight \\nculture was transferred to fresh BPW and incubated at 37 °C at 180 rpm until an optical \\ndensity (OD600 nm) of 1 was reached. For each CP E. coli, 9 g fresh caeca sample was \\nartificially contaminated with 1 mL of the 10−5 diluted bacterial suspension. For the control \\nstrain, six caeca samples were spiked. All spiked caeca samples were homogenized by \\nthorough mixing for 1 min. After sample preparation, the samples were blinded by a non-\\nparticipating person. All 18 samples were analyzed by two methods (Figure 1) as \\ndescribed below. First, 1 g of each spiked sample was added to 9 mL BPW and incubated \\nat 37 ± 2 °C for 16–18 h. Thereafter, the samples were separately subjected to investigations \\nusing the two different isolation procedures. For reliable interpretation, the 18 blinded \\nsamples were investigated with both methods in three independent approaches. \\n \\nFigure 1. The whole procedure of method A and method B to compare both and to validate the agar plates. BPW = Buffered \\nPeptone Water; McC+CTX+MEM = MacConkey Agar supplemen ted with 1mg/L cefotaxim and 0.125 mg/L meropenem; \\nMcC+MEM = MacConkey Agar supplemented with 0.125 mg/ L meropenem; LB+CTX = lysogeny broth supplemented \\nwith 1mg/L cefotaxim; LB+MEM = lysogeny broth supplemented with 0.125 mg/L meropenem. \\n2.2.3. Isolation and Detection Methods \\nMethod A represents the EURL-protocol for isolation of ESBL, AmpC and CP E. coli \\nfrom caeca, (accessed in December 2019) (https://www.eurl-ar.eu/protocols.aspx). For the \\nmethod comparison, the commercial chromID ® CARBA SMART agar (bioMérieux, \\nNürtingen, Germany) was used. This agar is composed of the CARBA agar and the OXA \\nagar in equal parts. Therefore, the EURL-protocol was adapted by applying 5 µl on a half \\nagar plate instead of 10 µl on a full agar plate. This adaption allowed the agar validation \\nas part of the method comparison (5 µL per half chromID® CARBA SMART agar plate, 10 \\nµL per a whole McC agar plate) (Figure 2). Additionally, a Real-Time PCR step was added \\nafter the enrichment in BPW (described below). \\nFigure 1. The whole procedure of method A and method B to compare both and to validate the agar plates. BPW = Buffered\\nPeptone Water; McC + CTX + MEM = MacConkey Agar supplemented with 1 mg/L cefotaxim and 0.125 mg/L meropenem;\\nMcC + MEM = MacConkey Agar supplemented with 0.125 mg/L meropenem; LB + CTX = lysogeny broth supplemented\\nwith 1 mg/L cefotaxim; LB+MEM = lysogeny broth supplemented with 0.125 mg/L meropenem.\\n2.2.3. Isolation and Detection Methods\\nMethod A represents the EURL-protocol for isolation of ESBL, AmpC and CP E. coli\\nfrom caeca, (https://www.eurl-ar.eu/protocols.aspx, accessed in December 2019). For the\\nmethod comparison, the commercial chromID® CARBA SMART agar (bioMérieux, Nürtin-\\ngen, Germany) was used. This agar is composed of the CARBA agar and the OXA agar in\\nequal parts. Therefore, the EURL-protocol was adapted by applying 5 µL on a half agar\\nplate instead of 10 µL on a full agar plate. This adaption allowed the agar validation as\\npart of the method comparison (5 µL per half chromID® CARBA SMART agar plate, 10 µL\\nper a whole McC agar plate) (Figure 2). Additionally, a Real-Time PCR step was added\\nafter the enrichment in BPW (described below).\\nMethod B represents the modiﬁed version of the isolation procedure. Therein, non-\\nselective enrichment in BPW has been maintained, followed by a second selective en-\\nrichment step. For that purpose, 10 µL of the BPW pre-enrichment has been applied to\\n10 mL LB + CTX and 10 mL LB + MEM. Both enrichments were incubated at 37 ± 2 ◦C for\\n16–18 h under microaerophilic conditions. Thereafter, 5 µL of the LB + CTX enrichment\\nwas inoculated on the CARBA side, and 5 µL of the LB + MEM enrichment on the OXA\\nside of the chromID® CARBA SMART plate (Figure 1). In addition, 10 µL of LB + CTX\\nor LB + MEM were inoculated on in-house selective agars McC + CTX + MEM or McC\\n+ MEM, respectively. Each sample was stroked on each agar in duplicate (six plates per\\nsample). Plates were incubated for 16–18 h at 37 ± 2 ◦C. Subsequently, one colony with'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 4}, page_content='Microorganisms 2021, 9, 1105 5 of 14\\nE. coli typical colony morphology was picked from each plate and enriched in 4 mL LB\\n+ MEM. The species of each colony was conﬁrmed by PCR and MALDI-ToF. Conﬁrmed\\ncolonies were incubated for 16–18 h at 37 ± 2 ◦C in LB and 500 µL aliquots were stored\\nin 800 µL of 80% glycerol for further analysis. Samples spiked with OXA-producers were\\nassessed as positive if growth was observed on the chromID® OXA side, likewise on McC\\n+ MEM. For the other CP E. coli, the sample was regarded positive if growth was observed\\nat least on chromID® CARBA side or on McC + CTX + MEM.\\nMicroorganisms 2021, 9, x FOR PEER REVIEW 5 of 15 \\n \\n \\n \\nFigure 2. Spreading the sample on the plate. The first spread (blue, first third of the plate) was \\nperformed by using a 10 µL-loop. A second spread (green, second third of the plate) was \\nperformed by another 10 µL-loop, which was turned the last third of the spread (yellow). \\nMethod B represents the modified version of the isolation procedure. Therein, non-\\nselective enrichment in BPW has been maintained, followed by a second selective \\nenrichment step. For that purpose, 10 µL of the BPW pre-enrichment has been applied to \\n10 mL LB + CTX and 10 mL LB + MEM. Both enrichments were incubated at 37 ± 2 °C for \\n16–18 h under microaerophilic conditions. Ther eafter, 5 µL of the LB + CTX enrichment \\nwas inoculated on the CARBA side, and 5 µL of the LB + MEM enrichment on the OXA \\nside of the chromID® CARBA SMART plate (Figure 1). In addition, 10 µL of LB + CTX or \\nLB + MEM were inoculated on in-house selective agars McC + CTX + MEM or McC + \\nMEM, respectively. Each sample was stroked on each agar in duplicate (six plates per \\nsample). Plates were incubated for 16–18 h at 37 ± 2 °C. Subsequently, one colony with E. \\ncoli typical colony morphology was picked from each plate and enriched in 4 mL LB + \\nMEM. The species of each colony was confirmed by PCR and MALDI-ToF. Confirmed \\ncolonies were incubated for 16–18 h at 37 ± 2 °C in LB and 500 µL aliquots were stored in \\n800 µL of 80% glycerol for further analysis. Samples spiked with OXA-producers were \\nassessed as positive if growth was observed on the chromID® OXA side, likewise on McC \\n+ MEM. For the other CP E. coli, the sample was regarded positive if growth was observed \\nat least on chromID® CARBA side or on McC + CTX + MEM. \\nTo confirm the validity of the results,  the whole experiment was independently \\nrepeated three times. Moreover, to ensure th at personal handling had no influence, both \\nmethods were carried out by two different people per approach. Moreover, each agar type \\nwas used in duplicate and deviations (i.e., colony morphology) were noted. If at least one \\ncolony could be isolated and confirmed from one of the duplicates, the sample was rated \\npositive for the corresponding method and agar type. \\n2.2.4. Typing \\nAntimicrobial Susceptibility-Testing (AST) \\nThe antimicrobial susceptibility of the isolates (Supplemental Material Table S1) was \\ndetermined by broth microdilution using defined antimicrobial substances and \\nconcentrations from the harmonized EU panel following Commission Implementing \\nDecision (CID) 2013/652/EU [plates EUVSEC  & EUVSEC2; TREK Diagnostic Systems \\n(Thermo Fisher Scientific, Schwerte, Germany)]. AST was conducted according to the EN \\nISO20776-1:2006 [15] and MIC values were interpreted based on EUCAST definitions \\n(www.eucast.org, November 2013) using epidemiological cut-off values fixed in CID \\n2013/652/2013. The strain ATCC 25922 was used for quality control. All isolates were \\ntested twice. First, before using them for ar tificial contamination and second, after their \\nsuccessful recovery from the samples to assess possible changes in their antimicrobial \\nresistance patterns. \\n  \\nFigure 2. Spreading the sample on the plate. The ﬁrst spread (blue, ﬁrst third of the plate) was\\nperformed by using a 10 µL-loop. A second spread (green, second third of the plate) was performed\\nby another 10 µL-loop, which was turned the last third of the spread (yellow).\\nTo conﬁrm the validity of the results, the whole experiment was independently re-\\npeated three times. Moreover, to ensure that personal handling had no inﬂuence, both\\nmethods were carried out by two different people per approach. Moreover, each agar type\\nwas used in duplicate and deviations (i.e., colony morphology) were noted. If at least one\\ncolony could be isolated and conﬁrmed from one of the duplicates, the sample was rated\\npositive for the corresponding method and agar type.\\n2.2.4. Typing\\nAntimicrobial Susceptibility-Testing (AST)\\nThe antimicrobial susceptibility of the isolates (Supplemental Material Table S1) was\\ndetermined by broth microdilution using deﬁned antimicrobial substances and concentra-\\ntions from the harmonized EU panel following Commission Implementing Decision (CID)\\n2013/652/EU [plates EUVSEC & EUVSEC2; TREK Diagnostic Systems (Thermo Fisher Sci-\\nentiﬁc, Schwerte, Germany)]. AST was conducted according to the EN ISO20776-1:2006 [15]\\nand MIC values were interpreted based on EUCAST deﬁnitions (www.eucast.org, Novem-\\nber 2013) using epidemiological cut-off values ﬁxed in CID 2013/652/2013. The strain\\nATCC 25922 was used for quality control. All isolates were tested twice. First, before\\nusing them for artiﬁcial contamination and second, after their successful recovery from the\\nsamples to assess possible changes in their antimicrobial resistance patterns.\\nMALDI-ToF MS\\nOne putative colony of CP E. coli of each agar plate (six per sample) was chosen\\nfor species conﬁrmation by MALDI-ToF MS. As matrix α-Cyano-4-hydroxycinnamic acid\\n(HCCA, Bruker, USA) was used, and analyses were performed by MALDI Microﬂex\\nBiotyper (Bruker Daltonics, Bremen, Germany) as recommended by the manufacturers.\\nMolecular Typing of CP E. coli\\nMolecular detection of the carbapenemase genes was carried out to conﬁrm the\\npresence/absence of blaGES, blaKPC, blaNDM, blaOXA-48 and blaVIM in enrichment cultures\\nand in presumptive colonies. Both sample enrichments and presumptive colonies were'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 5}, page_content='Microorganisms 2021, 9, 1105 6 of 14\\nused for extraction of template DNA by heat treatment of bacterial suspensions. For this\\nextraction, 800 µL were centrifuged by 10,000× g for 5 min. The cell pellet was resuspended\\nin 300 µL double distilled water and boiled for 10 min at 99◦C. Thereafter, the samples were\\ncooled on ice and centrifuged for 2 min at 16,000× g. Finally, 255 µL of the supernatant was\\nmixed with 45 µL trehalose for stable storage at−20 ◦C. Real-time PCR was performed on a\\nBio-Rad CFX system. Primers and probes were adapted from Swayne et al. (2011) [35] and\\nvan der Zee et al. (2014) [36]. Ampliﬁcation was conducted using Biozym qPCR Mastermix\\n(Biozym Scientiﬁc, Oldendorf, Germany) and the following conditions: Initially 2 min at\\n95 ◦C, 30 cycles with 5 s at 95 ◦C and 60 s at 60 ◦C for product ampliﬁcation.\\n2.2.5. Statistics\\nResults of both detection methods were expressed in percent (positive samples divided\\nby all tested samples). The EURL-method and the modiﬁed method were compared in\\nterms of their diagnostic accuracy indicated by sensitivity, speciﬁcity, false discovery\\nand omission rates, and overall diagnostic accuracy. At this point, the blinding of the\\nsamples was dissolved. If the detected CP E. coli were conﬁrmed using MALDI-ToF MS\\nand real-time PCR, a sample was classiﬁed as positive. The sensitivity was expressed as the\\nproportion of the positive samples correctly identiﬁed. The speciﬁcity was calculated as\\nthe proportion of the samples correctly identiﬁed as negative among all negative samples.\\nFalse discovery rates were calculated as the proportion of all false positive samples among\\nall positive samples. Likewise, false omission rates were calculated as the proportion of\\nall false negative samples among all negative samples. Overall diagnostic accuracy was\\ncalculated as the proportion of all tests that gave a correct result. All estimates as well\\nas their corresponding exact 95% conﬁdence intervals (Clopper-Pearson 95% conﬁdence\\nintervals as precision measures honoring small sample sizes) were estimated using the\\nepiR package in R [37].\\n3. Results\\n3.1. Modiﬁcation of the EURL-Protocol for Isolation of CP E. coli from Caeca Samples\\nIn order to determine the number of bacteria that survived the prevailing condition\\nin the pig feces, the recovery rate of the bacteria in feces was determined after storage at\\n6 ◦C. Based on the cfu counts, a reduction of 2 log units within 24 h was detected in pig\\nfeces samples stored at 6 ◦C. The addition of glycerin, sodium chloride, and trehalose to\\nthe feces samples did not result in a positive effect on the CPE survival rate (Figure 3).\\nIn a ﬁrst step, the suitability of two liquid media (LB- or Mossel-medium) as basis of\\nthis second enrichment were compared. The Mossel-medium was developed especially for\\nEnterobacteriaceae [31], whereas LB-medium is a nutrient-rich microbial broth used for\\nthe cultivation of E. coli. No differences in the recovery rate were observed in the direct\\ncomparison of LB- or Mossel-medium as second enrichment (data not shown). As LB is a\\ncommon medium used in many laboratories, it was used for further investigations.\\nThe addition of sublethal conditions of ZnSO 4 and MEM was intended to activate\\npotentially present resistance mechanisms in the liquid culture so that the corresponding\\nstrains may have a growth advantage when subsequently plated onto the selective solid\\nmedia. However, the slight selection pressure did not result in a better recovery rate.\\nTherefore, we used the second enrichment in LB + CTX as previously described [ 23].\\nTo isolate the OXA-48 producers as well, we tested another second enrichment in LB\\n+ MEM. Enterobacteriaceae are described as facultative anaerobes [ 38]. Therefore, the\\ntarget species can survive in both aerobic and anaerobic environments, in contrast to\\nsome representatives of the accompanying microbiota. The microaerophilic incubation\\nled to a median increase in isolated CP E. coli from 7 cfu/mL with aerobic incubation to\\n26.5 cfu/mL (Wilcoxon test, W = 0, p = < 0.05, n = 8). Likewise, the median number of CP\\nSalmonella increased from 81 cfu/mL with aerobic incubation to 144.5 cfu/mL (Wilcoxon\\ntest, W = 0, p = < 0.05, n = 8).'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 6}, page_content='Microorganisms 2021, 9, 1105 7 of 14\\nMicroorganisms 2021, 9, x FOR PEER REVIEW 7 of 15 \\n \\n \\nstrains may have a growth advantage when su bsequently plated onto the selective solid \\nmedia. However, the slight selection pressure  did not result in a better recovery rate. \\nTherefore, we used the second enrichment in LB + CTX as previously described [23]. To \\nisolate the OXA-48 producers as well, we tested another second enrichment in LB + MEM. \\nEnterobacteriaceae are described as facultat ive anaerobes [38]. Therefore, the target \\nspecies can survive in both aerobic and anae robic environments, in contrast to some \\nrepresentatives of the accompanying microbiota. The microaerophilic incubation led to a \\nm e d i a n  i n c r e a s e  i n  i s o l a t e d  C P  E .  c o l i  f r o m  7 cfu/mL with aerobic incubation to 26.5 \\ncfu/mL (Wilcoxon test, W = 0, p = < 0.05, n = 8). Likewise, the median number of CP \\nSalmonella increased from 81 cfu/mL with aerobic incubation to 144.5 cfu/mL (Wilcoxon \\ntest, W = 0, p = < 0.05, n = 8). \\n \\nFigure 3. The decreasing number of CPE ( E. coli and Salmonella each in average) in pig fa eces with and without some \\nadditives over ten days. \\nBased on the above-mentioned observatio ns, the procedure of the method was \\nmodified to promote the growth of CPE and to reduce the amount of accompanying \\nmicrobiota (Figure 4). The modified procedure includes a first, non-selective enrichment \\nin BPW, followed by two parallel selective enrichments (LB + CTX and LB + MEM) \\nincubated under microaerophilic conditions. Aliquots (10 µL) of the second enrichment \\nwere applied onto McC agar supplemented either with MEM or CTX + MEM. In the \\nfollowing chapter, this modified method was compared to the EURL-method. \\nFigure 3. The decreasing number of CPE ( E. coli and Salmonella each in average) in pig faeces with and without some\\nadditives over ten days.\\nBased on the above-mentioned observations, the procedure of the method was modi-\\nﬁed to promote the growth of CPE and to reduce the amount of accompanying microbiota\\n(Figure 4). The modiﬁed procedure includes a ﬁrst, non-selective enrichment in BPW,\\nfollowed by two parallel selective enrichments (LB + CTX and LB + MEM) incubated under\\nmicroaerophilic conditions. Aliquots (10 µL) of the second enrichment were applied onto\\nMcC agar supplemented either with MEM or CTX + MEM. In the following chapter, this\\nmodiﬁed method was compared to the EURL-method.\\n3.2. The Comparison of the Modiﬁed Method with the EURL-Method for Isolation of the CP E. coli\\nfrom Caeca Samples\\nTo determine the performance of the modiﬁed method (method B), its diagnostic\\naccuracy was compared to the reference method of the EURL-AR, the ofﬁcial protocol for\\nisolation of ESBL, AmpC and CP E. coli from caeca samples (method A). The complete\\nprocedure of the comparison and validation is schematically illustrated in Figure 1. In the\\nfollowing, the results of the three repetitions were considered together (sample size n = 54)\\nfor each method and agar type (Table 1).\\nTable 1. Summarized results on the repeated isolation of CPE from the spiked fecal samples (calculation by combining the OXA\\nand CARBA- site of the ChromID® SMART CARBA agars and of the McC + CTX + MEM and the McC + MEM plate of the\\nin-house agar). Method B is the modified method. Corresponding exact 95% confidence intervals are provided in brackets.\\nMethod A +\\nChromID® SMART\\nCARBA agars\\nMethod A +\\nin-house agars\\nMethod B +\\nin-house agars\\nMethod B +\\nChromID® SMART\\nCARBA agars\\nSensitivity 75 (57.8–87.9) 100 (90.2–100) 86.1 (70.5–95.3) 66.7 (49–81.4)\\nSpeciﬁcity 100 (81.5–100) 100 (81.5–100) 100 (81.5–100) 100 (81.5–100)\\nFalse discovery rate 0 (0–12.8) 0 (0–9.7) 0 (0–11.2) 0 (0–14.2)\\nFalse omission rate 33.3 (16.5–54) 0 (0–18.5) 21.7 (7.5–43.7) 40 (22.7–59.4)\\nAccuracy 83.3 (70.7–92.1) 100 (93.4–100) 90.7 (79.7–96.9) 77.8 (64.4–88)'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 7}, page_content='Microorganisms 2021, 9, 1105 8 of 14\\nMicroorganisms 2021, 9, x FOR PEER REVIEW 8 of 15 \\n \\n \\n \\nFigure 4. The final step-by-step guidance of the modified method (method B). BPW = Buffered \\nPeptone Water; McC + CTX + MEM = MacConkey Agar supplemented with 1mg/L cefotaxim and \\n0.125 mg/L meropenem; McC + MEM = MacConkey Agar supplemented with 0.125 mg/L \\nmeropenem; LB + CTX = lysogeny broth supplemented with 1mg/L cefotaxim; LB + MEM = \\nlysogeny broth supplemented with 0.125 mg/L meropenem. \\n3.2. The Comparison of the Modified Method with the EURL-Method for Isolation of \\nthe CP E. coli from Caeca Samples \\nTo determine the performance of the modi fied method (method B), its diagnostic \\naccuracy was compared to the reference method of the EURL-AR, the official protocol for \\nisolation of ESBL, AmpC and CP E. coli fr om caeca samples (method A). The complete \\nprocedure of the comparison and validation is schematically illustrated in Figure 1. In the \\nfollowing, the results of the three repetiti ons were considered together (sample size n = \\n54) for each method and agar type (Table 1). \\nTable 1. Summarized results on the repeated isolation of CPE from the spiked fecal samples (calculation by combining the \\nOXA and CARBA- site of the ChromID ® SMART CARBA agars and of the McC + CTX + MEM and the McC + MEM plate \\nof the in-house agar). Method  B is the modified method. Corresponding exac t 95% confidence intervals are provided in \\nbrackets. \\n Method A + ChromID ® \\nSMART CARBA agars \\nMethod A +  \\nin-house agars \\nMethod B +  \\nin-house agars \\nMethod B + ChromID ® \\nSMART CARBA agars \\nSensitivity 75 (57.8–87.9) 100 (90.2–100) 86.1 (70.5–95.3) 66.7 (49–81.4) \\nSpecificity 100 (81.5–100) 100 (81.5–100) 100 (81.5–100) 100 (81.5–100) \\nFalse discovery rate 0 (0–12.8) 0 (0–9.7) 0 (0–11.2) 0 (0–14.2) \\nFalse omission rate 33.3 (16.5–54) 0 (0–18.5) 21.7 (7.5–43.7) 40 (22.7–59.4) \\nAccuracy 83.3 (70.7–92.1) 100 (93.4–100) 90.7 (79.7–96.9) 77.8 (64.4–88) \\nFigure 4. The ﬁnal step-by-step guidance of the modiﬁed method (method B). BPW = Buffered\\nPeptone Water; McC + CTX + MEM = MacConkey Agar supplemented with 1 mg/L cefotaxim\\nand 0.125 mg/L meropenem; McC + MEM = MacConkey Agar supplemented with 0.125 mg/L\\nmeropenem; LB + CTX = lysogeny broth supplemented with 1 mg/L cefotaxim; LB + MEM =\\nlysogeny broth supplemented with 0.125 mg/L meropenem.\\nComparison of diagnostic accuracy measures show that the use of method A in\\ncombination with the in-house agar led to a signiﬁcantly higher sensitivity, relative to the\\nuse of method A with the chromID ® SMART CARBA agar. In addition to this overall\\nevaluation, the results for the individual approaches are given in Supplemental Material\\nTable S2. The best CPE detection was achieved by using the original EURL-protocol in\\ncombination with in-house prepared McC + MEM and McC + CTX + MEM agar plates.\\nAll negative samples (n/N) were correctly recognized, and CP E. coli were successfully\\nrecovered from each of the artiﬁcially contaminated samples (n/N). This corresponds to\\na 100% sensitivity and speciﬁcity. The sensitivity decreased to 75% when the in-house\\nagar plates were replaced by the commercial chromID® CARBA agar. However, speciﬁcity\\nremained at 100%. The modiﬁed method (B) reached a sensitivity of 86.1% when using\\nthe in-house agar and a sensitivity of 66.7% when using the commercial agar. However,\\nall negative samples (n/N) were also correctly identiﬁed leading to a speciﬁcity of 100%.\\nThe added value of this modiﬁed method is the simpliﬁcation of the isolation of putative\\ncolonies that are difﬁcult to isolate. This could be shown by counting the agar plates (108 per\\nagar type and method) with and without accompanying microbiota (i.e., Pseudomonas (P .)\\nputida, P . monteilli, P . aeruginosa, Proteus mirabilis) (Table 2). Using the EURL-procedure\\nin combination with the chromID ® CARBA agar, on 44.5% (48/108) of the agar plates\\ngrowth of accompanying microbiota was observed. That was reduced to 39.8% (43/108)\\nusing the modiﬁed method with the chromID® CARBA agar. In addition, for the chromID®\\nOXA agar, a reduction of accompanying microbiota from 23.2% (25/108) of the plates by'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 8}, page_content='Microorganisms 2021, 9, 1105 9 of 14\\nusing the EURL-procedure to 15.7% (17/108) using the modiﬁed method was observed.\\nUsing the in-house agars and the EURL-procedure, accompanying microbiota was observed\\non 37.0% (40/108) of the McC + MEM plates and on 33.4% (36/108) of the McC + CTX +\\nMEM plates. By changing to the modiﬁed method, we detected accompanying microbiota\\non 29.6% (32/108) of the McC + MEM plates and on 36.1% (39/108) of the McC + CTX +\\nMEM plates.\\nTable 2. Validation of the selective agars by counting the number of plates with growing of accompanying microbiota\\nfor each group per used method. McC + CTX + MEM = MacConkey Agar supplemented with 1 mg/L cefotaxim and\\n0.125 mg/L meropenem; McC + MEM = MacConkey Agar supplemented with 0.125 mg/L meropenem.\\nChromID® SMART\\nCARBA Side\\nChromID® SMART\\nOXA Side McC + MEM McC + CTX + MEM\\nMethod A 48/108\\n(44.45%)\\n25/108\\n(23.15%)\\n40/108\\n(37.04%)\\n36/108\\n(33.36%)\\nMethod B 43/108\\n(39.82%)\\n17/108\\n(15.74%)\\n32/108\\n(29.63%)\\n39/108\\n(36.14%)\\n3.3. Molecular Detection of the Carbapenemase Genes\\nAs previously described, the multiplex real-time PCR was used to pre-screen the\\nenrichments (BPW, LB + CTX, and LB + MEM). Discrepancies were detected between the\\nPCR and the results of the culture-based isolation. Comparing the PCR results with the\\ntrue status of the samples, a sensitivity of 61.1% and a speciﬁcity of 94.4% was calculated\\nfor the molecular screening of the heat-treated BPW enrichments for carbapenemase genes.\\nFourteen out of thirty-six positive samples were not correctly identiﬁed. One of the 18\\nnegative samples was incorrectly identiﬁed as positive for the blaGES.\\nWhen considering the screening of both second enrichments (results listed in Table 3),\\njust two of the false negatives remained false negative, and the false positive sample\\nremained as well. Therefore, the PCR achieved a sensitivity and a speciﬁcity of 94.4% by\\nusing both second enrichments as DNA template. Moreover, the real-time PCR showed\\nerroneous simultaneous detection of blaNDM and blaVIM in various samples (9.2%) that\\nwere only blaVIM-positive. However, a high cp-valuefor blaNDM was detected, suggesting\\nlow ampliﬁcation. The results for the weekly approaches are given in Supplemental\\nMaterial Table S3.\\nTable 3. Accumulated results on the weekly detection of carbapenemase genes from the spiked fecal samples. BPW was\\nused for the ﬁrst enrichment, LB + CTX and LB + MEM were used parallel as second enrichments. Corresponding exact 95%\\nconﬁdence intervals are provided in brackets.\\nBPW LB + CTX LB + MEM Both LB Enrichments\\nSensitivity 61.1 (43.5–76.9) 83.3 (67.2–93.6) 58.3 (40.8–74.4) 94.4 (81.3–99.3)\\nSpeciﬁty 94.4 (72.7–99.6) 94.4 (72.7–99.6) 100 (81.5–100) 94.4 (72.7–99.9)\\nFalse discovery rate 4.3 (0.1–21.9) 3.2 (0.1–16.7) 0 (0–16.1) 2.9 (0.1–14.9)\\nFalse omission rate 45.2 (27.3–64) 26.1 (10.2–48.4) 45.4 (28.1–63.6) 10.5 (1.3–33.1)\\nAccuracy 72.2 (58.4–83.5) 87.0 (75.1–94.7) 72.2 (58.4–83.5) 94.4 (84.6–98.8)\\n4. Discussion\\n4.1. Modiﬁcation Steps of the Ofﬁcial Isolation Protocol\\nAll modiﬁcations were tested to reduce the amount of accompanying microbiota or\\nto increase the recovery rate of the CPE. First, the recovery rate of the target bacteria in\\npig feces was determined at 6 ◦C and based on the bacterial growth calculated by the\\ndevelopment of the cfu/g every day. To increase the survival of bacteria for better isolation,\\nseveral additives to the pig feces were tested. In previous studies, the addition of sodium'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 9}, page_content='Microorganisms 2021, 9, 1105 10 of 14\\nchloride has been reported to support growth under difﬁcult conditions like low pH and\\nhigh lactate concentrations [29]. Another study evaluated trehalose for the protection of\\nE. coli against carbon stress [25]. The absence of a positive effect of these additives in our\\nstudy may be due to the lack of a clear deﬁnition of the individual inﬂuencing factors of\\neach fecal sample, i.e., the microbiota and other factors, which could have an inﬂuence on\\nthe nature of the samples. It should be noted here that the detection of CPE was increasingly\\nchallenging over the investigated period. One major challenge was the differentiation of\\nCPE from increasing amounts of accompanying ﬂora. Therefore, we recommend a timely\\nprocessing of samples, preferably within 24 h after collection as this was an important\\nfactor for reliable detection of CP E. coli in feces and food matrices.\\nAfter estimating bacterial survival, other options for improved detection of the target\\nbacteria were tested. To account for the low MIC values of CPE isolated from samples\\nof animal origin [ 22], we considered triggering gene expression by using a sublethal\\nconcentration of MEM. Moreover, the usage of ZnSO4 to push the metallo-β-lactamases was\\ntested. Both approaches did not improve the recovery rate. Previous experiments achieved\\na beneﬁcial effect by using a second selective enrichment step. This enrichment had been\\nconducted in LB+CTX [ 23]. As OXA-48 producers are not reliably able to survive the\\npresence of other β-lactams, like CTX, we considered another parallel second enrichment\\nusing LB+MEM [39,40]. The usefulness of the combination of both enrichments was also\\nproven by the sensitivity of 94.4% of the Real-Time PCR.\\nFurther, we exchanged the commercial selective agar chromID® CARBA agar with an\\nin-house prepared McC + CTX and McC + CTX + MEM. Our results support a previous\\nstudy on the suitability of chromID® CARBA agar for detecting CPE with only slightly\\nreduced susceptibility to carbapenems [ 22]. Within this study, it is presumed that the\\nconcentration of different ingredients and thus the selectivity is too high for CPE originating\\nfrom animals or food. CPE from livestock often exhibit lower MIC values for the respective\\ncarbapenems than clinical human isolates [ 22,24]. Our previous study investigated the\\nuse of chromID® CARBA after the end of the shelf life expecting a possible decrease in\\nantimicrobial concentration. As the selective agar is not speciﬁed in the EURL-AR reference\\nmethod (https://www.eurl-ar.eu/protocols.aspx, December 2019), the change to adjusted\\nagar might be easy adaptable. In this study, we analyzed the use of an in-house McC agar,\\nsupplemented with MEM and MEM+CTX, and commercial agar for the detection of CPE\\nfrom feces. The experiments were carried out with fresh prepared in-house agars (two\\nweeks). However, the comparison in this study is limited to only one commercial selective\\nagar. For commercial agars, a wide variation regarding their suitability for the isolation of\\nCPE from different samples has been reported [18,20,41,42]. For the chromID® CARBA agar,\\nsensitivity in the literature ranges from 30 to 100%, and speciﬁcity from 50 to 100%. Still,\\nthe chromID® CARBA agar is described as one of the most sensitive media [18,20,41,42]\\nand is therefore commonly used by German laboratories in the monitoring according to\\nCID 2013/652/EU.\\n4.2. The Comparison of Both Microbiological Methods\\nAlthough the isolation of CPE from livestock and food is rare in monitoring programs,\\nthe number of reports on successful recovery is increasing in recent years [6]. The isolation\\nmethods described differ from direct plating to different incubation conditions, supple-\\nments, and enrichments [23,43,44]. These non-harmonized detection methods in research\\nprojects in the animal sector complicate a realistic assessment of the occurrence of CPE in\\nlivestock. The EURL-AR has developed and provided a harmonized method for CP E. coli\\nisolation from meat and caeca samples, which is meant to be used in monitoring according\\nto CID 2013/652/EU. However, it is currently rarely used beyond the EU monitoring\\nprograms. Several publications reported on difﬁculties in detecting CPE with the EURL-\\nmethod [22–24]. We observed a sensitivity of 75% and speciﬁcity of 100% for the EURL-AR\\nprotocol with a commonly used commercial agar. This implies that 25% of CPE-positive\\nsamples were not recognized, which demonstrates the need for an improvement of the'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 10}, page_content='Microorganisms 2021, 9, 1105 11 of 14\\nreference method. Our results indicate that using the validated EURL-protocol in combina-\\ntion with the non-commercial McC agar supplemented with 0.125 mg/L MEM, and with\\n0.125 mg/L MEM and 1 mg/L CTX signiﬁcantly improved the sensitivity compared to\\nall other approaches. Selective enrichment under microaerobic conditions did not achieve\\nthe desired effect. This microaerobic incubation was promising because of the ability of\\nEnterobacteriaceae to grow under this condition, in contrast to e.g., Aeromonads, which\\nare often part of the accompanying microbiota [38]. Other options to favor Enterobacteri-\\naceae over other bacteria have been investigated. An incubation temperature of 44 ◦C was\\ndescribed in the report on the validation of selective McC agar supplemented with 1 mg/L\\nCTX for monitoring of ESBL- and AmpC producing E. coli in meat and caeca samples [45]\\nand in a previous study of Irrgang et al. 2019 [23]. The authors recognized that the growth\\nof CP Enterobacteriaceae other than CP E. coli might also be inhibited [ 23]. In terms of\\nharmonization, a method should be capable of also detecting other bacterial species, as all\\nCPE are of interest regarding the distribution of the resistances [6]. Our study focused on\\nE. coli as the EU monitoring for ESBL/pAmpC and carbapenemases targets this species.\\nThe reported problems during detection of CPE-containing samples indicate a low\\noccurrence of CPE in samples of animal origin [23,24]. Another rarely discussed reason for\\nreduced detection of CPE from feces and caecum could be that the bacteria in this environ-\\nment have lost the ability to grow on routine media. It is described thatE. coli, Klebsiella pneu-\\nmoniae, Salmonella enterica serovar Typhimurium, and other human/zoonotic pathogens\\ncould enter a distinct state, a so-called viable but non-culturable (VBNC) state [46,47].\\nIn combination with the physical stress exerted by caeca samples, the transition to a VBNC\\nstatus would be useful for the survival of CPE [47]. Overall, we need to be able to detect\\npositive samples and to isolate these CPE for a further characterization and risk assessment.\\nAnother approach to improve sensitivity might be a more sensitive molecular screen-\\ning method. Using pre-screening by PCR, efforts can focus on the cultivation of the isolates\\nfrom PCR-positive samples. This would decrease the costs and the personal time neces-\\nsary for the microbiological isolation. The multiplex Real-Time PCR applied in this study\\nshowed good results when screening the second enrichment (a sensitivity and speciﬁcity\\nof 94.4%). It might be possible that the use of a quantitative Real-Time PCR will provide\\nmore reliable results as a direct increase of the bacterial growth can be detected and used\\nfor adapting the time points for selective cultivation. The Real-Time PCR performed was\\nnot effective when applied on the initial BPW enrichment. To avoid the isolation step of\\na target bacterium and to immediately analyze the entire sample, the option of metage-\\nnomics becomes more attractive. Metagenomics would allow a simultaneous identiﬁcation\\nand typing of CPE and other pathogens [ 47]. It would be a powerful tool for monitor-\\ning purposes and might be considered in the long term to become part of the routine\\nmethod. Currently, this method is too expensive for routine use and requires additional\\nbioinformatic expertise.\\n5. Conclusions\\nAfter various approaches to optimize the recovery rate of CPE in pig feces, it could\\nbe concluded that potential additives did not have a positive effect. However, a positive\\neffect can be achieved by processing the samples in a timely manner. Moreover, our results\\nconﬁrmed the inﬂuence of the chosen media. Typically used commercial agar plates are\\noptimized for the isolation of CPE from human clinical samples with usually high MIC\\nvalues for carbapenems. Therefore, we recommend the EURL-method with a selective\\nmedium corresponding to the searched target bacterium. A second enrichment under\\nmicroaerophilic conditions reduced the accompanying microbiota, but the sensitivity of\\nthe whole procedure was decreased from 100% to 86.1%. The PCR-screening from a second\\nenrichment showed good results. Therefore, we recommend the modiﬁed method if a\\npresumptive CP E. coli cannot be isolated. However, a sensitive method is essential to\\navoid an underestimation of the CPE occurrence in livestock in Europe.'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 11}, page_content='Microorganisms 2021, 9, 1105 12 of 14\\nSupplementary Materials: The following are available online at https://www.mdpi.com/article/\\n10.3390/microorganisms9051105/s1, Supplemental Material Table S1: Detailed information about\\nthe used CPE. Supplemental Material Table S2: The repeated isolation of CPE from the spiked\\nfaecal samples. Supplemental Material Table S3: Summarized Real-time PCR results on the repeated\\ndetection of carbapenemase genes from the spiked faecal samples.\\nAuthor Contributions: Data curation, N.P .; formal analysis, L.H., S.S., and A.K.; investigation, N.P .,\\nY.K., T.S.-Z., K.J., and M.G.; project administration, A.I. and M.G.; visualization, N.P .; writing—\\noriginal draft, N.P ., J.A.H., and A.I.; writing—review and editing, K.J., Y.K., J.A.H., M.G., L.H.,\\nA.K., B.-A.T., S.S., D.M., and A.I. All authors have read and agreed to the published version of\\nthe manuscript.\\nFunding: This work was ﬁnancially supported by a grant of the German Federal Institute for Risk\\nAssessment (43-001 and 43-002). The work of Natalie Pauly was supported by the European Joint\\nProject (EJP) IMPART funded by the European Union’s Horizon 2020 research and innovation\\nprogramme under Grant Agreement No 773830. The work of Stefan Schwarz is supported by the\\nGerman Federal Ministry of Education and Research (BMBF) under project number and 01KI2009D\\nas part of the Research Network Zoonotic Infectious Diseases.\\nInstitutional Review Board Statement: Not applicable.\\nInformed Consent Statement: Not applicable.\\nData Availability Statement: MDPI: Research Data Policies.\\nAcknowledgments: We thank the laboratory team (Britta Lesniewsky, Anja Seeger, Silke Jahn, Romy\\nFuhrmeister, and Silvia Schmoger) of the NRL-AR for excellent technical assistance. We further\\nacknowledge the support of the participating company for the provision of the sample material.\\nConﬂicts of Interest: The authors declare no conﬂict of interest.\\nReferences\\n1. WHO. Critically Important Antimicrobials for Human Medicine, 6th ed.; World Health Organization: Geneva, Switzerland, 2019.\\n2. Madec, J.Y.; Haenni, M.; Nordmann, P .; Poirel, L. Extended-spectrum β-lactamase/AmpC- and carbapenemase-producing\\nEnterobacteriaceae in animals: A threat for humans? Clin. Microbiol. Infect. 2017, 23, 826–833. [CrossRef] [PubMed]\\n3. Aworh, M.K.; Kwaga, J.; Okolocha, E.; Mba, N.; Thakur, S. Prevalence and risk factors for multi-drug resistant Escherichia coli\\namong poultry workers in the Federal Capital Territory, Abuja, Nigeria.PLoS ONE 2019, 14, e0225379. [CrossRef]\\n4. EFSA/ECDC. The European Union summary report on antimicrobial resistance in zoonotic and indicator bacteria from hu-mans,\\nanimals and food in 2016. EFSA J. 2018, 16, e05182.\\n5. Köck, R.; Daniels-Haardt, I.; Becker, K.; Mellmann, A.; Friedrich, A.; Mevius, D.; Schwarz, S.; Jurke, A. Carbapenem-resistant\\nEnterobacteriaceae in wildlife, food-producing, and companion animals: A systematic review. Clin. Microbiol. Infect. 2018, 24,\\n1241–1250. [CrossRef]\\n6. Taneja, N.; Sharma, M. Antimicrobial resistance in the environment: The Indian scenario. Indian J. Med. Res. 2019, 149, 119–128.\\n[CrossRef]\\n7. Miriagou, V .; Cornaglia, G.; Edelstein, M.; Galani, I.; Giske, C.G.; Gniadkowski, M.; Malamou-Lada, E.; Martinez-Martinez, L.;\\nNavarro, F.; Nordmann, P .; et al. Acquired car-bapenemases in Gram-negative bacterial pathogens: Detection and surveillance\\nissues. Clin. Microbiol. Infect. 2010, 16, 112–122. [CrossRef]\\n8. Partridge, S.R.; Kwong, S.M.; Firth, N.; Jensen, S.O. Mobile Genetic Elements Associated with Antimicrobial Resistance.\\nClin. Microbiol. Rev. 2018, 31, e00088-17. [CrossRef] [PubMed]\\n9. Botelho, J.; Roberts, A.P .; Le ón-Sampedro, R.; Grosso, F.; Peixe, L. Carbapenemases on the move: It’s good to be on ICEs.\\nMob. DNA 2018, 9, 37. [CrossRef]\\n10. Palzkill, T. Metallo- β-lactamase structure and function. Ann. N. Y. Acad. Sci. 2013, 1277, 91–104. [CrossRef]\\n11. Salahuddin, P .; Kumar, A.; Kha, A.U. Structure, function of serine and Metallo-β-lactamases and their Inhibitors. Curr. Protein.\\nPept. Sci. 2018, 19, 130–144. [CrossRef]\\n12. Bush, K. Past and present perspectives on β-lactamases. Antimicrob. Agents Chemother. 2018, 62, e01076-18. [CrossRef]\\n13. Woodford, N.; Wareham, D.W.; Guerra, B.; Teale, C. Carbapenemase-producing Enterobacteriaceae and non-Enterobacteriaceae\\nfrom animals and the environment: An emerging public health risk of our own making? J. Antimicrob. Chemother. 2014, 69,\\n287–291. [CrossRef]\\n14. Grundmann, H.; Glasner, C.; Albiger, B.; Aanensen, D.M.; Tomlinson, C.T.; Andrasevi´ c, A.T.; Cantón, R.; Carmeli, Y.; Friedrich,\\nA.W.; Giske, C.G.; et al. Occurrence of carbapenemase-producing Klebsiella pneumoniae and Escherichia coli in the European\\nsurvey of carbapenemase-producing Enterobacteriaceae (EuSCAPE): A prospective, multinational study. Lancet Infect. Dis. 2017,\\n17, 153–163. [CrossRef]'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 12}, page_content='Microorganisms 2021, 9, 1105 13 of 14\\n15. EFSA and ECDC. The European Union summary report on antimicrobial resistance in zoonotic and indicator bacteria from\\nhumans, animals and food in 2017. EFSA J. 2019, 17, e05598.\\n16. Fischer, J.; Rodriguez, I.; Schmoger, S.; Friese, A.; Roesler, U.; Helmuth, R.; Guerra, B. Escherichia coli producing VIM-1\\ncar-bapenemase isolated on a pig farm. J. Antimicrob. Chemother. 2012, 67, 1793–1795. [CrossRef]\\n17. Schrijver, R.; Stijntjes, M.; Rodr íguez-Baño, J.; Tacconelli, E.; Rajendran, N.B.; Voss, A. Review of antimicrobial resistance\\nsurveillance programmes in livestock and meat in EU with focus on humans. Clin. Microbiol. Infect. 2018, 24, 577–590. [CrossRef]\\n18. Simner, P .J.; Gilmour, M.W.; DeGagne, P .; Nichol, K.; Karlowsky, J.A. Evaluation of Five Chromogenic Agar Media and the Rosco\\nRapid Carb Screen Kit for Detection and Conﬁrmation of Carbapenemase Production in Gram-Negative Bacilli. J. Clin. Microbiol.\\n2014, 53, 105–112. [CrossRef]\\n19. Wilkinson, K.M.; Winstanley, T.G.; Lanyon, C.; Cummings, S.P .; Raza, M.W.; Perry, J.D. Comparison of Four Chromogenic Culture\\nMedia for Carbapenemase-Producing Enterobacteriaceae. J. Clin. Microbiol. 2012, 50, 3102–3104. [CrossRef]\\n20. Vrioni, G.; Daniil, I.; Voulgari, E.; Ranellou, K.; Koumaki, V .; Ghirardi, S.; Kimouli, M.; Zambardi, G.; Tsakris, A. Comparative\\nevaluation of a prototype chromogenic medium (ChromID CARBA) for detecting carbapenemase-producing Enterobacte-riaceae\\nin surveillance rectal swabs. J. Clin. Microbiol. 2012, 50, 1841–1846. [CrossRef]\\n21. Papadimitriou-Olivgeris, M.; Bartzavali, C.; Christoﬁdou, M.; Bereksi, N.; Hey, J.; Zambardi, G.; Spiliopoulou, I. Performance of\\nchromID® CARBA medium for carbapenemases-producing enterobacteriaceae detection during rectal screening. Eur. J. Clin.\\nMicrobiol. Infect. Dis. 2014, 33, 35–40. [CrossRef]\\n22. Pauly, N.; Hammerl, J.A.; Grobbel, M.; Tenhagen, B.-A.; Käsbohrer, A.; Bisenius, S.; Fuchs, J.; Horlacher, S.; Lingstädt, H.;\\nMauermann, U.; et al. ChromID® CARBA Agar Fails to Detect Carbapenem-Resistant Enterobacteriaceae with Slightly Reduced\\nSusceptibility to Carbapenems. Front. Microbiol. 2020, 11, 1678. [CrossRef]\\n23. Irrgang, A.; Tenhagen, B.-A.; Pauly, N.; Schmoger, S.; Kaesbohrer, A.; Hammerl, J.A. Characterization of VIM-1-ProducingE. coli\\nIsolated from a German Fattening Pig Farm by an Improved Isolation Procedure. Front. Microbiol. 2019, 10, 2256. [CrossRef]\\n24. Pauly, N.; Hammerl, J.A.; Schwarz, S.; Grobbel, M.; Meemken, D.; Malorny, B.; Tenhagen, B.-A.; Käsbohrer, A.; Irrgang, A.\\nCo-occurrence of the blaVIM-1 and blaSHV-12 genes on an IncHI2 plasmid of an Escherichia coli isolate recovered from German\\nlivestock. J. Antimicrob. Chemother. 2021, 76, 531–533. [CrossRef] [PubMed]\\n25. Algara, M.M.; Kuczy ´ nska-Wi´ snik, D.; D˛ ebski, J.; Stojowska-Sw˛ edrzy ´ nska, K.; Sominka, H.; Bukrejewska, M.; Laskowska, E.\\nTrehalose protects Escherichia coli against carbon stress manifested by protein acetylation and aggregation. Mol. Microbiol. 2019,\\n112, 866–880. [CrossRef] [PubMed]\\n26. Kuczy ´ nska-Wi´ snik, D.; Stojowska, K.; Matuszewska, E.; Leszczy ´ nska, D.; Algara, M.M.; Augustynowicz, M.; Laskowska, E. Lack\\nof intracellular trehalose affects formation of Escherichia coli persister cells. Microbiol. Spectr. 2015, 161 Pt 4, 786–796. [CrossRef]\\n27. Achá, S.; Kühn, I.; Mbazima, G.; Colque-Navarro, P .; Möllby, R. Changes of viability and composition of the Escherichia coli ﬂora\\nin faecal samples during long time storage. J. Microbiol. Methods 2005, 63, 229–238. [CrossRef]\\n28. Hollander, D.H.; Nell, E.E. Improved Preservation of Treponema pallidum and Other Bacteria by Freezing with Glycerol.\\nAppl. Microbiol. 1954, 2, 164–170. [CrossRef] [PubMed]\\n29. Jordan, D. Sodium chloride enhances recovery and growth of acid-stressed E. coli O157:H7. Lett. Appl. Microbiol. 2001, 32,\\n312–315. [CrossRef]\\n30. Mañas, P .; Pagan, R.; Leguérinel, I.; Condon, S.; Mafart, P .; Sala, F. Effect of sodium chloride concentration on the heat resistance\\nand recovery of Salmonella typhimurium. Int. J. Food Microbiol. 2001, 63, 209–216. [CrossRef]\\n31. Mossel, D.A.; Harrewijn, G.A.; Zadelhoff, C.F.N.-V . Standardization of the selective inhibitory effect of surface active compounds\\nused in media for the detection of Enterobacteriaceae in foods and water. Health Lab. Sci. 1974, 11, 260–267.\\n32. Andersson, D.I.; Hughes, D. Microbiological effects of sublethal levels of antibiotics. Nat. Rev. Genet. 2014, 12, 465–478. [CrossRef]\\n[PubMed]\\n33. Nordmann, P .; Girlich, D.; Poirel, L. Detection of Carbapenemase Producers in Enterobacteriaceae by Use of a Novel Screening\\nMedium. J. Clin. Microbiol. 2012, 50, 2761–2766. [CrossRef] [PubMed]\\n34. Gullberg, E.; Albrecht, L.M.; Karlsson, C.; Sandegren, L.; Andersson, D.I. Selection of a Multidrug Resistance Plasmid by Sublethal\\nLevels of Antibiotics and Heavy Metals. mBio 2014, 5, e01918-14. [CrossRef] [PubMed]\\n35. Swayne, R.; Ludlam, H.; Shet, V .; Woodford, N.; Curran, M. Real-time TaqMan PCR for rapid detection of genes encoding ﬁve\\ntypes of non-metallo- (class A and D) carbapenemases in Enterobacteriaceae. Int. J. Antimicrob. Agents 2011, 38, 35–38. [CrossRef]\\n[PubMed]\\n36. Van Der Zee, A.; Roorda, L.; Bosman, G.; Fluit, A.C.; Hermans, M.; Smits, P .H.; Van Der Zanden, A.G.; Witt, R.T.; Van Coppenraet,\\nL.E.B.; Stuart, J.C.; et al. Multi-centre evaluation of real-time multiplex PCR for detection of carbapenemase genes OXA-48, VIM,\\nIMP , NDM and KPC.BMC Infect. Dis. 2014, 14, 27. [CrossRef]\\n37. R Core Team. R: A Language and Environment for Statistical Computing; R Foundation for Statistical Computing: Vienna, Austria,\\n2020; Available online: http://www.R-project.org/ (accessed on 11 March 2021).\\n38. Octavia, S.; Lan, R. The Family Enterobacteriaceae. In The Prokaryotes; Springer: Berlin/Heidelberg, Germany, 2014; pp. 225–286.\\nISBN 9783642389221.\\n39. Poirel, L.; Potron, A.; Nordmann, P . OXA-48-like carbapenemases: The phantom menace. J. Antimicrob. Chemother. 2012, 67,\\n1597–1606. [CrossRef]\\n40. Codjoe, F.S.; Donkor, E.S. Carbapenem Resistance: A Review. Med. Sci. 2017, 6, 1. [CrossRef]'), Document(metadata={'source': 'texts\\\\Pauly,method comparison, 2021.pdf', 'page': 13}, page_content='Microorganisms 2021, 9, 1105 14 of 14\\n41. Viau, R.; Frank, K.M.; Jacobs, M.R.; Wilson, B.; Kaye, K.; Donskey, C.J.; Perez, F.; Endimiani, A.; Bonomo, R.A. Intestinal Carriage\\nof Carbapenemase-Producing Organisms: Current Status of Surveillance Methods. Clin. Microbiol. Rev. 2015, 29, 1–27. [CrossRef]\\n42. Brand, P .; Gobeli, S.; Perreten, V . Pathotyping and antibiotic resistance of porcine enterovirulent Escherichia coli strains from\\nSwitzerland (2014–2015). Schweiz Arch Tierheilkd 2017, 159, 373–380. [CrossRef]\\n43. Mo, S.S.; Urdahl, A.M.; Madslien, K.; Sunde, M.; Nesse, L.L.; Slettemeås, J.S.; Norström, M. What does the fox say? Monitoring\\nantimicrobial resistance in the environment using wild red foxes as an indicator. PLoS ONE 2018, 13, e0198019. [CrossRef]\\n44. Hasman, H.; Agersø, Y.; Cavaco, L.M. Validation of selective MacConkey agar plates supplemented with 1 mg/L cefo-taxime\\nfor monitoring of ESBL- and AmpC-producing E. coli in meat and caeca samples. LABORATORY PROTOCOL Authored by\\nthe EURL-AR. 2017. Available online: https://www.eurl-ar.eu/CustomerData/Files/Folders/21-protocols/390_protocol-for-\\nvalidation-of-macconkey-and-ctx-agar-plates-ﬁnal03112017.pdf (accessed on November 2013).\\n45. Xu, H.-S.; Roberts, N.; Singleton, F.L.; Attwell, R.W.; Grimes, D.J.; Colwell, R.R. Survival and viability of nonculturableEscherichia\\ncoli andVibrio cholerae in the estuarine and marine environment. Microb. Ecol. 1982, 8, 313–323. [CrossRef] [PubMed]\\n46. Li, L.; Mendis, N.; Trigui, H.; Oliver, J.D.; Faucher, S.P . The importance of the viable but non-culturable state in human bacterial\\npathogens. Front. Microbiol. 2014, 5, 258. [CrossRef]\\n47. Grützke, J.; Malorny, B.; Hammerl, J.A.; Busch, A.; Tausch, S.H.; Tomaso, H.; Deneke, C. Fishing in the Soup—Pathogen Detection\\nin Food Safety Using Metabarcoding and Metagenomic Sequencing. Front. Microbiol. 2019, 10, 1805. [CrossRef] [PubMed]')]\n",
      "\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Content of Roschanski, VIM-1 Stall N2, 2019.pdf:\n",
      "[Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 0}, page_content='Detection of VIM-1-ProducingEnterobacter cloacae and\\nSalmonella enterica Serovars Infantis and Goldcoast at a\\nBreeding Pig Farm in Germany in 2017 and Their Molecular\\nRelationship to Former VIM-1-ProducingS. Infantis Isolates in\\nGerman Livestock Production\\nNicole Roschanski,a* Sead Hadziabdic,b Maria Borowiak,b Burkhard Malorny,b Bernd-Alois Tenhagen,b\\n Michaela Projahn,a*\\nAnnemarie Kaesbohrer,b,c Sebastian Guenther,a* Istvan Szabo,b Uwe Roesler,a Jennie Fischerb\\naInstitute for Animal Hygiene and Environmental Health, Freie Universitaet Berlin, Berlin, Germany\\nbDepartment of Biological Safety, German Federal Institute for Risk Assessment, BfR, Berlin, Germany\\ncInstitute of Veterinary Public Health, University of Veterinary Medicine, Vienna, Austria\\nABSTRACT In 2011, VIM-1-producingSalmonella enterica serovar Infantis andEsche-\\nrichia coli were isolated for the ﬁrst time in four German livestock farms. In 2015/\\n2016, highly related isolates were identiﬁed in German pig production. This raised\\nthe issue of potential reservoirs for these isolates, the relation of their mobile ge-\\nnetic elements, and potential links between the different affected farms/facilities.\\nIn a piglet-producing farm suspicious for being linked to someblaVIM-1 ﬁndings in\\nGermany, fecal and environmental samples were examined for the presence of\\ncarbapenemase-producingEnterobacteriaceae and Salmonella spp. Newly discovered\\nisolates were subjected to Illumina whole-genome sequencing (WGS) and S1 pulsed-\\nﬁeld gel electrophoresis (PFGE) hybridization experiments. WGS data of these iso-\\nlates were compared with those for the previously isolated VIM-1-producingSalmo-\\nnella Infantis isolates from pigs and poultry. Among 103 samples, one Salmonella\\nGoldcoast isolate, one Salmonella Infantis isolate, and one Enterobacter cloacae iso-\\nlate carrying the blaVIM-1 gene were detected. Comparative WGS analysis re-\\nvealed that the blaVIM-1 gene was part of a particular Tn21-like transposable element\\nin all isolates. It was located on IncHI2 (ST1) plasmids of /H11011290 to 300 kb with a\\nbackbone highly similar (98 to 100%) to that of reference pSE15-SA01028. SNP anal-\\nysis revealed a close relationship of all VIM-1-positiveS. Infantis isolates described\\nsince 2011. The ﬁndings of this study demonstrate that the occurrence of the\\nblaVIM-1 gene in German livestock is restricted neither to a certain bacterial species\\nnor to a certain Salmonella serovar but is linked to a particular Tn21-like transpos-\\nable element located on transferable pSE15-SA01028-like IncHI2 (ST1) plasmids, be-\\ning present in all of the investigated isolates from 2011 to 2017.\\nIMPORTANCE Carbapenems are considered one of few remaining treatment options\\nagainst multidrug-resistant Gram-negative pathogens in human clinical settings. The\\noccurrence of carbapenemase-producingEnterobacteriaceae in livestock and food is\\na major public health concern. Particularly the occurrence of VIM-1-producing Sal-\\nmonella Infantis in livestock farms is worrisome, as this zoonotic pathogen is one of\\nthe main causes for human salmonellosis in Europe. Investigations on the epidemiol-\\nogy of those carbapenemase-producing isolates and associated mobile genetic ele-\\nments through an in-depth molecular characterization are indispensable to under-\\nstand the transmission of carbapenemase-producingEnterobacteriaceae along the\\nfood chain and between different populations to develop strategies to prevent their\\nfurther spread.\\nCitation Roschanski N, Hadziabdic S, Borowiak\\nM, Malorny B, Tenhagen B-A, Projahn M,\\nKaesbohrer A, Guenther S, Szabo I, Roesler U,\\nFischer J. 2019. Detection of VIM-1-producing\\nEnterobacter cloacaeand Salmonella enterica\\nserovars Infantis and Goldcoast at a breeding\\npig farm in Germany in 2017 and their\\nmolecular relationship to former VIM-1-\\nproducingS. Infantis isolates in German\\nlivestock production. mSphere 4:e00089-19.\\nhttps://doi.org/10.1128/mSphere.00089-19.\\nEditor Ana\\nCristina Gales, Escola Paulista de\\nMedicina/Universidade Federal de São Paulo\\nCopyright© 2019 Roschanski et al. This is an\\nopen-access article distributed under the terms\\nof theCreative Commons Attribution 4.0\\nInternational\\nlicense.\\nAddress\\ncorrespondence to Jennie Fischer,\\njennie.ﬁscher@bfr.bund.de.\\n* Present address: Nicole Roschanski,\\nLandeslabor Schleswig-Holstein, Neumuenster,\\nGermany; Michaela Projahn, German Federal\\nInstitute for Risk Assessment, BfR, Department\\nof Biological Safety, Berlin, Germany; Sebastian\\nGuenther, University of Greifswald, Institute of\\nPharmacy, Greifswald, Germany.\\nReceived 14 February 2019\\nAccepted 23 May 2019\\nPublished\\nRESEARCH ARTICLE\\nClinical Science and Epidemiology\\nMay/June 2019 Volume 4 Issue 3 e00089-19\\n msphere.asm.org 1\\n12 June 2019\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 1}, page_content='KEYWORDS Salmonella, antimicrobial resistance, carbapenem resistance,\\ncarbapenemases, pigs, transmission\\nC\\narbapenems are among the few remaining treatment options for infections caused\\nby multidrug-resistant Gram-negative bacteria. Emergence of bacteria with ac-\\nquired resistance against carbapenems in human clinical settings is a major public\\nhealth issue (1). The discovery of VIM-1 carbapenemase-producingEscherichia coli and\\nSalmonella enterica subsp. enterica serovar Infantis (S. Infantis) in two German pig\\n(Salmonella isolates R25 and R27 andE. coli isolates R29 and R178) and one chicken-\\nfattening farm (Salmonellaisolate R3) in 2011 (2, 3) raised concerns about the spread\\nof\\nthis resistance gene in livestock and the development of a new reservoir for the\\nresistance to these last-line antibiotics.\\nThese ﬁrst ﬁndings triggered active monitoring programs in Germany and the\\nEuropean Union (CID 2013/652/EU) (4, 5). Additionally, samplings as well as focused\\nanalyses\\nof isolates submitted to the German National Reference Laboratories for\\nSalmonella (NRL-Salmonella) and for Antimicrobial Resistance (NRL-AR) are carried out\\nto detect further carbapenemase-producingEnterobacteriaceae (CPE) in food and live-\\nstock. Data related to CPE ﬁndings are continuously analyzed to carefully monitor the\\noccurrence of these CPE along the food chain. These efforts lead to the recovery of\\nfurther VIM-1-producingSalmonella and E. coli isolates in German livestock and food.\\nRetrospective investigation of bacterial cultures originating from the studies in 2011\\nidentiﬁed one additional VIM-1-producingS. Infantis isolate (isolate V363) in a sample\\nfrom a third pig-fattening farm as well as one serologically roughS. Infantis isolate\\n(isolate G-336-1a) from a second chicken fattening farm (6, 7). Analysis ofSalmonella\\nisolates\\nfrom routine diagnostics revealed two additional S. Infantis isolates from\\nminced pork (15-SA01028) in 2015 and from a sick piglet in 2016 (16-SA00749) (8, 9),\\nrespectively.\\nThis indicated that VIM-1-producingSalmonella isolates were still present\\nin German pig production. Three of the six affected livestock farms were additionally\\npositive forE. colistrains carryingblaVIM-1. TheblaVIM-1 gene was located on an/H11011300-kb\\nIncHI2 plasmid in the Salmonella isolates (sequence of plasmid pRH-R27 [GenBank\\naccession number LN555650.1] and plasmid pSE15-SA01028 [GenBank accession\\nnumber CP026661.1]) (3,7–10). All but oneE. coli isolate of sequence type 131 (ST131)\\nbelonged to a certainE. coliST88 clone. TheseE. coliisolates harbored theblaVIM-1 gene\\non smaller plasmids (180 to 230 kb) (6, 7, 11). One of the pig-fattening farms (S2, with\\nhighest\\nprevalence of VIM-1-producing Salmonella and E. coli) positive in 2011 was\\nintensively sampled and examined 4 years after the ﬁrst ﬁnding. Despite intensive\\nattempts, no VIM-1-positive isolate could be detected. Evaluation of farm-speciﬁc\\nkeeping and management parameters revealed that disinfection measures were still at\\na very high level, but the current pigs originated from a different pig breeding farm\\n(12).\\nMoreover,\\nin 2016, a VIM-1-producingE. coli ST10 strain was isolated in Germany\\nfrom retail seafood originating from Italy ( 13). The characteristics of the blaVIM-1-\\nassociated mobile genetic elements from this isolate shared no similarities with those\\nof the VIM-1-positive isolates from the German livestock sector. In this case, the\\nblaVIM-1-harboring class 1 integron carried furtheraacA4, aph(3=)-XV, aadA1, and catB2\\ngene cassettes in its variable region and was associated with a Tn3-like transposon on\\nan IncY plasmid. In the livestock isolates, theblaVIM-1-harboring class 1 integron carried\\nfurther aacA4-aadA1 gene cassettes and was associated with a Tn21-like transposon on\\nIncHI2 plasmids.\\nCarbapenems are not licensed for use in livestock. However, administration of any\\nother beta-lactam antibiotic or other antimicrobial classes with resistance genes ge-\\nnetically linked to carbapenemase-encoding genes might trigger the selection of CPE\\nand the spread of the respective mobile genetic elements. This may have contributed\\nto the recurrent detection of blaVIM-1-containing E. coli as well as S. Infantis over a\\nperiod of 5 years (2011 to 2016). However, the sporadic occurrence of CPE, their low\\nRoschanski et al.\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 2\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 2}, page_content='detection rates accompanied by the complexity of trade routes within the German pig\\nproduction system, complicates trace-back approaches. Nevertheless, based on analysis\\nof available background information of affected farms we identiﬁed a piglet-producing\\nfarm (N2) suspicious for being linked to at least two previously affected pig farms\\n(including S2) and collected fecal and environmental samples on this farm. In order to\\ncharacterize the mobile genetic elements and phylogeny of the isolates involved in this\\noccurrence, we carried out in-depth molecular analysis using whole-genome sequenc-\\ning (WGS) of isolates recovered from this piglet-producing farm and compared the data\\nwith sequence data from all seven VIM-1-positiveS. Infantis isolates previously de-\\nscribed.\\nRESULTS\\nPhenotypic and genotypic characterization of isolates andblaVIM-1-associated\\nmobile genetic elements. (i) Isolates from pig breeding farm N2.Using the mod-\\niﬁed DIN EN ISO 6579 method, sixSalmonella isolates were obtained from four samples\\n(Table 1). Isolates were identiﬁed by classical serology as eitherS. Infantis (n/H110051; isolate\\nN2-1) or S. Goldcoast (n/H110055; isolates N2-2 to N2-6). Results were conﬁrmed using the\\nSISTR Salmonellain silicotyping tool. TheS. Infantis isolate N2-1 (collected feces, rearing\\nTABLE 1Phenotypic and genotypic characteristics of isolates derived from farm N2 and previously describedblaVIM-1-positive S. Infantis\\nisolatesa\\nIsolate\\nYr of\\nisolation\\nSource, farm origin\\n(sample name)\\nSpecies or serovar\\n(MLST)\\nMICs (mg/liter)\\nof ETP, IMP,\\nand MEM\\n(April 2019)\\nMICs\\n(mg/liter)\\nof FOT\\nand TAZ\\nPresence of\\npAmpC- or\\ncarbapenemase-\\nencoding genes Reference\\nN2-8 2017 Collected feces,\\nswine farm\\nN2 (FD1-SK2)\\nEnterobacter cloacae 1, 4, 2 /H1102264,/H11022128blaACC-1, blaACT-7-like,\\nblaVIM-1\\nThis study\\nN2-6 2017 Manure, swine\\nfarm N2\\nS. Goldcoast (ST358) /H113490.015, 0.25,\\n0.006\\n32, 128 blaACC-1 This study\\nN2-5 2017 Collected feces,\\nswine farm\\nN2 (FD1-SK1)\\nS. Goldcoast (ST358) /H113490.015, 0.25,\\n/H113490.03\\n/H113490.25, 0.5 This study\\nN2-4 2017 Boot swabs,\\nswine farm\\nN2 (FD1-Sta)\\nS. Goldcoast (ST358) /H113490.015, 0.25,\\n/H113490.03\\n/H113490.25, 0.5 This study\\nN2-3 2017 Boot swabs,\\nswine farm\\nN2 (A4-Sta)\\nS. Goldcoast (ST358) /H113490.015, 0.25,\\n/H113490.03\\n/H113490.25, 0.5 This study\\nN2-2 2017 Boot swabs,\\nswine farm\\nN2 (A4-Sta)\\nS. Goldcoast (ST358) 0.5, 4, 2 /H1102264,/H11022128blaACC-1, blaVIM-1 This study\\nN2-1 2017 Collected feces,\\nswine farm\\nN2 (FD1-SK2)\\nS. Infantis (ST32) 0.125, 2, 0.5 /H1102264,/H11022128blaACC-1, blaVIM-1 This study\\n16-SA00749 2016 Sick piglet, swine\\nfarm N1\\nS. Infantis (ST32) 0.25, 4, 2 /H1102264,/H11022128blaACC-1, blaVIM-1 9\\n15-SA01028 2015 Minced pork\\nmeat\\nS. Infantis (ST32) 0.5, 4, 2 /H1102264,/H11022128blaACC-1, blaVIM-1 9\\nG-336-1a 2012 Collected dust,\\npoultry\\nfarm G78\\nS. Infantis (ST32)\\nserologically rough\\n/H110222, 8, 16/H1102264,/H11022128blaACC-1, blaVIM-1 6\\nV363 2012 Single animal\\nfeces,\\nswine\\nfarm S3\\nS. Infantis (ST32)\\nnonmotile\\n2, 8, 4 /H1102264,/H11022128blaACC-1, blaVIM-1 7\\nR27 2011 Pooled faces,\\nswine\\nfarm S2\\nS. Infantis (ST32)\\nnonmotile\\n0.25, 4, 0.5 /H1102264,/H11022128blaACC-1, blaVIM-1 3\\nR25 2011 Boot swabs,\\nswine\\nfarm S1\\nenvironment\\nS. Infantis (ST32)\\nnonmotile\\n0.25, 4, 0.5 /H1102264,/H11022128blaACC-1, blaVIM-1 3\\nR3 2011 Collected dust,\\npoultry\\nfarm G1\\nS. Infantis (ST32)\\nnonmotile\\n0.25, 4, 0.5 /H1102264,/H11022128blaACC-1, blaVIM-1 3\\naETP, ertapenem; IMP, imipenem; MEM, meropenem; FOT, cefotaxime; TAZ, ceftazidime.\\nAnalysis of VIM-1-Positive Strains in German Livestock\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 3\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 3}, page_content='quarter 1) as well asS. Goldcoast isolate N2-2 (boot swab, farrowing barn 4) tested\\npositive for theblaVIM-1 gene by PCR. FourS. Goldcoast isolates, detected in boot swabs\\n(n /H110052), collected feces (n/H110051), and manure (n/H110051), did not carry theblaVIM-1 gene. All\\nﬁve S. Goldcoast isolates belonged to ST358 (Table 1). TheS. Infantis isolate, like all\\nother previously described VIM-1-positiveS. Infantis isolates, belonged to ST32. Neither\\na blaVIM-1-negative S. Infantis isolate nor any other nonmotileSalmonella spp. were\\ndetected.\\nOne blaVIM-1-positive Enterobacter cloacae isolate (N2-8) was found through reanal-\\nysis of the collected fecal sample harboring the blaVIM-1-positive S. Infantis using\\nnonselective preenrichment followed by a selective cultivation in LB medium supple-\\nmented with 1 mg/liter of cefotaxime sodium salt (CTX).\\nAn overview of isolate characteristics is depicted inTable 1. Pulsed-ﬁeld gel elec-\\ntrophoresis\\n(PFGE) restriction patterns of all previously isolatedS. Infantis isolates and\\nthe recently discovered isolate N2-1 were highly similar (see Fig. S1 in the supplemental\\nmaterial). Likewise, the PFGE restriction patterns of all ﬁveS. Goldcoast isolates were\\nalike (Fig. S2). MIC testing of isolates harboring theblaVIM-1 gene revealed a low-level\\nresistance against the tested carbapenems but a high-level resistance against the\\nthird-generation cephalosporins tested (Table 1) in contrast to isolates lacking the\\nblaVIM-1 gene.\\n(ii) Phylogeny and SNP analysis ofS. Infantis isolates. Mapping of raw reads\\nagainst the PacBio chromosome sequence of isolate 15-SA01028 and subsequent single\\nnucleotide polymorphism (SNP) analysis revealed 0 to 51 SNPs per chromosome.\\nIsolates from 2011 and 2012 showed fewer SNPs (0 to 18), while their distance to\\nisolates obtained in 2015 to 2017 was 28 to 51 SNPs. AllblaVIM-1-harboring isolates\\nclustered separately with a distance of/H1102289 SNPs to the closestblaVIM-1-negative S.\\nInfantis relative, isolate 09-03100 (14)( Fig. 1 and Table S2).\\n(iii)\\nMobile genetic elements. S1 PFGE hybridization experiments revealed the\\npresence of blaVIM-1-harboring 290- to 300-kb plasmids for isolates N2-1 (S. Infantis),\\nN2-2 (S. Goldcoast), and N2-8 (E. cloacae) (Fig. S2). WGS results conﬁrmed presence of\\nIncHI2 (ST1) plasmids with consensus sequence lengths between 304 and 311 kb and\\nsequence similarities of 98 to 100% to reference plasmid pSE15-SA01028 (Table 2).\\nDetailed\\ncomparison of the plasmid sequences, visualized using the BLAST Ring Image\\nGenerator (BRIG) (Fig. 2), revealed that the main differences between plasmids are\\nFIG 1 Minimum spanning tree ofblaVIM-1 harboring S. Infantis isolates based on genome SNP analysis\\nwith the complete genome ofS. Infantis 15-SA01028. Branches are labeled with the number of SNP\\ndifferences. Darker greenish colors reﬂect most recent isolates obtained. Isolate 09-03100 served as the\\nnearest VIM-1-negative S. Infantis neighbor for comparison purposes.\\nRoschanski et al.\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 4\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 4}, page_content='based on structural changes in the blaVIM-1-harboring Tn21-like transposon. In all\\nplasmids the blaVIM-1 gene was part of an In110class 1 integron accompanied by the\\ngenes aacA4 and aadA1 in its variable region, as previously described for pRH-R27 (10).\\nIn\\np15SA-01028 and all isolates but G336-1 and N2-1, however, an insertion of a further\\nIn1516 class 1 integron harboring anereA and an aadA1 gene cassette was observed.\\nThis additional integron was located upstream of the IS1326 module and was itself\\nﬂanked upstream by cryptic coding DNA sequences, comprising several genes com-\\nmonly found associated with other mobile genetic elements (Fig. 3). Although the\\nblaVIM-1 gene was not present in theS. Goldcoast isolate N2-6 from manure, a 93%\\nidentity to pSE15-SA01028 could be observed for the plasmid harbored by this isolate\\n(286.971-bp consensus sequence length, conﬁrmed by an/H11011290-kb band in the S1 PFGE\\n[Table 2and Fig. S2]). In this isolate, absence of theblaVIM-1 gene is due to the lack of\\nthe complete Tn21-like transposon in this plasmid region that could be conﬁrmed\\nthrough closing the plasmid sequence via PCR sequencing at this position.\\nDISCUSSION\\nIn this study, twoblaVIM-1-producing Salmonella isolates (one S. Infantis and oneS.\\nGoldcoast) and oneblaVIM-1-producing Enterobacter cloacaeisolate were isolated within\\nan investigation of a farrow-to-wean farm suspicious for having connections to two\\npreviously pig-associatedblaVIM-1 ﬁndings in Germany.\\nThese ﬁndings indicate that in 2017 still a source, bearing potential to transfer\\nVIM-1-producing isolates to German pig producers, was present. However, the low\\nnumber of obtained VIM-1-producing isolates, despite an extensive sampling effort,\\nsuggests a very low prevalence of VIM-1-producing isolates in this farm. In-depth\\nmolecular analysis using WGS techniques revealed a close relationship of allblaVIM-1-\\nharboring S. Infantis isolates from German livestock recovered until now. The high\\ndegree of molecular similarity of isolates and associated mobile genetic elements hints\\nat a connection of affected farms that is not obvious through analysis of available\\nbackground information, especially for the poultry-related ﬁndings. SinceS. Infantis was\\ndetected in ﬁve pig-associated cases and two positive chicken-fattening farms, this\\nserovar might play a particular role in the occurrence ofblaVIM-1. However, comparative\\nanalysis in this study revealed thatblaVIM-1 in Germany is not restricted to a certain\\nserovar but might be regarded as a long-term multispecies-related occurrence, linked\\nto a certain pSE15-SA01028-like IncHI2 plasmid. All nineblaVIM-1-positive Salmonella\\nisolates as well as theblaVIM-1-positive E. cloacae isolate harbored this gene on/H11011290-\\nTABLE 2Occurrence and molecular characteristics of pSE15-SA01028-like plasmids in isolates derived from farm N2 and previously\\nblaVIM-1-positive isolates from food, pig, and poultry farms in Germany\\nIsolate\\nPresence of\\npAmpC- or\\ncarbapenemase-\\nencoding gene(s)\\nSize (kb) of\\nblaVIM-1-harboring\\nplasmidsa\\nPresence of\\nIncHI2 plasmidb\\nTotal consensus\\nsequence length\\n(bp)\\n% sequence\\nidentity to\\np15-SA01028 Tn21-like variant\\nN2-8 blaACC-1, blaACT-7-like, blaVIM-1 300 IncHI2 (ST1) 307.594 99 As pSE15-SA01028\\nN2-6 blaACC-1 IncHI2 (ST1) 286.971 92\\nN2-5 /H1102130 /H1102110\\nN2-4 /H1102130 /H1102110\\nN2-3 /H1102130 /H1102110\\nN2-2 blaACC-1, blaVIM-1 300 IncHI2 (ST1) 310.794 100 As pSE15-SA01028\\nN2-1 blaACC-1, blaVIM-1 290 IncHI2 (ST1) 304.382 98 Absence of In1516c\\n16-SA00749 blaACC-1, blaVIM-1 290 IncHI2 (ST1) 298.680 96 As pSE15-SA01028\\n15-SA01028 blaACC-1, blaVIM-1 300 IncHI2 (ST1) 310.921 100 Reference (CP026661)\\nG-336-1a blaACC-1, blaVIM-1 300 IncHI2 (ST1) 305.248 98 Absence of In1516c\\nV363 blaACC-1, blaVIM-1 300 IncHI2 (ST1) 310.162 100 As pSE15-SA01028\\nR27 blaACC-1, blaVIM-1 300 IncHI2 (ST1) 310.871 100 As pSE15-SA01028\\nR25 blaACC-1, blaVIM-1 300 IncHI2 (ST1) 310.921 100 As pSE15-SA01028\\nR3 blaACC-1, blaVIM-1 300 IncHI2 (ST1) 310.921 100 As pSE15-SA01028\\naBased on S1 restriction and subsequent hybridization.\\nbBased on CGE batch upload analysis.\\ncAs described by Falgenhauer et al. (10).\\nAnalysis\\nof VIM-1-Positive Strains in German Livestock\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 5\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 5}, page_content='to 300-kb IncH12 (ST1) plasmids with a backbone of 96 to 100% similarity to pSE15-\\nSA01028 and linked to a certain Tn21-like transposable element. Interestingly, mobility\\nof the Tn21-like transposon carrying theblaVIM-1 gene could be veriﬁed through the\\ndetection of S. Goldcoast N2-6 (isolated from manure) with a plasmid showing 92%\\nidentity to pSE15-SA01028 but lacking the complete blaVIM-1-containing Tn21-like\\ntransposon. However, since active transposition of Tn21-like transposable elements is a\\nreplicative mechanism, the loss of the whole transposon has possibly been mediated by\\nFIG 2 Comparative plasmid maps (BRIG) of isolates from farm N2 andblaVIM-1-harboring Salmonella isolates from previous studies, showing sequence identities\\nof /H1102290% to reference pSE15-SA01028 (GenBank accession numberCP026661). Isolates G336-1a and N2-1 lack the In1516class 1 integron harboringereA and\\naadA1 gene cassettes downstream of theblaVIM-1-containing class 1 integron. This integron was previously not described on pRH-R27 by Falgenhauer et al. (10).\\nIn\\nisolate N2-6, PCR-based sequence gap closure revealed that the wholeblaVIM-1-harboring Tn21-like transposon is absent in this plasmid region. Blue circles\\ndepict plasmids ofS. Infantis isolates from pigs, yellow circles ofS. Infantis from poultry farms, and green circles ofS. Goldcoast isolates, and the red circle depicts\\nthe plasmid of theE. cloacae isolate.\\nRoschanski et al.\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 6\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 6}, page_content='homologous recombination involving two ﬂanking IS1 elements (insA/insB [Fig. 2]), as\\nin\\nisolate N2-6, only oneinsA/insB copy is left. Mobility of this transposon was previously\\nalso proven throughin vitroexperiments, revealing acquisition of theblaVIM-1-carrying\\nTn21 on an IncI1 plasmid that was colocated in a VIM-1-producingE. coliisolate (6). The\\nsole\\npresence of this certainblaVIM-1-carrying Tn21 in any bacterial species or plasmid\\ncould be a hint of a potential relation to theblaVIM-1 ﬁndings in German livestock. It is\\nnoteworthy that theereA-aadA1-harboring integron downstream of theblaVIM-1-aacA4-\\naadA1 integron was previously not described on pRH-R27 by Falgenhauer et al. (10).\\nThis\\ndiscrepancy might be an artifact from Roche 454 short-read assembly or could be\\nbased on loss of this region during the sample preparation for sequencing by Falgen-\\nhauer et al. (10). The absence of this region could also be observed for isolates G336-1a\\nand\\nN2-1.\\nTransfer of the pSE15-SA01028-like IncHI2 plasmids seems to be less efﬁcient, since\\nin vitro transfer of the blaVIM-1 plasmid at least starting fromS. Infantis is difﬁcult to\\nprove (3, 10). However, the presence of this plasmid inS. Goldcoast, E. cloacae, and\\nseveral E. coli STs (6, 7, 11) unambiguously conﬁrms its transferabilityin vivo.\\nIn addition, the results of the SNP analysis of the VIM-1-producingS. Infantis isolates\\nindicate that transferin vivomight be a rare event and that there might have been just\\na sole acquisition event in one commonS. Infantis ancestor cell. All eightblaVIM-1-\\npositive S. Infantis isolates show fewer than 51 SNPs (Table S2 andFig. 1), conﬁrming\\ntheir\\nclose genetic relationship that had already been assumed on the basis of PFGE\\nanalysis (3). Moreover, they build a separate cluster (/H1102289 SNPs to the closest relative,\\n09-03100) within the S. Infantis population circulating in German food and livestock\\n(14).\\nAmong\\nthis cluster, isolates from 2011/2012 share the lowest number of SNPs (0 to\\n18 SNPs [Fig. 1and Table S2]). Assuming inS. Infantis a SNP rate at least as high as in\\nS. Typhimurium with 3 to 5 SNPs per year per chromosome (15), this sole transfer event\\nFIG 3 Observed variants of the Tn21-like transposon already described by Falgenhauer et al. (10), bearing theblaVIM-1 gene on a class 1 integron with additional\\nresistance genes aacA4 and aadA1 in its variable region. Cyan arrows represent transposon-associated genes, red ones antimicrobial resistance and class 1\\nintegron-associated genes, and gray arrows cryptic DNA sequences. The class 1 integron harboringereA and aadA1 gene cassettes, downstream of the\\nblaVIM-1-containing class 1 integron, was previously not described on pRH-R27 by Falgenhauer et al. (10) and was only present in isolates N2-2, N2-8, 16-SA00749,\\n15-SA01028,\\nV363, R27, R25, and R3.\\nAnalysis of VIM-1-Positive Strains in German Livestock\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 7\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 7}, page_content='might have occurred indeed shortly before or within the time frame of the ﬁrst ﬁndings\\nof VIM-1-positiveEnterobacteriaceae in 2011/2012. In fact, the number of SNPs between\\nthe isolates of our study suggests an even higher SNP rate for S. Infantis. ForSalmonella,\\ndisease outbreaks are usually characterized by SNP variations of 0 to 30 and varying\\nwithin each serovar and clonal lineage (16). Finally, SNP- andplasmid-based analyses\\nin this study indicate that this occurrence ofblaVIM-1 in Germany might be regarded\\nas a single low-level multispecies event that drags on for obviously more than 6\\nyears.\\nFollowing the hypothesis of a single transfer event intoS. Infantis, the acquisition of\\nthis VIM-1 metallo-beta-lactamase-encoding plasmid might have been associated with\\nan alteration ofSalmonella surface antigens leading to the nonserotypable phenotype\\nusing the Kaufmann-White scheme, as this is only observed for theS. Infantis isolates\\nfrom 2011/2012. A similar phenomenon was already described for anS. Typhimurium\\nstrain acquiring ablaVIM-2 gene. In this case, acquisition of theblaVIM-2 gene resulted in\\nalterations in micro- and macroscopic morphology, a reduced growth rate, and de-\\ncreased motility (17). Since S. Infantis isolates from 2015 and later show a classical\\nserological S. Infantis phenotype, subsequent genetic adaptations might have allowed\\nconquest of serological constraints.\\nAlthough no blaVIM-1-harboring E. coli was detected in this study, recent reports of\\nE. coliharboring highly similar or identicalblaVIM-1-bearing plasmids (6,11) suggest the\\npotential\\nexistence of a low-abundanceE. coli subpopulation in the German livestock\\nsector that might be able to pop up through antibiotic treatment like was assumed for\\nthe VIM-1-producingE. coli ST88 contamination in pig farm S2 (7, 12).\\nThe\\ndetection of theE. cloacaeonly through reanalysis of samples positive for VIM-1-\\nproducingSalmonellaunderlines that the detection of CPE in livestock samples is challeng-\\ning due to their low prevalence and the low-level expression of carbapenemase-encoding\\ngenes under nonselective pressure (MICs of CPE in this study ranged from 0.12 to\\n4 mg/liter). Low-level expression of theblaVIM-1 gene in isolates that had not undergone\\nantibiotic selective pressure was already reported forblaVIM-1 Salmonella and E. coli\\nisolates from German livestock and food (3, 13). However, a higher level of carbapenem\\nresistance\\nhas been observed through their cultivation in broth supplemented with a\\ncarbapenem (3). Differences in MICs between isolates of this study (although harboring\\na\\nhighly similar or identical plasmid) or after reanalysis of isolates several years after\\ntheir ﬁrst MIC testing underline that the level of carbapenem resistance might be\\ninﬂuenced by different origins and evolution of isolates (reﬂected through SNP differ-\\nences) or even isolate handling conditions (different isolation, cultivation, and storing\\nconditions in the different studies). These conditions might affect the expression of the\\nblaVIM-1 gene or the copy number of the plasmids (e.g., through antibiotic selective\\npressure).\\nIn human clinical settings in Germany, blaVIM-1 was ﬁrst described in 2007 (18);\\ntoday,\\nit is the second most frequently detected carbapenemase gene inEnterobacte-\\nriaceae, with E. cloacae being the main blaVIM-1 reservoir (19). In contrast, no VIM-1-\\nproducing Salmonella isolate\\nfrom humans in Germany has been described so far (20).\\nCurrently\\nthere is no literature available on blaVIM-1-harboring plasmids in human\\nclinical isolates in Germany. This raises questions on the role of theE. cloacae isolate\\nN2-8 serving either as one of the recipients that took up the plasmid fromS. Infantis or\\nas the initial donor of theblaVIM-1 plasmid. Further studies have to elucidate whether\\nthe blaVIM-1 gene in human clinicalE. cloacae isolates is also associated with pSE15-\\nSA01028-like plasmids. Answers to these questions require a “one health” approach,\\nincluding expanded in-depth comparative analysis, encompassing VIM-1-harboring\\nplasmids and isolates circulating in the human population and the environment in\\nGermany.\\nSince the ﬁrst ﬁndings of CPE in livestock, a number of studies have reported\\ncarbapenemase-producing bacteria with different carbapenemase-encoding genes in\\nfood-producing animals, including pigs, and companion animals worldwide (21–26).\\nWhether\\nthe globally increasing reports on CPE in livestock and along the food chain\\nRoschanski et al.\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 8\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 8}, page_content='are due to increased awareness of this issue followed by intensiﬁed screening activities\\nor if this reﬂects the current emergence of livestock as a new reservoir for CPE, probably\\ntriggered through their increased emergence in the human clinical sector, needs to be\\nfurther investigated.\\nIn Germany, the persistence of ablaVIM-1 gene located on a Tn21-like transposon on\\na transferable plasmid was shown in recent years. Evaluation of WGS data just allows\\nassumptions on the source of theblaVIM-1 gene on the farms; the agent that actually\\nserved as the initial bacterial source remains unknown. However,S. Infantis, which is\\nfrequently found in poultry and pigs in Germany and is among the main causes of\\nhuman salmonellosis in Europe (27, 28), may play a certain role in this case. The fact that\\nthe\\nVIM-1-producing S. Infantis was isolated in each of the affected farms but is rarely\\nfound through screening or retrospective analysis (6, 7) suggests that this serovar\\nserves\\nat least as an appropriate reservoir, stably hosting this particular blaVIM-1\\npSE15-SA01028-like plasmid. Finally, consideration must be given to how to handle the\\nsources of these VIM-1-producing isolates, such as the breeding farm under investiga-\\ntion in this study. Although persistence of CPE in animal surroundings was shown to\\nnot be dependent on selective pressure (29), antibiotic treatments such as those with\\namoxicillin\\nmight have triggered their persistence in this occurrence. Particular atten-\\ntion should be paid to detailed analysis of the epidemiological situation on the farm,\\ni.e., to determine in which groups of sows the isolates circulate. It should also be\\nexamined whether a source of entry from the environment via feed or other vectors can\\nbe identiﬁed. Based on these results, speciﬁc action should be taken to stop the\\nintroduction and circulation of these bacteria within the farm and to minimize the risk\\nof spreading the bacteria to fattening farms. Groups of sows identiﬁed to be infected\\nwith VIM-1-producingEnterobacteriaceae should be replaced and affected facilities or\\nstables should undergo strict cleaning, disinfection, and hygiene measures. Since\\nSalmonellaplays a special role in the current infection, a Salmonella vaccination\\nprogram might also be considered to support the eradication of these carbapenemase-\\nproducingSalmonella strains from the herd.\\nMATERIALS AND METHODS\\nSampling at the pig farm.Sampling of the breeding farm was performed during on-site visitation\\nby the Institute for Animal Hygiene and Environmental Health of Freie Universitaet Berlin. It covered\\nthree breeding centers, four gestation stalls, ﬁve farrowing barns (each containing approximately 75\\nfarrowing pens), and ﬁve rearing units. During the visit, one breeding center and one farrowing barn\\nwere not in use. The type and number of samples varied depending on the number of sows/piglets and\\nsubunits in each barn (Table S1). In the gestation stalls and the rearing quarters, mostly 10 to 15 sows\\nor weaners were housed per pen. From each of the units approximately ﬁve distinct fecal samples were\\ncollected and pooled per pen. Samples taken per pen were additionally pooled as shown in Table S1. In\\nthe breeding centers and the farrowing barns, where the sows were housed in individual stands, feces\\nfrom several animals were collected and pooled (Table S1). Boot swab samples were taken from the\\ncentral corridor of each housing unit. Liquid manure samples were collected from each of the three\\nmanure pits.\\nLaboratory analyses of the samples: screening for CPE andSalmonella spp. (i) Preenrichment\\nof samples.All samples were processed within 24 h after sampling. Of each pool of fecal samples, 20 g\\nwas inoculated in 180 ml of peptone water (buffered peptone water; Carl Roth GmbH, Karlsruhe,\\nGermany) without any supplement as recommended by the European Union Reference Laboratory for\\nAntimicrobial Resistance (EURL-AR [https://www.eurl-ar.eu/protocols.aspx]). Boot swabs were incubated\\nin\\n100 ml of buffered peptone water. From each liquid manure sample, 5 g was inoculated in 45 ml of\\npeptone water. After an overnight incubation at 37°C, aliquots of each culture were stored at/H1100280°C for\\nfuture investigations. All cultures were subjected to CPE detection procedures as follows.\\n(ii) DNA preparation, screening, and identiﬁcation of carbapenemase-encoding genes using\\nPCR. Each of the incubated peptone water cultures was tested with direct real-time PCR. The DNA\\npreparation as well as the real-time PCR based assay, detecting the carbapenemase genesblaVIM, blaKPC,\\nblaNDM, blaOXA-48, andblaGES, were performed as previously described (30). Subsequent bacterial species\\nidentiﬁcation\\nwas performed using matrix-assisted laser desorption ionization–time of ﬂight (MALDI-TOF)\\nmass spectrometry (MS) (MALDI Microﬂex LT and Biotyper database; Bruker Daltronics, Bremen,\\nGermany).\\n(iii) Isolation of CPE via incubation on selective agar plates.The overnight cultures were streaked\\non (i) ChromID Carba (bioMérieux, France) as well as (ii) MacConkey agar plates containing 1 mg/liter of\\ncefotaxime sodium salt (CTX; Merck KGaA, Darmstadt, Germany). On the following day, MALDI-TOF MS\\n(MALDI Microﬂex LT and Biotyper database; Bruker Daltonics, Bremen, Germany) was performed from\\nrandomly selected colonies (one colony per morphology was analyzed per plate) to determine bacterial\\nAnalysis of VIM-1-Positive Strains in German Livestock\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 9\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 9}, page_content='species. Subsequent conﬁrmation of the presence of the carbapenemase-encoding gene was\\ndetermined by real-time PCR as mentioned above (see “DNA preparation, screening, and identiﬁ-\\ncation of carbapenemase-encoding genes using PCR”).\\n(iv) Isolation of Salmonella following a modiﬁed DIN EN ISO 6579 protocol adapted to the\\ndetection of nonmotile Salmonella variants. As S. Infantis with different serological characteristics\\n(serologically rough, nonmotileS. entericagroup C with antigenic formula “6,7:-:-” or classically typed as\\nS. Infantis 6,7:r:1,5) were found in previously affected pig and poultry farms, a parallel in-depth screening\\nfor Salmonella was performed. Salmonella species isolates were obtained according to ISO 6579:\\n2002/H11001Amd 1:2007, with subsequent serotyping according to the White-Kauffmann-Le Minor scheme\\n(31).\\nSince\\nthe VIM-1-producingS. Infantis isolates from 2011/2012 were nonmotile, the DIN EN ISO 6579\\nprotocol was expanded and included both selective enrichment with Rappaport-Vassiliadis medium with\\nsoya (RVS-Bouillon) for nonmotileSalmonella variants and modiﬁed semisolid Rappaport-Vassiliadis\\nmedium (MSRV) for motile variants. Furthermore, cultures were incubated for 48 h on xylose-lysine-\\ndeoxycholate agar (XLD agar) with and without 1 mg/liter of CTX.\\n(v) Detailed analyses of single cultures.Samples that tested negative for CPE but positive for\\nVIM-1-producing Salmonella by using the modiﬁed DIN EN ISO 6579 protocol were analyzed a second\\ntime. The stored (/H1100280°C) cultures as well as the stored primary overnight cultures were revitalized (1:100)\\nin 5 ml of LB medium containing 1 mg/liter of CTX. The derived cultures were subsequently spread on\\nChromID Carba plates (bioMérieux, France) and MacConkey agar plates containing 1 mg/liter of CTX and\\n0.125 mg/liter of meropenem (MEM). Single colonies were selected for species identiﬁcation using\\nMALDI-TOF MS followed by a CPE status conﬁrmation using real-time PCR, as described in “DNA\\npreparation, screening, and identiﬁcation of carbapenemase-encoding genes using PCR” above.\\nPhenotypic and genotypic characterization of the carbapenemase-producing isolates. (i)Anti-\\nmicrobial susceptibility testing.Determination of MICs of the carbapenems ertapenem, meropenem,\\nand imipenem for presumptive CPE was carried out according to Commission Implementing Decision\\n2013/652/EU on the monitoring and reporting of antimicrobial resistance in zoonotic and commensal\\nbacteria. MICs were determined/redetermined using the broth microdilution method and following\\nguidelines described by the Clinical and Laboratory Standards Institute (CLSI) (32, 33), including, among\\nother\\nantimicrobial substances, the third-generation cephalosporins cefotaxime and ceftazidime and the\\ncarbapenems meropenem, ertapenem, and imipenem.\\n(ii) PFGE and Southern blot hybridization.CPE and Salmonella isolates underwent XbaI and S1\\nnuclease restriction of bacterial DNA and subsequent pulsed-ﬁeld gel electrophoresis (PFGE) (34) using\\nthe\\nCHEF-DR III system (Bio-Rad Laboratories GmbH, Munich, Germany) as previously described (35).\\nTo\\ndetermine the localization of theblaVIM-1 gene in these strains, Southern blot and hybridization\\nanalyses of S1 PFGE gels with a digoxigenin-labeledblaVIM-1 probe were conducted (35).\\nWGS.\\n(i) Library preparation.All Salmonella isolates and isolates harboring ablaVIM-1 gene were\\nsubjected to whole-genome sequencing (WGS) analysis. Liquid LB medium containing 1 mg/liter of CTX\\nwas inoculated with a single colony grown on LB agar. Broths were cultivated under shaking conditions\\n(180 to 220 rpm) at 37°C for 14 to 16 h. DNA was isolated using the PureLink genomic DNA minikit\\n(Invitrogen, Carlsbad, CA) followed by preparation of sequencing libraries using the Nextera XT DNA\\nsample preparation kit (Illumina, San Diego, CA) according to the manufacturer’s protocol. Paired-end\\nsequencing was performed in 2/H11003251 cycles on the Illumina MiSeq benchtop using the MiSeq Reagent\\nv3 600-cycle kit (Illumina).\\n(ii) Comparative analysis of WGS data of isolates andblaVIM-1-harboring plasmids. Raw reads\\nwere trimmed using trimmomatic (36), and trimmed sequencing data werede novo assembled using\\nSPAdes (https://cge.cbs.dtu.dk/services/SPAdes/). Assemblies led to an average contig number of 169\\nwith\\nan average coverage of 67. Contig sequences allowed determination of multilocus sequence\\ntype (MLST), plasmid multilocus sequence type (pMLST), and resistance gene proﬁles using the Bacterial\\nAnalysis Pipeline-Batch Upload from the Center for Genomic Epidemiology (CGE; http://www\\n.genomicepidemiology.org). Kmer-Finder (37) was used to conﬁrm MALDI-TOF species identiﬁcation and\\nSISTR\\n(38) for Salmonella serovar conﬁrmation. Mapping of raw reads against the PacBio bacterial\\nchromosome sequence of the meat-derivedblaVIM-1-harboring S. Infantis isolate 15-SA01028 (GenBank\\naccession numberCP026660.1)( 8) and subsequent SNP analysis ofSalmonella isolates were carried out\\nwith the BioNumerics software (v7.6; Applied Maths, Ghent, Belgium). TheblaVIM-1-negative S. Infantis\\nisolate 09-03100 was included in SNP analysis, as this isolate represents the closest relative among the\\ninvestigated isolates from the national strain collection of the NRL Salmonella (14). In terms of plasmid\\nsequence\\ncomparison, raw reads were mapped against the PacBio sequence of pSE15-SA01028 (GenBank\\naccession number CP026661.1), using CLC Genomics workbench 9.5. Visualization of differences in\\nplasmid\\nbackbones was performed with BLAST Ring Image Generator 0.95 (BRIG) (39). The absence of the\\nblaVIM-1-harboring Tn21-like transposon in the plasmid sequence of isolate N2-6 was conﬁrmed by\\nplasmid closure using PCR, followed by sequencing as previously described (35).\\nData\\navailability. Whole-genome raw sequence data of all blaVIM-1-positive isolates and the S.\\nInfantis blaVIM-1-negative isolate 09-03100 were submitted to the ENA database under the following\\naccession numbers: for N2-1,ERS2958092; N2-2,ERS2958093; N2-3,ERS2958094; N2-4,ERS2958095; N2-5,\\nERS2958096; N2-6,ERS2958097; N2-8,ERS2958098; R3,ERS2154041; R25,ERS2958099; R27,ERS2958100;\\nV363, ERS2958101; G-336-1a, ERS2101552; 15-SA1028, ERS2488743; 16-SA00749, ERS2488744; and 09-\\n03100, ERS2958102.\\nRoschanski et al.\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 10\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 10}, page_content='SUPPLEMENTAL MATERIAL\\nSupplemental material for this article may be found at https://doi.org/10.1128/\\nmSphere.00089-19.\\nFIG\\nS1, PDF ﬁle, 0.1 MB.\\nFIG S2, PDF ﬁle, 0.1 MB.\\nFIG S3, PDF ﬁle, 0.1 MB.\\nTABLE S1, PDF ﬁle, 0.1 MB.\\nTABLE S2, PDF ﬁle, 0.1 MB.\\nACKNOWLEDGMENTS\\nWe thank our technicians Heike Rose, Susann Sellenthin, Michael Kuehl, and Martha\\nBrom\\nfor committed assistance in the lab. We thank Carlus Deneke for his support\\nprocessing WGS data. The farmer is gratefully acknowledged for his kind cooperation.\\nThis work was partly funded by German Federal Ministry of Education and Research\\n(funding number 01Kl1013B and 01Kl1013C as part of the research consortium RESET).\\nREFERENCES\\n1. Tangden T, Giske CG. 2015. Global dissemination of extensively drug-\\nresistant carbapenemase-producing Enterobacteriaceae: clinical per-\\nspectives on detection, treatment and infection control. J Intern Med\\n277:501–512.https://doi.org/10.1111/joim.12342.\\n2.\\nFischer J, Rodriguez I, Schmoger S, Friese A, Roesler U, Helmuth R, Guerra\\nB. 2012. Escherichia coli producing VIM-1 carbapenemase isolated on a\\npig farm. J Antimicrob Chemother 67:1793–1795. https://doi.org/10\\n.1093/jac/dks108.\\n3.\\nFischer J, Rodriguez I, Schmoger S, Friese A, Roesler U, Helmuth R, Guerra\\nB. 2013. Salmonella enterica subsp. enterica producing VIM-1 carbapen-\\nemase isolated from livestock farms. J Antimicrob Chemother 68:\\n478 – 480.https://doi.org/10.1093/jac/dks393.\\n4.\\nEFSA Panel on Biological Hazards (BIOHAZ). 2013. Scientiﬁc opinion on\\ncarbapenem resistance in food animal ecosystems. EFSA J 11:3501.\\n5. European Commission. 2013. 2013/652/EU Commission Implementing\\nDecision of 12 November 2013 on the monitoring and reporting of\\nantimicrobial resistance in zoonotic and commensal bacteria. Off J Eur\\nUnion 2013:L303/26 –L303/39.\\n6. Roschanski N, Fischer J, Falgenhauer L, Pietsch M, Guenther S, Kreienbrock\\nL, Chakraborty T, Pfeifer Y, Guerra B, Roesler UH. 2018. Retrospective analysis\\nof bacterial cultures sampled in German chicken-fattening farms during the\\nyears 2011–2012 revealed additional VIM-1 carbapenemase-producing\\nEscherichia coliand a serologically roughSalmonella entericaserovar Infan-\\ntis. Front Microbiol 9:538.https://doi.org/10.3389/fmicb.2018.00538.\\n7.\\nFischer J, San Jose M, Roschanski N, Schmoger S, Baumann B, Irrgang A,\\nFriese A, Roesler U, Helmuth R, Guerra B. 2017. Spread and persistence\\nof VIM-1 carbapenemase-producing Enterobacteriaceae in three Ger-\\nman swine farms in 2011 and 2012. Vet Microbiol 200:118 –123.https://\\ndoi.org/10.1016/j.vetmic.2016.04.026.\\n8\\n. Borowiak M, Fischer J, Baumann B, Hammerl JA, Szabo I, Malorny B.\\n2018. Complete genome sequence of a VIM-1-producing Salmonella\\nenterica subsp. enterica serovar Infantis isolate derived from minced\\npork meat. Genome Announc 6:e00327-18. https://doi.org/10.1128/\\ngenomeA.00327-18.\\n9.\\nBorowiak M, Szabo I, Baumann B, Junker E, Hammerl JA, Kaesbohrer A,\\nMalorny B, Fischer J. 2017. VIM-1-producingSalmonella Infantis isolated\\nfrom swine and minced pork meat in Germany. J Antimicrob Chemother\\n72:2131–2133.https://doi.org/10.1093/jac/dkx101.\\n10.\\nFalgenhauer L, Ghosh H, Guerra B, Yao Y, Fritzenwanker M, Fischer J,\\nHelmuth R, Imirzalioglu C, Chakraborty T. 2017. Comparative genome\\nanalysis of IncHI2 VIM-1 carbapenemase-encoding plasmids ofEsche-\\nrichia coli and Salmonella enterica isolated from a livestock farm in\\nGermany. Vet Microbiol 200:114 –117.https://doi.org/10.1016/j.vetmic\\n.2015.09.001.\\n11.\\nIrrgang A, Fischer J, Grobbel M, Schmoger S, Skladnikiewicz-Ziemer T,\\nThomas K, Hensel A, Tenhagen B-A, Käsbohrer A. 2017. Recurrent de-\\ntection of VIM-1-producingEscherichia coliclone in German pig produc-\\ntion. J Antimicrob Chemother 72:944 –946.https://doi.org/10.1093/jac/\\ndkw479.\\n12.\\nRoschanski N, Friese A, Thieck M, Roesler U. 2016. Follow-up investiga-\\ntion of the ﬁrst VIM-1-positive pig farm in Germany— how is the situa-\\ntion 4 years after the ﬁrst detection? Clin Microbiol Infect 22:951–953.\\nhttps://doi.org/10.1016/j.cmi.2016.08.018.\\n13.\\nRoschanski N, Guenther S, Vu TTT, Fischer J, Semmler T, Huehn S, Alter\\nT, Roesler U. 2017. VIM-1 carbapenemase-producing Escherichia coli\\nisolated from retail seafood, Germany 2016. Euro Surveill 22:17-00032.\\nhttps://doi.org/10.2807/1560-7917.ES.2017.22.43.17-00032.\\n14.\\nFischer J, Borowiak M, Baumann B, Szabo I, Malorny B. 2017. Whole-\\ngenome sequencing analysis of multidrug-resistant Salmonella Infantis\\nisolates circulating in the German food-production chain, abstr P1080.\\n27th ECCMID, Vienna, Austria, 22 to 25 April 2017.\\n15. Hawkey J, Edwards DJ, Dimovski K, Hiley L, Billman-Jacobe H, Hogg G,\\nHolt KE. 2013. Evidence of microevolution of Salmonella Typhimurium\\nduring a series of egg-associated outbreaks linked to a single chicken\\nfarm. BMC Genomics 14:800.https://doi.org/10.1186/1471-2164-14-800.\\n16.\\nGymoese P, Sorensen G, Litrup E, Olsen JE, Nielsen EM, Torpdahl M. 2017.\\nInvestigation of outbreaks ofSalmonella enterica serovar Typhimurium\\nand its monophasic variants using whole-genome sequencing, Den-\\nmark. Emerg Infect Dis 23:1631–1639.https://doi.org/10.3201/eid2310\\n.161248.\\n17.\\nCordeiro NF, Chabalgoity JA, Yim L, Vignoli R. 2014. Synthesis of metallo-\\nbeta-lactamase VIM-2 is associated with a ﬁtness reduction inSalmonella\\nenterica serovar Typhimurium. Antimicrob Agents Chemother 58:\\n6528 – 6535.https://doi.org/10.1128/AAC.02847-14.\\n18.\\nWeile J, Rahmig H, Gfroer S, Schroeppel K, Knabbe C, Susa M. 2007. First\\ndetection of a VIM-1 metallo-beta-lactamase in a carbapenem-resistant\\nCitrobacter freundiiclinical isolate in an acute hospital in Germany. Scand\\nJ Infect Dis 39:264 –266.https://doi.org/10.1080/00365540600868388.\\n19.\\nPfennigwerth N. 2018. Bericht des Nationalen Referenzzentrums (NRZ)\\nfür gramnegative Krankenhauserreger. Epidemiol Bull 28:263–267.\\n20. Fernandez J, Guerra B, Rodicio MR. 2018. Resistance to carbapenems in\\nnon-typhoidal Salmonella enterica serovars from humans, animals and\\nfood. Vet Sci 5:E40.https://doi.org/10.3390/vetsci5020040.\\n2\\n1. Guerra B, Fischer J, Helmuth R. 2014. An emerging public health problem:\\nacquired carbapenemase-producing microorganisms are present in food-\\nproducing animals, their environment, companion animals and wild birds.\\nVet Microbiol 171:290 –297.https://doi.org/10.1016/j.vetmic.2014.02.001.\\n22.\\nZhang WJ, Lu Z, Schwarz S, Zhang RM, Wang XM, Si W, Yu S, Chen L, Liu\\nS. 2013. Complete sequence of thebla(NDM-1)-carrying plasmid pNDM-AB\\nfrom Acinetobacter baumannii of food animal origin. J Antimicrob Che-\\nmother 68:1681–1682. https://doi.org/10.1093/jac/dkt066.\\n23.\\nAl Bayssari C, Dabboussi F, Hamze M, Rolain JM. 2015. Emergence of\\ncarbapenemase-producing Pseudomonas aeruginosa and Acinetobacter\\nbaumannii in livestock animals in Lebanon. J Antimicrob Chemother\\n70:950 –951.https://doi.org/10.1093/jac/dku469.\\n24.\\nPruthvishree BS, Vinodh Kumar OR, Sinha DK, Malik YPS, Dubal ZB,\\nDesingu PA, Shivakumar M, Krishnaswamy N, Singh BR. 2017. Spatial\\nmolecular epidemiology of carbapenem-resistant and New Delhi me-\\ntallo beta-lactamase (blaNDM)-producing Escherichia coliin the piglets of\\nAnalysis of VIM-1-Positive Strains in German Livestock\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 11\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from '), Document(metadata={'source': 'texts\\\\Roschanski, VIM-1 Stall N2, 2019.pdf', 'page': 11}, page_content='organized farms in India. J Appl Microbiol 122:1537–1546.https://doi\\n.org/10.1111/jam.13455.\\n25.\\nPulss S, Semmler T, Prenger-Berninghoff E, Bauerfeind R, Ewers C. 2017.\\nFirst report of anEscherichia colistrain from swine carrying an OXA-181\\ncarbapenemase and the colistin resistance determinant MCR-1. Int J\\nAntimicrob Agents 50:232–236.https://doi.org/10.1016/j.ijantimicag.2017\\n.03.014.\\n26.\\nMollenkopf DF, Stull JW, Mathys DA, Bowman AS, Feicht SM, Grooters\\nSV, Daniels JB, Wittum TE. 2017. Carbapenemase-producing Enterobac-\\nteriaceae recovered from the environment of a swine farrow-to-ﬁnish\\noperation in the United States. Antimicrob Agents Chemother 61:\\ne01298-16.https://doi.org/10.1128/AAC.01298-16.\\n27.\\nHauser E, Tietze E, Helmuth R, Junker E, Prager R, Schroeter A, Rabsch W,\\nFruth A, Toboldt A, Malorny B. 2012. Clonal dissemination ofSalmonella\\nenterica serovar Infantis in Germany. Foodborne Pathog Dis 9:352–360.\\nhttps://doi.org/10.1089/fpd.2011.1038.\\n28.\\nEuropean Food Safety Authority. 2017. The European Union summary\\nreport on trends and sources of zoonoses, zoonotic agents and food-\\nborne outbreaks in 2016. EFSA J 15:5077.\\n29. Hadziabdic S, Fischer J, Malorny B, Borowiak M, Guerra B, Kaesbohrer A,\\nGonzalez-Zorn B, Szabo I. 2018. In vivo transfer and microevolution of\\navian native IncA/C2 blaNDM-1-carrying plasmid pRH-1238 during a\\nbroiler chicken infection study. Antimicrob Agents Chemother 62:\\ne02128-17.https://doi.org/10.1128/AAC.02128-17.\\n30.\\nRoschanski N, Friese A, von Salviati-Claudius C, Hering J, Kaesbohrer A,\\nKreienbrock L, Roesler U. 2017. Prevalence of carbapenemase producing\\nEnterobacteriaceae isolated from German pig-fattening farms during the\\nyears 2011–2013. Vet Microbiol 200:124 –129.https://doi.org/10.1016/j\\n.vetmic.2015.11.030.\\n31.\\nGrimont PAD, Weill F-X. 2007. Antigenic formulae of the Salmonella\\nserovars, 9th ed. WHO Collaborating Centre for Reference and Research\\nonSalmonella, Paris, France.\\n32. Clinical and Laboratory Standards Institute. 2018. Methods for dilution\\nantimicrobial susceptibility tests for bacteria that grow aerobically,\\nstandard M7, 11th ed. Clinical and Laboratory Standards Institute,\\nWayne, PA.\\n33. Clinical and Laboratory Standars Institute. 2018. Performance standards\\nfor antimicrobial disk and dilution susceptibility tests for bacteria iso-\\nlated from animals, standard Vet01-A5, 5th ed. Clinical and Laboratory\\nStandards Institute, Wayne, PA.\\n34. Ribot EM, Fair MA, Gautom R, Cameron DN, Hunter SB, Swaminathan B,\\nBarrett TJ. 2006. Standardization of pulsed-ﬁeld gel electrophoresis pro-\\ntocols for the subtyping ofEscherichia coli O157:H7, Salmonella, and\\nShigella for PulseNet. Foodborne Pathog Dis 3:59 – 67.https://doi.org/10\\n.1089/fpd.2006.3.59.\\n3\\n5. Rodriguez I, Barownick W, Helmuth R, Mendoza MC, Rodicio MR, Schroeter\\nA, Guerra B. 2009. Extended-spectrum beta-lactamases and AmpC beta-\\nlactamases in ceftiofur-resistantSalmonella entericaisolates from food and\\nlivestock obtained in Germany during 2003– 07. J Antimicrob Chemother\\n64:301–309.https://doi.org/10.1093/jac/dkp195.\\n36.\\nBolger AM, Lohse M, Usadel B. 2014. Trimmomatic: a ﬂexible trimmer for\\nIllumina sequence data. Bioinformatics 30:2114 –2120.https://doi.org/10\\n.1093/bioinformatics/btu170.\\n37.\\nLarsen MV, Cosentino S, Lukjancenko O, Saputra D, Rasmussen S, Has-\\nman H, Sicheritz-Ponten T, Aarestrup FM, Ussery DW, Lund O. 2014.\\nBenchmarking of methods for genomic taxonomy. J Clin Microbiol\\n52:1529 –1539.https://doi.org/10.1128/JCM.02981-13.\\n38.\\nAshton PM, Nair S, Peters TM, Bale JA, Powell DG, Painset A, Tewolde R,\\nSchaefer U, Jenkins C, Dallman TJ, de Pinna EM, Grant KA,Salmonella\\nWhole Genome Sequencing Implementation Group. 2016. Identiﬁcation\\nofSalmonella for public health surveillance using whole genome se-\\nquencing. PeerJ 4:e1752.https://doi.org/10.7717/peerj.1752.\\n39.\\nAlikhan NF, Petty NK, Ben Zakour NL, Beatson SA. 2011. BLAST Ring\\nImage Generator (BRIG): simple prokaryote genome comparisons. BMC\\nGenomics 12:402.https://doi.org/10.1186/1471-2164-12-402.\\nRoschanski et al.\\nMay/June 2019 Volume 4 Issue 3 e00089-19 msphere.asm.org 12\\n on January 10, 2020 by guest http://msphere.asm.org/ Downloaded from ')]\n",
      "\n",
      "--------------------------------------------------------------------------------\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import nest_asyncio\n",
    "import asyncio\n",
    "from langchain_community.document_loaders import Docx2txtLoader, PyPDFLoader\n",
    "\n",
    "# Apply nest_asyncio\n",
    "nest_asyncio.apply()\n",
    "\n",
    "directory = \"texts\"\n",
    "\n",
    "# Function to load PDF files\n",
    "async def load_pdf(file_path):\n",
    "    loader = PyPDFLoader(file_path)\n",
    "    pages = []\n",
    "    async for page in loader.alazy_load():\n",
    "        pages.append(page)\n",
    "    return pages\n",
    "\n",
    "# Load all files\n",
    "texts = {}\n",
    "for filename in os.listdir(directory):\n",
    "    file_path = os.path.join(directory, filename)\n",
    "    if filename.endswith(\".docx\"):\n",
    "        loader = Docx2txtLoader(file_path)\n",
    "        texts[filename] = loader.load()\n",
    "    elif filename.endswith(\".pdf\"):\n",
    "        texts[filename] = asyncio.run(load_pdf(file_path))\n",
    "\n",
    "# Print loaded texts\n",
    "for filename, text in texts.items():\n",
    "    print(f\"Content of {filename}:\")\n",
    "    print(text[:500])  # Print first 500 characters of each file\n",
    "    print(\"\\n\" + \"-\"*80 + \"\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "antibiotics-08-00023.pdf\n",
      "IMPART, plate validation, 2022.pdf\n",
      "Irrgang, GES-5, 2020.pdf\n",
      "Irrgang, VIM_2019.pdf\n",
      "Pauly, 19-AB01133.pdf\n",
      "Pauly, Carba plate validation, 2020.pdf\n",
      "Pauly,method comparison, 2021.pdf\n",
      "Roschanski, VIM-1 Stall N2, 2019.pdf\n"
     ]
    }
   ],
   "source": [
    "# Combine page content and clean up the texts dictionary\n",
    "for filename, documents in texts.items():\n",
    "    combined_text = \" \".join([doc.page_content for doc in documents])\n",
    "    texts[filename] = combined_text\n",
    "\n",
    "\n",
    "for filename in texts.keys():\n",
    "    print(filename)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import getpass\n",
    "import os\n",
    "\n",
    "os.environ[\"OPENAI_API_KEY\"] = getpass.getpass(\"Enter your OpenAI API key: \")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_openai import ChatOpenAI\n",
    "llm = ChatOpenAI(model=\"gpt-4o\", temperature=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total tokens used: 127612\n",
      "Prompt tokens: 127372\n",
      "Completion tokens: 240\n",
      "Total cost (USD): 0.32083\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'antibiotics-08-00023.pdf': {'isolation': 'no',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'},\n",
       " 'IMPART, plate validation, 2022.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'yes'},\n",
       " 'Irrgang, GES-5, 2020.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'},\n",
       " 'Irrgang, VIM_2019.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'},\n",
       " 'Pauly, 19-AB01133.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'},\n",
       " 'Pauly, Carba plate validation, 2020.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'yes'},\n",
       " 'Pauly,method comparison, 2021.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'yes'},\n",
       " 'Roschanski, VIM-1 Stall N2, 2019.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'}}"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "## Inclusion / Exclusion\n",
    "\n",
    "from langchain_core.output_parsers import JsonOutputParser\n",
    "from langchain_core.prompts import PromptTemplate\n",
    "from pydantic import BaseModel, Field\n",
    "\n",
    "# Define your desired data structure.\n",
    "class selection(BaseModel):\n",
    "    isolation: str = Field(description=\"Answer the first question with either 'yes' or 'no'\")\n",
    "    method: str = Field(description=\"Answer the second question with either 'yes' or 'no'\")\n",
    "    study_type: str = Field(description=\"Answer the third question with either 'yes' or 'no'\")\n",
    "\n",
    "# Set up the parser\n",
    "parser = JsonOutputParser(pydantic_object=selection)\n",
    "\n",
    "# Create the prompt template\n",
    "prompt = PromptTemplate(\n",
    "    template=\"\"\"\n",
    "    Read the following study and answer these questions in JSON format:\n",
    "    1. Does this study explicitly isolate strains of carbapenemase-producing bacteria?\n",
    "    2. Does this study state the bacteriological method used for the isolation of the carbapenemase-producing bacteria?\n",
    "    3. Does this study fall in one or multiple of the following categories: protocol validation, protocol development, method validation, method comparison, plate validation?\n",
    "    {format_instructions}\n",
    "    Text:{text}\\n\"\"\",\n",
    "    input_variables=[\"text\"],\n",
    "    partial_variables={\"format_instructions\": parser.get_format_instructions()},\n",
    ")\n",
    "\n",
    "# Combine the prompt with the model and parser\n",
    "chain = prompt | llm | parser\n",
    "\n",
    "from langchain_community.callbacks import get_openai_callback\n",
    "\n",
    "def process_texts(texts):\n",
    "    results = {}\n",
    "    with get_openai_callback() as cb:\n",
    "        for filename, text in texts.items():\n",
    "            result = chain.invoke({\"text\": text})\n",
    "            results[filename] = result\n",
    "        print(f\"Total tokens used: {cb.total_tokens}\")\n",
    "        print(f\"Prompt tokens: {cb.prompt_tokens}\")\n",
    "        print(f\"Completion tokens: {cb.completion_tokens}\")\n",
    "        print(f\"Total cost (USD): {cb.total_cost}\")\n",
    "    return results\n",
    "\n",
    "# Run the function on the texts dictionary\n",
    "extraction_selection = process_texts(texts)\n",
    "extraction_selection"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Accepted files: ['IMPART, plate validation, 2022.pdf', 'Irrgang, GES-5, 2020.pdf', 'Irrgang, VIM_2019.pdf', 'Pauly, 19-AB01133.pdf', 'Pauly, Carba plate validation, 2020.pdf', 'Pauly,method comparison, 2021.pdf', 'Roschanski, VIM-1 Stall N2, 2019.pdf']\n",
      "Rejected files: ['antibiotics-08-00023.pdf']\n"
     ]
    }
   ],
   "source": [
    "def categorize_files(extraction_selection):\n",
    "    accepted = []\n",
    "    rejected = []\n",
    "    for filename, response in extraction_selection.items():\n",
    "        if (response['isolation'] == 'yes' and response['method'] == 'yes') or response['study_type'] == 'yes':\n",
    "            accepted.append(filename)\n",
    "        else:\n",
    "            rejected.append(filename)\n",
    "    return accepted, rejected\n",
    "\n",
    "accepted, rejected = categorize_files(extraction_selection)\n",
    "\n",
    "print(\"Accepted files:\", accepted)\n",
    "print(\"Rejected files:\", rejected)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'IMPART, plate validation, 2022.pdf': 'accepted', 'Irrgang, GES-5, 2020.pdf': 'accepted', 'Irrgang, VIM_2019.pdf': 'accepted', 'Pauly, 19-AB01133.pdf': 'accepted', 'Pauly, Carba plate validation, 2020.pdf': 'accepted', 'Pauly,method comparison, 2021.pdf': 'accepted', 'Roschanski, VIM-1 Stall N2, 2019.pdf': 'accepted', 'antibiotics-08-00023.pdf': 'rejected'}\n"
     ]
    }
   ],
   "source": [
    "# Create a dictionary with filename and acceptance status\n",
    "acceptance_status = {filename: 'accepted' for filename in accepted}\n",
    "acceptance_status.update({filename: 'rejected' for filename in rejected})\n",
    "\n",
    "# Print the dictionary\n",
    "print(acceptance_status)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Matches: {'antibiotics-08-00023.pdf'}\n",
      "Number of matches: 1 of 1\n"
     ]
    }
   ],
   "source": [
    "# Compare ground_truth_rejected with rejected and count matches\n",
    "\n",
    "ground_truth_rejected = ['antibiotics-08-00023.pdf']\n",
    "\n",
    "matches = set(ground_truth_rejected) & set(rejected)\n",
    "match_count = len(matches)\n",
    "total_count = len(ground_truth_rejected)\n",
    "\n",
    "print(f\"Matches: {matches}\")\n",
    "print(f\"Number of matches: {match_count} of {total_count}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "from typing import List, Optional\n",
    "from pydantic import BaseModel, Field\n",
    "\n",
    "class CPE(BaseModel):\n",
    "    \"\"\"Information about studies on carbapenemase-producing bacteria\"\"\"\n",
    "    country: Optional[str] = Field(default=None, description=\"Origin of samples.\")\n",
    "    type_of_study: Optional[str] = Field(default=None, description=\"Classify the study into: sample investigation, protocol development, method comparison, method validation. Multiple labels possible separated by comma.\") #o3\n",
    "    sample_type: Optional[str] = Field(default=None, description=\"Which sample types were used in the study? Pick from 'livestock' (eg. fecal samples, feces, caecum, manure, cattle, pigs, poultry, etc.), 'livestock environment' (eg. farm or barn environment, etc.), 'companion animal' (eg. dog, cat, etc.), 'food' (eg. meat), 'environment' (eg. water, soil, etc.) or 'aquaculture'. Multiple answers possible separated by comma.\") #ok, 1 Fehler bei Pauly\n",
    "    matrix_description: Optional[str] = Field(default=None, description=\"Matrix description of study. Which animal and sample material was used?\") #ok, 1 Fehler bei Pauly\n",
    "    bacterial_species: Optional[str] = Field(default=None, description=\"Species of bacteria. Multiple answers possible separated by comma.\") #fertig, o1\n",
    "    genes: Optional[str] = Field(default=None, description=\"Genes or carbapenemase detected in this study.\") # potentially picklist, otherwise o3\n",
    "    protocol_reference: Optional[str] = Field(default=None, description=\"Is there a reference for the protocol used? Eg. EURL-AR, DIN EN ISO, etc. Yes or No. Provide reference, if possible.\") #fertig, o1\n",
    "    isolation_procedure: Optional[str] = Field(default=None, description=\"Is the method for the isolation of carbapenemase-producing bacteria culture-based, non-culture-based or both?\") #fertig, o1\n",
    "    method_validation: Optional[str] = Field(default=None, description=\"Was the method for the isolation of carbapenemase-producing bacteria validated? Yes or no. Provide values for sensitivity and specificity.\") #neu, testen\n",
    "    isolate_characterization: Optional[str] = Field(default=None, description=\"Which methods (eg. PFGE, NGS, etc.) were used for the characterization of carbapenemase-producing bacteria? Do not include information on species identification (MALDI-ToF). Multiple answers possible separated by comma.\") #fertig, o1\n",
    "    MLST_isolates: Optional[str] = Field(default=None, description=\"What was the MLST (Multi Locus Sequence Typing) of the isolates?\") #fertig, 4o\n",
    "    gene_localization: Optional[str] = Field(default=None, description=\"Was the characterization of isolates for carbapenemase-producing bacteria based on a gene located on a plasmid and / or on the chromosome? Multiple answers possible separated by comma.\") # 2 Fehler.: schauen on es nach beschränkung des textes besser wird.\n",
    "    plasmid_type: Optional[str] = Field(default=None, description=\"If plasmid was detected, on which type of plasmid (Inc group) was the gene located?\") #fertig, o1    \n",
    "    plasmid_size: Optional[str] = Field(default=None, description=\"If plasmid was detected, what was the size (bp or kbp) of the plasmid?\") #fertig, o1\n",
    "    plasmid_transferable: Optional[str] = Field(default=None, description=\"If plasmid was detected, was the CPE-gene harbouring plasmid transferable by conjugation and / or transformation? Multiple answers possible separated by comma.\") #fertig, o1\n",
    "    LOD_culture_based: Optional[str] = Field(default=None, description=\"Was there a LOD (limit of detection) for the isolation method provided?\") #fertig, 4o\n",
    "    confirmation_carbapenemase: Optional[str] = Field(default=None, description=\"How was carbapenemase-production confirmed (eg. PCR, real time PCR, MIC, etc.)? Do not include species confirmation (MALDI-ToF). Do not include characterization of the isolates.\") #müssen wir auf ergebnis warten  \n",
    "    sample_dilution: Optional[str] = Field(default=None, description=\"Dilution of the sample for the isolation method (eg. 1:10). Do not include dilution of the enrichment.\") #fertig, o1\n",
    "    sample_weight: Optional[str] = Field(default=None, description=\"Weight of samples. If multiple samples exist, separate weights by comma.\") #fertig, o1\n",
    "    first_enrichtment: Optional[str] = Field(default=None, description=\"Was the first enrichment or preenrichment selective or non selective? If selective, which antimicrobial was used with which concentration?\") #fertig, 4o\n",
    "    first_enrichtment_condition: Optional[str] = Field(default=None, description=\"Which condition of temperature and time for first enrichment or preenrichment?\") #fertig, 4o\n",
    "    second_enrichment: Optional[str] = Field(default=None, description=\"Was an other enrichment carried out? Yes or No. If you can not find the information it is 'no'.\") #fertig, 4o\n",
    "    second_enrichtment_selective: Optional[str] = Field(default=None, description=\"Was the second enrichment selective or non selective? If selective: which antimicrobial was used with which concentration?\") #fertig, 4o\n",
    "    second_enrichtment_condition: Optional[str] = Field(default=None, description=\"Which condition of temperature and time for second enrichment?\") #fertig, 4o\n",
    "    selective_plate: Optional[str] = Field(default=None, description=\"Which selective agars where used? Provide name of antimicrobial used at which concentration for each agar plate. Multiple answers possible separated by comma.\") #fertig, 4o; concentration fehlt zum teil; separieren in name of agar plate; name of antimicrobial & concentration\n",
    "    ncb_method: Optional[str] = Field(default=None, description=\"Which non-culture-based method (eg. PCR, real time PCR, NGS, WGS etc.) for the isolation of carbapenemase-producing bacteria was used?\") # Testen! Evtl mit regel arbeiten; o1 war besser\n",
    "    which_step: Optional[str] = Field(default=None, description=\"At which step was the non-culture-based method applied? After first enrichment, after second enrichment, post-isolation.\") #Neu, testen, o1\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total tokens used: 143119\n",
      "Prompt tokens: 140238\n",
      "Completion tokens: 2881\n",
      "Total cost (USD): 0.379405\n"
     ]
    }
   ],
   "source": [
    "from typing import List, Optional\n",
    "from pydantic import BaseModel, Field\n",
    "from langchain_core.output_parsers import JsonOutputParser\n",
    "from langchain_core.prompts import PromptTemplate\n",
    "\n",
    "# Set up the parser\n",
    "parser = JsonOutputParser(pydantic_object=CPE)\n",
    "\n",
    "prompt = PromptTemplate(\n",
    "    template=(\n",
    "        \"You are an expert extraction algorithm. \"\n",
    "        \"Only extract relevant information from the text. \"\n",
    "        \"If you do not know the value of an attribute asked to extract, \"\n",
    "        \"return null for the attribute's value.\"\n",
    "        \"{format_instructions}\\n\\n\"\n",
    "        \"Text: {text}\\n\"\n",
    "    ),\n",
    "    input_variables=[\"text\"],\n",
    "    partial_variables={\n",
    "        \"format_instructions\": parser.get_format_instructions()\n",
    "    },\n",
    ")\n",
    "\n",
    "# Combine the prompt with the model and parser\n",
    "chain = prompt | llm | parser\n",
    "\n",
    "from langchain_community.callbacks import get_openai_callback\n",
    "\n",
    "def process_texts(texts):\n",
    "    results = {}\n",
    "    with get_openai_callback() as cb:\n",
    "        for filename, text in texts.items():\n",
    "            result = chain.invoke({\"text\": text})\n",
    "            results[filename] = result\n",
    "        print(f\"Total tokens used: {cb.total_tokens}\")\n",
    "        print(f\"Prompt tokens: {cb.prompt_tokens}\")\n",
    "        print(f\"Completion tokens: {cb.completion_tokens}\")\n",
    "        print(f\"Total cost (USD): {cb.total_cost}\")\n",
    "    return results\n",
    "\n",
    "# Run the function on the texts dictionary\n",
    "extraction_cpe = process_texts(texts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "filename",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "acceptance_status",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "country",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "type_of_study",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "sample_type",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "matrix_description",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "bacterial_species",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "genes",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "protocol_reference",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "isolation_procedure",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "method_validation",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "isolate_characterization",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "MLST_isolates",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "gene_localization",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "plasmid_type",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "plasmid_size",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "plasmid_transferable",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "LOD_culture_based",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "confirmation_carbapenemase",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "sample_dilution",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "sample_weight",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "first_enrichtment",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "first_enrichtment_condition",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "second_enrichment",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "second_enrichtment_selective",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "second_enrichtment_condition",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "selective_plate",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "ncb_method",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "which_step",
         "rawType": "object",
         "type": "unknown"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "cbe8bb9a-7215-4000-8383-5869f3257438",
       "rows": [
        [
         "antibiotics-08-00023.pdf",
         "antibiotics-08-00023.pdf",
         "rejected",
         "Portugal",
         "sample investigation",
         "livestock",
         "livestock manure from poultry, pig, dairy farms and slaughterhouses",
         "Enterobacteriaceae, E. coli, Citrobacter freundii, Raoultella ornithinolytica, Salmonella enterica, C. koseri, Enterobacter cloacae, Morganella morganii, C. braakii, E. hermannii, E. vulneris, Klebsiella oxytoca, Kluyvera spp.",
         "catI, catII, qnrS, qnrB, oqxB, tet(A), tet(M), sul1, sul2, sul3, dfrIa, dfrIb, dfrIc, dfrIIa, dfrIIb, dfrIIc, dfrIIIa, dfrIIIb, dfrIIIc",
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null
        ],
        [
         "IMPART, plate validation, 2022.pdf",
         "IMPART, plate validation, 2022.pdf",
         "accepted",
         "Europe",
         "method comparison",
         "livestock, food",
         "spiked pig caecal and turkey meat samples",
         "Enterobacteriaceae, Salmonella, Escherichia coli, Klebsiella pneumoniae",
         "blaIMP, blaOXA-48, blaVIM-1, blaKPC-2, blaNDM-1",
         "EURL-AR",
         "culture-based",
         "Yes, sensitivity and specificity values provided",
         "species identification, susceptibility tests, genotyping",
         null,
         "plasmid",
         "IncHI2",
         null,
         "conjugation",
         "< 100 CFU/g",
         "PCR, real time PCR",
         "1:10",
         null,
         "non selective",
         "37°C overnight",
         "No",
         null,
         null,
         "Brilliance™ CRE Agar, CHROMID® CARBA, CHROMagar™ mSuperCARBA™, Chromatic™ CRE, CHROMID® OXA-48, Chromatic™ OXA-48",
         null,
         null
        ],
        [
         "Irrgang, GES-5, 2020.pdf",
         "Irrgang, GES-5, 2020.pdf",
         "accepted",
         "Germany",
         "sample investigation",
         "livestock",
         "fecal sample of a fattening pig",
         "Escherichia coli",
         "blaGES-1, blaGES-5, blaGES-5B",
         "EURL-AR",
         "culture-based",
         null,
         "real-time PCR, Sanger sequencing, Nanopore sequencing, Illumina sequencing",
         "ST1084",
         "plasmid",
         "IncF",
         "12 kb, 227 kb",
         null,
         null,
         "real-time PCR, PCR sequencing",
         null,
         null,
         "selective",
         null,
         "no",
         null,
         null,
         null,
         null,
         null
        ],
        [
         "Irrgang, VIM_2019.pdf",
         "Irrgang, VIM_2019.pdf",
         "accepted",
         "Germany",
         "sample investigation, method comparison",
         "livestock, livestock environment",
         "fecal samples from pigs, barn surrounding environment",
         "Escherichia coli",
         "blaVIM−1",
         "EURL-AR",
         "culture-based",
         null,
         "PFGE, whole-genome sequencing",
         "ST-88",
         "plasmid, chromosome",
         "IncHI2",
         "207 kb, 208 kb, 212 kb, 238 kb, 253 kb, 375 kb",
         "non-self-transmissible",
         null,
         "real-time PCR",
         "1:10",
         "10 g",
         "non selective",
         "16–20 h at 37°C",
         "Yes",
         "selective with 1 mg/L cefotaxime",
         "37°C, 44°C",
         "chromID Carba, MacConkey agar with 1 mg/L CTX",
         "real-time PCR",
         "post-isolation"
        ],
        [
         "Pauly, 19-AB01133.pdf",
         "Pauly, 19-AB01133.pdf",
         "accepted",
         "Germany",
         "sample investigation",
         "food, livestock",
         "pork shoulder from a pig raised in Germany",
         "Escherichia coli",
         "blaVIM-1, blaSHV-5, blaCMY-13",
         "EURL-AR",
         "culture-based",
         "Yes",
         "WGS, S1-PFGE",
         "ST5869",
         "plasmid",
         "IncA/C2",
         "190 kb",
         "conjugation, transformation",
         null,
         "real-time PCR",
         null,
         null,
         "selective",
         "37 ±2 °C for 16–18 h",
         "Yes",
         "selective",
         null,
         "McConkey agar supplemented with 0.125 mg/L MEM and 1 mg/L CTX",
         null,
         null
        ],
        [
         "Pauly, Carba plate validation, 2020.pdf",
         "Pauly, Carba plate validation, 2020.pdf",
         "accepted",
         "Germany",
         "method validation",
         "livestock, food",
         "cecum content, meat",
         "Enterobacteriaceae, Escherichia coli, Salmonella Corvallis, Vibrio parahaemolyticus, Klebsiella pneumoniae",
         "blaKPC, blaVIM, blaNDM, blaIMP",
         "EURL-AR",
         "culture-based",
         "Yes",
         null,
         null,
         "plasmid",
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         null,
         "No",
         null,
         null,
         "ChromID CARBA, MacConkey agar with cefotaxime and meropenem",
         null,
         null
        ],
        [
         "Pauly,method comparison, 2021.pdf",
         "Pauly,method comparison, 2021.pdf",
         "accepted",
         "Germany",
         "method comparison, method validation",
         "livestock",
         "pig caecum",
         "E. coli, Salmonella enterica subsp. enterica",
         "blaVIM-1, blaGES-5, blaKPC-2, blaNDM-1, blaOXA-48",
         "EURL-AR",
         "culture-based",
         "Yes, sensitivity 100%, specificity 100%",
         "PCR, MALDI-ToF",
         null,
         "plasmid",
         "IncHI2",
         null,
         null,
         null,
         "PCR, real time PCR",
         "1:10",
         "9 g",
         "non selective",
         "37 ± 2 °C for 16–18 h",
         "Yes",
         "selective, LB + CTX, LB + MEM",
         "37 ± 2 °C for 16–18 h under microaerophilic conditions",
         "MacConkey Agar supplemented with 0.125 mg/L meropenem, 1 mg/L cefotaxime",
         "real time PCR",
         "post-isolation"
        ],
        [
         "Roschanski, VIM-1 Stall N2, 2019.pdf",
         "Roschanski, VIM-1 Stall N2, 2019.pdf",
         "accepted",
         "Germany",
         "sample investigation",
         "livestock, livestock environment",
         "Breeding pig farm, fecal and environmental samples",
         "Enterobacter cloacae, Salmonella enterica Serovars Infantis, Salmonella enterica Serovars Goldcoast",
         "blaVIM-1",
         "DIN EN ISO 6579",
         "culture-based",
         null,
         "WGS, PFGE",
         "ST32, ST358",
         "plasmid",
         "IncHI2 (ST1)",
         "290-300 kb",
         "Yes",
         null,
         "PCR",
         null,
         null,
         "non selective",
         "37°C overnight",
         "No",
         null,
         null,
         "ChromID Carba, MacConkey with cefotaxime",
         null,
         null
        ]
       ],
       "shape": {
        "columns": 29,
        "rows": 8
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>filename</th>\n",
       "      <th>acceptance_status</th>\n",
       "      <th>country</th>\n",
       "      <th>type_of_study</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>matrix_description</th>\n",
       "      <th>bacterial_species</th>\n",
       "      <th>genes</th>\n",
       "      <th>protocol_reference</th>\n",
       "      <th>isolation_procedure</th>\n",
       "      <th>...</th>\n",
       "      <th>sample_dilution</th>\n",
       "      <th>sample_weight</th>\n",
       "      <th>first_enrichtment</th>\n",
       "      <th>first_enrichtment_condition</th>\n",
       "      <th>second_enrichment</th>\n",
       "      <th>second_enrichtment_selective</th>\n",
       "      <th>second_enrichtment_condition</th>\n",
       "      <th>selective_plate</th>\n",
       "      <th>ncb_method</th>\n",
       "      <th>which_step</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>antibiotics-08-00023.pdf</th>\n",
       "      <td>antibiotics-08-00023.pdf</td>\n",
       "      <td>rejected</td>\n",
       "      <td>Portugal</td>\n",
       "      <td>sample investigation</td>\n",
       "      <td>livestock</td>\n",
       "      <td>livestock manure from poultry, pig, dairy farm...</td>\n",
       "      <td>Enterobacteriaceae, E. coli, Citrobacter freun...</td>\n",
       "      <td>catI, catII, qnrS, qnrB, oqxB, tet(A), tet(M),...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>IMPART, plate validation, 2022.pdf</th>\n",
       "      <td>IMPART, plate validation, 2022.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Europe</td>\n",
       "      <td>method comparison</td>\n",
       "      <td>livestock, food</td>\n",
       "      <td>spiked pig caecal and turkey meat samples</td>\n",
       "      <td>Enterobacteriaceae, Salmonella, Escherichia co...</td>\n",
       "      <td>blaIMP, blaOXA-48, blaVIM-1, blaKPC-2, blaNDM-1</td>\n",
       "      <td>EURL-AR</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>1:10</td>\n",
       "      <td>None</td>\n",
       "      <td>non selective</td>\n",
       "      <td>37°C overnight</td>\n",
       "      <td>No</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Brilliance™ CRE Agar, CHROMID® CARBA, CHROMaga...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Irrgang, GES-5, 2020.pdf</th>\n",
       "      <td>Irrgang, GES-5, 2020.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>sample investigation</td>\n",
       "      <td>livestock</td>\n",
       "      <td>fecal sample of a fattening pig</td>\n",
       "      <td>Escherichia coli</td>\n",
       "      <td>blaGES-1, blaGES-5, blaGES-5B</td>\n",
       "      <td>EURL-AR</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>selective</td>\n",
       "      <td>None</td>\n",
       "      <td>no</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Irrgang, VIM_2019.pdf</th>\n",
       "      <td>Irrgang, VIM_2019.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>sample investigation, method comparison</td>\n",
       "      <td>livestock, livestock environment</td>\n",
       "      <td>fecal samples from pigs, barn surrounding envi...</td>\n",
       "      <td>Escherichia coli</td>\n",
       "      <td>blaVIM−1</td>\n",
       "      <td>EURL-AR</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>1:10</td>\n",
       "      <td>10 g</td>\n",
       "      <td>non selective</td>\n",
       "      <td>16–20 h at 37°C</td>\n",
       "      <td>Yes</td>\n",
       "      <td>selective with 1 mg/L cefotaxime</td>\n",
       "      <td>37°C, 44°C</td>\n",
       "      <td>chromID Carba, MacConkey agar with 1 mg/L CTX</td>\n",
       "      <td>real-time PCR</td>\n",
       "      <td>post-isolation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Pauly, 19-AB01133.pdf</th>\n",
       "      <td>Pauly, 19-AB01133.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>sample investigation</td>\n",
       "      <td>food, livestock</td>\n",
       "      <td>pork shoulder from a pig raised in Germany</td>\n",
       "      <td>Escherichia coli</td>\n",
       "      <td>blaVIM-1, blaSHV-5, blaCMY-13</td>\n",
       "      <td>EURL-AR</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>selective</td>\n",
       "      <td>37 ±2 °C for 16–18 h</td>\n",
       "      <td>Yes</td>\n",
       "      <td>selective</td>\n",
       "      <td>None</td>\n",
       "      <td>McConkey agar supplemented with 0.125 mg/L MEM...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Pauly, Carba plate validation, 2020.pdf</th>\n",
       "      <td>Pauly, Carba plate validation, 2020.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>method validation</td>\n",
       "      <td>livestock, food</td>\n",
       "      <td>cecum content, meat</td>\n",
       "      <td>Enterobacteriaceae, Escherichia coli, Salmonel...</td>\n",
       "      <td>blaKPC, blaVIM, blaNDM, blaIMP</td>\n",
       "      <td>EURL-AR</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>No</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>ChromID CARBA, MacConkey agar with cefotaxime ...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Pauly,method comparison, 2021.pdf</th>\n",
       "      <td>Pauly,method comparison, 2021.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>method comparison, method validation</td>\n",
       "      <td>livestock</td>\n",
       "      <td>pig caecum</td>\n",
       "      <td>E. coli, Salmonella enterica subsp. enterica</td>\n",
       "      <td>blaVIM-1, blaGES-5, blaKPC-2, blaNDM-1, blaOXA-48</td>\n",
       "      <td>EURL-AR</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>1:10</td>\n",
       "      <td>9 g</td>\n",
       "      <td>non selective</td>\n",
       "      <td>37 ± 2 °C for 16–18 h</td>\n",
       "      <td>Yes</td>\n",
       "      <td>selective, LB + CTX, LB + MEM</td>\n",
       "      <td>37 ± 2 °C for 16–18 h under microaerophilic co...</td>\n",
       "      <td>MacConkey Agar supplemented with 0.125 mg/L me...</td>\n",
       "      <td>real time PCR</td>\n",
       "      <td>post-isolation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Roschanski, VIM-1 Stall N2, 2019.pdf</th>\n",
       "      <td>Roschanski, VIM-1 Stall N2, 2019.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>sample investigation</td>\n",
       "      <td>livestock, livestock environment</td>\n",
       "      <td>Breeding pig farm, fecal and environmental sam...</td>\n",
       "      <td>Enterobacter cloacae, Salmonella enterica Sero...</td>\n",
       "      <td>blaVIM-1</td>\n",
       "      <td>DIN EN ISO 6579</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>non selective</td>\n",
       "      <td>37°C overnight</td>\n",
       "      <td>No</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>ChromID Carba, MacConkey with cefotaxime</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>8 rows × 29 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                        filename  \\\n",
       "antibiotics-08-00023.pdf                                antibiotics-08-00023.pdf   \n",
       "IMPART, plate validation, 2022.pdf            IMPART, plate validation, 2022.pdf   \n",
       "Irrgang, GES-5, 2020.pdf                                Irrgang, GES-5, 2020.pdf   \n",
       "Irrgang, VIM_2019.pdf                                      Irrgang, VIM_2019.pdf   \n",
       "Pauly, 19-AB01133.pdf                                      Pauly, 19-AB01133.pdf   \n",
       "Pauly, Carba plate validation, 2020.pdf  Pauly, Carba plate validation, 2020.pdf   \n",
       "Pauly,method comparison, 2021.pdf              Pauly,method comparison, 2021.pdf   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf        Roschanski, VIM-1 Stall N2, 2019.pdf   \n",
       "\n",
       "                                        acceptance_status   country  \\\n",
       "antibiotics-08-00023.pdf                         rejected  Portugal   \n",
       "IMPART, plate validation, 2022.pdf               accepted    Europe   \n",
       "Irrgang, GES-5, 2020.pdf                         accepted   Germany   \n",
       "Irrgang, VIM_2019.pdf                            accepted   Germany   \n",
       "Pauly, 19-AB01133.pdf                            accepted   Germany   \n",
       "Pauly, Carba plate validation, 2020.pdf          accepted   Germany   \n",
       "Pauly,method comparison, 2021.pdf                accepted   Germany   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf             accepted   Germany   \n",
       "\n",
       "                                                                   type_of_study  \\\n",
       "antibiotics-08-00023.pdf                                    sample investigation   \n",
       "IMPART, plate validation, 2022.pdf                             method comparison   \n",
       "Irrgang, GES-5, 2020.pdf                                    sample investigation   \n",
       "Irrgang, VIM_2019.pdf                    sample investigation, method comparison   \n",
       "Pauly, 19-AB01133.pdf                                       sample investigation   \n",
       "Pauly, Carba plate validation, 2020.pdf                        method validation   \n",
       "Pauly,method comparison, 2021.pdf           method comparison, method validation   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                        sample investigation   \n",
       "\n",
       "                                                              sample_type  \\\n",
       "antibiotics-08-00023.pdf                                        livestock   \n",
       "IMPART, plate validation, 2022.pdf                        livestock, food   \n",
       "Irrgang, GES-5, 2020.pdf                                        livestock   \n",
       "Irrgang, VIM_2019.pdf                    livestock, livestock environment   \n",
       "Pauly, 19-AB01133.pdf                                     food, livestock   \n",
       "Pauly, Carba plate validation, 2020.pdf                   livestock, food   \n",
       "Pauly,method comparison, 2021.pdf                               livestock   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     livestock, livestock environment   \n",
       "\n",
       "                                                                        matrix_description  \\\n",
       "antibiotics-08-00023.pdf                 livestock manure from poultry, pig, dairy farm...   \n",
       "IMPART, plate validation, 2022.pdf               spiked pig caecal and turkey meat samples   \n",
       "Irrgang, GES-5, 2020.pdf                                   fecal sample of a fattening pig   \n",
       "Irrgang, VIM_2019.pdf                    fecal samples from pigs, barn surrounding envi...   \n",
       "Pauly, 19-AB01133.pdf                           pork shoulder from a pig raised in Germany   \n",
       "Pauly, Carba plate validation, 2020.pdf                                cecum content, meat   \n",
       "Pauly,method comparison, 2021.pdf                                               pig caecum   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     Breeding pig farm, fecal and environmental sam...   \n",
       "\n",
       "                                                                         bacterial_species  \\\n",
       "antibiotics-08-00023.pdf                 Enterobacteriaceae, E. coli, Citrobacter freun...   \n",
       "IMPART, plate validation, 2022.pdf       Enterobacteriaceae, Salmonella, Escherichia co...   \n",
       "Irrgang, GES-5, 2020.pdf                                                  Escherichia coli   \n",
       "Irrgang, VIM_2019.pdf                                                     Escherichia coli   \n",
       "Pauly, 19-AB01133.pdf                                                     Escherichia coli   \n",
       "Pauly, Carba plate validation, 2020.pdf  Enterobacteriaceae, Escherichia coli, Salmonel...   \n",
       "Pauly,method comparison, 2021.pdf             E. coli, Salmonella enterica subsp. enterica   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     Enterobacter cloacae, Salmonella enterica Sero...   \n",
       "\n",
       "                                                                                     genes  \\\n",
       "antibiotics-08-00023.pdf                 catI, catII, qnrS, qnrB, oqxB, tet(A), tet(M),...   \n",
       "IMPART, plate validation, 2022.pdf         blaIMP, blaOXA-48, blaVIM-1, blaKPC-2, blaNDM-1   \n",
       "Irrgang, GES-5, 2020.pdf                                     blaGES-1, blaGES-5, blaGES-5B   \n",
       "Irrgang, VIM_2019.pdf                                                             blaVIM−1   \n",
       "Pauly, 19-AB01133.pdf                                        blaVIM-1, blaSHV-5, blaCMY-13   \n",
       "Pauly, Carba plate validation, 2020.pdf                     blaKPC, blaVIM, blaNDM, blaIMP   \n",
       "Pauly,method comparison, 2021.pdf        blaVIM-1, blaGES-5, blaKPC-2, blaNDM-1, blaOXA-48   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                              blaVIM-1   \n",
       "\n",
       "                                        protocol_reference  \\\n",
       "antibiotics-08-00023.pdf                              None   \n",
       "IMPART, plate validation, 2022.pdf                 EURL-AR   \n",
       "Irrgang, GES-5, 2020.pdf                           EURL-AR   \n",
       "Irrgang, VIM_2019.pdf                              EURL-AR   \n",
       "Pauly, 19-AB01133.pdf                              EURL-AR   \n",
       "Pauly, Carba plate validation, 2020.pdf            EURL-AR   \n",
       "Pauly,method comparison, 2021.pdf                  EURL-AR   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf       DIN EN ISO 6579   \n",
       "\n",
       "                                        isolation_procedure  ...  \\\n",
       "antibiotics-08-00023.pdf                               None  ...   \n",
       "IMPART, plate validation, 2022.pdf            culture-based  ...   \n",
       "Irrgang, GES-5, 2020.pdf                      culture-based  ...   \n",
       "Irrgang, VIM_2019.pdf                         culture-based  ...   \n",
       "Pauly, 19-AB01133.pdf                         culture-based  ...   \n",
       "Pauly, Carba plate validation, 2020.pdf       culture-based  ...   \n",
       "Pauly,method comparison, 2021.pdf             culture-based  ...   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf          culture-based  ...   \n",
       "\n",
       "                                        sample_dilution sample_weight  \\\n",
       "antibiotics-08-00023.pdf                           None          None   \n",
       "IMPART, plate validation, 2022.pdf                 1:10          None   \n",
       "Irrgang, GES-5, 2020.pdf                           None          None   \n",
       "Irrgang, VIM_2019.pdf                              1:10          10 g   \n",
       "Pauly, 19-AB01133.pdf                              None          None   \n",
       "Pauly, Carba plate validation, 2020.pdf            None          None   \n",
       "Pauly,method comparison, 2021.pdf                  1:10           9 g   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf               None          None   \n",
       "\n",
       "                                        first_enrichtment  \\\n",
       "antibiotics-08-00023.pdf                             None   \n",
       "IMPART, plate validation, 2022.pdf          non selective   \n",
       "Irrgang, GES-5, 2020.pdf                        selective   \n",
       "Irrgang, VIM_2019.pdf                       non selective   \n",
       "Pauly, 19-AB01133.pdf                           selective   \n",
       "Pauly, Carba plate validation, 2020.pdf              None   \n",
       "Pauly,method comparison, 2021.pdf           non selective   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf        non selective   \n",
       "\n",
       "                                        first_enrichtment_condition  \\\n",
       "antibiotics-08-00023.pdf                                       None   \n",
       "IMPART, plate validation, 2022.pdf                   37°C overnight   \n",
       "Irrgang, GES-5, 2020.pdf                                       None   \n",
       "Irrgang, VIM_2019.pdf                               16–20 h at 37°C   \n",
       "Pauly, 19-AB01133.pdf                          37 ±2 °C for 16–18 h   \n",
       "Pauly, Carba plate validation, 2020.pdf                        None   \n",
       "Pauly,method comparison, 2021.pdf             37 ± 2 °C for 16–18 h   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                 37°C overnight   \n",
       "\n",
       "                                        second_enrichment  \\\n",
       "antibiotics-08-00023.pdf                             None   \n",
       "IMPART, plate validation, 2022.pdf                     No   \n",
       "Irrgang, GES-5, 2020.pdf                               no   \n",
       "Irrgang, VIM_2019.pdf                                 Yes   \n",
       "Pauly, 19-AB01133.pdf                                 Yes   \n",
       "Pauly, Carba plate validation, 2020.pdf                No   \n",
       "Pauly,method comparison, 2021.pdf                     Yes   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                   No   \n",
       "\n",
       "                                             second_enrichtment_selective  \\\n",
       "antibiotics-08-00023.pdf                                             None   \n",
       "IMPART, plate validation, 2022.pdf                                   None   \n",
       "Irrgang, GES-5, 2020.pdf                                             None   \n",
       "Irrgang, VIM_2019.pdf                    selective with 1 mg/L cefotaxime   \n",
       "Pauly, 19-AB01133.pdf                                           selective   \n",
       "Pauly, Carba plate validation, 2020.pdf                              None   \n",
       "Pauly,method comparison, 2021.pdf           selective, LB + CTX, LB + MEM   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                 None   \n",
       "\n",
       "                                                              second_enrichtment_condition  \\\n",
       "antibiotics-08-00023.pdf                                                              None   \n",
       "IMPART, plate validation, 2022.pdf                                                    None   \n",
       "Irrgang, GES-5, 2020.pdf                                                              None   \n",
       "Irrgang, VIM_2019.pdf                                                           37°C, 44°C   \n",
       "Pauly, 19-AB01133.pdf                                                                 None   \n",
       "Pauly, Carba plate validation, 2020.pdf                                               None   \n",
       "Pauly,method comparison, 2021.pdf        37 ± 2 °C for 16–18 h under microaerophilic co...   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                                  None   \n",
       "\n",
       "                                                                           selective_plate  \\\n",
       "antibiotics-08-00023.pdf                                                              None   \n",
       "IMPART, plate validation, 2022.pdf       Brilliance™ CRE Agar, CHROMID® CARBA, CHROMaga...   \n",
       "Irrgang, GES-5, 2020.pdf                                                              None   \n",
       "Irrgang, VIM_2019.pdf                        chromID Carba, MacConkey agar with 1 mg/L CTX   \n",
       "Pauly, 19-AB01133.pdf                    McConkey agar supplemented with 0.125 mg/L MEM...   \n",
       "Pauly, Carba plate validation, 2020.pdf  ChromID CARBA, MacConkey agar with cefotaxime ...   \n",
       "Pauly,method comparison, 2021.pdf        MacConkey Agar supplemented with 0.125 mg/L me...   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf              ChromID Carba, MacConkey with cefotaxime   \n",
       "\n",
       "                                            ncb_method      which_step  \n",
       "antibiotics-08-00023.pdf                          None            None  \n",
       "IMPART, plate validation, 2022.pdf                None            None  \n",
       "Irrgang, GES-5, 2020.pdf                          None            None  \n",
       "Irrgang, VIM_2019.pdf                    real-time PCR  post-isolation  \n",
       "Pauly, 19-AB01133.pdf                             None            None  \n",
       "Pauly, Carba plate validation, 2020.pdf           None            None  \n",
       "Pauly,method comparison, 2021.pdf        real time PCR  post-isolation  \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf              None            None  \n",
       "\n",
       "[8 rows x 29 columns]"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Convert the extraction_cpe_o1 dictionary to a list of dictionaries suitable for DataFrame creation\n",
    "data = [item for item in extraction_cpe.values()]\n",
    "index = list(extraction_cpe.keys())\n",
    "\n",
    "# Create the DataFrame\n",
    "df_extraction = pd.DataFrame(data, index=index)\n",
    "\n",
    "# Add filename and acceptance_status as the first and second columns\n",
    "df_extraction.insert(0, 'filename', df_extraction.index)\n",
    "df_extraction.insert(1, 'acceptance_status', df_extraction['filename'].map(acceptance_status))\n",
    "\n",
    "\n",
    "df_extraction"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
