{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!pip install --upgrade --quiet langchain langchain-openai"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import nest_asyncio\n",
    "import asyncio\n",
    "from langchain_community.document_loaders import Docx2txtLoader, PyPDFLoader\n",
    "\n",
    "# Apply nest_asyncio\n",
    "nest_asyncio.apply()\n",
    "\n",
    "directory = \"texts\"\n",
    "\n",
    "# Function to load PDF files\n",
    "async def load_pdf(file_path):\n",
    "    loader = PyPDFLoader(file_path)\n",
    "    pages = []\n",
    "    async for page in loader.alazy_load():\n",
    "        pages.append(page)\n",
    "    return pages\n",
    "\n",
    "# Load all files\n",
    "texts = {}\n",
    "for filename in os.listdir(directory):\n",
    "    file_path = os.path.join(directory, filename)\n",
    "    if filename.endswith(\".docx\"):\n",
    "        loader = Docx2txtLoader(file_path)\n",
    "        texts[filename] = loader.load()\n",
    "    elif filename.endswith(\".pdf\"):\n",
    "        texts[filename] = asyncio.run(load_pdf(file_path))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "antibiotics-08-00023.pdf\n",
      "IMPART, plate validation, 2022.pdf\n",
      "Irrgang, GES-5, 2020.pdf\n",
      "Irrgang, VIM_2019.pdf\n",
      "Pauly, 19-AB01133.pdf\n",
      "Pauly, Carba plate validation, 2020.pdf\n",
      "Pauly,method comparison, 2021.pdf\n",
      "Roschanski, VIM-1 Stall N2, 2019.pdf\n"
     ]
    }
   ],
   "source": [
    "# Combine page content and clean up the texts dictionary\n",
    "for filename, documents in texts.items():\n",
    "    combined_text = \" \".join([doc.page_content for doc in documents])\n",
    "    texts[filename] = combined_text\n",
    "\n",
    "\n",
    "for filename in texts.keys():\n",
    "    print(filename)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import getpass\n",
    "import os\n",
    "\n",
    "os.environ[\"OPENAI_API_KEY\"] = getpass.getpass(\"Enter your OpenAI API key: \")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_openai import ChatOpenAI\n",
    "llm = ChatOpenAI(model=\"gpt-4o\", temperature=0)\n",
    "llm_o3 = ChatOpenAI(model=\"o3-mini\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total tokens used: 127613\n",
      "Prompt tokens: 127372\n",
      "Completion tokens: 241\n",
      "Total cost (USD): 0.32084\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'antibiotics-08-00023.pdf': {'isolation': 'no',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'},\n",
       " 'IMPART, plate validation, 2022.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'yes'},\n",
       " 'Irrgang, GES-5, 2020.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'},\n",
       " 'Irrgang, VIM_2019.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'},\n",
       " 'Pauly, 19-AB01133.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'},\n",
       " 'Pauly, Carba plate validation, 2020.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'yes'},\n",
       " 'Pauly,method comparison, 2021.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'method comparison'},\n",
       " 'Roschanski, VIM-1 Stall N2, 2019.pdf': {'isolation': 'yes',\n",
       "  'method': 'yes',\n",
       "  'study_type': 'no'}}"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "## Inclusion / Exclusion\n",
    "\n",
    "from langchain_core.output_parsers import JsonOutputParser\n",
    "from langchain_core.prompts import PromptTemplate\n",
    "from pydantic import BaseModel, Field\n",
    "\n",
    "# Define your desired data structure.\n",
    "class selection(BaseModel):\n",
    "    isolation: str = Field(description=\"Answer the first question with either 'yes' or 'no'\")\n",
    "    method: str = Field(description=\"Answer the second question with either 'yes' or 'no'\")\n",
    "    study_type: str = Field(description=\"Answer the third question with either 'yes' or 'no'\")\n",
    "\n",
    "# Set up the parser\n",
    "parser = JsonOutputParser(pydantic_object=selection)\n",
    "\n",
    "# Create the prompt template\n",
    "prompt = PromptTemplate(\n",
    "    template=\"\"\"\n",
    "    Read the following study and answer these questions in JSON format:\n",
    "    1. Does this study explicitly isolate strains of carbapenemase-producing bacteria?\n",
    "    2. Does this study state the bacteriological method used for the isolation of the carbapenemase-producing bacteria?\n",
    "    3. Does this study fall in one or multiple of the following categories: protocol validation, protocol development, method validation, method comparison, plate validation?\n",
    "    {format_instructions}\n",
    "    Text:{text}\\n\"\"\",\n",
    "    input_variables=[\"text\"],\n",
    "    partial_variables={\"format_instructions\": parser.get_format_instructions()},\n",
    ")\n",
    "\n",
    "# Combine the prompt with the model and parser\n",
    "chain = prompt | llm | parser\n",
    "\n",
    "from langchain_community.callbacks import get_openai_callback\n",
    "\n",
    "def process_texts(texts):\n",
    "    results = {}\n",
    "    with get_openai_callback() as cb:\n",
    "        for filename, text in texts.items():\n",
    "            result = chain.invoke({\"text\": text})\n",
    "            results[filename] = result\n",
    "        print(f\"Total tokens used: {cb.total_tokens}\")\n",
    "        print(f\"Prompt tokens: {cb.prompt_tokens}\")\n",
    "        print(f\"Completion tokens: {cb.completion_tokens}\")\n",
    "        print(f\"Total cost (USD): {cb.total_cost}\")\n",
    "    return results\n",
    "\n",
    "# Run the function on the texts dictionary\n",
    "extraction_selection = process_texts(texts)\n",
    "extraction_selection"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Accepted files: ['IMPART, plate validation, 2022.pdf', 'Irrgang, GES-5, 2020.pdf', 'Irrgang, VIM_2019.pdf', 'Pauly, 19-AB01133.pdf', 'Pauly, Carba plate validation, 2020.pdf', 'Pauly,method comparison, 2021.pdf', 'Roschanski, VIM-1 Stall N2, 2019.pdf']\n",
      "Rejected files: ['antibiotics-08-00023.pdf']\n"
     ]
    }
   ],
   "source": [
    "def categorize_files(extraction_selection):\n",
    "    accepted = []\n",
    "    rejected = []\n",
    "    for filename, response in extraction_selection.items():\n",
    "        if (response['isolation'] == 'yes' and response['method'] == 'yes') or response['study_type'] == 'yes':\n",
    "            accepted.append(filename)\n",
    "        else:\n",
    "            rejected.append(filename)\n",
    "    return accepted, rejected\n",
    "\n",
    "accepted, rejected = categorize_files(extraction_selection)\n",
    "\n",
    "print(\"Accepted files:\", accepted)\n",
    "print(\"Rejected files:\", rejected)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'IMPART, plate validation, 2022.pdf': 'accepted', 'Irrgang, GES-5, 2020.pdf': 'accepted', 'Irrgang, VIM_2019.pdf': 'accepted', 'Pauly, 19-AB01133.pdf': 'accepted', 'Pauly, Carba plate validation, 2020.pdf': 'accepted', 'Pauly,method comparison, 2021.pdf': 'accepted', 'Roschanski, VIM-1 Stall N2, 2019.pdf': 'accepted', 'antibiotics-08-00023.pdf': 'rejected'}\n"
     ]
    }
   ],
   "source": [
    "# Create a dictionary with filename and acceptance status\n",
    "acceptance_status = {filename: 'accepted' for filename in accepted}\n",
    "acceptance_status.update({filename: 'rejected' for filename in rejected})\n",
    "\n",
    "# Print the dictionary\n",
    "print(acceptance_status)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Matches: {'antibiotics-08-00023.pdf'}\n",
      "Number of matches: 1 of 1\n"
     ]
    }
   ],
   "source": [
    "# Compare ground_truth_rejected with rejected and count matches\n",
    "\n",
    "ground_truth_rejected = ['antibiotics-08-00023.pdf']\n",
    "\n",
    "matches = set(ground_truth_rejected) & set(rejected)\n",
    "match_count = len(matches)\n",
    "total_count = len(ground_truth_rejected)\n",
    "\n",
    "print(f\"Matches: {matches}\")\n",
    "print(f\"Number of matches: {match_count} of {total_count}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from typing import List, Optional\n",
    "from pydantic import BaseModel, Field\n",
    "\n",
    "class CPE(BaseModel):\n",
    "    \"\"\"Information about studies on carbapenemase-producing bacteria\"\"\"\n",
    "    country: Optional[str] = Field(default=None, description=\"Origin of samples.\")\n",
    "    type_of_study: Optional[str] = Field(default=None, description=\"Classify the study into: sample investigation, protocol development, method comparison, method validation. Multiple labels possible separated by comma.\")\n",
    "    sample_type: Optional[str] = Field(default=None, description=\"Which sample types were used in the study? Pick from 'livestock' (eg. fecal samples, feces, caecum, manure, cattle, pigs, poultry, etc.), 'livestock environment' (eg. farm or barn environment, etc.), 'companion animal' (eg. dog, cat, etc.), 'food' (eg. meat), 'environment' (eg. water, soil, etc.) or 'aquaculture'. Multiple answers possible separated by comma.\")\n",
    "    matrix_description: Optional[str] = Field(default=None, description=\"Matrix description of study. Which animal and sample material was used?\")\n",
    "    bacterial_species: Optional[str] = Field(default=None, description=\"Species of bacteria. Multiple answers possible separated by comma.\")\n",
    "    genes: Optional[str] = Field(default=None, description=\"Genes or carbapenemase detected in this study.\")\n",
    "    protocol_reference: Optional[str] = Field(default=None, description=\"Is there a reference for the protocol used? Eg. EURL-AR, DIN EN ISO, etc. Yes or No. Provide reference, if possible.\")\n",
    "    isolation_procedure: Optional[str] = Field(default=None, description=\"Is the method for the isolation of carbapenemase-producing bacteria culture-based, non-culture-based or both?\")\n",
    "    method_validation: Optional[str] = Field(default=None, description=\"Was the method for the isolation of carbapenemase-producing bacteria validated? Yes or no. Provide values for sensitivity and specificity.\")\n",
    "    isolate_characterization: Optional[str] = Field(default=None, description=\"Which methods (eg. PFGE, NGS, etc.) were used for the characterization of carbapenemase-producing bacteria? Do not include information on species identification (MALDI-ToF). Multiple answers possible separated by comma.\")\n",
    "    MLST_isolates: Optional[str] = Field(default=None, description=\"What was the MLST (Multi Locus Sequence Typing) of the isolates?\")\n",
    "    gene_localization: Optional[str] = Field(default=None, description=\"Was the characterization of isolates for carbapenemase-producing bacteria based on a gene located on a plasmid and / or on the chromosome? Multiple answers possible separated by comma.\")\n",
    "    plasmid_type: Optional[str] = Field(default=None, description=\"If plasmid was detected, on which type of plasmid (Inc group) was the gene located?\")   \n",
    "    plasmid_size: Optional[str] = Field(default=None, description=\"If plasmid was detected, what was the size (bp or kbp) of the plasmid?\")\n",
    "    plasmid_transferable: Optional[str] = Field(default=None, description=\"If plasmid was detected, was the CPE-gene harbouring plasmid transferable by conjugation and / or transformation? Multiple answers possible separated by comma.\")\n",
    "    LOD_culture_based: Optional[str] = Field(default=None, description=\"Was there a LOD (limit of detection) for the isolation method provided?\")\n",
    "    confirmation_carbapenemase: Optional[str] = Field(default=None, description=\"How was carbapenemase-production confirmed (eg. PCR, real time PCR, MIC, etc.)? Do not include species confirmation (MALDI-ToF). Do not include characterization of the isolates.\")  \n",
    "    sample_dilution: Optional[str] = Field(default=None, description=\"Dilution of the sample for the isolation method (eg. 1:10). Do not include dilution of the enrichment.\")\n",
    "    sample_weight: Optional[str] = Field(default=None, description=\"Weight of samples. If multiple samples exist, separate weights by comma.\")\n",
    "    first_enrichtment: Optional[str] = Field(default=None, description=\"Was the first enrichment or preenrichment selective or non selective? If selective, which antimicrobial was used with which concentration?\")\n",
    "    first_enrichtment_condition: Optional[str] = Field(default=None, description=\"Which condition of temperature and time for first enrichment or preenrichment?\")\n",
    "    second_enrichment: Optional[str] = Field(default=None, description=\"Was an other enrichment carried out? Yes or No. If you can not find the information it is 'no'.\")\n",
    "    second_enrichtment_selective: Optional[str] = Field(default=None, description=\"Was the second enrichment selective or non selective? If selective: which antimicrobial was used with which concentration?\")\n",
    "    second_enrichtment_condition: Optional[str] = Field(default=None, description=\"Which condition of temperature and time for second enrichment?\")\n",
    "    selective_plate: Optional[str] = Field(default=None, description=\"Which selective agars where used? Provide name of antimicrobial used at which concentration for each agar plate. Multiple answers possible separated by comma.\")\n",
    "    ncb_method: Optional[str] = Field(default=None, description=\"Which non-culture-based method (eg. PCR, real time PCR, NGS, WGS etc.) for the isolation of carbapenemase-producing bacteria was used?\") \n",
    "    which_step: Optional[str] = Field(default=None, description=\"At which step was the non-culture-based method applied? After first enrichment, after second enrichment, post-isolation.\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total tokens used: 166028\n",
      "Prompt tokens: 140230\n",
      "Completion tokens: 25798\n",
      "Total cost (USD): 0.0\n"
     ]
    }
   ],
   "source": [
    "from typing import List, Optional\n",
    "from pydantic import BaseModel, Field\n",
    "from langchain_core.output_parsers import JsonOutputParser\n",
    "from langchain_core.prompts import PromptTemplate\n",
    "\n",
    "# Set up the parser\n",
    "parser = JsonOutputParser(pydantic_object=CPE)\n",
    "\n",
    "prompt = PromptTemplate(\n",
    "    template=(\n",
    "        \"You are an expert extraction algorithm. \"\n",
    "        \"Only extract relevant information from the text. \"\n",
    "        \"If you do not know the value of an attribute asked to extract, \"\n",
    "        \"return null for the attribute's value.\"\n",
    "        \"{format_instructions}\\n\\n\"\n",
    "        \"Text: {text}\\n\"\n",
    "    ),\n",
    "    input_variables=[\"text\"],\n",
    "    partial_variables={\n",
    "        \"format_instructions\": parser.get_format_instructions()\n",
    "    },\n",
    ")\n",
    "\n",
    "# Combine the prompt with the model and parser\n",
    "chain = prompt | llm_o3 | parser\n",
    "\n",
    "from langchain_community.callbacks import get_openai_callback\n",
    "\n",
    "def process_texts(texts):\n",
    "    results = {}\n",
    "    with get_openai_callback() as cb:\n",
    "        for filename, text in texts.items():\n",
    "            result = chain.invoke({\"text\": text})\n",
    "            results[filename] = result\n",
    "        print(f\"Total tokens used: {cb.total_tokens}\")\n",
    "        print(f\"Prompt tokens: {cb.prompt_tokens}\")\n",
    "        print(f\"Completion tokens: {cb.completion_tokens}\")\n",
    "        print(f\"Total cost (USD): {cb.total_cost}\")\n",
    "    return results\n",
    "\n",
    "# Run the function on the texts dictionary\n",
    "extraction_cpe = process_texts(texts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "filename",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "acceptance_status",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "country",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "type_of_study",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "sample_type",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "matrix_description",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "bacterial_species",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "genes",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "protocol_reference",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "isolation_procedure",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "method_validation",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "isolate_characterization",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "MLST_isolates",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "gene_localization",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "plasmid_type",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "plasmid_size",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "plasmid_transferable",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "LOD_culture_based",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "confirmation_carbapenemase",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "sample_dilution",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "sample_weight",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "first_enrichtment",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "first_enrichtment_condition",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "second_enrichment",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "second_enrichtment_selective",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "second_enrichtment_condition",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "selective_plate",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "ncb_method",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "which_step",
         "rawType": "object",
         "type": "unknown"
        }
       ],
       "conversionMethod": "pd.DataFrame",
       "ref": "6ea0a431-1dab-4811-981a-0449433fa490",
       "rows": [
        [
         "antibiotics-08-00023.pdf",
         "antibiotics-08-00023.pdf",
         "rejected",
         "Portugal",
         "Prevalence study of multidrug‐resistant Enterobacteriaceae in livestock manure and related environments",
         "Livestock manure, slurry, and slaughterhouse wastewater",
         "Samples collected from dairy cattle, pig, and poultry farms as well as slaughterhouses; manure or slurry from open tanks or waste pools and wastewater from animal slaughter processes",
         "Enterobacteriaceae including Escherichia coli, Citrobacter freundii, Raoultella ornithinolytica, Salmonella enterica, Enterobacter cloacae, Morganella morganii, among others",
         "Multiple resistance genes were screened including chloramphenicol resistance genes (catI, catII), tetracycline resistance genes (tet(A), tet(M), etc.), sulphonamide resistance genes (sul1, sul2, sul3), trimethoprim resistance genes (dfr groups), and quinolone resistance genes (qnrB, qnrS, oqxB) as well as integron markers (intI1, intI2)",
         "Protocols referenced include those described in previous studies and CLSI guidelines; species identification was performed using API 20E galleries",
         "Decimal dilutions prepared in sterile 0.9% NaCl; filtration of 100 mL through 0.45 µm cellulose membranes followed by plating on selective VRBG agar and incubation at 37°C",
         "Kirby–Bauer disk diffusion method as defined by CLSI guidelines with quality control strains",
         "Colonies re‐streaked for purity and identified using standard biochemical tests (API 20E)",
         null,
         null,
         null,
         null,
         null,
         null,
         "No isolates resistant to imipenem and meropenem were detected",
         "Decimal dilutions in sterile 0.9% NaCl",
         null,
         null,
         null,
         "no",
         null,
         null,
         "VRBG (Violet Red Bile Glucose) selective agar",
         "Multiplex PCR assays for screening of resistance genes",
         "Non‐culture based detection applied post-isolation for genetic characterization"
        ],
        [
         "IMPART, plate validation, 2022.pdf",
         "IMPART, plate validation, 2022.pdf",
         "accepted",
         "France, Norway, Sweden, Denmark, Germany, Netherlands, United Kingdom, Poland, Italy",
         "multicenter evaluation study, method comparison",
         "livestock (pig caecal content) and food (SPF turkey meat)",
         "Pig caecal samples from pigs and meat samples from specific pathogen free (SPF) turkeys",
         "Escherichia coli, Klebsiella pneumoniae, Salmonella Kentucky, Enterobacter spp.",
         "blaIMP, blaOXA-48, blaVIM-1, blaNDM-1, blaKPC-2, blaGES-5, blaKPC-3",
         "EURL-AR protocol (EU Decision 2013/652/EU)",
         "culture-based isolation after non‐selective pre‐enrichment in buffered peptone water",
         "Validated using control strains (e.g. E. coli ATCC 25922, E. coli 16874 blaOXA-48) with sensitivity and specificity evaluations",
         "MALDI-TOF MS, antimicrobial susceptibility testing (broth microdilution/disk diffusion) and genotypic identification by PCR and real-time PCR",
         null,
         "plasmid",
         null,
         null,
         "yes",
         null,
         "Confirmed by PCR and real-time PCR applied on sub-cultured colonies",
         "1:10 dilution in buffered peptone water",
         null,
         "non-selective pre-enrichment in buffered peptone water",
         "overnight incubation at 37°C",
         "no",
         null,
         null,
         "Brilliance™ CRE Agar, CHROMID® CARBA Agar, CHROMID® OXA-48 Agar, Chromatic™ CRE, Chromatic™ OXA-48, CHROMagar™ mSuperCARBA™",
         "PCR and real-time PCR",
         "post-isolation"
        ],
        [
         "Irrgang, GES-5, 2020.pdf",
         "Irrgang, GES-5, 2020.pdf",
         "accepted",
         "Germany",
         "sample investigation",
         "livestock",
         "fecal sample from a fattening pig",
         "Escherichia coli",
         "blaGES-1, blaGES-5, blaGES-5B",
         "EURL-AR",
         "culture-based",
         null,
         "real-time PCR, diagnostic PCR, Sanger sequencing, Nanopore sequencing, Illumina sequencing, hybrid assembly, MLST",
         "ST1084",
         "plasmid",
         null,
         "12 kb",
         null,
         null,
         "PCR (real-time and diagnostic) and MIC determination",
         null,
         null,
         null,
         null,
         "no",
         null,
         null,
         null,
         null,
         null
        ],
        [
         "Irrgang, VIM_2019.pdf",
         "Irrgang, VIM_2019.pdf",
         "accepted",
         "Germany",
         "sample investigation, protocol development",
         "livestock, livestock environment",
         "Pig fecal samples (composite fecal material), boot swabs and environmental wipes from a pig fattening farm",
         "Escherichia coli",
         "blaVIM-1",
         "EURL-AR",
         "Modified culture-dependent method combining unselective pre-enrichment in buffered peptone water (diluted 1:10) with a selective enrichment step in LB supplemented with cefotaxime (CTX) (at 37°C or 44°C) and plating on chromID R-Carba and MacConkey agar",
         null,
         "PFGE, MLST, whole-genome sequencing and real-time PCR confirmation",
         "ST-88",
         "plasmid, chromosome",
         "IncHI2",
         "207-375 kb",
         "non-self-transmissible",
         null,
         "real-time PCR",
         "1:10",
         "10 g",
         "non-selective pre-enrichment",
         "16-20 h at 37°C",
         "yes",
         "selective",
         "24 h at 44°C",
         "chromID R-Carba, MacConkey agar with CTX",
         "real-time PCR",
         "applied on the enrichment broth"
        ],
        [
         "Pauly, 19-AB01133.pdf",
         "Pauly, 19-AB01133.pdf",
         "accepted",
         "Germany",
         "sample investigation",
         "food",
         "Pork shoulder meat sample from a fattened pig",
         "Escherichia coli",
         "blaVIM-1, blaSHV-5, blaCMY-13",
         "European Union Reference Laboratory for Antimicrobial Resistance (EURL-AR)",
         "culture-based",
         null,
         "MALDI-ToF, PCR, S1-PFGE, Southern blot hybridization, whole genome sequencing, filter-mating and transformation studies",
         "ST5869",
         "plasmid",
         "IncA/C2",
         "190 kb",
         "conjugative",
         null,
         "PCR, Southern blot hybridization and MIC testing",
         null,
         null,
         null,
         null,
         "yes",
         "selective with CTX and MEM",
         "16–18 h at 37±2°C (plate incubation)",
         "McConkey agar with 0.125 mg/L meropenem, McConkey agar with 0.125 mg/L meropenem and 1 mg/L CTX, chromID® CARBA SMART",
         null,
         null
        ],
        [
         "Pauly, Carba plate validation, 2020.pdf",
         "Pauly, Carba plate validation, 2020.pdf",
         "accepted",
         "Germany",
         "method comparison, method validation",
         "livestock, food",
         "cecum content and meat",
         "Escherichia coli, Salmonella, Vibrio, Klebsiella",
         "blaVIM-1, blaNDM-1, blaKPC-2",
         "EURL-AR protocol",
         "culture-based",
         "Method validated with improved sensitivity and specificity using McC+CTX+MEM agar compared to ChromID R⃝CARBA agar",
         "Broth microdilution for MIC determination and PCR for carbapenemase gene detection",
         null,
         null,
         null,
         null,
         null,
         null,
         "PCR and MIC testing",
         "1:10,000",
         null,
         null,
         null,
         null,
         null,
         null,
         "ChromID R⃝CARBA agar, MacConkey agar with 1 mg/L cefotaxime and 0.125 mg/L meropenem",
         null,
         null
        ],
        [
         "Pauly,method comparison, 2021.pdf",
         "Pauly,method comparison, 2021.pdf",
         "accepted",
         "Germany",
         "protocol development, method comparison",
         "livestock",
         "Pig caeca",
         "Escherichia coli",
         "blaVIM-1, blaGES-5, blaKPC-2, blaNDM-1, blaOXA-48",
         "EURL-AR protocol for isolation of ESBL, AmpC and CP E. coli",
         "culture-based",
         "Validated with sensitivity ranging from 66.7% to 100% and specificity of 100%",
         "PCR, MALDI-ToF MS",
         null,
         null,
         null,
         null,
         null,
         null,
         "Real-time PCR",
         "1:10",
         "9 g",
         "non-selective",
         "37°C for 16–18 hours",
         "yes",
         "selective using LB+CTX and LB+MEM",
         "37±2°C for 16–18 hours under microaerophilic conditions",
         "ChromID® CARBA SMART agar, McC + CTX+MEM, McC + MEM",
         "Real-time PCR",
         "After enrichment steps (both first and second enrichment)"
        ],
        [
         "Roschanski, VIM-1 Stall N2, 2019.pdf",
         "Roschanski, VIM-1 Stall N2, 2019.pdf",
         "accepted",
         "Germany",
         "sample investigation",
         "fecal, environmental, boot swabs, manure, dust",
         "Pig farm; samples from feces, boot swabs, manure, and environmental dust",
         "Enterobacter cloacae, Salmonella enterica serovar Infantis, Salmonella enterica serovar Goldcoast",
         "blaVIM-1",
         "Modified DIN EN ISO 6579 protocol; EURL-AR guidelines",
         "Culture-based isolation using selective preenrichment followed by plating on ChromID Carba and MacConkey agar with cefotaxime",
         null,
         "Whole-genome sequencing (WGS), pulsed-field gel electrophoresis (PFGE) and S1 PFGE hybridization",
         "ST32 (Salmonella Infantis), ST358 (Salmonella Goldcoast)",
         "plasmid",
         "IncHI2 (ST1)",
         "290-300 kb",
         "transferable in vivo",
         null,
         "Real-time PCR",
         null,
         "20 g (feces), 5 g (manure)",
         "non selective preenrichment",
         "overnight incubation at 37°C",
         "no",
         null,
         null,
         "ChromID Carba, MacConkey agar with cefotaxime",
         "Real-time PCR",
         "Applied to preenrichment cultures"
        ]
       ],
       "shape": {
        "columns": 29,
        "rows": 8
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>filename</th>\n",
       "      <th>acceptance_status</th>\n",
       "      <th>country</th>\n",
       "      <th>type_of_study</th>\n",
       "      <th>sample_type</th>\n",
       "      <th>matrix_description</th>\n",
       "      <th>bacterial_species</th>\n",
       "      <th>genes</th>\n",
       "      <th>protocol_reference</th>\n",
       "      <th>isolation_procedure</th>\n",
       "      <th>...</th>\n",
       "      <th>sample_dilution</th>\n",
       "      <th>sample_weight</th>\n",
       "      <th>first_enrichtment</th>\n",
       "      <th>first_enrichtment_condition</th>\n",
       "      <th>second_enrichment</th>\n",
       "      <th>second_enrichtment_selective</th>\n",
       "      <th>second_enrichtment_condition</th>\n",
       "      <th>selective_plate</th>\n",
       "      <th>ncb_method</th>\n",
       "      <th>which_step</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>antibiotics-08-00023.pdf</th>\n",
       "      <td>antibiotics-08-00023.pdf</td>\n",
       "      <td>rejected</td>\n",
       "      <td>Portugal</td>\n",
       "      <td>Prevalence study of multidrug‐resistant Entero...</td>\n",
       "      <td>Livestock manure, slurry, and slaughterhouse w...</td>\n",
       "      <td>Samples collected from dairy cattle, pig, and ...</td>\n",
       "      <td>Enterobacteriaceae including Escherichia coli,...</td>\n",
       "      <td>Multiple resistance genes were screened includ...</td>\n",
       "      <td>Protocols referenced include those described i...</td>\n",
       "      <td>Decimal dilutions prepared in sterile 0.9% NaC...</td>\n",
       "      <td>...</td>\n",
       "      <td>Decimal dilutions in sterile 0.9% NaCl</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>no</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>VRBG (Violet Red Bile Glucose) selective agar</td>\n",
       "      <td>Multiplex PCR assays for screening of resistan...</td>\n",
       "      <td>Non‐culture based detection applied post-isola...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>IMPART, plate validation, 2022.pdf</th>\n",
       "      <td>IMPART, plate validation, 2022.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>France, Norway, Sweden, Denmark, Germany, Neth...</td>\n",
       "      <td>multicenter evaluation study, method comparison</td>\n",
       "      <td>livestock (pig caecal content) and food (SPF t...</td>\n",
       "      <td>Pig caecal samples from pigs and meat samples ...</td>\n",
       "      <td>Escherichia coli, Klebsiella pneumoniae, Salmo...</td>\n",
       "      <td>blaIMP, blaOXA-48, blaVIM-1, blaNDM-1, blaKPC-...</td>\n",
       "      <td>EURL-AR protocol (EU Decision 2013/652/EU)</td>\n",
       "      <td>culture-based isolation after non‐selective pr...</td>\n",
       "      <td>...</td>\n",
       "      <td>1:10 dilution in buffered peptone water</td>\n",
       "      <td>None</td>\n",
       "      <td>non-selective pre-enrichment in buffered pepto...</td>\n",
       "      <td>overnight incubation at 37°C</td>\n",
       "      <td>no</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>Brilliance™ CRE Agar, CHROMID® CARBA Agar, CHR...</td>\n",
       "      <td>PCR and real-time PCR</td>\n",
       "      <td>post-isolation</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Irrgang, GES-5, 2020.pdf</th>\n",
       "      <td>Irrgang, GES-5, 2020.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>sample investigation</td>\n",
       "      <td>livestock</td>\n",
       "      <td>fecal sample from a fattening pig</td>\n",
       "      <td>Escherichia coli</td>\n",
       "      <td>blaGES-1, blaGES-5, blaGES-5B</td>\n",
       "      <td>EURL-AR</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>no</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Irrgang, VIM_2019.pdf</th>\n",
       "      <td>Irrgang, VIM_2019.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>sample investigation, protocol development</td>\n",
       "      <td>livestock, livestock environment</td>\n",
       "      <td>Pig fecal samples (composite fecal material), ...</td>\n",
       "      <td>Escherichia coli</td>\n",
       "      <td>blaVIM-1</td>\n",
       "      <td>EURL-AR</td>\n",
       "      <td>Modified culture-dependent method combining un...</td>\n",
       "      <td>...</td>\n",
       "      <td>1:10</td>\n",
       "      <td>10 g</td>\n",
       "      <td>non-selective pre-enrichment</td>\n",
       "      <td>16-20 h at 37°C</td>\n",
       "      <td>yes</td>\n",
       "      <td>selective</td>\n",
       "      <td>24 h at 44°C</td>\n",
       "      <td>chromID R-Carba, MacConkey agar with CTX</td>\n",
       "      <td>real-time PCR</td>\n",
       "      <td>applied on the enrichment broth</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Pauly, 19-AB01133.pdf</th>\n",
       "      <td>Pauly, 19-AB01133.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>sample investigation</td>\n",
       "      <td>food</td>\n",
       "      <td>Pork shoulder meat sample from a fattened pig</td>\n",
       "      <td>Escherichia coli</td>\n",
       "      <td>blaVIM-1, blaSHV-5, blaCMY-13</td>\n",
       "      <td>European Union Reference Laboratory for Antimi...</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>yes</td>\n",
       "      <td>selective with CTX and MEM</td>\n",
       "      <td>16–18 h at 37±2°C (plate incubation)</td>\n",
       "      <td>McConkey agar with 0.125 mg/L meropenem, McCon...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Pauly, Carba plate validation, 2020.pdf</th>\n",
       "      <td>Pauly, Carba plate validation, 2020.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>method comparison, method validation</td>\n",
       "      <td>livestock, food</td>\n",
       "      <td>cecum content and meat</td>\n",
       "      <td>Escherichia coli, Salmonella, Vibrio, Klebsiella</td>\n",
       "      <td>blaVIM-1, blaNDM-1, blaKPC-2</td>\n",
       "      <td>EURL-AR protocol</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>1:10,000</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>ChromID R⃝CARBA agar, MacConkey agar with 1 mg...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Pauly,method comparison, 2021.pdf</th>\n",
       "      <td>Pauly,method comparison, 2021.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>protocol development, method comparison</td>\n",
       "      <td>livestock</td>\n",
       "      <td>Pig caeca</td>\n",
       "      <td>Escherichia coli</td>\n",
       "      <td>blaVIM-1, blaGES-5, blaKPC-2, blaNDM-1, blaOXA-48</td>\n",
       "      <td>EURL-AR protocol for isolation of ESBL, AmpC a...</td>\n",
       "      <td>culture-based</td>\n",
       "      <td>...</td>\n",
       "      <td>1:10</td>\n",
       "      <td>9 g</td>\n",
       "      <td>non-selective</td>\n",
       "      <td>37°C for 16–18 hours</td>\n",
       "      <td>yes</td>\n",
       "      <td>selective using LB+CTX and LB+MEM</td>\n",
       "      <td>37±2°C for 16–18 hours under microaerophilic c...</td>\n",
       "      <td>ChromID® CARBA SMART agar, McC + CTX+MEM, McC ...</td>\n",
       "      <td>Real-time PCR</td>\n",
       "      <td>After enrichment steps (both first and second ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Roschanski, VIM-1 Stall N2, 2019.pdf</th>\n",
       "      <td>Roschanski, VIM-1 Stall N2, 2019.pdf</td>\n",
       "      <td>accepted</td>\n",
       "      <td>Germany</td>\n",
       "      <td>sample investigation</td>\n",
       "      <td>fecal, environmental, boot swabs, manure, dust</td>\n",
       "      <td>Pig farm; samples from feces, boot swabs, manu...</td>\n",
       "      <td>Enterobacter cloacae, Salmonella enterica sero...</td>\n",
       "      <td>blaVIM-1</td>\n",
       "      <td>Modified DIN EN ISO 6579 protocol; EURL-AR gui...</td>\n",
       "      <td>Culture-based isolation using selective preenr...</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>20 g (feces), 5 g (manure)</td>\n",
       "      <td>non selective preenrichment</td>\n",
       "      <td>overnight incubation at 37°C</td>\n",
       "      <td>no</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>ChromID Carba, MacConkey agar with cefotaxime</td>\n",
       "      <td>Real-time PCR</td>\n",
       "      <td>Applied to preenrichment cultures</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>8 rows × 29 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                                        filename  \\\n",
       "antibiotics-08-00023.pdf                                antibiotics-08-00023.pdf   \n",
       "IMPART, plate validation, 2022.pdf            IMPART, plate validation, 2022.pdf   \n",
       "Irrgang, GES-5, 2020.pdf                                Irrgang, GES-5, 2020.pdf   \n",
       "Irrgang, VIM_2019.pdf                                      Irrgang, VIM_2019.pdf   \n",
       "Pauly, 19-AB01133.pdf                                      Pauly, 19-AB01133.pdf   \n",
       "Pauly, Carba plate validation, 2020.pdf  Pauly, Carba plate validation, 2020.pdf   \n",
       "Pauly,method comparison, 2021.pdf              Pauly,method comparison, 2021.pdf   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf        Roschanski, VIM-1 Stall N2, 2019.pdf   \n",
       "\n",
       "                                        acceptance_status  \\\n",
       "antibiotics-08-00023.pdf                         rejected   \n",
       "IMPART, plate validation, 2022.pdf               accepted   \n",
       "Irrgang, GES-5, 2020.pdf                         accepted   \n",
       "Irrgang, VIM_2019.pdf                            accepted   \n",
       "Pauly, 19-AB01133.pdf                            accepted   \n",
       "Pauly, Carba plate validation, 2020.pdf          accepted   \n",
       "Pauly,method comparison, 2021.pdf                accepted   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf             accepted   \n",
       "\n",
       "                                                                                   country  \\\n",
       "antibiotics-08-00023.pdf                                                          Portugal   \n",
       "IMPART, plate validation, 2022.pdf       France, Norway, Sweden, Denmark, Germany, Neth...   \n",
       "Irrgang, GES-5, 2020.pdf                                                           Germany   \n",
       "Irrgang, VIM_2019.pdf                                                              Germany   \n",
       "Pauly, 19-AB01133.pdf                                                              Germany   \n",
       "Pauly, Carba plate validation, 2020.pdf                                            Germany   \n",
       "Pauly,method comparison, 2021.pdf                                                  Germany   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                               Germany   \n",
       "\n",
       "                                                                             type_of_study  \\\n",
       "antibiotics-08-00023.pdf                 Prevalence study of multidrug‐resistant Entero...   \n",
       "IMPART, plate validation, 2022.pdf         multicenter evaluation study, method comparison   \n",
       "Irrgang, GES-5, 2020.pdf                                              sample investigation   \n",
       "Irrgang, VIM_2019.pdf                           sample investigation, protocol development   \n",
       "Pauly, 19-AB01133.pdf                                                 sample investigation   \n",
       "Pauly, Carba plate validation, 2020.pdf               method comparison, method validation   \n",
       "Pauly,method comparison, 2021.pdf                  protocol development, method comparison   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                  sample investigation   \n",
       "\n",
       "                                                                               sample_type  \\\n",
       "antibiotics-08-00023.pdf                 Livestock manure, slurry, and slaughterhouse w...   \n",
       "IMPART, plate validation, 2022.pdf       livestock (pig caecal content) and food (SPF t...   \n",
       "Irrgang, GES-5, 2020.pdf                                                         livestock   \n",
       "Irrgang, VIM_2019.pdf                                     livestock, livestock environment   \n",
       "Pauly, 19-AB01133.pdf                                                                 food   \n",
       "Pauly, Carba plate validation, 2020.pdf                                    livestock, food   \n",
       "Pauly,method comparison, 2021.pdf                                                livestock   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf        fecal, environmental, boot swabs, manure, dust   \n",
       "\n",
       "                                                                        matrix_description  \\\n",
       "antibiotics-08-00023.pdf                 Samples collected from dairy cattle, pig, and ...   \n",
       "IMPART, plate validation, 2022.pdf       Pig caecal samples from pigs and meat samples ...   \n",
       "Irrgang, GES-5, 2020.pdf                                 fecal sample from a fattening pig   \n",
       "Irrgang, VIM_2019.pdf                    Pig fecal samples (composite fecal material), ...   \n",
       "Pauly, 19-AB01133.pdf                        Pork shoulder meat sample from a fattened pig   \n",
       "Pauly, Carba plate validation, 2020.pdf                             cecum content and meat   \n",
       "Pauly,method comparison, 2021.pdf                                                Pig caeca   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     Pig farm; samples from feces, boot swabs, manu...   \n",
       "\n",
       "                                                                         bacterial_species  \\\n",
       "antibiotics-08-00023.pdf                 Enterobacteriaceae including Escherichia coli,...   \n",
       "IMPART, plate validation, 2022.pdf       Escherichia coli, Klebsiella pneumoniae, Salmo...   \n",
       "Irrgang, GES-5, 2020.pdf                                                  Escherichia coli   \n",
       "Irrgang, VIM_2019.pdf                                                     Escherichia coli   \n",
       "Pauly, 19-AB01133.pdf                                                     Escherichia coli   \n",
       "Pauly, Carba plate validation, 2020.pdf   Escherichia coli, Salmonella, Vibrio, Klebsiella   \n",
       "Pauly,method comparison, 2021.pdf                                         Escherichia coli   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     Enterobacter cloacae, Salmonella enterica sero...   \n",
       "\n",
       "                                                                                     genes  \\\n",
       "antibiotics-08-00023.pdf                 Multiple resistance genes were screened includ...   \n",
       "IMPART, plate validation, 2022.pdf       blaIMP, blaOXA-48, blaVIM-1, blaNDM-1, blaKPC-...   \n",
       "Irrgang, GES-5, 2020.pdf                                     blaGES-1, blaGES-5, blaGES-5B   \n",
       "Irrgang, VIM_2019.pdf                                                             blaVIM-1   \n",
       "Pauly, 19-AB01133.pdf                                        blaVIM-1, blaSHV-5, blaCMY-13   \n",
       "Pauly, Carba plate validation, 2020.pdf                       blaVIM-1, blaNDM-1, blaKPC-2   \n",
       "Pauly,method comparison, 2021.pdf        blaVIM-1, blaGES-5, blaKPC-2, blaNDM-1, blaOXA-48   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                              blaVIM-1   \n",
       "\n",
       "                                                                        protocol_reference  \\\n",
       "antibiotics-08-00023.pdf                 Protocols referenced include those described i...   \n",
       "IMPART, plate validation, 2022.pdf              EURL-AR protocol (EU Decision 2013/652/EU)   \n",
       "Irrgang, GES-5, 2020.pdf                                                           EURL-AR   \n",
       "Irrgang, VIM_2019.pdf                                                              EURL-AR   \n",
       "Pauly, 19-AB01133.pdf                    European Union Reference Laboratory for Antimi...   \n",
       "Pauly, Carba plate validation, 2020.pdf                                   EURL-AR protocol   \n",
       "Pauly,method comparison, 2021.pdf        EURL-AR protocol for isolation of ESBL, AmpC a...   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     Modified DIN EN ISO 6579 protocol; EURL-AR gui...   \n",
       "\n",
       "                                                                       isolation_procedure  \\\n",
       "antibiotics-08-00023.pdf                 Decimal dilutions prepared in sterile 0.9% NaC...   \n",
       "IMPART, plate validation, 2022.pdf       culture-based isolation after non‐selective pr...   \n",
       "Irrgang, GES-5, 2020.pdf                                                     culture-based   \n",
       "Irrgang, VIM_2019.pdf                    Modified culture-dependent method combining un...   \n",
       "Pauly, 19-AB01133.pdf                                                        culture-based   \n",
       "Pauly, Carba plate validation, 2020.pdf                                      culture-based   \n",
       "Pauly,method comparison, 2021.pdf                                            culture-based   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     Culture-based isolation using selective preenr...   \n",
       "\n",
       "                                         ...  \\\n",
       "antibiotics-08-00023.pdf                 ...   \n",
       "IMPART, plate validation, 2022.pdf       ...   \n",
       "Irrgang, GES-5, 2020.pdf                 ...   \n",
       "Irrgang, VIM_2019.pdf                    ...   \n",
       "Pauly, 19-AB01133.pdf                    ...   \n",
       "Pauly, Carba plate validation, 2020.pdf  ...   \n",
       "Pauly,method comparison, 2021.pdf        ...   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     ...   \n",
       "\n",
       "                                                                 sample_dilution  \\\n",
       "antibiotics-08-00023.pdf                  Decimal dilutions in sterile 0.9% NaCl   \n",
       "IMPART, plate validation, 2022.pdf       1:10 dilution in buffered peptone water   \n",
       "Irrgang, GES-5, 2020.pdf                                                    None   \n",
       "Irrgang, VIM_2019.pdf                                                       1:10   \n",
       "Pauly, 19-AB01133.pdf                                                       None   \n",
       "Pauly, Carba plate validation, 2020.pdf                                 1:10,000   \n",
       "Pauly,method comparison, 2021.pdf                                           1:10   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                        None   \n",
       "\n",
       "                                                      sample_weight  \\\n",
       "antibiotics-08-00023.pdf                                       None   \n",
       "IMPART, plate validation, 2022.pdf                             None   \n",
       "Irrgang, GES-5, 2020.pdf                                       None   \n",
       "Irrgang, VIM_2019.pdf                                          10 g   \n",
       "Pauly, 19-AB01133.pdf                                          None   \n",
       "Pauly, Carba plate validation, 2020.pdf                        None   \n",
       "Pauly,method comparison, 2021.pdf                               9 g   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     20 g (feces), 5 g (manure)   \n",
       "\n",
       "                                                                         first_enrichtment  \\\n",
       "antibiotics-08-00023.pdf                                                              None   \n",
       "IMPART, plate validation, 2022.pdf       non-selective pre-enrichment in buffered pepto...   \n",
       "Irrgang, GES-5, 2020.pdf                                                              None   \n",
       "Irrgang, VIM_2019.pdf                                         non-selective pre-enrichment   \n",
       "Pauly, 19-AB01133.pdf                                                                 None   \n",
       "Pauly, Carba plate validation, 2020.pdf                                               None   \n",
       "Pauly,method comparison, 2021.pdf                                            non-selective   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                           non selective preenrichment   \n",
       "\n",
       "                                          first_enrichtment_condition  \\\n",
       "antibiotics-08-00023.pdf                                         None   \n",
       "IMPART, plate validation, 2022.pdf       overnight incubation at 37°C   \n",
       "Irrgang, GES-5, 2020.pdf                                         None   \n",
       "Irrgang, VIM_2019.pdf                                 16-20 h at 37°C   \n",
       "Pauly, 19-AB01133.pdf                                            None   \n",
       "Pauly, Carba plate validation, 2020.pdf                          None   \n",
       "Pauly,method comparison, 2021.pdf                37°C for 16–18 hours   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf     overnight incubation at 37°C   \n",
       "\n",
       "                                        second_enrichment  \\\n",
       "antibiotics-08-00023.pdf                               no   \n",
       "IMPART, plate validation, 2022.pdf                     no   \n",
       "Irrgang, GES-5, 2020.pdf                               no   \n",
       "Irrgang, VIM_2019.pdf                                 yes   \n",
       "Pauly, 19-AB01133.pdf                                 yes   \n",
       "Pauly, Carba plate validation, 2020.pdf              None   \n",
       "Pauly,method comparison, 2021.pdf                     yes   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                   no   \n",
       "\n",
       "                                              second_enrichtment_selective  \\\n",
       "antibiotics-08-00023.pdf                                              None   \n",
       "IMPART, plate validation, 2022.pdf                                    None   \n",
       "Irrgang, GES-5, 2020.pdf                                              None   \n",
       "Irrgang, VIM_2019.pdf                                            selective   \n",
       "Pauly, 19-AB01133.pdf                           selective with CTX and MEM   \n",
       "Pauly, Carba plate validation, 2020.pdf                               None   \n",
       "Pauly,method comparison, 2021.pdf        selective using LB+CTX and LB+MEM   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                  None   \n",
       "\n",
       "                                                              second_enrichtment_condition  \\\n",
       "antibiotics-08-00023.pdf                                                              None   \n",
       "IMPART, plate validation, 2022.pdf                                                    None   \n",
       "Irrgang, GES-5, 2020.pdf                                                              None   \n",
       "Irrgang, VIM_2019.pdf                                                         24 h at 44°C   \n",
       "Pauly, 19-AB01133.pdf                                 16–18 h at 37±2°C (plate incubation)   \n",
       "Pauly, Carba plate validation, 2020.pdf                                               None   \n",
       "Pauly,method comparison, 2021.pdf        37±2°C for 16–18 hours under microaerophilic c...   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                                  None   \n",
       "\n",
       "                                                                           selective_plate  \\\n",
       "antibiotics-08-00023.pdf                     VRBG (Violet Red Bile Glucose) selective agar   \n",
       "IMPART, plate validation, 2022.pdf       Brilliance™ CRE Agar, CHROMID® CARBA Agar, CHR...   \n",
       "Irrgang, GES-5, 2020.pdf                                                              None   \n",
       "Irrgang, VIM_2019.pdf                             chromID R-Carba, MacConkey agar with CTX   \n",
       "Pauly, 19-AB01133.pdf                    McConkey agar with 0.125 mg/L meropenem, McCon...   \n",
       "Pauly, Carba plate validation, 2020.pdf  ChromID R⃝CARBA agar, MacConkey agar with 1 mg...   \n",
       "Pauly,method comparison, 2021.pdf        ChromID® CARBA SMART agar, McC + CTX+MEM, McC ...   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf         ChromID Carba, MacConkey agar with cefotaxime   \n",
       "\n",
       "                                                                                ncb_method  \\\n",
       "antibiotics-08-00023.pdf                 Multiplex PCR assays for screening of resistan...   \n",
       "IMPART, plate validation, 2022.pdf                                   PCR and real-time PCR   \n",
       "Irrgang, GES-5, 2020.pdf                                                              None   \n",
       "Irrgang, VIM_2019.pdf                                                        real-time PCR   \n",
       "Pauly, 19-AB01133.pdf                                                                 None   \n",
       "Pauly, Carba plate validation, 2020.pdf                                               None   \n",
       "Pauly,method comparison, 2021.pdf                                            Real-time PCR   \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                                         Real-time PCR   \n",
       "\n",
       "                                                                                which_step  \n",
       "antibiotics-08-00023.pdf                 Non‐culture based detection applied post-isola...  \n",
       "IMPART, plate validation, 2022.pdf                                          post-isolation  \n",
       "Irrgang, GES-5, 2020.pdf                                                              None  \n",
       "Irrgang, VIM_2019.pdf                                      applied on the enrichment broth  \n",
       "Pauly, 19-AB01133.pdf                                                                 None  \n",
       "Pauly, Carba plate validation, 2020.pdf                                               None  \n",
       "Pauly,method comparison, 2021.pdf        After enrichment steps (both first and second ...  \n",
       "Roschanski, VIM-1 Stall N2, 2019.pdf                     Applied to preenrichment cultures  \n",
       "\n",
       "[8 rows x 29 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Convert the extraction_cpe_o1 dictionary to a list of dictionaries suitable for DataFrame creation\n",
    "data = [item for item in extraction_cpe.values()]\n",
    "index = list(extraction_cpe.keys())\n",
    "\n",
    "# Create the DataFrame\n",
    "df_extraction = pd.DataFrame(data, index=index)\n",
    "\n",
    "# Add filename and acceptance_status as the first and second columns\n",
    "df_extraction.insert(0, 'filename', df_extraction.index)\n",
    "df_extraction.insert(1, 'acceptance_status', df_extraction['filename'].map(acceptance_status))\n",
    "\n",
    "\n",
    "df_extraction"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
